var data={"title":"Methicillin-resistant Staphylococcus aureus (MRSA): Microbiology","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Methicillin-resistant Staphylococcus aureus (MRSA): Microbiology</div><dl id=\"topicContributors\"><dt><span> </span>Author:</dt><dd><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/contributors\" class=\"contributor contributor_credentials\">Franklin D Lowy, MD</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/contributors\" class=\"contributor contributor_credentials\">Daniel J Sexton, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/contributors\" class=\"contributor contributor_credentials\">Elinor L Baron, MD, DTMH</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Jun 15, 2017.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H1\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Methicillin is a semisynthetic beta-lactamase&ndash;resistant penicillin that was introduced in 1959; shortly thereafter, isolates of <em>Staphylococcus aureus</em> and coagulase-negative staphylococci with methicillin resistance were described. Outbreaks of methicillin-resistant <em>S. aureus</em> (MRSA) infection occurred in Europe in the early 1960s [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/1\" class=\"abstract_t\">1</a>]. Subsequently, these organisms have emerged as major nosocomial and community-acquired pathogens.</p><p>Three pandemic MRSA clones have been traced back to the original 1959 MRSA isolates in Denmark and England [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/2\" class=\"abstract_t\">2</a>]. In addition, molecular typing of MRSA strains collected from many geographic areas has revealed that five major MRSA clones emerged worldwide by 2002 [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/3\" class=\"abstract_t\">3</a>]. The epidemic community-associated MRSA strains in the early 2000s appear to have emerged from earlier epidemic clones [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/4-6\" class=\"abstract_t\">4-6</a>].</p><p>Our understanding of the genetic mechanisms responsible for methicillin resistance will be reviewed here. Virulence determinants for community-acquired MRSA are discussed separately, as are mechanisms for <em>S. aureus</em> with reduced susceptibility to <a href=\"topic.htm?path=vancomycin-drug-information\" class=\"drug drug_general\">vancomycin</a>. (See <a href=\"topic.htm?path=virulence-determinants-of-community-acquired-methicillin-resistant-staphylococcus-aureus\" class=\"medical medical_review\">&quot;Virulence determinants of community-acquired methicillin-resistant Staphylococcus aureus&quot;</a> and <a href=\"topic.htm?path=staphylococcus-aureus-bacteremia-with-reduced-susceptibility-to-vancomycin\" class=\"medical medical_review\">&quot;Staphylococcus aureus bacteremia with reduced susceptibility to vancomycin&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H2\"><span class=\"h1\">DEFINITION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Methicillin resistance requires the presence of the <em>mec</em> gene; strains lacking a <em>mec</em> gene are not methicillin resistant. Methicillin resistance is defined in the clinical microbiology laboratory as an <a href=\"topic.htm?path=oxacillin-drug-information\" class=\"drug drug_general\">oxacillin</a> minimum inhibitory concentration (MIC) &ge;4 <span class=\"nowrap\">mcg/mL</span> [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/7\" class=\"abstract_t\">7</a>]. Other methods of detection, such as the use of the <a href=\"topic.htm?path=cefoxitin-drug-information\" class=\"drug drug_general\">cefoxitin</a> disk diffusion test or one of several polymerase chain reactions to detect the <em>mec</em> gene, are also used. Isolates resistant to oxacillin or methicillin are also resistant to all beta-lactam agents, including cephalosporins (with the exception of <a href=\"topic.htm?path=ceftaroline-drug-information\" class=\"drug drug_general\">ceftaroline</a> and ceftobiprole, fifth-generation cephalosporins). MICs of &le;2 <span class=\"nowrap\">mcg/mL</span> are considered susceptible.</p><p class=\"headingAnchor\" id=\"H3\"><span class=\"h1\">MOLECULAR MECHANISM OF METHICILLIN RESISTANCE</span></p><p class=\"headingAnchor\" id=\"H4\"><span class=\"h2\">mec gene</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The presence of the <em>mec</em> gene is an absolute requirement for <em>S. aureus</em> to express methicillin resistance. The <em>mec</em> gene is absent from susceptible strains and present in all resistant strains [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/8-10\" class=\"abstract_t\">8-10</a>]. The structural component of the <em>mec</em> gene, <em>mecA</em>, encodes the penicillin-binding protein 2a (PBP2a) that establishes resistance to methicillin and other semisynthetic penicillinase-resistant beta-lactams.</p><p>The mechanism of <a href=\"topic.htm?path=oxacillin-drug-information\" class=\"drug drug_general\">oxacillin</a> resistance may be different in borderline-resistant strains, in which the <em>mec</em> gene is not present or is present in a very small resistant subpopulation (referred to as heteroresistance). (See <a href=\"#H10\" class=\"local\">'Borderline resistance'</a> below.)</p><p>The complete genomes for numerous strains of methicillin-resistant <em>S. aureus </em>(MRSA) have been published; three classes of pathogenicity islands and several different superantigens have been identified [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/11-13\" class=\"abstract_t\">11-13</a>]. The <em>mec</em> gene consists of a structural component, <em>mecA</em>, and, in many cases, two regulatory components that control expression of the gene:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><em>mecR1</em>-<em>mecI</em> is a negative regulator of <em>mecA</em> transcription. Mutations in this set of genes results in more highly resistant strains.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The beta-lactamase genes (<em>blaI</em>, <em>blaRI</em>, and <em>blaZ</em>) control expression of beta-lactamase and, because of sequence similarity to the <em>mecR1</em>-<em>mecI </em>genes, also can downregulate <em>mecA</em> gene transcription [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/14\" class=\"abstract_t\">14</a>]. However, beta-lactamase produces resistance by a mechanism different from <em>mec</em>; it hydrolyzes the beta-lactam ring. Because this negative regulation is not tightly controlled, expression of resistance following exposure to beta-lactams is relatively rapid. The looser control allows the <em>mec</em> gene to synthesize protein, which, under conditions of greater regulatory control, would not be permitted.</p><p/><p>In addition to these regulatory genes, there is a series of five auxiliary genes that can modify expression of methicillin resistance. These are the <em>fem</em> (factor essential for the expression of methicillin resistance) <em>A</em> to <em>E</em> genes. (See <a href=\"#H9\" class=\"local\">'fem genes'</a> below.)</p><p class=\"headingAnchor\" id=\"H5\"><span class=\"h3\">Penicillin-binding protein 2a</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Penicillin-binding proteins are peptidase enzymes located in the bacterial membrane that catalyze the transpeptidation reactions of peptidoglycan during cell wall synthesis. <em>mecA</em> encodes penicillin-binding protein (PBP) 2a, an inducible protein that establishes resistance to the semisynthetic penicillinase resistant beta-lactams: methicillin, <a href=\"topic.htm?path=nafcillin-drug-information\" class=\"drug drug_general\">nafcillin</a>, <a href=\"topic.htm?path=oxacillin-drug-information\" class=\"drug drug_general\">oxacillin</a>, and all cephalosporins [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/15,16\" class=\"abstract_t\">15,16</a>]. In contrast with the other four PBPs (1-4), PBP2a has a low affinity for beta-lactam antibiotics. (See <a href=\"topic.htm?path=resistance-of-streptococcus-pneumoniae-to-beta-lactam-antibiotics\" class=\"medical medical_review\">&quot;Resistance of Streptococcus pneumoniae to beta-lactam antibiotics&quot;</a>.)</p><p>In susceptible staphylococcal isolates, the beta-lactams covalently bind to PBPs 1-3, thereby inactivating enzyme activity, preventing transpeptidation, and ultimately contributing to bacterial death. PBP2a, with its low affinity for the beta-lactams, can substitute for the enzymatic activity of these PBPs and allow completion of cell wall assembly. The resulting peptidoglycan has a structurally different muropeptide composition, a change that does not appear to affect cellular function [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/17\" class=\"abstract_t\">17</a>].</p><p class=\"headingAnchor\" id=\"H6\"><span class=\"h3\">Staphylococcal chromosomal cassette mec (SCCmec)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The <em>mec</em> gene is part of a 21 to 67 kb mobile chromosomal element called the staphylococcal cassette chromosome (SCC<em>mec</em>). The majority of healthcare-associated (mostly nosocomial) MRSA clones is associated with SCC<em>mec</em> types I, II, and III and are multidrug resistant [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/18\" class=\"abstract_t\">18</a>]. In contrast, most community-associated MRSA (CA-MRSA) strains have type IV or V SCC<em>mec</em> and were formerly susceptible to other antibiotic families; this is no longer the case [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/12,19-23\" class=\"abstract_t\">12,19-23</a>].</p><p>Methicillin resistance may reduce the virulence of healthcare-associated MRSA by interfering with <em>agr</em> quorum sensing [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/24\" class=\"abstract_t\">24</a>]. In one study, healthcare-associated MRSA strains carrying SCC<em>mec</em> type II produced reduced amounts of cytolytic toxins as measured by an in vitro T cell survival assay and in vivo murine bacteremia model. Alteration of the cell wall appeared to affect the <em>agr</em> quorum sensing system, resulting in diminished virulence. This effect has not been observed among community-associated MRSA isolates and may help explain the failure of hospital-acquired MRSA (HA-MRSA) to spread into the community.</p><p>Sequencing SCC<em>mec</em> from many MRSA strains has resulted in identification of 11 different SCC<em>mec</em> types (I-XI) that vary in genetic makeup and size [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/25-27\" class=\"abstract_t\">25-27</a>]. Transfer of SCC<em>mec</em> from MRSA into well-adapted strains of methicillin-susceptible <em>S. aureus</em> (MSSA) has occurred on a number of occasions, resulting in new MRSA isolates that spread rapidly in healthcare institutions. This transfer is uncommon, restricted to a limited number of staphylococcal lineages. In 2011, a novel <em>mecA</em> homologue was identified in isolates obtained from both cows and humans [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/28\" class=\"abstract_t\">28</a>]. This paper (as well as others) identified livestock as a reservoir of MRSA with the potential for transmission to humans [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/29\" class=\"abstract_t\">29</a>].</p><p class=\"headingAnchor\" id=\"H7\"><span class=\"h3\">Origin in coagulase-negative staphylococci</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>It is believed that the <em>mec</em> gene was acquired from these closely related staphylococcal species via a limited number of genetic events. The <em>mec</em> gene is essentially the same in all staphylococcal species. Several studies point to coagulase-negative staphylococci (CoNS) as the origin of methicillin resistance in <em>S. aureus</em>. Segments of DNA from an insertion sequence found in CoNS (IS 1272) have been identified in MRSA [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/30,31\" class=\"abstract_t\">30,31</a>]. Insertion sequence elements are DNA segments that encode enzymes that allow for site-specific recombination. In addition, one study reported 88 percent amino acid homology for the MRSA <em>mec</em> gene in <em>Staphylococcus sciuri</em> or <em>Staphylococcus fleuretti</em> [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/32\" class=\"abstract_t\">32</a>], another species of CoNS [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/33\" class=\"abstract_t\">33</a>]. The presence of different insertion sequence fragments within the <em>mec</em> gene makes transposition a likely mechanism of transfer for the gene between species [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/34\" class=\"abstract_t\">34</a>].</p><p>Sequencing of the MRSA strain USA300, the epidemic clone that is now the most common cause of community-acquired MRSA infections in the United States, has shown that additional virulence and resistance genes have also been acquired from CoNS [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/13\" class=\"abstract_t\">13</a>]. These genes include molecular variants of enterotoxins Q and K and a mobile element (the arginine catabolic mobile element [ACME]) that encodes an arginine deaminase pathway and an oligopeptide permease system. The authors hypothesize that these genes enable the strain to evade host immune responses and contribute to its ability to survive and spread in host tissue. The virulence of USA300 appears to be linked to the differential expression of selected virulence determinants that were already present in the progenitor strain (USA500) of USA300 [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/5\" class=\"abstract_t\">5</a>].</p><p class=\"headingAnchor\" id=\"H8\"><span class=\"h1\">EXPRESSION OF METHICILLIN RESISTANCE</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Despite the presence in methicillin-resistant <em>S. aureus </em>(MRSA) of the <em>mec</em> gene, the phenotypic expression of methicillin resistance varies. There are three different forms for the expression of methicillin resistance: homogeneous, heterogeneous, and borderline (see <a href=\"#H10\" class=\"local\">'Borderline resistance'</a> below). Most clinical isolates of MRSA from a given patient are heterogeneous in their expression of methicillin resistance.</p><p>As an example, under routine growth conditions (37&ordm;C, unsupplemented media), &ge;99.9 percent of MRSA appear to be susceptible to the beta-lactams (eg, <a href=\"topic.htm?path=oxacillin-drug-information\" class=\"drug drug_general\">oxacillin</a> 4 <span class=\"nowrap\">mcg/mL)</span>. However, if the cells are grown at 30 to 35&ordm;C or in the presence of 6.5% sodium chloride, they become more homogeneously resistant and express beta-lactam resistance at a much higher frequency [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/34\" class=\"abstract_t\">34</a>]. In addition, growth of heterogeneous strains in the presence of a beta-lactam results in the selection of a homogeneous phenotype. Serial passage of these cells in the absence of antibiotic leads to slow reversion back to the heterogeneous state.</p><p>A similar phenomenon has been observed in experimental endocarditis. Treatment of rabbits infected with MRSA with a beta-lactam results in a greater percentage of the total bacterial population being resistant than in untreated controls [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/35\" class=\"abstract_t\">35</a>]. There was also a correlation between the potential efficacy of the antibiotic and its affinity for binding to PBP2a in vitro.</p><p class=\"headingAnchor\" id=\"H9\"><span class=\"h2\">fem genes</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The <em>fem</em> genes and other genes are necessary for the homogeneous expression of resistance [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/36\" class=\"abstract_t\">36</a>]. These auxiliary genes affect different steps in the synthesis of peptidoglycan. Inactivation of these genes can convert a homogeneously resistant strain to a heterogeneous resistant one [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/37\" class=\"abstract_t\">37</a>].</p><p class=\"headingAnchor\" id=\"H10\"><span class=\"h2\">Borderline resistance</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Borderline resistance refers to isolates that are at the margin of resistance. This type of resistance may be due to one of several mechanisms:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Some strains with borderline resistance possess the <em>mecA</em> gene. In these strains, the resistant subpopulation may be extremely small and therefore more susceptible to beta-lactams.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In strains that lack the <em>mecA</em> gene (and therefore penicillin-binding protein [PBP] 2a), there may be alterations in or overexpression of the other PBPs, resulting in reduced affinity for beta-lactams or the availability of more enzyme for peptidoglycan synthesis. Overproduction of beta-lactamase with slow hydrolysis of the beta-lactam antibiotic has also been hypothesized as a potential mechanism for borderline resistance in <em>mecA</em>-negative strains [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/34,38,39\" class=\"abstract_t\">34,38,39</a>].</p><p/><p class=\"headingAnchor\" id=\"H11\"><span class=\"h1\">LABORATORY DETECTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The most accurate methods to detect methicillin-resistant <em>S. aureus </em>(MRSA) are polymerase chain reaction (PCR) for detection of the <em>mecA</em> gene and latex agglutination tests for the protein product of <em>mecA</em>, penicillin-binding protein 2a [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/40-43\" class=\"abstract_t\">40-43</a>]. When these tests are not available, traditional microbiology laboratory techniques are acceptable, such as broth microdilution, agar screening plates containing salt (4%) and <a href=\"topic.htm?path=oxacillin-drug-information\" class=\"drug drug_general\">oxacillin</a> (6 <span class=\"nowrap\">mcg/mL),</span> and <a href=\"topic.htm?path=cefoxitin-drug-information\" class=\"drug drug_general\">cefoxitin</a> disk diffusion tests [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/44\" class=\"abstract_t\">44</a>].</p><p class=\"headingAnchor\" id=\"H12\"><span class=\"h2\">Rapid testing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Rapid testing for detection of MRSA is discussed in detail separately. (See <a href=\"topic.htm?path=rapid-detection-of-methicillin-resistant-staphylococcus-aureus\" class=\"medical medical_review\">&quot;Rapid detection of methicillin-resistant Staphylococcus aureus&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H13\"><span class=\"h2\">Culture</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Surveillance cultures are performed at body sites (mostly the anterior nares) that are frequently colonized with MRSA. The majority of patients with asymptomatic MRSA colonization will be detected by screening culture from the anterior nares (sensitivity 73 to 93 percent) [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/45-47\" class=\"abstract_t\">45-47</a>]. The sensitivity can be increased by screening additional body sites or open lesions, such as surgical wounds, pressure (decubitus) ulcers, and areas of skin breakdown.</p><p>In patients without open lesions, a one-year study of multisite screening (nares, rectum, and axilla) found that nares culture alone missed 27 percent of MRSA carriers [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/46\" class=\"abstract_t\">46</a>]. Throat swabs may also be helpful. In a screening study of almost 3000 individuals for <em>S. aureus</em> carriage, 37 percent had nasal carriage and 13 percent were colonized only in the throat, increasing the sensitivity of screening by 26 percent [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/48\" class=\"abstract_t\">48</a>]. A similar increase in yield with throat swabs (22 percent) was seen in the small subset of MRSA carriers.</p><p>Traditional methods used to process surveillance cultures take 48 to 72 hours to yield results. However, newly available techniques shorten the amount of time required to detect MRSA in surveillance cultures. A chromogenic selective agar containing <a href=\"topic.htm?path=cefoxitin-drug-information\" class=\"drug drug_general\">cefoxitin</a> detects a majority of MRSA isolates within 24 hours, while commercially available real-time PCR tests for <em>mecA</em> can detect MRSA within two hours [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/40\" class=\"abstract_t\">40</a>].</p><p class=\"headingAnchor\" id=\"H14\"><span class=\"h2\">Antimicrobial susceptibility testing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>In the past, most healthcare-associated MRSA strains were multidrug resistant. Isolates that are resistant to <a href=\"topic.htm?path=oxacillin-drug-information\" class=\"drug drug_general\">oxacillin</a> but remain susceptible to most non&ndash;beta-lactam agents (eg, <a href=\"topic.htm?path=trimethoprim-sulfamethoxazole-co-trimoxazole-drug-information\" class=\"drug drug_general\">trimethoprim-sulfamethoxazole</a>, <a href=\"topic.htm?path=clindamycin-drug-information\" class=\"drug drug_general\">clindamycin</a>, and <a href=\"topic.htm?path=ciprofloxacin-drug-information\" class=\"drug drug_general\">ciprofloxacin</a>) are often community-associated MRSA; however, these strains have also become increasingly antibiotic resistant. Such strains should be tested using a confirmatory test such as a <em>mecA</em> probe, PCR assay for <em>mecA</em>, oxacillin-salt agar screening plates, or <a href=\"topic.htm?path=cefoxitin-drug-information\" class=\"drug drug_general\">cefoxitin</a> disk diffusion tests.</p><p>Antibiotic susceptibility testing for methicillin resistance has been modified to enhance the detection of these isolates, a large number of which are heterogeneously resistant to methicillin. Susceptibility testing now includes use of the more stable <a href=\"topic.htm?path=oxacillin-drug-information\" class=\"drug drug_general\">oxacillin</a> rather than methicillin disk, incubation at &le;35&ordm;C for 24 rather than 18 hours, and the incorporation of 6.5% sodium chloride into the media. It is important to remember that isolates resistant to oxacillin are also resistant to all beta-lactam agents including cephalosporins.</p><p>Specific recommendations exist for the different methods of susceptibility testing such as automated turbidimetric or disk diffusion assays [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/44\" class=\"abstract_t\">44</a>]. Increasingly clinical isolates are screened for methicillin resistance by PCR or with probes specific for segments of the <em>mec</em> gene. Kits for the detection of the <em>mec</em> gene are now common.</p><p class=\"headingAnchor\" id=\"H2311893849\"><span class=\"h3\">Elevated vancomycin MIC</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Increased mortality and treatment failure have been reported in patients treated with <a href=\"topic.htm?path=vancomycin-drug-information\" class=\"drug drug_general\">vancomycin</a> for MRSA bacteremia when their isolates have elevated minimum inhibitory concentration (MIC). Risk factors that predict vancomycin MIC of 2 <span class=\"nowrap\">mcg/mL</span> include age &gt;50 years, administration of vancomycin for &gt;48 hours in the week prior to bacteremia, chronic liver disease, history of MRSA bacteremia, and presence of a central line [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/49\" class=\"abstract_t\">49</a>]. (See <a href=\"topic.htm?path=staphylococcus-aureus-bacteremia-with-reduced-susceptibility-to-vancomycin\" class=\"medical medical_review\">&quot;Staphylococcus aureus bacteremia with reduced susceptibility to vancomycin&quot;</a>.)</p><p>Additional data suggest that increased risk for treatment failure among patients with hospital-acquired MRSA infections compared with community-acquired MRSA may be independent of the <a href=\"topic.htm?path=vancomycin-drug-information\" class=\"drug drug_general\">vancomycin</a> MIC, suggesting a potential intrinsic strain-specific defect [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/50\" class=\"abstract_t\">50</a>]. The perceived risk of treatment failure may be independent of antibiotic choice, as patients with methicillin-susceptible <em>S. aureus</em> bacteremia and high vancomycin MIC have an equal risk for treatment failure as patients with MRSA bacteremia and high vancomycin MIC [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/51\" class=\"abstract_t\">51</a>]. A systematic review and meta-analysis of 38 studies of <em>S. aureus</em> bacteremia concluded that patients with <em>S. aureus</em> bacteremia and vancomycin MIC &ge;1.5 had no difference in outcomes compared with patients with <em>S. aureus</em> bacteremia and vancomycin MIC &lt;1.5, regardless of assay type [<a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/52\" class=\"abstract_t\">52</a>].</p><p class=\"headingAnchor\" id=\"H190972270\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Shortly after the introduction of methicillin in 1959, isolates resistant to this agent were reported. Outbreaks of methicillin-resistant <em>Staphylococcus aureus</em> (MRSA) infections occurred in Europe in the early 1960s. Since these original descriptions, MRSA as well as coagulase-negative staphylococci, which are commonly resistant to methicillin, have emerged as major nosocomial and, in the case of MRSA, community-acquired pathogens. (See <a href=\"#H1\" class=\"local\">'Introduction'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Methicillin resistance is defined in the clinical microbiology laboratory as an <a href=\"topic.htm?path=oxacillin-drug-information\" class=\"drug drug_general\">oxacillin</a> minimum inhibitory concentration &ge;4 <span class=\"nowrap\">mcg/mL</span>. Isolates resistant to oxacillin or methicillin are also resistant to all beta-lactam agents, including cephalosporins. (See <a href=\"#H2\" class=\"local\">'Definition'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Methicillin resistance requires the presence of the <em>mec</em> gene; strains lacking a <em>mec</em> gene are not methicillin resistant. The structural component of the <em>mec</em> gene, <em>mecA</em>, encodes the penicillin-binding protein 2a (PBP2a) that establishes resistance to methicillin and other semisynthetic penicillinase resistant beta-lactams. (See <a href=\"#H2\" class=\"local\">'Definition'</a> above and <a href=\"#H4\" class=\"local\">'mec gene'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The most accurate methods to detect MRSA are polymerase chain reaction for detection of the <em>mecA</em> gene and latex agglutination tests for the protein product of <em>mecA</em>, PBP2a. When these tests are not available, traditional microbiology laboratory techniques are acceptable, such as oxacillin-salt agar screening plates and <a href=\"topic.htm?path=cefoxitin-drug-information\" class=\"drug drug_general\">cefoxitin</a> disk diffusion tests. Rapid testing for detection of MRSA is discussed in detail separately. (See <a href=\"#H11\" class=\"local\">'Laboratory detection'</a> above and <a href=\"topic.htm?path=rapid-detection-of-methicillin-resistant-staphylococcus-aureus\" class=\"medical medical_review\">&quot;Rapid detection of methicillin-resistant Staphylococcus aureus&quot;</a>.)</p><p/></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/1\" class=\"nounderline abstract_t\">Benner EJ, Kayser FH. Growing clinical significance of methcillin-resistant Staphylococcus aureus. Lancet 1968; 2:741.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/2\" class=\"nounderline abstract_t\">Cris&oacute;stomo MI, Westh H, Tomasz A, et al. The evolution of methicillin resistance in Staphylococcus aureus: similarity of genetic backgrounds in historically early methicillin-susceptible and -resistant isolates and contemporary epidemic clones. Proc Natl Acad Sci U S A 2001; 98:9865.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/3\" class=\"nounderline abstract_t\">Enright MC, Robinson DA, Randle G, et al. The evolutionary history of methicillin-resistant Staphylococcus aureus (MRSA). Proc Natl Acad Sci U S A 2002; 99:7687.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/4\" class=\"nounderline abstract_t\">Robinson DA, Kearns AM, Holmes A, et al. Re-emergence of early pandemic Staphylococcus aureus as a community-acquired meticillin-resistant clone. Lancet 2005; 365:1256.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/5\" class=\"nounderline abstract_t\">Li M, Diep BA, Villaruz AE, et al. Evolution of virulence in epidemic community-associated methicillin-resistant Staphylococcus aureus. Proc Natl Acad Sci U S A 2009; 106:5883.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/6\" class=\"nounderline abstract_t\">Francois P, Bento M, Renzi G, et al. Evaluation of three molecular assays for rapid identification of methicillin-resistant Staphylococcus aureus. J Clin Microbiol 2007; 45:2011.</a></li><li class=\"breakAll\">Clinical and Laboratory Standards Institute. Performance Standards for Antimicrobial Susceptibility Testing: Sixteenth Informational Supplement. M100-S16 Methods for dilution antimicrobial susceptibility tests for bacteria that grow aerobically: Approved Standard. Vol 26. No 3. CLSI, Wayne, PA 2006.</li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/8\" class=\"nounderline abstract_t\">Inglis B, Matthews PR, Stewart PR. The expression in Staphylococcus aureus of cloned DNA encoding methicillin resistance. J Gen Microbiol 1988; 134:1465.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/9\" class=\"nounderline abstract_t\">Tesch W, Str&auml;ssle A, Berger-B&auml;chi B, et al. Cloning and expression of methicillin resistance from Staphylococcus epidermidis in Staphylococcus carnosus. Antimicrob Agents Chemother 1988; 32:1494.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/10\" class=\"nounderline abstract_t\">Peacock SJ, Paterson GK. Mechanisms of Methicillin Resistance in Staphylococcus aureus. Annu Rev Biochem 2015; 84:577.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/11\" class=\"nounderline abstract_t\">Kuroda M, Ohta T, Uchiyama I, et al. Whole genome sequencing of meticillin-resistant Staphylococcus aureus. Lancet 2001; 357:1225.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/12\" class=\"nounderline abstract_t\">Baba T, Takeuchi F, Kuroda M, et al. Genome and virulence determinants of high virulence community-acquired MRSA. Lancet 2002; 359:1819.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/13\" class=\"nounderline abstract_t\">Diep BA, Gill SR, Chang RF, et al. Complete genome sequence of USA300, an epidemic clone of community-acquired meticillin-resistant Staphylococcus aureus. Lancet 2006; 367:731.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/14\" class=\"nounderline abstract_t\">Zhang HZ, Hackbarth CJ, Chansky KM, Chambers HF. A proteolytic transmembrane signaling pathway and resistance to beta-lactams in staphylococci. Science 2001; 291:1962.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/15\" class=\"nounderline abstract_t\">Hartman BJ, Tomasz A. Low-affinity penicillin-binding protein associated with beta-lactam resistance in Staphylococcus aureus. J Bacteriol 1984; 158:513.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/16\" class=\"nounderline abstract_t\">Utsui Y, Yokota T. Role of an altered penicillin-binding protein in methicillin- and cephem-resistant Staphylococcus aureus. Antimicrob Agents Chemother 1985; 28:397.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/17\" class=\"nounderline abstract_t\">de Jonge BL, Tomasz A. Abnormal peptidoglycan produced in a methicillin-resistant strain of Staphylococcus aureus grown in the presence of methicillin: functional role for penicillin-binding protein 2A in cell wall synthesis. Antimicrob Agents Chemother 1993; 37:342.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/18\" class=\"nounderline abstract_t\">Michel M, Gutmann L. Methicillin-resistant Staphylococcus aureus and vancomycin-resistant enterococci: therapeutic realities and possibilities. Lancet 1997; 349:1901.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/19\" class=\"nounderline abstract_t\">Daum RS, Ito T, Hiramatsu K, et al. A novel methicillin-resistance cassette in community-acquired methicillin-resistant Staphylococcus aureus isolates of diverse genetic backgrounds. J Infect Dis 2002; 186:1344.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/20\" class=\"nounderline abstract_t\">Ma XX, Ito T, Tiensasitorn C, et al. Novel type of staphylococcal cassette chromosome mec identified in community-acquired methicillin-resistant Staphylococcus aureus strains. Antimicrob Agents Chemother 2002; 46:1147.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/21\" class=\"nounderline abstract_t\">Naimi TS, LeDell KH, Como-Sabetti K, et al. Comparison of community- and health care-associated methicillin-resistant Staphylococcus aureus infection. JAMA 2003; 290:2976.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/22\" class=\"nounderline abstract_t\">Salmenlinna S, Lyytik&auml;inen O, Vuopio-Varkila J. Community-acquired methicillin-resistant Staphylococcus aureus, Finland. Emerg Infect Dis 2002; 8:602.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/23\" class=\"nounderline abstract_t\">Sattler CA, Mason EO Jr, Kaplan SL. Prospective comparison of risk factors and demographic and clinical characteristics of community-acquired, methicillin-resistant versus methicillin-susceptible Staphylococcus aureus infection in children. Pediatr Infect Dis J 2002; 21:910.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/24\" class=\"nounderline abstract_t\">Rudkin JK, Edwards AM, Bowden MG, et al. Methicillin resistance reduces the virulence of healthcare-associated methicillin-resistant Staphylococcus aureus by interfering with the agr quorum sensing system. J Infect Dis 2012; 205:798.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/25\" class=\"nounderline abstract_t\">Oliveira DC, Tomasz A, de Lencastre H. The evolution of pandemic clones of methicillin-resistant Staphylococcus aureus: identification of two ancestral genetic backgrounds and the associated mec elements. Microb Drug Resist 2001; 7:349.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/26\" class=\"nounderline abstract_t\">Ito T, Ma XX, Takeuchi F, et al. Novel type V staphylococcal cassette chromosome mec driven by a novel cassette chromosome recombinase, ccrC. Antimicrob Agents Chemother 2004; 48:2637.</a></li><li class=\"breakAll\">SCCmec: International Working Group on the Staphylococcal Cassette Chromosome elements http://www.sccmec.org/Pages/SCC_HomeEN.html (Accessed on February 21, 2014).</li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/28\" class=\"nounderline abstract_t\">Garc&iacute;a-&Aacute;lvarez L, Holden MT, Lindsay H, et al. Meticillin-resistant Staphylococcus aureus with a novel mecA homologue in human and bovine populations in the UK and Denmark: a descriptive study. Lancet Infect Dis 2011; 11:595.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/29\" class=\"nounderline abstract_t\">van Cleef BA, Monnet DL, Voss A, et al. Livestock-associated methicillin-resistant Staphylococcus aureus in humans, Europe. Emerg Infect Dis 2011; 17:502.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/30\" class=\"nounderline abstract_t\">Archer GL, Niemeyer DM. Origin and evolution of DNA associated with resistance to methicillin in staphylococci. Trends Microbiol 1994; 2:343.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/31\" class=\"nounderline abstract_t\">Archer GL, Niemeyer DM, Thanassi JA, Pucci MJ. Dissemination among staphylococci of DNA sequences associated with methicillin resistance. Antimicrob Agents Chemother 1994; 38:447.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/32\" class=\"nounderline abstract_t\">Tsubakishita S, Kuwahara-Arai K, Sasaki T, Hiramatsu K. Origin and molecular evolution of the determinant of methicillin resistance in staphylococci. Antimicrob Agents Chemother 2010; 54:4352.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/33\" class=\"nounderline abstract_t\">Wu S, Piscitelli C, de Lencastre H, Tomasz A. Tracking the evolutionary origin of the methicillin resistance gene: cloning and sequencing of a homologue of mecA from a methicillin susceptible strain of Staphylococcus sciuri. Microb Drug Resist 1996; 2:435.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/34\" class=\"nounderline abstract_t\">Chambers HF. Methicillin resistance in staphylococci: molecular and biochemical basis and clinical implications. Clin Microbiol Rev 1997; 10:781.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/35\" class=\"nounderline abstract_t\">Chambers HF, Sachdeva M, Kennedy S. Binding affinity for penicillin-binding protein 2a correlates with in vivo activity of beta-lactam antibiotics against methicillin-resistant Staphylococcus aureus. J Infect Dis 1990; 162:705.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/36\" class=\"nounderline abstract_t\">de Lencastre H, Tomasz A. Reassessment of the number of auxiliary genes essential for expression of high-level methicillin resistance in Staphylococcus aureus. Antimicrob Agents Chemother 1994; 38:2590.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/37\" class=\"nounderline abstract_t\">Berger-B&auml;chi B. Expression of resistance to methicillin. Trends Microbiol 1994; 2:389.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/38\" class=\"nounderline abstract_t\">Tomasz A, Drugeon HB, de Lencastre HM, et al. New mechanism for methicillin resistance in Staphylococcus aureus: clinical isolates that lack the PBP 2a gene and contain normal penicillin-binding proteins with modified penicillin-binding capacity. Antimicrob Agents Chemother 1989; 33:1869.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/39\" class=\"nounderline abstract_t\">Henze UU, Berger-B&auml;chi B. Staphylococcus aureus penicillin-binding protein 4 and intrinsic beta-lactam resistance. Antimicrob Agents Chemother 1995; 39:2415.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/40\" class=\"nounderline abstract_t\">Warren DK, Liao RS, Merz LR, et al. Detection of methicillin-resistant Staphylococcus aureus directly from nasal swab specimens by a real-time PCR assay. J Clin Microbiol 2004; 42:5578.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/41\" class=\"nounderline abstract_t\">Harbarth S, Masuet-Aumatell C, Schrenzel J, et al. Evaluation of rapid screening and pre-emptive contact isolation for detecting and controlling methicillin-resistant Staphylococcus aureus in critical care: an interventional cohort study. Crit Care 2006; 10:R25.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/42\" class=\"nounderline abstract_t\">Miller MB, Meyer H, Rogers E, Gilligan PH. Comparison of conventional susceptibility testing, penicillin-binding protein 2a latex agglutination testing, and mecA real-time PCR for detection of oxacillin resistance in Staphylococcus aureus and coagulase-negative Staphylococcus. J Clin Microbiol 2005; 43:3450.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/43\" class=\"nounderline abstract_t\">Sakoulas G, Gold HS, Venkataraman L, et al. Methicillin-resistant Staphylococcus aureus: comparison of susceptibility testing methods and analysis of mecA-positive susceptible strains. J Clin Microbiol 2001; 39:3946.</a></li><li class=\"breakAll\">National Committee for Clinical Laboratory Standards. Performance standards for antimicrobial susceptibility testing. CLSI approved standard M100-S23. Clinical and Laboratory Standards Institute, Wayne, PA 2013.</li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/45\" class=\"nounderline abstract_t\">Sanford MD, Widmer AF, Bale MJ, et al. Efficient detection and long-term persistence of the carriage of methicillin-resistant Staphylococcus aureus. Clin Infect Dis 1994; 19:1123.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/46\" class=\"nounderline abstract_t\">Eveillard M, de Lassence A, Lancien E, et al. Evaluation of a strategy of screening multiple anatomical sites for methicillin-resistant Staphylococcus aureus at admission to a teaching hospital. Infect Control Hosp Epidemiol 2006; 27:181.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/47\" class=\"nounderline abstract_t\">Huang SS, Rifas-Shiman SL, Warren DK, et al. Improving methicillin-resistant Staphylococcus aureus surveillance and reporting in intensive care units. J Infect Dis 2007; 195:330.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/48\" class=\"nounderline abstract_t\">Mertz D, Frei R, Jaussi B, et al. Throat swabs are necessary to reliably detect carriers of Staphylococcus aureus. Clin Infect Dis 2007; 45:475.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/49\" class=\"nounderline abstract_t\">Lubin AS, Snydman DR, Ruthazer R, et al. Predicting high vancomycin minimum inhibitory concentration in methicillin-resistant Staphylococcus aureus bloodstream infections. Clin Infect Dis 2011; 52:997.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/50\" class=\"nounderline abstract_t\">Chen SY, Liao CH, Wang JL, et al. Methicillin-resistant Staphylococcus aureus (MRSA) staphylococcal cassette chromosome mec genotype effects outcomes of patients with healthcare-associated MRSA bacteremia independently of vancomycin minimum inhibitory concentration. Clin Infect Dis 2012; 55:1329.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/51\" class=\"nounderline abstract_t\">Holmes NE, Turnidge JD, Munckhof WJ, et al. Antibiotic choice may not explain poorer outcomes in patients with Staphylococcus aureus bacteremia and high vancomycin minimum inhibitory concentrations. J Infect Dis 2011; 204:340.</a></li><li><a href=\"https://www.uptodate.com/contents/methicillin-resistant-staphylococcus-aureus-mrsa-microbiology/abstract/52\" class=\"nounderline abstract_t\">Kalil AC, Van Schooneveld TC, Fey PD, Rupp ME. Association between vancomycin minimum inhibitory concentration and mortality among patients with Staphylococcus aureus bloodstream infections: a systematic review and meta-analysis. JAMA 2014; 312:1552.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 3153 Version 13.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H190972270\"><span>SUMMARY</span></a></li><li><a href=\"#H1\" id=\"outline-link-H1\">INTRODUCTION</a></li><li><a href=\"#H2\" id=\"outline-link-H2\">DEFINITION</a></li><li><a href=\"#H3\" id=\"outline-link-H3\">MOLECULAR MECHANISM OF METHICILLIN RESISTANCE</a><ul><li><a href=\"#H4\" id=\"outline-link-H4\">mec gene</a><ul><li><a href=\"#H5\" id=\"outline-link-H5\">- Penicillin-binding protein 2a</a></li><li><a href=\"#H6\" id=\"outline-link-H6\">- Staphylococcal chromosomal cassette mec (SCCmec)</a></li><li><a href=\"#H7\" id=\"outline-link-H7\">- Origin in coagulase-negative staphylococci</a></li></ul></li></ul></li><li><a href=\"#H8\" id=\"outline-link-H8\">EXPRESSION OF METHICILLIN RESISTANCE</a><ul><li><a href=\"#H9\" id=\"outline-link-H9\">fem genes</a></li><li><a href=\"#H10\" id=\"outline-link-H10\">Borderline resistance</a></li></ul></li><li><a href=\"#H11\" id=\"outline-link-H11\">LABORATORY DETECTION</a><ul><li><a href=\"#H12\" id=\"outline-link-H12\">Rapid testing</a></li><li><a href=\"#H13\" id=\"outline-link-H13\">Culture</a></li><li><a href=\"#H14\" id=\"outline-link-H14\">Antimicrobial susceptibility testing</a><ul><li><a href=\"#H2311893849\" id=\"outline-link-H2311893849\">- Elevated vancomycin MIC</a></li></ul></li></ul></li><li><a href=\"#H190972270\" id=\"outline-link-H190972270\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=rapid-detection-of-methicillin-resistant-staphylococcus-aureus\" class=\"medical medical_review\">Rapid detection of methicillin-resistant Staphylococcus aureus</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=resistance-of-streptococcus-pneumoniae-to-beta-lactam-antibiotics\" class=\"medical medical_review\">Resistance of Streptococcus pneumoniae to beta-lactam antibiotics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=staphylococcus-aureus-bacteremia-with-reduced-susceptibility-to-vancomycin\" class=\"medical medical_review\">Staphylococcus aureus bacteremia with reduced susceptibility to vancomycin</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=virulence-determinants-of-community-acquired-methicillin-resistant-staphylococcus-aureus\" class=\"medical medical_review\">Virulence determinants of community-acquired methicillin-resistant Staphylococcus aureus</a></li></ul></div></div>","javascript":null}