var data={"title":"Microbiology of enterococci","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Microbiology of enterococci</div><dl id=\"topicContributors\"><dt><span> </span>Author:</dt><dd><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/contributors\" class=\"contributor contributor_credentials\">Barbara E Murray, MD</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/contributors\" class=\"contributor contributor_credentials\">Daniel J Sexton, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/contributors\" class=\"contributor contributor_credentials\">Elinor L Baron, MD, DTMH</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Aug 31, 2017.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H1\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A number of the characteristic features of enterococci (formerly called group D streptococci) have been recognized for at least 100 years, including their presence in feces and sewage, their ability to cause endocarditis and urinary tract infections (UTIs), and their ability to survive harsh environmental conditions including drying, high temperatures, and exposure to some antiseptics [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/1,2\" class=\"abstract_t\">1,2</a>].</p><p>The common name &quot;enterococcus,&quot; derived from the French &quot;ent&eacute;rocoque&quot; used in an 1899 publication, points out that these bacteria are cocci of intestinal (enteric) origin, and the 1906 use of <em>Streptococcus faecalis</em> and the 1919 use of <em>Streptococcus faecium</em> also emphasized their presence in feces.</p><p>Subsequent studies confirmed these early observations. Most humans and animals have enterococci in their intestinal tract; the counts in healthy humans often range from 10<sup>5</sup> to 10<sup>7</sup> bacteria per gram of stool.</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Enterococci, particularly <em>Enterococcus faecalis</em>, are a common cause of endocarditis (5 to 15 percent of community-acquired endocarditis and up to 30 percent of nosocomially acquired endocarditis) and can be a common cause of nosocomial urinary tract infections (being recovered from up to 15 to 20 percent of UTIs in the hospital setting) [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/3,4\" class=\"abstract_t\">3,4</a>].</p><p/><p class=\"headingAnchor\" id=\"H2\"><span class=\"h1\">MICROBIOLOGY CLASSIFICATION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Enterococci are gram-positive cocci that form short- to medium-length chains. In the 1930s, Lancefield devised a serologic typing system that classified enterococci as group D streptococci, along with some organisms (eg, <em>Streptococcus gallolyticus</em>, formerly <em>S. bovis</em>) that differed from enterococci in a number of other features. In 1937, Sherman divided streptococci into the pyogenic, viridans, lactic, and enterococcal groups. Enterococci were characterized by the following features:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Their reaction with group D antiserum</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Their ability to grow in high salt, at 10&ordm;C and 40&ordm;C, and at high pH</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Their ability to survive exposure to 60&ordm;C for 30 minutes</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Their ability to hydrolyze esculin in the presence of bile</p><p/><p>In the ensuing 45 to 50 years, enterococci continued to be taxonomically classified in the genus <em>Streptococcus</em>. In the mid-1980s, with advances in taxonomic techniques based upon nucleic acid analyses, it was proposed that <em>S. faecalis</em> and <em>S. faecium </em>be removed from the genus <em>Streptococcus</em>, transferred to a new genus, <em>Enterococcus</em>, and reclassified as <em>E. faecalis</em> and <em>E. faecium</em>, respectively [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/5\" class=\"abstract_t\">5</a>]. Subsequently, other enterococcal species were also reclassified from the genus <em>Streptococcus</em> to the genus <em>Enterococcus</em>; many new species have also been recognized.</p><p class=\"headingAnchor\" id=\"H3\"><span class=\"h2\">Problems with classification</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The characteristic features of enterococci as being bile esculin positive, reactive with group D antiserum, and able to grow in 6.5 percent sodium chloride were used by clinical laboratories for decades to distinguish enterococci from other catalase-negative gram-positive cocci. However, some problems arise with the use of these criteria. As an example, the presence of group D antigen can be difficult to detect in some enterococcal isolates. In addition, other organisms such as lactococci also can grow in high salt <span class=\"nowrap\">and/or</span> be bile esculin positive; however, lactococci do not grow (or grow very slowly) at 45&ordm;C.</p><p>Two other genera that share a number of biochemical features with enterococci, including their ability to produce the group D antigen, are <em>Pediococcus </em>and <em>Leuconostoc</em>, which can also grow at 45&ordm;C. These organisms are intrinsically <a href=\"topic.htm?path=vancomycin-drug-information\" class=\"drug drug_general\">vancomycin</a> resistant, which was a reliable distinguishing trait prior to the emergence of vancomycin resistant enterococci (VRE). These genera are now best distinguished from VRE by the addition of tests for production of pyrrolidonyl arylamidase (PYR) and leucine aminopeptidase (LAP). Expected results are: <span class=\"nowrap\">+/+</span> with enterococci; <span class=\"nowrap\">-/+</span> for <em>Pediococcus</em>; and <span class=\"nowrap\">-/-</span> for <em>Leuconostoc</em>; <em>Leuconostoc </em>also produce gas from glucose [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/6\" class=\"abstract_t\">6</a>].</p><p class=\"headingAnchor\" id=\"H4\"><span class=\"h2\">Identification at the species level</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Many laboratories do not identify enterococci to the species level. Prior to the emergence of <a href=\"topic.htm?path=vancomycin-drug-information\" class=\"drug drug_general\">vancomycin</a> resistance, the majority of clinical enterococcal isolates were <em>E. faecalis</em> (85 to 90 percent), but the incidence of <em>E. faecium</em> in the United States has increased (now about 38 percent of healthcare-associated enterococcal isolates), coincident with emergence of <a href=\"topic.htm?path=ampicillin-drug-information\" class=\"drug drug_general\">ampicillin</a> and vancomycin resistance [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/7\" class=\"abstract_t\">7</a>]. Although commercial kits for <em>E. faecalis</em> are generally reliable, commercial strips and kits for non&ndash;<em>E. faecalis</em> have been less reliable.</p><p>With the increase in non-faecalis (primarily <em>E. faecium</em>) and the increased attention paid to vancomycin-resistant organisms, including <em>E. gallinarum</em> and <em>E. casseliflavus</em>, there is renewed interest in identifying these organisms. One reason for this interest is related to species-specific antimicrobial resistance differences that may not be apparent from routine antibiograms.</p><p>For example, <em>E. faecium</em> isolates have, as a species characteristic, moderately high level resistance (minimum inhibitory concentrations approximately 250 <span class=\"nowrap\">mcg/mL)</span> to <a href=\"topic.htm?path=tobramycin-drug-information\" class=\"drug drug_general\">tobramycin</a>, which prevents synergism of tobramycin with beta-lactams. Since this trait will not be detected by the clinical laboratory, use of tobramycin for endocarditis would require identification of the infecting organism to the species level. (See <a href=\"topic.htm?path=antimicrobial-therapy-of-native-valve-endocarditis#H11\" class=\"medical medical_review\">&quot;Antimicrobial therapy of native valve endocarditis&quot;, section on 'Enterococci'</a>.)</p><p>Another reason for species identification is that the appearance of a non-faecalis, non-faecium species in different patients may be a clue to an outbreak secondary to nosocomial spread. Repeated isolation of an unusual species from a single patient may also serve as a clue to relapse from an underlying focus (eg, renal abscess, endocarditis) rather than reinfection by a new organism.</p><p>Enterococcal species were traditionally distinguished based upon results from a complex set of biochemical reactions, including fermentation of carbohydrates, hydrolysis of arginine, tolerance to tellurite, and motility and pigmentation:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Tellurite tolerance with production of black colonies and utilization of pyruvate are characteristic of <em>E. faecalis</em>.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The primary motile species are <em>E. casseliflavus</em> (<em>E. flavescens</em>) and <em>E. gallinarum</em>; yellow pigment is produced by <em>E. casseliflavus</em> and <em>E. mundtii</em>.</p><p/><p>Motility and pigment are particularly useful for distinguishing <em>E. casseliflavus</em> and <em>E. gallinarum</em> from <em>E. faecium</em>, which can be misidentified by kits that do not include these tests. However, occasional isolates display atypical reactions.</p><p>There are a number of difficulties with identification of enterococci to the species level. These include the necessity to perform multiple reactions, the analysis of which is best performed by computer application, and the difficulty in identifying enterococci with atypical phenotypic characteristics [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/8\" class=\"abstract_t\">8</a>]. In many locations, genetic methods have largely replaced these biochemical tests for species identification. These include gene probes, polymerase chain reaction (PCR) for specific genes, 16S rRNA sequencing, and matrix-assisted laser <span class=\"nowrap\">desorption/ionization&ndash;time</span> of flight (MALDI-TOF) [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/9-14\" class=\"abstract_t\">9-14</a>].</p><p>On a practical level, almost all <em>E. faecalis</em> isolates are susceptible to <a href=\"topic.htm?path=ampicillin-drug-information\" class=\"drug drug_general\">ampicillin</a> but resistant to <a href=\"topic.htm?path=quinupristin-and-dalfopristin-drug-information\" class=\"drug drug_general\">quinupristin-dalfopristin</a>, whereas most clinical <em>E. faecium</em> isolates are resistant to ampicillin yet susceptible to quinupristin-dalfopristin.</p><p class=\"headingAnchor\" id=\"H5\"><span class=\"h1\">ANTIBIOTIC RESISTANCE</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Enterococci come well equipped with a variety of intrinsic (ie, naturally occurring species characteristics) antibiotic resistances. They are also capable of acquiring new resistance genes <span class=\"nowrap\">and/or</span> mutations, particularly after exposure to antibiotics. (See <a href=\"topic.htm?path=mechanisms-of-antibiotic-resistance-in-enterococci\" class=\"medical medical_review\">&quot;Mechanisms of antibiotic resistance in enterococci&quot;</a>.)</p><p><em>E. faecium</em> isolates resistant to antibiotics have been recovered from commercial chicken and meat products in the United States. Two studies, for example, reported <a href=\"topic.htm?path=quinupristin-and-dalfopristin-drug-information\" class=\"drug drug_general\">quinupristin-dalfopristin</a>&ndash;resistant <em>E. faecium</em> in up to 78 percent of isolates from the poultry production environment [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/15\" class=\"abstract_t\">15</a>] and 58 percent of chicken carcasses purchased in supermarkets from four states in the United States [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/16\" class=\"abstract_t\">16</a>]. In the latter report, such resistant organisms were cultured in 3 of 334 (0.9 percent) stool specimens from outpatients [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/16\" class=\"abstract_t\">16</a>]. In Europe, <em>E. faecium</em> resistant to <a href=\"topic.htm?path=vancomycin-drug-information\" class=\"drug drug_general\">vancomycin</a> have been identified frequently in food animals, although vancomycin-resistant enterococci belonging to clones common in food animal feces rarely have been found to infect humans.</p><p>It is unclear whether the ingestion of these organisms leads to persistent intestinal carriage in humans. This issue was addressed in a study in which 12 healthy volunteers were fed glycopeptide and streptogramin-resistant <em>E. faecium</em> strains recovered from chickens or pork [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/17\" class=\"abstract_t\">17</a>]. The strains were detectable in the stool of 8 of the 12 volunteers at one week, 1 of 12 at two weeks, and none at five weeks. This observation illustrates the potential for spread of these organisms along with their resistance genes.</p><p class=\"headingAnchor\" id=\"H6\"><span class=\"h2\">Nosocomial infection</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Enterococci are the second to third most common organism recovered from hospital-acquired (nosocomial) infections, particularly in more severely ill patients who have been hospitalized for long periods of time <span class=\"nowrap\">and/or</span> have received multiple antibiotics [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/4,18\" class=\"abstract_t\">4,18</a>]. (See <a href=\"topic.htm?path=vancomycin-resistant-enterococci-epidemiology-prevention-and-control\" class=\"medical medical_review\">&quot;Vancomycin-resistant enterococci: Epidemiology, prevention, and control&quot;</a>.)</p><p>Until the appearance of <a href=\"topic.htm?path=vancomycin-drug-information\" class=\"drug drug_general\">vancomycin</a> resistance, <em>E. faecalis</em> accounted for 85 to 90 percent of enterococcal isolates and <em>E. faecium</em> for &le;10 percent. However, <em>E. faecium</em> has been implicated in up to 40 percent of bloodstream isolates in high-risk populations such as liver or stem cell transplant recipients and up to 35 percent of enterococcal isolates from nosocomial infections [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/19,20\" class=\"abstract_t\">19,20</a>].</p><p>The rise of <em>E. faecium</em> in the hospital setting has been attributed to its many antimicrobial resistances, which allow these organisms to survive and proliferate when antibiotic-susceptible organisms are eliminated [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/21\" class=\"abstract_t\">21</a>]. Initially, most clinical isolates of multiresistant <em>E. faecium</em> appeared to belong to a single clonal complex (CC17) of related multilocus sequence types (MLST). Subsequently, two distinct clades of <em>E. faecium</em> were identified (human commensal [clade B] and animal associated [clade A]) whose core genes differ by an average of 3 to 5 percent and which diverged from each other thousands of years ago [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/22,23\" class=\"abstract_t\">22,23</a>]; it is estimated that, sometime around the 1940s, a subclade designated &quot;A1&quot; emerged from the animal-associated clade (&quot;A2&quot;), and subclade A1 contains almost all the clinical and multidrug-resistant <em>E. faecium</em> strains [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/24\" class=\"abstract_t\">24</a>]. In addition to having more antibiotic resistance genes such as resistance to <a href=\"topic.htm?path=ampicillin-drug-information\" class=\"drug drug_general\">ampicillin</a> and high levels of <a href=\"topic.htm?path=vancomycin-drug-information\" class=\"drug drug_general\">vancomycin</a>, the hospital-associated clade also has genes that animal and community human fecal isolates do not have, including proven and putative virulence and colonization factors, and a much higher rate of mutation. This suggests that the hospital-associated clade is not just more antibiotic resistant but also more capable of adapting to antibiotics, the environment, and possibly to the human immune response [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/24-28\" class=\"abstract_t\">24-28</a>].</p><p>Enterococci can often persist in the face of antibiotics use and therefore have enhanced opportunity to cause opportunistic infection [<a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/29\" class=\"abstract_t\">29</a>], particularly in patients with altered immune defenses <span class=\"nowrap\">and/or</span> multiple potential portals of entry. Enterococci also have intrinsic factors such as a number of potential adhesin genes, which may explain the propensity of enterococci to cause endocarditis.</p><p>Once enterococci cause infection, their antibiotic resistances can create even greater dilemmas. Our worst therapeutic problems arise when a multidrug-resistant hospital-acquired <em>E. faecium</em> causes endocarditis. (See <a href=\"topic.htm?path=treatment-of-enterococcal-infections\" class=\"medical medical_review\">&quot;Treatment of enterococcal infections&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H90634767\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Enterococci (formerly called group D streptococci) are gram-positive cocci of intestinal origin that usually form short chains and are a common cause of endocarditis and nosocomial infections. (See <a href=\"#H1\" class=\"local\">'Introduction'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Microbiologic characteristics of enterococci include reactivity with group D antiserum in approximately 80 percent; ability to grow in high salt, at 10&ordm;C and 40&ordm;C, and at high pH; and survival at 60&ordm;C for 30 minutes. They hydrolyze esculin in the presence of bile and produce pyrrolidonyl arylamidase (PYR). (See <a href=\"#H2\" class=\"local\">'Microbiology classification'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Species differentiation, traditionally based on results from a complex set of biochemical reactions but more recently on DNA-based testing, is important, as species-specific antimicrobial resistance differences can affect therapeutic decisions. The identification of non-faecium, non-faecalis species can be helpful in detecting nosocomial outbreaks or relapsing infection. (See <a href=\"#H4\" class=\"local\">'Identification at the species level'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Almost all <em>E. faecalis</em> isolates are susceptible to <a href=\"topic.htm?path=ampicillin-drug-information\" class=\"drug drug_general\">ampicillin</a> but resistant to <a href=\"topic.htm?path=quinupristin-and-dalfopristin-drug-information\" class=\"drug drug_general\">quinupristin-dalfopristin</a>. Most clinical <em>E. faecium</em> isolates are resistant to ampicillin, susceptible to quinupristin-dalfopristin, and have moderately high-level resistance to <a href=\"topic.htm?path=tobramycin-drug-information\" class=\"drug drug_general\">tobramycin</a>; in the United States, most clinical <em>E. faecium</em> are resistant to moderate to high levels of <a href=\"topic.htm?path=vancomycin-drug-information\" class=\"drug drug_general\">vancomycin</a>. <em>E. gallinarum</em> and <em>E. casseliflavus</em> are intrinsically resistant to low levels of vancomycin. (See <a href=\"#H4\" class=\"local\">'Identification at the species level'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Enterococci resistant to antibiotics have been recovered from commercial chicken and meat products in the United States, but it is unclear whether the ingestion of these organisms leads to persistent intestinal carriage in humans. (See <a href=\"#H5\" class=\"local\">'Antibiotic resistance'</a> above and <a href=\"topic.htm?path=mechanisms-of-antibiotic-resistance-in-enterococci\" class=\"medical medical_review\">&quot;Mechanisms of antibiotic resistance in enterococci&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Their many antimicrobial resistances, which allow them to survive, proliferate, and fill the void when antibiotic-susceptible organisms are eliminated, likely contribute to the frequency of enterococci in nosocomial infections. (See <a href=\"#H6\" class=\"local\">'Nosocomial infection'</a> above.)</p><p/></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/1\" class=\"nounderline abstract_t\">Murray BE. The life and times of the Enterococcus. Clin Microbiol Rev 1990; 3:46.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/2\" class=\"nounderline abstract_t\">Wade JJ. Enterococcus faecium in hospitals. Eur J Clin Microbiol Infect Dis 1997; 16:113.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/3\" class=\"nounderline abstract_t\">Gross PA, Harkavy LM, Barden GE, Flower MF. The epidemiology of nosocomial enterococcal urinary tract infection. Am J Med Sci 1976; 272:75.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/4\" class=\"nounderline abstract_t\">Emori TG, Gaynes RP. An overview of nosocomial infections, including the role of the microbiology laboratory. Clin Microbiol Rev 1993; 6:428.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/5\" class=\"nounderline abstract_t\">Schleifer KH, Kilpper-Balz R. Transfer of Streptococcus faecalis and Streptococcus faecium to the genus Enterococcus nom. rev. as Enterococcus faecalis comb. nov. and Enterococcus faecium comb. Int J Syst Bacteriol 1984; 34:31.</a></li><li class=\"breakAll\">Teixeira LM, Carvalho MS, Facklam RR. Enterococcus. In: Manual of Clinical Microbiology, Murray PR, Baron EJ, Jorgensen JH, LLandry ML, Pfaller MA (Eds), ASM Press, Washington, DC 2007.</li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/7\" class=\"nounderline abstract_t\">Hidron AI, Edwards JR, Patel J, et al. NHSN annual update: antimicrobial-resistant pathogens associated with healthcare-associated infections: annual summary of data reported to the National Healthcare Safety Network at the Centers for Disease Control and Prevention, 2006-2007. Infect Control Hosp Epidemiol 2008; 29:996.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/8\" class=\"nounderline abstract_t\">Teixeira LM, Facklam RR, Steigerwalt AG, et al. Correlation between phenotypic characteristics and DNA relatedness within Enterococcus faecium strains. J Clin Microbiol 1995; 33:1520.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/9\" class=\"nounderline abstract_t\">Coque TM, Murray BE. Identification of Enterococcus faecalis strains by DNA hybridization and pulsed-field gel electrophoresis. J Clin Microbiol 1995; 33:3368.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/10\" class=\"nounderline abstract_t\">Tyrrell GJ, Bethune RN, Willey B, Low DE. Species identification of enterococci via intergenic ribosomal PCR. J Clin Microbiol 1997; 35:1054.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/11\" class=\"nounderline abstract_t\">Depardieu F, Perichon B, Courvalin P. Detection of the van alphabet and identification of enterococci and staphylococci at the species level by multiplex PCR. J Clin Microbiol 2004; 42:5857.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/12\" class=\"nounderline abstract_t\">Martinez RM, Bauerle ER, Fang FC, Butler-Wu SM. Evaluation of three rapid diagnostic methods for direct identification of microorganisms in positive blood cultures. J Clin Microbiol 2014; 52:2521.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/13\" class=\"nounderline abstract_t\">Deck MK, Anderson ES, Buckner RJ, et al. Rapid detection of Enterococcus spp. direct from blood culture bottles using Enterococcus QuickFISH method: a multicenter investigation. Diagn Microbiol Infect Dis 2014; 78:338.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/14\" class=\"nounderline abstract_t\">Fang H, Ohlsson AK, Ullberg M, Ozenci V. Evaluation of species-specific PCR, Bruker MS, VITEK MS and the VITEK 2 system for the identification of clinical Enterococcus isolates. Eur J Clin Microbiol Infect Dis 2012; 31:3073.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/15\" class=\"nounderline abstract_t\">Hayes JR, McIntosh AC, Qaiyumi S, et al. High-frequency recovery of quinupristin-dalfopristin-resistant Enterococcus faecium isolates from the poultry production environment. J Clin Microbiol 2001; 39:2298.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/16\" class=\"nounderline abstract_t\">McDonald LC, Rossiter S, Mackinson C, et al. Quinupristin-dalfopristin-resistant Enterococcus faecium on chicken and in human stool specimens. N Engl J Med 2001; 345:1155.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/17\" class=\"nounderline abstract_t\">S&oslash;rensen TL, Blom M, Monnet DL, et al. Transient intestinal carriage after ingestion of antibiotic-resistant Enterococcus faecium from chicken and pork. N Engl J Med 2001; 345:1161.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/18\" class=\"nounderline abstract_t\">Schaberg DR, Culver DH, Gaynes RP. Major trends in the microbial etiology of nosocomial infection. Am J Med 1991; 91:72S.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/19\" class=\"nounderline abstract_t\">Mikulska M, Del Bono V, Raiola AM, et al. Blood stream infections in allogeneic hematopoietic stem cell transplant recipients: reemergence of Gram-negative rods and increasing antibiotic resistance. Biol Blood Marrow Transplant 2009; 15:47.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/20\" class=\"nounderline abstract_t\">Bedini A, Codeluppi M, Cocchi S, et al. Gram-positive bloodstream infections in liver transplant recipients: incidence, risk factors, and impact on survival. Transplant Proc 2007; 39:1947.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/21\" class=\"nounderline abstract_t\">Suppola JP, Volin L, Valtonen VV, Vaara M. Overgrowth of Enterococcus faecium in the feces of patients with hematologic malignancies. Clin Infect Dis 1996; 23:694.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/22\" class=\"nounderline abstract_t\">Galloway-Pe&ntilde;a J, Roh JH, Latorre M, et al. Genomic and SNP analyses demonstrate a distant separation of the hospital and community-associated clades of Enterococcus faecium. PLoS One 2012; 7:e30187.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/23\" class=\"nounderline abstract_t\">Palmer KL, Godfrey P, Griggs A, et al. Comparative genomics of enterococci: variation in Enterococcus faecalis, clade structure in E. faecium, and defining characteristics of E. gallinarum and E. casseliflavus. MBio 2012; 3:e00318.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/24\" class=\"nounderline abstract_t\">Lebreton F, van Schaik W, McGuire AM, et al. Emergence of epidemic multidrug-resistant Enterococcus faecium from animal and commensal strains. MBio 2013; 4.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/25\" class=\"nounderline abstract_t\">Rice LB, Carias L, Rudin S, et al. A potential virulence gene, hylEfm, predominates in Enterococcus faecium of clinical origin. J Infect Dis 2003; 187:508.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/26\" class=\"nounderline abstract_t\">Nallapareddy SR, Weinstock GM, Murray BE. Clinical isolates of Enterococcus faecium exhibit strain-specific collagen binding mediated by Acm, a new member of the MSCRAMM family. Mol Microbiol 2003; 47:1733.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/27\" class=\"nounderline abstract_t\">Leavis HL, Bonten MJ, Willems RJ. Identification of high-risk enterococcal clonal complexes: global dispersion and antibiotic resistance. Curr Opin Microbiol 2006; 9:454.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/28\" class=\"nounderline abstract_t\">Zhang X, Top J, de Been M, et al. Identification of a genetic determinant in clinical Enterococcus faecium strains that contributes to intestinal colonization during antibiotic treatment. J Infect Dis 2013; 207:1780.</a></li><li><a href=\"https://www.uptodate.com/contents/microbiology-of-enterococci/abstract/29\" class=\"nounderline abstract_t\">Sillanp&auml;&auml; J, Prakash VP, Nallapareddy SR, Murray BE. Distribution of genes encoding MSCRAMMs and Pili in clinical and natural populations of Enterococcus faecium. J Clin Microbiol 2009; 47:896.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 3160 Version 13.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H90634767\"><span>SUMMARY</span></a></li><li><a href=\"#H1\" id=\"outline-link-H1\">INTRODUCTION</a></li><li><a href=\"#H2\" id=\"outline-link-H2\">MICROBIOLOGY CLASSIFICATION</a><ul><li><a href=\"#H3\" id=\"outline-link-H3\">Problems with classification</a></li><li><a href=\"#H4\" id=\"outline-link-H4\">Identification at the species level</a></li></ul></li><li><a href=\"#H5\" id=\"outline-link-H5\">ANTIBIOTIC RESISTANCE</a><ul><li><a href=\"#H6\" id=\"outline-link-H6\">Nosocomial infection</a></li></ul></li><li><a href=\"#H90634767\" id=\"outline-link-H90634767\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=antimicrobial-therapy-of-native-valve-endocarditis\" class=\"medical medical_review\">Antimicrobial therapy of native valve endocarditis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=mechanisms-of-antibiotic-resistance-in-enterococci\" class=\"medical medical_review\">Mechanisms of antibiotic resistance in enterococci</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=treatment-of-enterococcal-infections\" class=\"medical medical_review\">Treatment of enterococcal infections</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=vancomycin-resistant-enterococci-epidemiology-prevention-and-control\" class=\"medical medical_review\">Vancomycin-resistant enterococci: Epidemiology, prevention, and control</a></li></ul></div></div>","javascript":null}