var data={"title":"Overview of genetic variation","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Overview of genetic variation</div><dl id=\"topicContributors\"><dt><span> </span>Author:</dt><dd><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/contributors\" class=\"contributor contributor_credentials\">Benjamin A Raby, MD, MPH</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/contributors\" class=\"contributor contributor_credentials\">Anne Slavotinek, MBBS, PhD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/contributors\" class=\"contributor contributor_credentials\">Jennifer S Tirnauer, MD</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Mar 27, 2018.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H1\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The human genome is rich in variation. Until recently, only a handful of common variants, termed polymorphisms (chromosome or sequence variants found in more than 1 percent of the population), have been implicated in common disease, although the majority of heritable phenotypic variation is due to alterations in DNA sequence. In contrast, the genetics of Mendelian conditions caused by rare mutations that occur sporadically or that are inherited have been better characterized.</p><p>The Human Genome Project, completed in 2003, elucidated the complete sequence of the human reference genome [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/1\" class=\"abstract_t\">1</a>]. This sequence, together with the availability of DNA genotyping and sequencing methods capable of surveying variation in large numbers of individuals on a genome-wide scale, has led to rapid expansion of the list of common genetic variants that contribute to disease susceptibility, disease progression, and variability in treatment response.</p><p>This topic will review the types of genetic variation implicated in health and disease, with a particular focus on common genetic variants, including short tandem repeats (STRs), single nucleotide polymorphisms (SNPs), and copy number variants (CNVs). </p><p>A glossary of genetic terms is available separately. (See <a href=\"topic.htm?path=genetics-glossary-of-terms\" class=\"medical medical_review\">&quot;Genetics: Glossary of terms&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H2\"><span class=\"h1\">RARE CHROMOSOMAL ABERRATIONS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Chromosomal abnormalities that are visible using light microscopy can be classified into those due to aberrant chromosome number and those due to abnormal chromosome structure. These variants can result in well-characterized syndromes, although many structural abnormalities are rare or unique. Chromosome aberrations typically result from either chromosome recombination errors during meiosis (for germline mutations) or aberrant chromosomal segregation during meiosis or mitosis. Large-scale variants can be evaluated using a variety of cytogenetic approaches, including karyotyping, fluorescence in-situ hybridization (FISH), and array comparative genome hybridization (array CGH) applications. (See <a href=\"#H4\" class=\"local\">'Aneuploidy'</a> below and <a href=\"#H6\" class=\"local\">'Structural chromosome abnormalities'</a> below and <a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Cytogenetics and molecular genetics&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H3\"><span class=\"h2\">Numerical chromosome abnormalities</span></p><p class=\"headingAnchor\" id=\"H4\"><span class=\"h3\">Aneuploidy</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Aneuploidy refers to an abnormal copy number of one or more chromosome(s). This is thought to result during cell division, either through non-disjunction of paired chromosomes in meiosis (<a href=\"image.htm?imageKey=PC%2F52882%7EPC%2F81597\" class=\"graphic graphic_figure graphicRef52882 graphicRef81597 \">figure 1A-B</a>) or through anaphase lag. Normal somatic cells (ie, non-egg and sperm cells) typically are diploid for all 22 autosomes (non-sex-linked chromosomes) and have two sex chromosomes, either XX or XY for the female and male, respectively. Monosomy refers to loss of a chromosome. Trisomy refers to gain of a chromosome.</p><p>All autosomal monosomies (loss of an autosomal chromosome) are lethal in the embryonic period, unless they occur in the setting of mosaicism (ie, a mixture of monosomic and karyotypically normal cell types) in the developing embryo. </p><p>Some autosomal trisomies are compatible with life:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Trisomy 21, also known as Down syndrome, is often compatible with survival into adulthood. Down syndrome is characterized by variable expression of a range of clinical manifestations, including cognitive impairment, muscle hypotonia, congenital heart disease, and characteristic facial features (round facies, upslanting palpebral (eye) fissures, flattened nasal bridge, and macroglossia) [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/2\" class=\"abstract_t\">2</a>]. (See <a href=\"topic.htm?path=congenital-cytogenetic-abnormalities\" class=\"medical medical_review\">&quot;Congenital cytogenetic abnormalities&quot;</a> and <a href=\"topic.htm?path=down-syndrome-clinical-features-and-diagnosis\" class=\"medical medical_review\">&quot;Down syndrome: Clinical features and diagnosis&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Trisomy 18 (Edwards syndrome) and trisomy 13 (Patau syndrome) are caused by one additional copy of chromosome 18 and 13, respectively. Although babies with both of these autosomal trisomies have been reported to survive to term, there is a significantly shortened life expectancy. </p><p/><p>Sex-linked aneuploidies compatible with life include the following:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Females with only one X chromosome (monosomy X or XO) have Turner syndrome and can have a normal life span. Turner syndrome occurs in approximately one in every 2500 female births and is characterized by short stature, webbed neck, and gonadal dysfunction [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/3\" class=\"abstract_t\">3</a>]. Additional manifestations may include congenital heart disease (coarctation of the aorta), renal anomalies, lymphedema of the hands and feet, and hypothyroidism. Overall intellect is considered to be normal, although there may be individual specific strengths (for example, verbal skills) and challenges (for example, tasks that require visuospatial or math skills). Management guidelines for females with Turner syndrome have been published [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/4\" class=\"abstract_t\">4</a>]. (See <a href=\"topic.htm?path=clinical-manifestations-and-diagnosis-of-turner-syndrome\" class=\"medical medical_review\">&quot;Clinical manifestations and diagnosis of Turner syndrome&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>47,XXY, also known as Klinefelter syndrome, occurs in approximately 1 out of every 500 to 1 in 1000 male births. 47,XXY males are infertile due to hypogonadism, although residual foci of spermatogonia and testicular sperm extraction in combination with intracytoplasmic sperm injection may allow some men with Klinefelter syndrome to father children [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/5\" class=\"abstract_t\">5</a>]. Other manifestations include cognitive or behavioral problems, microorchidism, and gynecomastia. (See <a href=\"topic.htm?path=causes-of-primary-hypogonadism-in-males#H3\" class=\"medical medical_review\">&quot;Causes of primary hypogonadism in males&quot;, section on 'Klinefelter syndrome'</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The presence of an extra Y chromosome (47,XYY) is usually of little clinical consequence, other than increased growth velocity in early childhood. Males who are 47,XYY generally have normal physical attributes and are fertile. The diagnosis is usually only made when genetic testing is performed for other reasons. (See <a href=\"topic.htm?path=the-child-with-tall-stature-and-or-abnormally-rapid-growth#H21\" class=\"medical medical_review\">&quot;The child with tall stature and/or abnormally rapid growth&quot;, section on '47,XYY syndrome'</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Triple X syndrome (47,XXX) occurs in one in 1000 female births. 47,XXX females, similar to 47,XXY males, display increased growth velocity but are usually otherwise asymptomatic, although mild learning disabilities with difficulties with motor coordination and auditory processing disorders have been reported [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/6\" class=\"abstract_t\">6</a>]. </p><p/><p class=\"headingAnchor\" id=\"H5\"><span class=\"h3\">Polyploidy</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Polyploidy refers to abnormal copy number of all chromosomes, typically manifesting as triploidy (69,XXX, 69,XXY, or 69,XYY) resulting from ovum fertilization by two spermatozoa (dispermy). Approximately 1 to 3 percent of pregnancies are triploid, which results in spontaneous abortion in early- to mid-pregnancy.</p><p class=\"headingAnchor\" id=\"H6\"><span class=\"h2\">Structural chromosome abnormalities</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Structural chromosome abnormalities are a class of DNA sequence alterations that span more than 1000 base pairs (ie, one kilobase [kb]). There are a number of different categories of structural chromosomal abnormalities that include the following:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Copy number variations</strong> &mdash; A copy number variation refers to a nucleotide sequence, typically of at least 1000 bases in length, that varies in copy number between different individuals. A deleted region is termed a &ldquo;loss&rdquo;; regions that are duplicated one or more times are termed &ldquo;gains.&rdquo; (See <a href=\"#H14\" class=\"local\">'Copy number variations (CNVs)'</a> below and <a href=\"topic.htm?path=genomic-disorders-an-overview#H16722199\" class=\"medical medical_review\">&quot;Genomic disorders: An overview&quot;, section on 'Copy number variations'</a>.) </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Chromosomal translocations</strong> &mdash; A chromosomal translocation occurs when a portion of one chromosome becomes fused with a portion of another non-homologous chromosome (<a href=\"image.htm?imageKey=PC%2F71519\" class=\"graphic graphic_figure graphicRef71519 \">figure 2</a>). A commonly-used example is the translocation observed in chronic myeloid leukemia (CML) (<a href=\"image.htm?imageKey=PC%2F70807\" class=\"graphic graphic_figure graphicRef70807 \">figure 3</a>). The Philadelphia chromosome results from a reciprocal translocation between chromosomes 9 and 22 [t(9;22)(q34;q11)], causing a fusion gene and predisposing to CML. (See <a href=\"topic.htm?path=chromosomal-translocations-deletions-and-inversions#H6\" class=\"medical medical_review\">&quot;Chromosomal translocations, deletions, and inversions&quot;, section on 'Translocations'</a> and <a href=\"topic.htm?path=cellular-and-molecular-biology-of-chronic-myeloid-leukemia\" class=\"medical medical_review\">&quot;Cellular and molecular biology of chronic myeloid leukemia&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Chromosome deletions</strong> &mdash; Chromosomal deletion refers to loss of a portion of a chromosome; typically many genes are affected. When disease-causing genes are deleted, the resultant haploinsufficiency can cause clinical effects, for example deletion of the peripheral myelin protein 22 or <em>PMP22</em> gene causes hereditary neuropathy with liability to pressure palsies. (See <a href=\"topic.htm?path=chromosomal-translocations-deletions-and-inversions#H10\" class=\"medical medical_review\">&quot;Chromosomal translocations, deletions, and inversions&quot;, section on 'Deletions'</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Chromosome duplications</strong> &mdash; A chromosomal duplication occurs when a region of the chromosome is duplicated, resulting in the presence of twice as many copies of the duplicated region than seen on the normal chromosome. When disease-causing genes are duplicated, the resultant increase in copy number can cause clinical effects. For example, duplication of the peripheral myelin protein 22 or <em>PMP22</em> gene can cause Charcot-Marie-Tooth disease type IA. (See <a href=\"topic.htm?path=chromosomal-translocations-deletions-and-inversions\" class=\"medical medical_review\">&quot;Chromosomal translocations, deletions, and inversions&quot;</a> and <a href=\"topic.htm?path=charcot-marie-tooth-disease-genetics-clinical-features-and-diagnosis\" class=\"medical medical_review\">&quot;Charcot-Marie-Tooth disease: Genetics, clinical features, and diagnosis&quot;</a>.) </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Chromosome inversions</strong> &mdash; A chromosome inversion occurs when a region of a chromosome is excised and reinserted into the chromosome in the opposite orientation (<a href=\"image.htm?imageKey=OBGYN%2F65339\" class=\"graphic graphic_figure graphicRef65339 \">figure 4</a>). The number of copies of each gene is normal, but the change in order may affect gene regulation. Inversions of disease-causing genes can interrupt transcripts or change their expression level, for example a gene inversion associated with factor VIII deficiency has been identified in 40 percent of patients with hemophilia A [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/7\" class=\"abstract_t\">7</a>]. (See <a href=\"topic.htm?path=chromosomal-translocations-deletions-and-inversions#H13\" class=\"medical medical_review\">&quot;Chromosomal translocations, deletions, and inversions&quot;, section on 'Inversions'</a> and <a href=\"topic.htm?path=genetics-of-hemophilia-a-and-b\" class=\"medical medical_review\">&quot;Genetics of hemophilia A and B&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Ring chromosome formation</strong> &mdash; A ring chromosome can form when the two ends of the chromosome become fused, creating a ring structure. Ring chromosomes commonly result from breakage and loss of chromosome material at the ends of both &lsquo;p&rsquo; and &lsquo;q&rsquo; arms of the same chromosome, with subsequent rejoining of the ends of the arms. Ring chromosomes are often unstable and frequently associated with deleterious phenotypic effects.</p><p/><p>These and other structural chromosome differences, such as acentric or dicentric chromosomes or isochromosomes, result through abnormal chromosome breakage repair or non-homologous recombination errors (<a href=\"image.htm?imageKey=PC%2F67644\" class=\"graphic graphic_figure graphicRef67644 \">figure 5</a>). Non-homologous recombination &ldquo;hotspots&rdquo; are thought to contribute to several disorders, including Duchenne muscular dystrophy and osteogenesis imperfecta [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/8-10\" class=\"abstract_t\">8-10</a>]. Gene inversions related to factor VIII deficiency may result from mechanisms related to abnormal chromatin folding patterns [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/11\" class=\"abstract_t\">11</a>]. (See <a href=\"topic.htm?path=duchenne-and-becker-muscular-dystrophy-clinical-features-and-diagnosis\" class=\"medical medical_review\">&quot;Duchenne and Becker muscular dystrophy: Clinical features and diagnosis&quot;</a> and <a href=\"topic.htm?path=inherited-syndromes-associated-with-cardiac-disease\" class=\"medical medical_review\">&quot;Inherited syndromes associated with cardiac disease&quot;</a> and <a href=\"topic.htm?path=osteogenesis-imperfecta-clinical-features-and-diagnosis\" class=\"medical medical_review\">&quot;Osteogenesis imperfecta: Clinical features and diagnosis&quot;</a>.)</p><p>Structural abnormalities influence gene function in several ways, including deletion or duplication of phenotypically relevant genes, or gene fusion to create a novel gene and transcript. Fusion may result in an inappropriate regulatory sequence and aberrant gene expression, or juxtaposition of two coding sequences to produce novel fusion proteins (<a href=\"image.htm?imageKey=PC%2F65092\" class=\"graphic graphic_figure graphicRef65092 \">figure 6</a>) [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/12,13\" class=\"abstract_t\">12,13</a>].</p><p>Gene fusion is the cause of CML and some of the primary (neoplastic) hypereosinophilic syndromes. CML results from fusion of the <em>BCR</em> and <em>ABL </em>genes. Some cases of primary hypereosinophilic syndrome are caused by fusion of the FIP1-like-1 (<em>FIP1L1</em>) and platelet-derived growth factor receptor alpha (<em>PDGFRA</em>) genes. The resulting fusion proteins in both cases are hyperfunctional tyrosine kinases with uncontrolled activity that leads to dysregulated clonal cellular expansion. The activity of both kinases can be blocked by <a href=\"topic.htm?path=imatinib-drug-information\" class=\"drug drug_general\">imatinib</a>, resulting in clinical and molecular remission of both conditions. (See <a href=\"topic.htm?path=cellular-and-molecular-biology-of-chronic-myeloid-leukemia\" class=\"medical medical_review\">&quot;Cellular and molecular biology of chronic myeloid leukemia&quot;</a> and <a href=\"topic.htm?path=hypereosinophilic-syndromes-clinical-manifestations-pathophysiology-and-diagnosis\" class=\"medical medical_review\">&quot;Hypereosinophilic syndromes: Clinical manifestations, pathophysiology, and diagnosis&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H7\"><span class=\"h1\">COMMON GENETIC VARIATION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Large chromosomal aberrations can dramatically impact clinical phenotype and are frequently implicated in rare genetic syndromes. Smaller-sized variants, however, are far more abundant, and are responsible for most of the genetic variance for heritable traits in general populations. These variants include single nucleotide substitutions (mutations or single nucleotide polymorphisms), insertions and deletions (indels), copy number variants, and short tandem repeats. In many cases, the genetic variant detected is a marker for another genetic defect at a nearby locus rather than a cause of the phenotypic abnormality. </p><p class=\"headingAnchor\" id=\"H10\"><span class=\"h2\">Single nucleotide polymorphisms (SNPs)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Single nucleotide polymorphisms (SNPs; ie, substitutions of one nucleotide for another) represent the most abundant form of genetic variation and are responsible for much of the heritable phenotypic variation observed in human populations. Estimates based on whole genome sequencing efforts suggest that individuals carry between 2.8 and 3.9 million single-base pair variants, with higher estimates observed in individuals of African descent [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/14,15\" class=\"abstract_t\">14,15</a>]. </p><p>Estimates suggest that unrelated haploid genomes differ every 185 to 2000 nucleotides. This wide range is due in part to the variable rate of polymorphism across the genome, with some regions demonstrating high heterozygosity (such as the MHC cluster on chromosome 6), while others display more restricted variation. It has been proposed that this is likely to do evolutionary constraint of critical functional elements [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/15-19\" class=\"abstract_t\">15-19</a>]. Local heterozygosity rates within genes also differ substantially, with lower rates of base substitution in coding regions of genes compared to untranslated (non-coding) regions, upstream promoter sequences, or exon-intron boundaries [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/18\" class=\"abstract_t\">18</a>].</p><p>By convention, single-base pair changes that achieve a population frequency of at least 1 percent are referred to as SNPs. Less-frequent substitutions that interrupt gene function, or that have clinical consequences, are labeled as mutations, even though many of these are observed at a fixed frequency (albeit below 1 percent) across diverse populations. The distinction between a SNP and a mutation is largely semantic. Differences in the frequency distribution of SNPs result from the combination of effects of natural selection, genetic drift, and other population genetic effects. The functional impact of a variant is not determined by its frequency, but rather by its location within genes or non-genic regulatory regions. Single-base substitutions resulting in SNPs or mutations arise through shared mechanisms, primarily single-base slip mispairing during DNA replication and CpG-mediated cytosine deamination [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/20\" class=\"abstract_t\">20</a>].</p><p class=\"headingAnchor\" id=\"H11\"><span class=\"h3\">Monogenic disease</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>More than 65 percent of mutations known to cause disease are single nucleotide substitutions [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/21\" class=\"abstract_t\">21</a>]. The majority of such loci identified to date represent highly penetrant, rare coding variants that alter the amino acid composition of proteins, resulting in deleterious phenotypic consequences such as sickle cell anemia, alpha-1 antitrypsin deficiency, and hemochromatosis.</p><p class=\"headingAnchor\" id=\"H12\"><span class=\"h3\">Genetic basis of complex disease</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Identification of common variation that confers susceptibility to common, complex diseases, such as hypertension, has been more challenging. These common variants typically confer much weaker effects than variants associated with monogenic diseases and must interact with other genes and environmental triggers to establish disease. Mapping of these variants has been facilitated by advances in high-throughput genotyping with SNPs and next-generation sequencing.</p><p>It is anticipated that the number of proven disease susceptibility variants will grow exponentially over the next few years. Genome-wide association studies (GWAS) have emerged as a powerful means of identifying common disease-susceptibility variants and have been applied across a wide range of conditions [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/22\" class=\"abstract_t\">22</a>]. Such studies are supported by the availability of large, well-characterized populations; robust microarray technology enabling genotyping of hundreds of thousands of variants; and the development of powerful analytic tools and statistical methods. (See <a href=\"topic.htm?path=genetic-association-studies-principles-and-applications\" class=\"medical medical_review\">&quot;Genetic association studies: Principles and applications&quot;</a>.) </p><p>The National Heart, Lung, and Blood Institute maintains a searchable database called GRASP (Genome-Wide Repository of Associations Between SNPs and Phenotypes) that includes all GWAS studies published in English with at least 25,000 variants [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/23,24\" class=\"abstract_t\">23,24</a>]. The Build 2.0.0.0 data release includes all studies published through December 2014. It contains information from 2082 studies, with results for 177 broad phenotype categories. In addition to trait-related results, more than 1 million results are also available related to quantitative trait locus (QTL) studies of gene-expression (eQTL), metabolomics (mQTL), DNA methylation (meQTL), protein glycosylation (gQTL), protein levels (pQTL), and microRNA expression (mirQTL). GRASP is searchable by phenotype, and all results and study details are freely available for download.</p><p>GWAS have been complemented by other gene identification strategies, such as next-generation sequencing and systems biology approaches, as it has been recognized that susceptibility loci identified by GWAS account for only a small proportion of the estimated heritability of common, complex traits [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/22,25,26\" class=\"abstract_t\">22,25,26</a>]. These other gene identification strategies are discussed separately. (See <a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">&quot;Next-generation DNA sequencing (NGS): Principles and clinical applications&quot;</a> and <a href=\"topic.htm?path=personalized-medicine\" class=\"medical medical_review\">&quot;Personalized medicine&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H13\"><span class=\"h2\">Insertion/deletion polymorphism</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span><span class=\"nowrap\">Insertion/deletion</span> (or indel) polymorphisms are polymorphic sites of gains or losses of one or more nucleotides (<a href=\"image.htm?imageKey=PC%2F69699\" class=\"graphic graphic_figure graphicRef69699 \">figure 7</a>). Most indels range in length from 1 to 4 nucleotides; the vast majority are a single nucleotide.</p><p>Indels are distributed throughout the genome and represent approximately 10 percent of human DNA variation [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/15\" class=\"abstract_t\">15</a>]. Though less common than single-base substitutions (~350,000 per individual, <span class=\"nowrap\">~1/10<sup>th</sup></span> of the number of SNPs), indels represent a considerable proportion of sequence variation in human populations. Like single-base substitutions, most indels (particularly insertions) arise through slip-mispairing [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/27,28\" class=\"abstract_t\">27,28</a>]. When situated in the coding sequence of a gene, indels typically result in frameshift mutations that can alter protein structure and function (<a href=\"image.htm?imageKey=PC%2F69699\" class=\"graphic graphic_figure graphicRef69699 \">figure 7</a>). If a frameshift mutation causes a premature stop codon, the transcript may undergo nonsense mediated decay, leading to loss of an allele. Indels constitute a substantial proportion of variation contributing to disease pathogenesis, representing more than 25 percent of the disease-associated variants in the Human Gene Mutation Database [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/29\" class=\"abstract_t\">29</a>].</p><p>Most pathogenic indels are frameshift mutations. However, insertions or deletions of three or six nucleotides can have pathogenic consequences despite maintaining normal reading frames. As an example, a 3-nucleotide (CTT) deletion at codon 508 (phenylalanine) of the <em>CFTR</em> gene (DeltaF508) is responsible for 70 percent of cystic fibrosis cases in Caucasian populations. This mutation prevents normal trafficking of mature CFTR protein to the cell surface [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/30-32\" class=\"abstract_t\">30-32</a>]. (See <a href=\"topic.htm?path=cystic-fibrosis-genetics-and-pathogenesis\" class=\"medical medical_review\">&quot;Cystic fibrosis: Genetics and pathogenesis&quot;</a>.)</p><p>Larger indels have also been implicated in human disease, including a 32 base pair deletion in the coding region of chemokine-receptor 5 (<em>CCR5</em>), observed in 16 percent of Caucasians. <em>CCR5</em> Delta32 substantially impairs HIV infection of T cells by impairing normal CCR5 function, a T-cell surface receptor, which serves as a co-receptor for HIV transfection. Thus, homozygotes for this indel are protected from HIV infection, and the rate of disease progression is slowed among heterozygote carriers [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/33-35\" class=\"abstract_t\">33-35</a>].</p><p class=\"headingAnchor\" id=\"H14\"><span class=\"h2\">Copy number variations (CNVs)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Copy number variations (CNVs) are a prevalent type of structural genetic variation (<a href=\"image.htm?imageKey=PC%2F67644\" class=\"graphic graphic_figure graphicRef67644 \">figure 5</a>). They involve DNA segments spanning thousands to millions of bases whose copy number varies between different individuals. These submicroscopic genomic differences in the number of copies of one or more sections of DNA result in DNA gains or losses. CNVs may be associated with phenotypic effects, but in other cases they are clinically silent. An extensive discussion of CNVs is presented elsewhere. (See <a href=\"topic.htm?path=genomic-disorders-an-overview#H16722199\" class=\"medical medical_review\">&quot;Genomic disorders: An overview&quot;, section on 'Copy number variations'</a>.)</p><p class=\"headingAnchor\" id=\"H1423422\"><span class=\"h2\">Short tandem repeat markers</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Short tandem repeats (STRs) are sequences of two to six base pairs that repeat in tandem and are flanked by non-repetitive sequence. Genetic variation is conferred by the variable length of the sequence due to variation in the number of tandem repeats (<a href=\"image.htm?imageKey=PC%2F82363\" class=\"graphic graphic_figure graphicRef82363 \">figure 8</a>). This polymorphism is generated by slip-mispairing during DNA replication, with formation of DNA loops and back-priming of the leading strand [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/36,37\" class=\"abstract_t\">36,37</a>]. The longer a segment of repetitive DNA, the more susceptible it is to slip-mispairing. Thus, mutation rates for STRs are substantially higher than for single-base pair mutations. Although previously widely used, STRs have largely been replaced by single nucleotide polymorphisms (SNPs) and copy number variants (CNVs) as markers of genetic variation. </p><p>Disease-causing expansions of tandem repeat sequences (trinucleotide repeats) have been described in coding regions of genes, intronic sequences, and 5' untranslated regions (5&rsquo;UTR) of sequence [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/38-40\" class=\"abstract_t\">38-40</a>] and are mutated in several neurogenetic conditions. These include Fragile X syndrome (due to expansion of a CGG repeat sequence in the 5'UTR of the <em>FMR1</em> gene), myotonic dystrophy (due to expansion of a CTG repeat in the 3'UTR of the <em>DMPK</em> gene) (<a href=\"image.htm?imageKey=PC%2F82363\" class=\"graphic graphic_figure graphicRef82363 \">figure 8</a>) [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/41-43\" class=\"abstract_t\">41-43</a>], and Huntington disease (due to CAG expansion within the coding region of the <em>HD</em> gene that encodes Huntingtin) [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/38\" class=\"abstract_t\">38</a>]. (See <a href=\"topic.htm?path=myotonic-dystrophy-etiology-clinical-features-and-diagnosis\" class=\"medical medical_review\">&quot;Myotonic dystrophy: Etiology, clinical features, and diagnosis&quot;</a> and <a href=\"topic.htm?path=prenatal-screening-and-diagnosis-for-fragile-x-syndrome\" class=\"medical medical_review\">&quot;Prenatal screening and diagnosis for fragile X syndrome&quot;</a> and <a href=\"topic.htm?path=huntington-disease-genetics-and-pathogenesis\" class=\"medical medical_review\">&quot;Huntington disease: Genetics and pathogenesis&quot;</a>.)</p><p>A unique clinical feature often associated with disease due to expanded trinucleotide repeats is that of genetic anticipation, whereby disease severity worsens or age of symptom onset decreases in subsequent generations. As an example, in myotonic dystrophy, mild manifestations of the disease in an older adult patient may only be recognized following diagnosis of the condition in a severely affected grandchild. This phenomenon reflects increasing expansion of the number of trinucleotide repeats with subsequent generations. The converse phenomenon of repeat length contraction has also been observed, and may explain variable penetrance observed in myotonic dystrophy, although this phenomenon is very rare [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/42\" class=\"abstract_t\">42</a>]. Parent of origin biases have also been observed in several of these conditions, likely due to gender-related differences in repeat expansion rates.</p><p class=\"headingAnchor\" id=\"H1423724\"><span class=\"h3\">Microsatellite markers</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>As tens of thousands of common and highly polymorphic STR loci have been identified throughout the genome, and because they are readily amenable to polymerase chain reaction (PCR)-based genotyping, STRs (otherwise known as microsatellite markers) were previously used as genetic markers for genome mapping and family-based linkage analysis studies. In linkage analysis, a panel of 300 to 600 densely spaced markers was genotyped in large pedigrees that were ascertained because of multiple affected individuals. Chromosome recombination rates were tracked across families at a genome-wide level to identify chromosomal segments that segregated with disease status through multiple generations (<a href=\"image.htm?imageKey=PC%2F75378%7EPC%2F57437\" class=\"graphic graphic_figure graphicRef75378 graphicRef57437 \">figure 9A-B</a>). The markers themselves are rarely the disease-causing variation, but the markers that are most strongly linked with disease can be used to delineate the genomic region harboring the causative mutation. An example of the successful use of linkage for gene mapping was the location of the <em>CFTR</em> gene on chromosome 7 [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/44\" class=\"abstract_t\">44</a>]. (See <a href=\"topic.htm?path=cystic-fibrosis-genetics-and-pathogenesis\" class=\"medical medical_review\">&quot;Cystic fibrosis: Genetics and pathogenesis&quot;</a>.)</p><p>Hundreds of disease-causing traits have since been identified by this methodology. Similar approaches for mapping complex common diseases such as diabetes, cancer, asthma, and hypertension, have been less reliably successful, and it has become more common to perform SNP analysis, array comparative genomic hybridization (array CGH), or next-generation sequencing for gene identification in both rare and common inherited disorders. (See <a href=\"#H10\" class=\"local\">'Single nucleotide polymorphisms (SNPs)'</a> above and <a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics#H15538652\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Cytogenetics and molecular genetics&quot;, section on 'Array comparative genomic hybridization'</a> and <a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">&quot;Next-generation DNA sequencing (NGS): Principles and clinical applications&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H3587710938\"><span class=\"h1\">CLINICAL CLASSIFICATION OF DNA VARIANTS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>All of the aforementioned variant types have the potential to cause disease or contribute to disease susceptibility risk. A variant's capacity to contribute to disease pathogenesis is determined by its functional impact on gene expression or protein function. A variety of classification conventions have been applied by clinical laboratories when reporting back results of genetic testing. In an attempt to standardize this, the American College of Medical Genetics and Genomics (ACMG), together with the Association for Molecular Pathology and the College of American Pathologists, developed a set of guidelines for variant reporting, and have recommended use of a five-tier terminology system for the clinical classification of genetic variants, consisting of the following designations [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/45\" class=\"abstract_t\">45</a>]:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Pathogenic &ndash; A pathogenic variant is a disease-causing variant, as determined by very strong genetic and experimental evidence, including consistent familial co-segregation with disease and definitive functional studies. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Likely pathogenic &ndash; A likely pathogenic variant is a variant with strong, but not definitive, evidence of pathogenicity based on its similarity to known pathogenic variants, co-segregation with disease in families or populations, and functional evidence. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Uncertain significance &ndash; A variant of uncertain significance (VUS) is a variant for which the specific criteria for the above four criteria are not met, or when contradictory lines of evidence in support of both benign and pathogenic classifications are present.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Likely benign &ndash; A likely benign variant is a variant with multiple supporting (but not conclusive) lines of evidence suggesting it is not disease-causing. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Benign &ndash; A benign variant is a variant with conclusive evidence that it is not disease-causing, as determined typically (but not exclusively) by a high prevalence of the variant in the general (healthy) population, at a prevalence that exceeds that of the suspected disease. </p><p/><p>The recommendations were accompanied by a detailed description of the process for variant classification [<a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/45\" class=\"abstract_t\">45</a>]. This process combines multiple lines of evidence, including allele frequency data from the general population, evidence for co-segregation of variant with disease in families, results from prediction algorithms that considered the functional potential and cross-species conservation, and experimental studies demonstrating altered gene or protein function.</p><p>The implications of these different types of variants and our approach to discussing these implications with individuals who undergo genomic sequencing is presented separately. (See <a href=\"topic.htm?path=secondary-findings-from-genetic-testing\" class=\"medical medical_review\">&quot;Secondary findings from genetic testing&quot;</a>.)</p><p class=\"headingAnchor\" id=\"PATIENT_INFORMATION\"><span class=\"h1\">INFORMATION FOR PATIENTS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>UpToDate offers two types of patient education materials, &quot;The Basics&quot; and &quot;Beyond the Basics.&quot; The Basics patient education pieces are written in plain language, at the 5<sup>th</sup> to 6<sup>th</sup> grade reading level, and they answer the four or five key questions a patient might have about a given condition. These articles are best for patients who want a general overview and who prefer short, easy-to-read materials. Beyond the Basics patient education pieces are longer, more sophisticated, and more detailed. These articles are written at the 10<sup>th</sup> to 12<sup>th</sup> grade reading level and are best for patients who want in-depth information and are comfortable with some medical jargon.</p><p>Here are the patient education articles that are relevant to this topic. We encourage you to print or e-mail these topics to your patients. (You can also locate patient education articles on a variety of subjects by searching on &quot;patient info&quot; and the keyword(s) of interest.)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Basics topics (see <a href=\"topic.htm?path=genetic-testing-the-basics\" class=\"medical medical_basics\">&quot;Patient education: Genetic testing (The Basics)&quot;</a>)</p><p/><p class=\"headingAnchor\" id=\"H15\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Large chromosomal abnormalities involve changes in chromosome number or structure. Aneuploidy is an abnormal number of one or more chromosome(s). Autosomal monosomies (loss of a non-sex linked chromosome), are incompatible with life unless accompanied by mosaicism. Turner syndrome is caused by the loss of one X chromosome. Trisomy 21 (Down syndrome) is the only trisomy for a whole human autosome in which affected individuals frequently survive into adulthood. (See <a href=\"#H4\" class=\"local\">'Aneuploidy'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Structural chromosome abnormalities include copy number variations, as well as chromosomal translocations, deletions, duplications, and inversions. Gene fusion caused by a structural chromosome aberration may result in a novel protein. Both chronic myeloid leukemia and some forms of primary hypereosinophilic syndrome are related to a hyperfunctional tyrosine kinase fusion protein. (See <a href=\"#H6\" class=\"local\">'Structural chromosome abnormalities'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Smaller chromosome abnormalities (single nucleotide polymorphisms, indels, short tandem repeats) and copy number variations are responsible for most genetic variation. (See <a href=\"#H7\" class=\"local\">'Common genetic variation'</a> above.)</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Single nucleotide polymorphisms (SNPs) are single-base pair changes that are seen at a population frequency of at least 1 percent; these are responsible for much of the heritable phenotypic variation in human populations. More than 65 percent of the currently-identified mutations that cause disease are highly penetrant single nucleotide substitutions, resulting in amino acid substitutions and abnormal proteins. (See <a href=\"#H10\" class=\"local\">'Single nucleotide polymorphisms (SNPs)'</a> above.)</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span><span class=\"nowrap\">Insertion/deletion</span> (indel) polymorphisms are polymorphic sites of gains or losses of one or more nucleotides. Most pathogenic indels are frameshift mutations. For example a 32 base pair deletion (<em>CCR5</em> Delta32) conveys protection against HIV transfection. (See <a href=\"#H13\" class=\"local\">'Insertion/deletion polymorphism'</a> above.)</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Short tandem repeats (STRs) are sequences of two to six base pairs that repeat in tandem and are flanked by non-repetitive sequences. When used as markers of genetic loci for genome mapping and linkage analysis studies, STRs are known as microsatellites. Trinucleotide repeats, a type of STR, have been identified in several neurogenetic conditions (for example, Fragile X, myotonic dystrophy, and Huntington disease) and may be associated with genetic anticipation (increasing disease severity and earlier age of onset over subsequent generations). (See <a href=\"#H1423422\" class=\"local\">'Short tandem repeat markers'</a> above.)</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Copy number variants (CNVs) are the most prevalent structural genetic variation, with DNA segments that are multiple kilobases in length varying in copy number between individuals. CNVs are particularly common in gene families related to immune and inflammatory responses, cell signaling and cell adhesion, structural proteins, and olfactory receptors. (See <a href=\"#H14\" class=\"local\">'Copy number variations (CNVs)'</a> above and <a href=\"topic.htm?path=genomic-disorders-an-overview#H16722199\" class=\"medical medical_review\">&quot;Genomic disorders: An overview&quot;, section on 'Copy number variations'</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Genome-wide association studies (GWAS), next-generation DNA sequencing, and systems biology approaches have facilitated the identification of susceptibility loci for complex diseases. These gene variations with low penetrance can result in phenotypic expression, depending on interaction with other genes and environmental triggers. (See <a href=\"#H12\" class=\"local\">'Genetic basis of complex disease'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Variants may be classified as pathogenic, likely pathogenic, of uncertain significance, likely benign, or benign. The implications of these findings and their disclosure to the patient are discussed separately. (See <a href=\"#H3587710938\" class=\"local\">'Clinical classification of DNA variants'</a> above and <a href=\"topic.htm?path=secondary-findings-from-genetic-testing\" class=\"medical medical_review\">&quot;Secondary findings from genetic testing&quot;</a>.)</p></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/1\" class=\"nounderline abstract_t\">Collins FS, Green ED, Guttmacher AE, et al. A vision for the future of genomics research. Nature 2003; 422:835.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/2\" class=\"nounderline abstract_t\">Ranweiler R. Assessment and care of the newborn with Down syndrome. Adv Neonatal Care 2009; 9:17.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/3\" class=\"nounderline abstract_t\">Kesler SR. Turner syndrome. Child Adolesc Psychiatr Clin N Am 2007; 16:709.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/4\" class=\"nounderline abstract_t\">Hjerrild BE, Mortensen KH, Gravholt CH. Turner syndrome and clinical treatment. Br Med Bull 2008; 86:77.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/5\" class=\"nounderline abstract_t\">Wikstr&ouml;m AM, Dunkel L. Klinefelter syndrome. Best Pract Res Clin Endocrinol Metab 2011; 25:239.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/6\" class=\"nounderline abstract_t\">Otter M, Schrander-Stumpel CT, Curfs LM. Triple X syndrome: a review of the literature. Eur J Hum Genet 2010; 18:265.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/7\" class=\"nounderline abstract_t\">Higuchi M, Kazazian HH Jr, Kasch L, et al. Molecular characterization of severe hemophilia A suggests that about half the mutations are not within the coding regions and splice junctions of the factor VIII gene. Proc Natl Acad Sci U S A 1991; 88:7405.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/8\" class=\"nounderline abstract_t\">Hu XY, Ray PN, Worton RG. Mechanisms of tandem duplication in the Duchenne muscular dystrophy gene include both homologous and nonhomologous intrachromosomal recombination. EMBO J 1991; 10:2471.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/9\" class=\"nounderline abstract_t\">Lau YL, Srivastava G, Wong V, et al. Deletions, duplications and novel restriction fragment length polymorphism in Duchenne and Becker muscular dystrophies. Clin Genet 1992; 41:252.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/10\" class=\"nounderline abstract_t\">Cohn DH, Zhang X, Byers PH. Homology-mediated recombination between type I collagen gene exons results in an internal tandem duplication and lethal osteogenesis imperfecta. Hum Mutat 1993; 2:21.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/11\" class=\"nounderline abstract_t\">Jennings MW, Jones RW, Wood WG, Weatherall DJ. Analysis of an inversion within the human beta globin gene cluster. Nucleic Acids Res 1985; 13:2897.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/12\" class=\"nounderline abstract_t\">Cools J, DeAngelo DJ, Gotlib J, et al. A tyrosine kinase created by fusion of the PDGFRA and FIP1L1 genes as a therapeutic target of imatinib in idiopathic hypereosinophilic syndrome. N Engl J Med 2003; 348:1201.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/13\" class=\"nounderline abstract_t\">Van den Berghe H. Letter: The Ph1-chromosome: translocation to chromosome 9. Lancet 1973; 302:1030.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/14\" class=\"nounderline abstract_t\">Lupski JR, Reid JG, Gonzaga-Jauregui C, et al. Whole-genome sequencing in a patient with Charcot-Marie-Tooth neuropathy. N Engl J Med 2010; 362:1181.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/15\" class=\"nounderline abstract_t\">1000 Genomes Project Consortium, Abecasis GR, Altshuler D, et al. A map of human genome variation from population-scale sequencing. Nature 2010; 467:1061.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/16\" class=\"nounderline abstract_t\">Chakravarti A. It's raining SNPs, hallelujah? Nat Genet 1998; 19:216.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/17\" class=\"nounderline abstract_t\">Li WH, Sadler LA. Low nucleotide diversity in man. Genetics 1991; 129:513.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/18\" class=\"nounderline abstract_t\">Stephens JC, Schneider JA, Tanguay DA, et al. Haplotype variation and linkage disequilibrium in 313 human genes. Science 2001; 293:489.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/19\" class=\"nounderline abstract_t\">Halushka MK, Fan JB, Bentley K, et al. Patterns of single-nucleotide polymorphisms in candidate genes for blood-pressure homeostasis. Nat Genet 1999; 22:239.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/20\" class=\"nounderline abstract_t\">Rideout WM 3rd, Coetzee GA, Olumi AF, Jones PA. 5-Methylcytosine as an endogenous mutagen in the human LDL receptor and p53 genes. Science 1990; 249:1288.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/21\" class=\"nounderline abstract_t\">Cooper DN, Krawczak M. Human Gene Mutation Database. Hum Genet 1996; 98:629.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/22\" class=\"nounderline abstract_t\">Prins BP, Lagou V, Asselbergs FW, et al. Genetics of coronary artery disease: genome-wide association studies and beyond. Atherosclerosis 2012; 225:1.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/23\" class=\"nounderline abstract_t\">Leslie R, O'Donnell CJ, Johnson AD. GRASP: analysis of genotype-phenotype results from 1390 genome-wide association studies and corresponding open access database. Bioinformatics 2014; 30:i185.</a></li><li class=\"breakAll\">https://grasp.nhlbi.nih.gov/Overview.aspx (Accessed on March 18, 2018).</li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/25\" class=\"nounderline abstract_t\">Guerra SG, Vyse TJ, Cunninghame Graham DS. The genetics of lupus: a functional perspective. Arthritis Res Ther 2012; 14:211.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/26\" class=\"nounderline abstract_t\">Marian AJ. Elements of 'missing heritability'. Curr Opin Cardiol 2012; 27:197.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/27\" class=\"nounderline abstract_t\">Krawczak M, Cooper DN. Gene deletions causing human genetic disease: mechanisms of mutagenesis and the role of the local DNA sequence environment. Hum Genet 1991; 86:425.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/28\" class=\"nounderline abstract_t\">Taylor MS, Ponting CP, Copley RR. Occurrence and consequences of coding sequence insertions and deletions in Mammalian genomes. Genome Res 2004; 14:555.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/29\" class=\"nounderline abstract_t\">Antonarakis SE, Krawczak M, Cooper DN. Disease-causing mutations in the human genome. Eur J Pediatr 2000; 159 Suppl 3:S173.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/30\" class=\"nounderline abstract_t\">Kerem B, Rommens JM, Buchanan JA, et al. Identification of the cystic fibrosis gene: genetic analysis. Science 1989; 245:1073.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/31\" class=\"nounderline abstract_t\">Riordan JR, Rommens JM, Kerem B, et al. Identification of the cystic fibrosis gene: cloning and characterization of complementary DNA. Science 1989; 245:1066.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/32\" class=\"nounderline abstract_t\">Rommens JM, Zengerling S, Burns J, et al. Identification and regional localization of DNA markers on chromosome 7 for the cloning of the cystic fibrosis gene. Am J Hum Genet 1988; 43:645.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/33\" class=\"nounderline abstract_t\">Liu R, Paxton WA, Choe S, et al. Homozygous defect in HIV-1 coreceptor accounts for resistance of some multiply-exposed individuals to HIV-1 infection. Cell 1996; 86:367.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/34\" class=\"nounderline abstract_t\">Samson M, Libert F, Doranz BJ, et al. Resistance to HIV-1 infection in caucasian individuals bearing mutant alleles of the CCR-5 chemokine receptor gene. Nature 1996; 382:722.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/35\" class=\"nounderline abstract_t\">Eugen-Olsen J, Iversen AK, Garred P, et al. Heterozygosity for a deletion in the CKR-5 gene leads to prolonged AIDS-free survival and slower CD4 T-cell decline in a cohort of HIV-seropositive individuals. AIDS 1997; 11:305.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/36\" class=\"nounderline abstract_t\">Schl&ouml;tterer C. Evolutionary dynamics of microsatellite DNA. Chromosoma 2000; 109:365.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/37\" class=\"nounderline abstract_t\">Ellegren H. Microsatellite mutations in the germline: implications for evolutionary inference. Trends Genet 2000; 16:551.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/38\" class=\"nounderline abstract_t\">A novel gene containing a trinucleotide repeat that is expanded and unstable on Huntington's disease chromosomes. The Huntington's Disease Collaborative Research Group. Cell 1993; 72:971.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/39\" class=\"nounderline abstract_t\">Lalioti MD, Scott HS, Buresi C, et al. Dodecamer repeat expansion in cystatin B gene in progressive myoclonus epilepsy. Nature 1997; 386:847.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/40\" class=\"nounderline abstract_t\">Muragaki Y, Mundlos S, Upton J, Olsen BR. Altered growth and branching patterns in synpolydactyly caused by mutations in HOXD13. Science 1996; 272:548.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/41\" class=\"nounderline abstract_t\">Verkerk AJ, Pieretti M, Sutcliffe JS, et al. Identification of a gene (FMR-1) containing a CGG repeat coincident with a breakpoint cluster region exhibiting length variation in fragile X syndrome. Cell 1991; 65:905.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/42\" class=\"nounderline abstract_t\">Buxton J, Shelbourne P, Davies J, et al. Detection of an unstable fragment of DNA specific to individuals with myotonic dystrophy. Nature 1992; 355:547.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/43\" class=\"nounderline abstract_t\">Harley HG, Brook JD, Rundle SA, et al. Expansion of an unstable DNA region and phenotypic variation in myotonic dystrophy. Nature 1992; 355:545.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/44\" class=\"nounderline abstract_t\">Iannuzzi MC, Collins FS. Reverse genetics and cystic fibrosis. Am J Respir Cell Mol Biol 1990; 2:309.</a></li><li><a href=\"https://www.uptodate.com/contents/overview-of-genetic-variation/abstract/45\" class=\"nounderline abstract_t\">Richards S, Aziz N, Bale S, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Genet Med 2015; 17:405.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 2895 Version 19.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H15\"><span>SUMMARY</span></a></li><li><a href=\"#H1\" id=\"outline-link-H1\">INTRODUCTION</a></li><li><a href=\"#H2\" id=\"outline-link-H2\">RARE CHROMOSOMAL ABERRATIONS</a><ul><li><a href=\"#H3\" id=\"outline-link-H3\">Numerical chromosome abnormalities</a><ul><li><a href=\"#H4\" id=\"outline-link-H4\">- Aneuploidy</a></li><li><a href=\"#H5\" id=\"outline-link-H5\">- Polyploidy</a></li></ul></li><li><a href=\"#H6\" id=\"outline-link-H6\">Structural chromosome abnormalities</a></li></ul></li><li><a href=\"#H7\" id=\"outline-link-H7\">COMMON GENETIC VARIATION</a><ul><li><a href=\"#H10\" id=\"outline-link-H10\">Single nucleotide polymorphisms (SNPs)</a><ul><li><a href=\"#H11\" id=\"outline-link-H11\">- Monogenic disease</a></li><li><a href=\"#H12\" id=\"outline-link-H12\">- Genetic basis of complex disease</a></li></ul></li><li><a href=\"#H13\" id=\"outline-link-H13\">Insertion/deletion polymorphism</a></li><li><a href=\"#H14\" id=\"outline-link-H14\">Copy number variations (CNVs)</a></li><li><a href=\"#H1423422\" id=\"outline-link-H1423422\">Short tandem repeat markers</a><ul><li><a href=\"#H1423724\" id=\"outline-link-H1423724\">- Microsatellite markers</a></li></ul></li></ul></li><li><a href=\"#H3587710938\" id=\"outline-link-H3587710938\">CLINICAL CLASSIFICATION OF DNA VARIANTS</a></li><li><a href=\"#PATIENT_INFORMATION\" id=\"outline-link-H1459528211\">INFORMATION FOR PATIENTS</a></li><li><a href=\"#H15\" id=\"outline-link-H15\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>GRAPHICS <a href=\"#\" id=\"viewAllGraphicsLink\">View All</a></h2><div id=\"outlineGraphics\"><ul><li><div id=\"PC/2895|FIG\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"FIGURES\">FIGURES</a></div><ul><li><a href=\"image.htm?imageKey=PC/52882\" class=\"graphic graphic_figure\">- Normal assortment</a></li><li><a href=\"image.htm?imageKey=PC/81597\" class=\"graphic graphic_figure\">- Non-disjunction</a></li><li><a href=\"image.htm?imageKey=PC/71519\" class=\"graphic graphic_figure\">- Chromosomal translocations</a></li><li><a href=\"image.htm?imageKey=PC/70807\" class=\"graphic graphic_figure\">- Translocation</a></li><li><a href=\"image.htm?imageKey=OBGYN/65339\" class=\"graphic graphic_figure\">- Inversions and recombinations</a></li><li><a href=\"image.htm?imageKey=PC/67644\" class=\"graphic graphic_figure\">- Nonallelic homologous recombination</a></li><li><a href=\"image.htm?imageKey=PC/65092\" class=\"graphic graphic_figure\">- Gene fusion</a></li><li><a href=\"image.htm?imageKey=PC/69699\" class=\"graphic graphic_figure\">- Pathogenic indel</a></li><li><a href=\"image.htm?imageKey=PC/82363\" class=\"graphic graphic_figure\">- Short tandem repeats</a></li><li><a href=\"image.htm?imageKey=PC/75378\" class=\"graphic graphic_figure\">- Linkage one pedigree</a></li><li><a href=\"image.htm?imageKey=PC/57437\" class=\"graphic graphic_figure\">- Link disease gene location</a></li></ul></li></ul></div></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=causes-of-primary-hypogonadism-in-males\" class=\"medical medical_review\">Causes of primary hypogonadism in males</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=cellular-and-molecular-biology-of-chronic-myeloid-leukemia\" class=\"medical medical_review\">Cellular and molecular biology of chronic myeloid leukemia</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=charcot-marie-tooth-disease-genetics-clinical-features-and-diagnosis\" class=\"medical medical_review\">Charcot-Marie-Tooth disease: Genetics, clinical features, and diagnosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=chromosomal-translocations-deletions-and-inversions\" class=\"medical medical_review\">Chromosomal translocations, deletions, and inversions</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=clinical-manifestations-and-diagnosis-of-turner-syndrome\" class=\"medical medical_review\">Clinical manifestations and diagnosis of Turner syndrome</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=congenital-cytogenetic-abnormalities\" class=\"medical medical_review\">Congenital cytogenetic abnormalities</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=cystic-fibrosis-genetics-and-pathogenesis\" class=\"medical medical_review\">Cystic fibrosis: Genetics and pathogenesis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=down-syndrome-clinical-features-and-diagnosis\" class=\"medical medical_review\">Down syndrome: Clinical features and diagnosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=duchenne-and-becker-muscular-dystrophy-clinical-features-and-diagnosis\" class=\"medical medical_review\">Duchenne and Becker muscular dystrophy: Clinical features and diagnosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetic-association-studies-principles-and-applications\" class=\"medical medical_review\">Genetic association studies: Principles and applications</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetics-of-hemophilia-a-and-b\" class=\"medical medical_review\">Genetics of hemophilia A and B</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetics-glossary-of-terms\" class=\"medical medical_review\">Genetics: Glossary of terms</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genomic-disorders-an-overview\" class=\"medical medical_review\">Genomic disorders: An overview</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=huntington-disease-genetics-and-pathogenesis\" class=\"medical medical_review\">Huntington disease: Genetics and pathogenesis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=hypereosinophilic-syndromes-clinical-manifestations-pathophysiology-and-diagnosis\" class=\"medical medical_review\">Hypereosinophilic syndromes: Clinical manifestations, pathophysiology, and diagnosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=inherited-syndromes-associated-with-cardiac-disease\" class=\"medical medical_review\">Inherited syndromes associated with cardiac disease</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=myotonic-dystrophy-etiology-clinical-features-and-diagnosis\" class=\"medical medical_review\">Myotonic dystrophy: Etiology, clinical features, and diagnosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">Next-generation DNA sequencing (NGS): Principles and clinical applications</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=osteogenesis-imperfecta-clinical-features-and-diagnosis\" class=\"medical medical_review\">Osteogenesis imperfecta: Clinical features and diagnosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetic-testing-the-basics\" class=\"medical medical_basics\">Patient education: Genetic testing (The Basics)</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=personalized-medicine\" class=\"medical medical_review\">Personalized medicine</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=prenatal-screening-and-diagnosis-for-fragile-x-syndrome\" class=\"medical medical_review\">Prenatal screening and diagnosis for fragile X syndrome</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=secondary-findings-from-genetic-testing\" class=\"medical medical_review\">Secondary findings from genetic testing</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=the-child-with-tall-stature-and-or-abnormally-rapid-growth\" class=\"medical medical_review\">The child with tall stature and/or abnormally rapid growth</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">Tools for genetics and genomics: Cytogenetics and molecular genetics</a></li></ul></div></div>","javascript":null}