var data={"title":"Principles of complex trait genetics","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Principles of complex trait genetics</div><dl id=\"topicContributors\"><dt><span> </span>Authors:</dt><dd><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/contributors\" class=\"contributor contributor_credentials\">Juan C Celed&oacute;n, MD, DrPH</a></dd><dd><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/contributors\" class=\"contributor contributor_credentials\">Gary M Hunninghake, MD, MPH</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/contributors\" class=\"contributor contributor_credentials\">Benjamin A Raby, MD, MPH</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/contributors\" class=\"contributor contributor_credentials\">Jennifer S Tirnauer, MD</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Apr 10, 2017.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H3540509\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Most human genetic traits can be classified as either monogenic or complex. Monogenic traits are strongly influenced by variation within a single gene and are recognized by their classic patterns of inheritance within families. While monogenic traits formed the basis for &quot;classic&quot; genetics, it has become clear that conditions whose inheritance strictly conforms to Mendelian principles are relatively rare. (See <a href=\"topic.htm?path=inheritance-patterns-of-monogenic-disorders-mendelian-and-non-mendelian\" class=\"medical medical_review\">&quot;Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)&quot;</a> and <a href=\"topic.htm?path=genetics-glossary-of-terms\" class=\"medical medical_review\">&quot;Genetics: Glossary of terms&quot;</a>.)</p><p>Complex traits are believed to result from variation within multiple genes and their interaction with behavioral and environmental factors. Complex traits do not follow readily predictable patterns of inheritance.</p><p>This distinction between monogenic and complex traits, while useful, can be overly simplistic. Traits that appear to be monogenic can be influenced by variation in multiple genes (&quot;modifier genes&quot;) [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/1\" class=\"abstract_t\">1</a>]; complex traits can be predominantly influenced by variation in a single gene [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/2\" class=\"abstract_t\">2</a>].</p><p>This topic will review the challenges related to the identification of complex trait susceptibility genes, the factors that contribute to phenotypic complexity, and current understanding of the genetic architecture of complex genetic traits. Genetic traits with monogenic inheritance, either with a Mendelian or a non-Mendelian pattern, are discussed separately. (See <a href=\"topic.htm?path=inheritance-patterns-of-monogenic-disorders-mendelian-and-non-mendelian\" class=\"medical medical_review\">&quot;Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)&quot;</a> and <a href=\"topic.htm?path=inheritance-patterns-of-monogenic-disorders-mendelian-and-non-mendelian#H3744740437\" class=\"medical medical_review\">&quot;Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)&quot;, section on 'Causes of non-Mendelian inheritance'</a>.)</p><p class=\"headingAnchor\" id=\"H3540524\"><span class=\"h1\">SPECTRUM OF GENETIC VARIATION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Most monogenic diseases are caused by mutations that reduce the function or stability of a single protein by altering its three-dimensional structure. These mutations include point mutations (eg, changes in single nucleotides that alter the amino acid sequence), insertions, or deletions in the DNA sequence that encodes the protein; or changes in the non-coding DNA that interfere with gene splicing. (See <a href=\"topic.htm?path=inheritance-patterns-of-monogenic-disorders-mendelian-and-non-mendelian\" class=\"medical medical_review\">&quot;Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)&quot;</a> and <a href=\"topic.htm?path=basic-principles-of-genetic-disease#H1497131\" class=\"medical medical_review\">&quot;Basic principles of genetic disease&quot;, section on 'DNA sequence variation'</a> and <a href=\"topic.htm?path=inheritance-patterns-of-monogenic-disorders-mendelian-and-non-mendelian#H3744740437\" class=\"medical medical_review\">&quot;Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)&quot;, section on 'Causes of non-Mendelian inheritance'</a>.)</p><p>In complex genetic traits, the spectrum of genetic mutation (ie, the types of mutations and the number of mutations involved in a single disease) is broad and more complicated, and it is often difficult to identify a single causal genetic change. These traits tend to be polygenic. Further, although coding mutations have been identified in complex diseases, most of the genetic variations mapped for complex genetic traits tend to be non-coding (located in non-genic, regulatory or intronic regions) and likely act by altering the control of gene expression (eg, interfering with a transcription factor binding site), the stability of the RNA or protein product, or posttranslational modification of the protein product, rather than altering the amino acid sequence directly. (See <a href=\"#H3540531\" class=\"local\">'Determinants of genetic effect'</a> below.)</p><p>The genetic variation in complex traits is also more difficult to analyze. Several factors contribute to difficulty in the recognition of functional regulatory variation:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Regulatory variants reside in noncoding sequences of the genome. Functional sequence annotation of noncoding sequences is frequently less comprehensive than annotation for coding regions.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Numerous genetic variants have been identified that are strongly associated with a complex disease, but are not definitively related to a known gene (eg, the 9p21 locus in coronary artery disease) [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/3\" class=\"abstract_t\">3</a>]. The mechanisms by which such variants affect complex traits are more difficult to uncover [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/4\" class=\"abstract_t\">4</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Susceptibility loci for complex traits may influence the regulation of several genes within a genomic region [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/5\" class=\"abstract_t\">5</a>]. This suggests that some regulatory regions of DNA, rather than affecting a single gene, could play a role in regulating the regional transcription patterns of multiple genes.</p><p/><p class=\"headingAnchor\" id=\"H3540531\"><span class=\"h2\">Determinants of genetic effect</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Whereas Mendelian traits are characterized by a strong genotype-phenotype correlation, complex genetic traits typically exhibit weak correlations between genotype and phenotype. Data from genome-wide association studies (GWAS) of complex traits demonstrate that many disease susceptibility variants, by themselves, confer only a modestly increased risk (ie, relative risk ranging between 1.1 and 1.5). (See <a href=\"#H3540594\" class=\"local\">'Approaches to identifying genetic determinants of complex traits'</a> below.)</p><p>The relatively weak genotype-phenotype correlations for complex traits may be explained by the following hypotheses: </p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Regulatory variation</strong> &ndash; Changes in the non-coding (ie, regulatory) regions of genes may alter the quantity of a protein without changing its function. The effect on cellular phenotype may be more subtle than the effect of coding mutations that affect protein structure.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Polygenic effects</strong> &ndash; Most complex genetic traits are likely the result of variation within multiple genes. Any individual genetic variant might only confer a relatively small effect; the cumulative effect of multiple genetic variants would result in a much larger overall effect on the trait.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Gene-gene interactions</strong> &ndash; Some genetic variants may only have an effect that is contingent upon the presence of additional genetic variants at nearby or distant loci. Determining these interactions between genes presents an enormous mathematical challenge.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Gene-environment interactions</strong> &ndash; Environmental exposures (broadly defined as all non-genetic exposures) are likely to have a strong influence on the manifestation of most complex traits. These can include behavioral exposures and exposures about which the individual is unaware. The effect of a particular environmental exposure on the expression of a genetic determinant has been demonstrated for some conditions (ie, the effect of an MMP12 variant on chronic obstructive pulmonary disease [COPD] risk is dependent on cigarette smoke exposure) [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/6\" class=\"abstract_t\">6</a>]. Determining how these gene-environment interactions are manifested also presents a substantial mathematical challenge.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Rare (uncommon) susceptibility variants</strong> &ndash; Rare (uncommon) susceptibility variants are uncommon genetic changes that may play a minor role in a disease phenotype in the population as a whole, yet can have a large effect on the phenotype of single individual. These rare variants are difficult to incorporate into population-based genotype-phenotype correlations because of limited statistical power to detect their effect with usual sample size and challenges in developing models to examine their joint effects. (See <a href=\"#H3540587\" class=\"local\">'Common disease-rare variant hypothesis'</a> below.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Epigenetic mechanisms</strong> &ndash; Epigenetic mechanisms influence the expression of a gene without changing any DNA sequence. Such mechanisms may be heritable or post-natal, and include DNA methylation and histone modification. Environmental or behavioral exposures may lead to human disease through epigenetics. For example, cigarette smoking can induce increased DNA methylation and thus reduced expression of genes acting as tumor suppressors. Moreover, epigenetic mechanisms could mask the identification of causative genetic loci. As an example, lack of awareness of epigenetic effects on the inheritance of telomere length would have obscured the identification of the causative <em>PARN</em> genetic loci in familial pulmonary fibrosis kindred [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/7\" class=\"abstract_t\">7</a>]. </p><p/><p class=\"headingAnchor\" id=\"H3540573\"><span class=\"h1\">COMMON VERSUS RARE ALLELES</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The polygenic nature of complex traits is widely accepted, but controversy exists regarding the likely frequency of the most important susceptibility alleles for complex traits [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/8,9\" class=\"abstract_t\">8,9</a>]. The relative contribution of common variants (minor allele frequency [MAF] &ge;5 percent) or rare variants (MAF &le;1 percent) as critical determinants of complex traits is unclear.</p><p class=\"headingAnchor\" id=\"H707825542\"><span class=\"h2\">Impact on disease</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The comparative impact of common or rare alleles likely depends on the complex trait of interest and whether &quot;impact&quot; is defined on the basis of population attributable risk (favoring common variants) or risk prediction (favoring rare variants with larger effects). Population attributable risk (PAR) refers to the proportion of subjects affected by a disease in a population that can be explained by exposure to a particular risk factor. Risk prediction relates to the likelihood that an individual with a particular risk factor will develop a disease. A classic example from epidemiology helps illustrate the differences between these two concepts:</p><p>Lung cancer risk is increased in both people who smoke and in people who have an occupational history of asbestos exposure [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/10\" class=\"abstract_t\">10</a>]. The risk for lung cancer in an individual who worked in an asbestos factory is significantly greater than the risk for lung cancer in the average smoker. However, cigarette smoking is considerably more common than having worked in an asbestos factory. Thus, although exposure to asbestos confers a greater risk for lung cancer (better risk prediction) than a history of smoking at an individual level, reducing exposure to cigarette smoking would have a much greater impact in reducing the number of lung cancer cases than eliminating asbestos. Viewed through this framework, it is clear that there is value to identifying common and rare determinants of complex genetic traits, as they both affect the development of human diseases.</p><p class=\"headingAnchor\" id=\"H3540580\"><span class=\"h2\">Common disease-common variant hypothesis</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The common disease-common variant hypothesis posits that common genetic variants underlie susceptibility to most common traits (eg, cardiovascular disease, diabetes, or asthma) [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/11\" class=\"abstract_t\">11</a>]. This hypothesis is rooted in population genetics theory, which states that the current human population is the result of a global expansion from a smaller founding population from sub-Saharan Africa. Thus, the current human population shares common genetic variants that were present in the founding population, and common complex traits are the result of inheriting these variants [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/12\" class=\"abstract_t\">12</a>].</p><p>To a certain extent, the identification of nearly &gt;14,000 highly replicable genetic loci from genome-wide association studies (GWAS) supports the common disease-common variant hypothesis [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/13\" class=\"abstract_t\">13</a>]. However, rare variants may partly explain findings for common variants in GWAS [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/14\" class=\"abstract_t\">14</a>] and explain some of the unexplained or &quot;missing&quot; heritability for most complex traits. (See <a href=\"#H3540594\" class=\"local\">'Approaches to identifying genetic determinants of complex traits'</a> below and <a href=\"topic.htm?path=genetic-association-studies-principles-and-applications#H6\" class=\"medical medical_review\">&quot;Genetic association studies: Principles and applications&quot;, section on 'Missing heritability'</a>.)</p><p class=\"headingAnchor\" id=\"H3540587\"><span class=\"h2\">Common disease-rare variant hypothesis</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>To date, more than &gt;14,000 common susceptibility variants have been mapped. For a given complex trait (eg, asthma), however, the variants identified explain only a small fraction of the total heritability (proportion of phenotypic variation explained by genetics). Moreover, they appear to be of limited value in predicting disease risk. As an example, while up to 80 percent of the variability in height is felt to be due to genetic factors [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/15,16\" class=\"abstract_t\">15,16</a>], the combined effect of 20 genetic loci identified by GWAS explained less than 3 percent of the heritability of height [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/17\" class=\"abstract_t\">17</a>].</p><p>Since the loci identified by an initial GWAS likely represent the loci with the greatest contribution to the heritability, it has been argued that further study of common genetic variation in increasingly larger samples is unlikely to uncover a substantial portion of the &quot;missing heritability&quot; of complex traits [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/8\" class=\"abstract_t\">8</a>], which may be at least partly due to the effects of rare variants. Rare variants may be more valuable in predicting disease risk at an individual level than common variants [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/18-20\" class=\"abstract_t\">18-20</a>].</p><p class=\"headingAnchor\" id=\"H2886789\"><span class=\"h2\">Other hypotheses</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>As stated above, the causes of a significant proportion of the heritability of complex traits have yet to be identified. There are plausible explanations for this, including gene-by-gene or gene-by-environment interactions and epigenetic<strong> </strong>mechanisms. Most GWAS performed to date have analyzed data from the perspective of identifying those associations found most frequently, thereby detecting variants with the most significant p-values but not necessarily the strongest genetic effects. Similarly, virtually all published GWAS use a univariate modeling approach that tests for association one variant at a time, ignoring possible effects of two or more variants acting in combination.</p><p>The inherent complexity of common diseases suggests that consideration of these issues will be fruitful. However, the required statistical methods needed for such analyses; the computational resources needed to search for complex gene-gene and gene-environment interactions; as well as resources to integrate epigenetic and genomics data at a genome-wide scale, are only now being developed.</p><p class=\"headingAnchor\" id=\"H3540594\"><span class=\"h1\">APPROACHES TO IDENTIFYING GENETIC DETERMINANTS OF COMPLEX TRAITS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Linkage analysis, a statistical technique that successfully identified causal genes for over 2000 monogenic diseases, has had limited success for gene mapping of complex traits (eg, identification of maturity onset diabetes of the young [MODY] 1, 2, and 3 for diabetes) [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/21\" class=\"abstract_t\">21</a>]. Identifying genetic determinants for complex traits has proven to be very challenging. (See <a href=\"topic.htm?path=genetics-glossary-of-terms\" class=\"medical medical_review\">&quot;Genetics: Glossary of terms&quot;</a>.)</p><p>Until the mid-1990s, the two main approaches to identify susceptibility variants for complex traits were positional cloning (genome-wide linkage analysis followed by fine mapping analyses of linkage and association) and candidate-gene association studies. The candidate gene approach (ie, making an educated guess about the role of a gene in a disease based on prior knowledge of the functions of that gene) has been occasionally successful (eg, identification of APOE4 for Alzheimer's disease) [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/22\" class=\"abstract_t\">22</a>]. However, many findings have been inconsistently replicated and applications are limited to conditions with known biology.</p><p>A genetic mapping technique called genome-wide association study (GWAS), involves the simultaneous testing of multiple genetic variants spread across the human genome for association with a particular disease or trait. Whereas candidate gene approaches require a priori knowledge about the functions of a gene, GWAS studies are a way to scan all the genes in the genome without bias. (See <a href=\"topic.htm?path=genetic-association-studies-principles-and-applications\" class=\"medical medical_review\">&quot;Genetic association studies: Principles and applications&quot;</a>.)</p><p>GWAS became feasible upon compilation of high resolution linkage disequilibrium maps of common genetic variants (through the International HapMap Project) [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/23\" class=\"abstract_t\">23</a>], and with the advent of high throughput genome-wide genotyping technologies [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/24\" class=\"abstract_t\">24</a>]. GWAS have catalogued associations for over 100,000 SNPs, including presentation of more than 1400 highly-replicable genetic loci for thousands of human diseases and complex traits [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/13\" class=\"abstract_t\">13</a>].</p><p>GWAS have identified susceptibility loci with [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/25\" class=\"abstract_t\">25</a>] and without [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/26\" class=\"abstract_t\">26</a>] known function. In addition, loci identified by GWAS are providing surprising genetic links among seemingly unrelated complex traits (eg, a single nucleotide polymorphism [SNP] in the transcription factor 2 gene, also known as hepatocyte nuclear factor-1-beta [HF1B], is associated with both type 2 diabetes and prostate cancer) [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/27\" class=\"abstract_t\">27</a>]. Although much work remains to be done with regard to functional characterization of well-replicated loci, GWAS have yielded novel insights into the pathogenesis of numerous complex traits. (See <a href=\"topic.htm?path=overview-of-genetic-variation#H10\" class=\"medical medical_review\">&quot;Overview of genetic variation&quot;, section on 'Single nucleotide polymorphisms (SNPs)'</a>.)</p><p>The hallmark of a significant finding of genetic association for a complex trait is the ability to reproduce an original finding of association in independent populations. Demonstration that a statistically-significant genotype-phenotype correlation can be found in an independent population is considered the most rigorous proof of genetic association and has become a standard in the field. In contrast, all singular findings of genetic association should be considered speculative. An important caveat to this rule is that certain susceptibility loci may only be relevant to certain ethnic groups. Thus, an observation about a genotype-phenotype association should first be tested in a cohort belonging to the same ethnic group as the initial analysis [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/28\" class=\"abstract_t\">28</a>].</p><p>Although GWAS have demonstrated success in mapping the associations between common variants and complex traits, newer techniques such as next-generation sequencing, which can be used to sequence the entire exome (eg, whole exome sequencing [WES]) or the entire genome (eg, whole genome sequencing [WGS]) hold promise in improving our ability to assess the contribution of rare variants to the heritability of complex traits [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/29\" class=\"abstract_t\">29</a>]. These studies may require novel methodologies (eg, aggregation of data from multiple genetic variants across a single gene or pathway) and sample collection strategies (eg, large sample sizes, family-based studies, or assessments of population isolates) to improve the statistical power to detect true associations [<a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/30\" class=\"abstract_t\">30</a>]. (See <a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">&quot;Next-generation DNA sequencing (NGS): Principles and clinical applications&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H3540615\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Complex trait conditions are those that cannot be explained by alteration of a single gene (ie, monogenic). They are typically polygenic, with additional influences from gene-gene and gene-environment interactions. (See <a href=\"#H3540509\" class=\"local\">'Introduction'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Complex genetic traits involve a broader spectrum of mutations than typically seen with monogenic conditions. This spectrum of mutations includes variations in regulatory genes residing in noncoding DNA sequences. (See <a href=\"#H3540524\" class=\"local\">'Spectrum of genetic variation'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The correlation between genotype and phenotype for complex genetic traits is weaker than for Mendelian traits. This weaker correlation is potentially influenced by polygenic effects, gene-gene and gene-environment interactions, epigenetic phenomena, and unidentified rare disease-susceptibility variants. (See <a href=\"#H3540531\" class=\"local\">'Determinants of genetic effect'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Complex genetic traits may be associated with commonly occurring allelic variants that individually have a weak effect or with rare variants that have a stronger effect. Common variants have a greater impact on population-attributable risk, while uncommon variants have a greater impact on risk prediction at the level of the individual. (See <a href=\"#H3540573\" class=\"local\">'Common versus rare alleles'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Techniques to identify complex trait genes include linkage analyses, candidate gene testing, genome-wide association studies (GWAS), and next generation sequencing (NGS) or the entire exome or genome. Replication of findings in another cohort is an essential element to validate findings from an initial GWAS or NGS study. (See <a href=\"#H3540594\" class=\"local\">'Approaches to identifying genetic determinants of complex traits'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Monogenic conditions are discussed separately. (See <a href=\"topic.htm?path=inheritance-patterns-of-monogenic-disorders-mendelian-and-non-mendelian\" class=\"medical medical_review\">&quot;Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)&quot;</a> and <a href=\"topic.htm?path=inheritance-patterns-of-monogenic-disorders-mendelian-and-non-mendelian#H3744740437\" class=\"medical medical_review\">&quot;Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)&quot;, section on 'Causes of non-Mendelian inheritance'</a>.)</p></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/1\" class=\"nounderline abstract_t\">Gu Y, Harley IT, Henderson LB, et al. Identification of IFRD1 as a modifier gene for cystic fibrosis lung disease. Nature 2009; 458:1039.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/2\" class=\"nounderline abstract_t\">Stefansson H, Rye DB, Hicks A, et al. A genetic risk factor for periodic limb movements in sleep. N Engl J Med 2007; 357:639.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/3\" class=\"nounderline abstract_t\">Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. Nature 2007; 447:661.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/4\" class=\"nounderline abstract_t\">Harismendy O, Notani D, Song X, et al. 9p21 DNA variants associated with coronary artery disease impair interferon-&gamma; signalling response. Nature 2011; 470:264.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/5\" class=\"nounderline abstract_t\">Verlaan DJ, Berlivet S, Hunninghake GM, et al. Allele-specific chromatin remodeling in the ZPBP2/GSDMB/ORMDL3 locus associated with the risk of asthma and autoimmune disease. Am J Hum Genet 2009; 85:377.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/6\" class=\"nounderline abstract_t\">Hunninghake GM, Cho MH, Tesfaigzi Y, et al. MMP12, lung function, and COPD in high-risk populations. N Engl J Med 2009; 361:2599.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/7\" class=\"nounderline abstract_t\">Xing C, Garcia CK. Epigenetic inheritance of telomere length obscures identification of causative PARN locus. J Med Genet 2016; 53:356.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/8\" class=\"nounderline abstract_t\">Goldstein DB. Common genetic variation and human traits. N Engl J Med 2009; 360:1696.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/9\" class=\"nounderline abstract_t\">Hirschhorn JN. Genomewide association studies--illuminating biologic pathways. N Engl J Med 2009; 360:1699.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/10\" class=\"nounderline abstract_t\">DOLL R. Mortality from lung cancer in asbestos workers. Br J Ind Med 1955; 12:81.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/11\" class=\"nounderline abstract_t\">Reich DE, Lander ES. On the allelic spectrum of human disease. Trends Genet 2001; 17:502.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/12\" class=\"nounderline abstract_t\">Cargill M, Daley GQ. Mining for SNPs: putting the common variants--common disease hypothesis to the test. Pharmacogenomics 2000; 1:27.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/13\" class=\"nounderline abstract_t\">Welter D, MacArthur J, Morales J, et al. The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. Nucleic Acids Res 2014; 42:D1001.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/14\" class=\"nounderline abstract_t\">Dickson SP, Wang K, Krantz I, et al. Rare variants create synthetic genome-wide associations. PLoS Biol 2010; 8:e1000294.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/15\" class=\"nounderline abstract_t\">Carmichael CM, McGue M. A cross-sectional examination of height, weight, and body mass index in adult twins. J Gerontol A Biol Sci Med Sci 1995; 50:B237.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/16\" class=\"nounderline abstract_t\">Silventoinen K, Sammalisto S, Perola M, et al. Heritability of adult body height: a comparative study of twin cohorts in eight countries. Twin Res 2003; 6:399.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/17\" class=\"nounderline abstract_t\">Weedon MN, Lango H, Lindgren CM, et al. Genome-wide association analysis identifies 20 loci that influence adult height. Nat Genet 2008; 40:575.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/18\" class=\"nounderline abstract_t\">Meigs JB, Shrader P, Sullivan LM, et al. Genotype score in addition to common risk factors for prediction of type 2 diabetes. N Engl J Med 2008; 359:2208.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/19\" class=\"nounderline abstract_t\">Paynter NP, Chasman DI, Buring JE, et al. Cardiovascular disease risk prediction with and without knowledge of genetic variation at chromosome 9p21.3. Ann Intern Med 2009; 150:65.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/20\" class=\"nounderline abstract_t\">Kraft P, Hunter DJ. Genetic risk prediction--are we there yet? N Engl J Med 2009; 360:1701.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/21\" class=\"nounderline abstract_t\">Bell GI, Polonsky KS. Diabetes mellitus and genetically programmed defects in beta-cell function. Nature 2001; 414:788.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/22\" class=\"nounderline abstract_t\">Strittmatter WJ, Saunders AM, Schmechel D, et al. Apolipoprotein E: high-avidity binding to beta-amyloid and increased frequency of type 4 allele in late-onset familial Alzheimer disease. Proc Natl Acad Sci U S A 1993; 90:1977.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/23\" class=\"nounderline abstract_t\">International HapMap Consortium. A haplotype map of the human genome. Nature 2005; 437:1299.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/24\" class=\"nounderline abstract_t\">Wang DG, Fan JB, Siao CJ, et al. Large-scale identification, mapping, and genotyping of single-nucleotide polymorphisms in the human genome. Science 1998; 280:1077.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/25\" class=\"nounderline abstract_t\">Kathiresan S, Melander O, Anevski D, et al. Polymorphisms associated with cholesterol and risk of cardiovascular events. N Engl J Med 2008; 358:1240.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/26\" class=\"nounderline abstract_t\">Duerr RH, Taylor KD, Brant SR, et al. A genome-wide association study identifies IL23R as an inflammatory bowel disease gene. Science 2006; 314:1461.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/27\" class=\"nounderline abstract_t\">Gudmundsson J, Sulem P, Steinthorsdottir V, et al. Two variants on chromosome 17 confer prostate cancer risk, and the one in TCF2 protects against type 2 diabetes. Nat Genet 2007; 39:977.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/28\" class=\"nounderline abstract_t\">Yasuda K, Miyake K, Horikawa Y, et al. Variants in KCNQ1 are associated with susceptibility to type 2 diabetes mellitus. Nat Genet 2008; 40:1092.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/29\" class=\"nounderline abstract_t\">Tennessen JA, Bigham AW, O'Connor TD, et al. Evolution and functional impact of rare coding variation from deep sequencing of human exomes. Science 2012; 337:64.</a></li><li><a href=\"https://www.uptodate.com/contents/principles-of-complex-trait-genetics/abstract/30\" class=\"nounderline abstract_t\">Panoutsopoulou K, Tachmazidou I, Zeggini E. In search of low-frequency and rare variants affecting complex traits. Hum Mol Genet 2013; 22:R16.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 14600 Version 15.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H3540615\"><span>SUMMARY</span></a></li><li><a href=\"#H3540509\" id=\"outline-link-H3540509\">INTRODUCTION</a></li><li><a href=\"#H3540524\" id=\"outline-link-H3540524\">SPECTRUM OF GENETIC VARIATION</a><ul><li><a href=\"#H3540531\" id=\"outline-link-H3540531\">Determinants of genetic effect</a></li></ul></li><li><a href=\"#H3540573\" id=\"outline-link-H3540573\">COMMON VERSUS RARE ALLELES</a><ul><li><a href=\"#H707825542\" id=\"outline-link-H707825542\">Impact on disease</a></li><li><a href=\"#H3540580\" id=\"outline-link-H3540580\">Common disease-common variant hypothesis</a></li><li><a href=\"#H3540587\" id=\"outline-link-H3540587\">Common disease-rare variant hypothesis</a></li><li><a href=\"#H2886789\" id=\"outline-link-H2886789\">Other hypotheses</a></li></ul></li><li><a href=\"#H3540594\" id=\"outline-link-H3540594\">APPROACHES TO IDENTIFYING GENETIC DETERMINANTS OF COMPLEX TRAITS</a></li><li><a href=\"#H3540615\" id=\"outline-link-H3540615\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=basic-principles-of-genetic-disease\" class=\"medical medical_review\">Basic principles of genetic disease</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetic-association-studies-principles-and-applications\" class=\"medical medical_review\">Genetic association studies: Principles and applications</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetics-glossary-of-terms\" class=\"medical medical_review\">Genetics: Glossary of terms</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=inheritance-patterns-of-monogenic-disorders-mendelian-and-non-mendelian\" class=\"medical medical_review\">Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">Next-generation DNA sequencing (NGS): Principles and clinical applications</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=overview-of-genetic-variation\" class=\"medical medical_review\">Overview of genetic variation</a></li></ul></div></div>","javascript":null}