var data={"title":"Tools for genetics and genomics: Specially bred and genetically engineered mice","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Tools for genetics and genomics: Specially bred and genetically engineered mice</div><dl id=\"topicContributors\"><dt><span> </span>Author:</dt><dd><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/contributors\" class=\"contributor contributor_credentials\">Robert D Blank, MD, PhD</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/contributors\" class=\"contributor contributor_credentials\">Benjamin A Raby, MD, MPH</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/contributors\" class=\"contributor contributor_credentials\">Jennifer S Tirnauer, MD</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Jun 23, 2017.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H1\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Laboratory mice are among the most widely used model systems in biomedical research.</p><p>The breeding strategies and genetics of laboratory mice are introduced in this topic review. Detailed information regarding specific mouse strains and breeding constructs are available online at <a href=\"http://www.jax.org/&amp;token=1CuGwd+3iviUWL4zVz92B0704Gp7+rRf2KXMp82lEZk=&amp;TOPIC_ID=2894\" target=\"_blank\" class=\"external\">The Jackson Laboratory</a> website.</p><p>Other model systems for studying human disease are discussed separately (eg, yeast, worm, fruit fly, zebrafish). (See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-model-systems\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Model systems&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H2\"><span class=\"h1\">INBRED MICE</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Strains of inbred mice are model organisms for genetic studies, with hundreds of inbred strains in existence. For the purposes of genetic investigations, the key properties of inbred mice are that they are essentially isogenic and homozygous. Isogenicity means that sex-matched individuals have the same genotype across the entire genome. Homozygosity means that both chromosomes carry the same allele of each locus.</p><p>Concepts of inbreeding were introduced in a classic paper from 1931 [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/1\" class=\"abstract_t\">1</a>]. A brief summary of the principles and standards of inbreeding includes the following:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The generally accepted criterion for a strain to be considered inbred is 20 generations of brother-sister mating, or alternative breeding schemes that lead to an equivalent reduction of heterozygosity. Although historically 20 generations of full-sib inbreeding were considered to be sufficient to fix the genotype of an inbred line, with an expected residual heterozygosity of less than 0.1 percent, it has now been shown that considerably more inbreeding is needed to fix the genome [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/2\" class=\"abstract_t\">2</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Many inbred strains are complex hybrids whose ancestry includes mice that would not have interbred in the wild. For this reason, the amount of genetic variability among laboratory mice is extremely large.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Nomenclature standards have been developed regarding mouse strain and substrain designations. Substrains are stocks separated after inbred status has been achieved. In general, the strain name is given first, followed by a slash, and then the substrain designation and when applicable, by information regarding mutations or transgenes.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Not all mice having official names are inbred, and some nominal &quot;substrain&quot; differences are in fact strain differences for which nomenclature exceptions have been made because of historical precedent (eg, <span class=\"nowrap\">C57BL/6</span> and <span class=\"nowrap\">C57BL/10)</span>.</p><p/><p>Strictly speaking, new mutations and residual heterozygosity may contribute to non-isogenicity even within an inbred strain, but in most settings this is not an important issue. However, the potential accumulation of differences over time underlies the need to identify both strain and substrain when identifying mice. For these reasons, a single genotype suffices to describe all sex-matched individuals of a single substrain.</p><p>F1 animals, the first generation obtained from a cross between parents of different inbred strains, are isogenic but not homozygous. They have all inherited the same alleles from their inbred parents, but one chromosome of each pair is derived from the maternal strain and the other is derived from the paternal strain. The mitochondrial genome is derived from the maternal strain, and it is therefore important to specify the maternal and paternal strains. By convention, in notating crosses, the dam (female) is listed first and the sire (male) second.</p><p>These features allow investigation of both linkage and association in experimental mouse crosses:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Linkage is the determination of the chromosomal location of a gene influencing a trait</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Association is the relationship between a specific allele and the trait</p><p/><p>In analyzing the genetic basis of disease, we seek knowledge of both of these complementary properties. A standard mouse intercross is a biallelic study, in which only two alleles (one derived from each parent) are segregating. In contrast, natural human populations are multiallelic.</p><p>It is not necessary for an experimental mouse intercross to be limited to two progenitor strains, however. A greater range of alleles can be achieved by increasing the number of progenitors to four or more [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/3\" class=\"abstract_t\">3</a>]. In addition, the intercross can be continued beyond F2, increasing the precision with which genes can be mapped [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/4\" class=\"abstract_t\">4</a>].</p><p>Finally, the dense linkage map of the mouse, together with extensive genotypic data of the inbred mouse strains, makes most mouse crosses fully informative. In contrast, human crosses depend on finding markers in which different alleles can be identified within the study population.</p><p>The biallelic nature of many mouse crosses, the ability to control breeding by designating matings, and the known genotypic data of inbred mouse strains greatly simplifies interpretation of the genetic results of experimental breeding relative to the analysis of a human pedigree study. However, this relative simplicity also poses important limitations for the use of inbred mice as models for human genetics.</p><p class=\"headingAnchor\" id=\"H3\"><span class=\"h1\">SPECIAL BREEDING SCHEMES</span></p><p class=\"headingAnchor\" id=\"H4\"><span class=\"h2\">Recombinant inbred strains</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Recombinant inbred (RI) strains are inbred strains established from brother-sister pairs of F2 intercross mice obtained from inbred progenitors [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/5,6\" class=\"abstract_t\">5,6</a>]. The breeding scheme is illustrated in a figure (<a href=\"image.htm?imageKey=ALLRG%2F62083\" class=\"graphic graphic_figure graphicRef62083 \">figure 1</a>) [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/7,8\" class=\"abstract_t\">7,8</a>]. They are isogenic and homozygous, like other inbred strains, and at any given locus they carry either of the progenitor strain alleles.</p><p>RI strains are constructed as a series from a single initial intercross. Subsequently, because of crossing over and segregation, the effectively random locations of crossovers during the inbreeding process results in each strain in the series having a different complement of alleles derived from each of the progenitors. Any individual strain in a two-progenitor series is expected to have half of its genome derived from each progenitor strain. More than twenty generations of brother-sister inbreeding are needed for establishment of RI strains. RI strains are powerful mapping tools for monogenic traits and are also useful in mapping polygenic traits [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/2,9,10\" class=\"abstract_t\">2,9,10</a>].</p><p class=\"headingAnchor\" id=\"H5\"><span class=\"h2\">Congenic strains</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Congenic strains are those in which a specific genetic locus is transferred onto a different recipient strain [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/6,11\" class=\"abstract_t\">6,11</a>]. The breeding scheme is illustrated in a figure (<a href=\"image.htm?imageKey=ALLRG%2F63654\" class=\"graphic graphic_figure graphicRef63654 \">figure 2</a>) [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/7,8\" class=\"abstract_t\">7,8</a>]. Since linkage frequently carries neighboring genes along with the target locus into the recipient, this type of strain is generated to evaluate the effect of a gene in isolation from the effect of strain background.</p><p>Because the donor segment includes more than just a single gene, only relative isolation of the target gene is achieved. Traditionally, 10 generations of backcrossing to the recipient strain, with selection at each generation for the presence of the desired donor locus, followed by brother-sister mating thereafter, has been the standard of inbreeding for congenic strains. This standard is theoretically equivalent to 20 generations of brother-sister inbreeding.</p><p>Many investigators use marker-assisted selection to accelerate the pace at which donor alleles unlinked to the target locus are lost. This approach relies on the fact that while the expected loss of donor alleles is 50 percent at each generation, the actual loss of donor alleles in an individual mouse may be more or less than theoretical expectation. By selecting as breeders those individuals with the least number of donor alleles, the number of generations needed to achieve 99.9 percent homozygosity at unlinked loci can be reduced by three generations, corresponding to a time saving of nearly one year. This approach has therefore been called &quot;speed congenic&quot; breeding [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/12-14\" class=\"abstract_t\">12-14</a>]. Congenic strains first became widely used in immunologic experiments, in which the major histocompatibility complex was transferred between inbred strains. It has subsequently become standard practice to construct congenic strains for quantitative trait loci identified in intercrosses, utilizing negative selection for unlinked donor alleles.</p><p class=\"headingAnchor\" id=\"H6\"><span class=\"h2\">Recombinant congenic strains</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Recombinant congenic (RC) strains combine elements of breeding from both RI and congenic breeding schemes. Initially, several generations of backcrossing (usually two) are performed, followed by subsequent brother-sister inbreeding (<a href=\"image.htm?imageKey=PC%2F75129\" class=\"graphic graphic_algorithm graphicRef75129 \">algorithm 1</a>) [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/15-17\" class=\"abstract_t\">15-17</a>].</p><p>As in congenic strains, the initial backcrosses reduce the contribution of the donor strain to the final genotype of individual RC strains. As in RI strains and in contrast to congenic strains, there is generally no selection for which specific donor chromosome segments are included in RC strains. Conceptually, one might consider RC strains to be &quot;unbalanced&quot; RI strains; while RI strains derive half of their alleles from each progenitor, 2nd backcross (N3) RC strain derive one-eighth of their alleles from the donor progenitor and the remainder from the recipient progenitor. RC strains are used in mapping polygenic traits and studying epistasis, or interactions between genes [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/10,18-20\" class=\"abstract_t\">10,18-20</a>].</p><p class=\"headingAnchor\" id=\"H7\"><span class=\"h2\">Chromosome substitution strains</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Chromosome substitution (CS) strains are intended to facilitate mapping of quantitative traits, by moving chromosomes from a donor strain to a recipient strain. As for RC strains, the contributions of the donor chromosomes are unequal. Unlike the situation for RC strains, the donor contribution is theoretically restricted to a single chromosome, obviating the need for gross mapping of the relevant loci [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/21\" class=\"abstract_t\">21</a>]. Constructing CS strains relies on the same principles used in generating congenic strains, except that selection for markers spanning the donor chromosome are used simultaneously, rather than a single marker as in the case of congenic strains. Few CS strains have been bred thus far, so their use has been limited [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/22-25\" class=\"abstract_t\">22-25</a>].</p><p class=\"headingAnchor\" id=\"H8\"><span class=\"h2\">The collaborative cross</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>An ongoing effort has been to produce a mouse resource for genetic mapping that overcomes the limitations of inbreeding [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/26\" class=\"abstract_t\">26</a>]. The objective is to generate a series of 1000 eight-progenitor recombinant inbred strains. The progenitors are chosen to include more than 80 percent of the known interstrain diversity among present inbred mouse strains. F1 animals produced from such strains are isogenic, outbred, and possess haplotype blocks whose length approaches those of natural outbred populations.</p><p>Combinational mating among these inbred strains will allow 4 to the 1000th power outbred genotypes to be produced in multiple animals, where &quot;4&quot; represents two sexes and two cross directions. The number of potential genotypes would therefore be orders of magnitude larger than is necessary for application of the methods of genomewide association studies (GWAS). (See <a href=\"topic.htm?path=genetic-association-studies-principles-and-applications\" class=\"medical medical_review\">&quot;Genetic association studies: Principles and applications&quot;</a>.)</p><p>The short haplotype blocks will allow localization of functionally important genetic variation to short genomic segments. Thus, the advantages of isogeneity will be maintained, including the need to perform genotyping only once, and the ability to estimate phenotype from the pooling of multiple animals. The collaborative cross mice, however, will overcome limitations of the genetic resolving power of currently available mouse strains.</p><p>As of early 2010, more than 1000 incipient RI strains were in production [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/27-29\" class=\"abstract_t\">27-29</a>], but by early 2012 extinction reduced that number to 458 [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/30\" class=\"abstract_t\">30</a>]. The first group of strains is approaching the N23 generation of inbreeding. Not every strain survives the breeding program, with many experiencing reproductive failure between the N4 and N8 generations. The RI lines are being produced and maintained at several sites worldwide. Subsequent work has confirmed that the theoretical potential of the collaborative cross is in fact being realized [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/30,31\" class=\"abstract_t\">30,31</a>]. The first 48 collaborative cross strains are now available for distribution; details are available at the <a href=\"http://www.csbio.unc.edu/CCstatus/index.py?run=AvailableLines&amp;token=MOSx24E6HB+txKpO0eF0Gr4ba9n0xihmaulye6YPShUUT2tkNlrerfv4IBCJFHpyNT2UeC8e27R2jCbk1DpRRAeYCjfxqSHOEHiz7PzEnDQ=&amp;TOPIC_ID=2894\" target=\"_blank\" class=\"external\">UNC Systems Genetics Core website</a>.</p><p class=\"headingAnchor\" id=\"H9\"><span class=\"h1\">OUTBRED MICE</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Some commonly used mice are outbred (ie, bred specifically to maintain heterozygosity over much of the genome). Examples include Swiss and CD-1 mice, which are commercially available from major vendors. Such mice are generally healthy, vigorous, and long-lived. In general, such mice are preferred for investigations in which genetic constitution is not an issue. The diversity outcross, a population of mice derived from the collaborative cross, but with subsequent breeding undertaken to preserve heterozygosity, provides a curated outbred population that allows high-resolution genetic mapping [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/32\" class=\"abstract_t\">32</a>].</p><p class=\"headingAnchor\" id=\"H10\"><span class=\"h1\">LARGE SCALE MUTAGENESIS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Large scale mutagenesis projects represent an alternative strategy for finding genes that affect specific phenotypes. Typically, single base mutations are produced. Mutagenesis projects depend critically on the ability to recognize abnormal phenotypes reliably and easily, as many mice must be examined to find the desired mutants. Consequently, the mutagenesis strategy relies on generating viable mutants at high frequency, using robust methods to screen mutants, and applying efficient methods for mapping the mutated genes.</p><p>Notable successes have been reported [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/33\" class=\"abstract_t\">33</a>]. As an example, the Golgi protein GMAP-210, encoded by Trip11, provides a specific highlight of a mouse mutagenesis screen identifying a gene responsible for a human disease. The mutant mouse gene caused shortened limbs, failure of osteogenesis at multiple sites, and apoptosis of growth plate chondrocytes, culminating in lethality [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/34\" class=\"abstract_t\">34</a>]. These investigators recognized that the mutant mice resemble humans with achondrogenesis type 1A, and found mutations in the human Trip11 gene in 10 patients with the disorder who were not related to each other.</p><p>Various project websites with information regarding other studies include:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>MRC Harwell &ndash; <a href=\"http://www.har.mrc.ac.uk/&amp;token=qb2eot8BkfUYyBHKKvgkz+BUzDII6bhRVwwGmM9sUfSJVfhe9hpG2QsT8EBr3PDo&amp;TOPIC_ID=2894\" target=\"_blank\" class=\"external\">www.har.mrc.ac.uk/</a></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The Jackson Laboratory &ndash; <a href=\"http://www.jax.org/&amp;token=1CuGwd+3iviUWL4zVz92B0704Gp7+rRf2KXMp82lEZk=&amp;TOPIC_ID=2894\" target=\"_blank\" class=\"external\">www.jax.org</a></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Centre for Modeling Human Disease at Toronto Centre for Phenogenomics &ndash; <a href=\"http://www.cmhd.ca/enu_mutagenesis/index.html&amp;token=DPkwxpoKVYlxzvr+H5hPUMfrZv+pVFGpFQVuQjRXqWc32jfGyI1weDEMshqd0PiZolV+68+hqJgJ45pZgQ9gyQ==&amp;TOPIC_ID=2894\" target=\"_blank\" class=\"external\">www.cmhd.ca/enu_mutagenesis/index.html</a></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Tennessee Mouse Genome Consortium &ndash; <a href=\"http://www.tnmouse.org/navigate.html&amp;token=j0iZCPRE4Doc0mP56hELhJ/IsUOl7EX65c44guuBssJTR0/XhKf7Qvvyp4vRXP2g&amp;TOPIC_ID=2894\" target=\"_blank\" class=\"external\">www.tnmouse.org/navigate.html</a></p><p/><p class=\"headingAnchor\" id=\"H11\"><span class=\"h1\">GENETICALLY ENGINEERED MICE</span></p><p class=\"headingAnchor\" id=\"H12\"><span class=\"h2\">Technical underpinnings</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>To understand the various genetically engineered mouse systems, it is necessary to appreciate the technologies used to construct them. Fundamental to all such experiments is the ability to carry out molecular cloning and molecular biological manipulations [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/35-37\" class=\"abstract_t\">35-37</a>]. Specific methods used to introduce engineered DNA into mice are available in standard laboratory references [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/38,39\" class=\"abstract_t\">38,39</a>].</p><p>The fundamental technology at the core of most genetically engineered mice is the ability to introduce foreign DNA into pluripotent recipient cells. In practice, this is accomplished either by direct microinjection of DNA into a fertilized egg (<a href=\"image.htm?imageKey=PC%2F77166\" class=\"graphic graphic_picture graphicRef77166 \">picture 1</a>), or by transfection into mouse embryonic stem (ES) cells.</p><p>ES cells are pluripotent cells that can give rise to all tissues, including the germline [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/40-42\" class=\"abstract_t\">40-42</a>]. ES cells can be grown in culture and subjected to transfection and selection in the same manner as other cultured cells. Typically, targeted transfection relies upon a combination of selection for resistance to selective agents, Southern blotting, and polymerase chain reaction (PCR) to verify that exogenous DNA has been incorporated into the correct target site. ES cells shown to harbor the desired construct are then injected into mouse blastocysts, to yield chimeric embryos. The blastocysts can then be implanted into foster mothers, which then carry the pregnancy to term.</p><p class=\"headingAnchor\" id=\"H13\"><span class=\"h2\">Transgenic mice</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Transgenic mice are those into which foreign DNA has been incorporated into the genome. According to this broad definition, all types of genetically engineered mice are transgenic. In a more narrow sense, transgenic mice have an &quot;extra&quot; gene introduced to accomplish any of several experimental objectives:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>To correct pathology caused by mutation, thus proving that the transgene complements the preexisting mutation [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/43\" class=\"abstract_t\">43</a>]</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>To introduce a reporter gene under specified genetic control to identify tissues and times at which the included control is active [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/44-46\" class=\"abstract_t\">44-46</a>]</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>To introduce an abnormal gene, thus creating a disease model [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/47\" class=\"abstract_t\">47</a>]</p><p/><p>Simple transgenics, in which no attempt is made to target the construct to a specific site in the genome, can be generated by microinjecting DNA directly into one of the pronuclei of fertilized eggs in vitro. Surviving blastocysts are implanted into foster mothers and pups analyzed for presence of the transgene. Early work was typically performed using outbred eggs, but now much microinjection work is performed using eggs obtained from <span class=\"nowrap\">FVB/N</span> mice, which have large pronuclei and breed well [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/48\" class=\"abstract_t\">48</a>].</p><p class=\"headingAnchor\" id=\"H14\"><span class=\"h2\">Knockout mice</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>In contrast to simple transgenic animals, knockout mice depend upon successful gene targeting to disrupt the target gene [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/49\" class=\"abstract_t\">49</a>]. This is achieved by using genetic selection to enrich transfected ES cells for successful targeting. Targeting vectors for mammalian cells use a combination of positive selection (incorporation and retention of a selected marker) and negative selection (loss of a second selected marker) (<a href=\"image.htm?imageKey=PC%2F79203\" class=\"graphic graphic_figure graphicRef79203 \">figure 3</a>). Technical aspects of constructing the necessary targeting vectors have advanced in recent years, and feature bacterial artificial chromosome clones prepared from <span class=\"nowrap\">129/Sv</span> mice [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/50\" class=\"abstract_t\">50</a>] and a &quot;recombineering&quot; host-vector system in which some steps are accomplished in vivo in bacteria [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/51\" class=\"abstract_t\">51</a>].</p><p>The motivation for doing this is to study the consequences of loss of function of the targeted gene directly in vivo. In contrast to experiments using cultured cells, the knockout allows the consequences of target gene disruption to be evaluated in the context of whole-organism physiology. In this setting, it is possible to study physiological adaptation to the knockout and discover effects in tissues in which pathology might not have been suspected a priori. Comparison and contrast among knockout phenotypes for related target genes help to identify both the unique and redundant functions of their products.</p><p class=\"headingAnchor\" id=\"H15\"><span class=\"h2\">Knock-in mice</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Knock-in mice are theoretically similar to knockout mice, with the important difference that an altered rather than a null version of the target gene is substituted for the naturally occurring allele. Knock-in technology allows examination of the effects of different mutations on the same gene. This is particularly informative if the mutations thought to cause human disease result in gain of function rather than loss of function (eg, oncogenes) or if the goal is to investigate a mutant in a single tissue. (See <a href=\"#H16\" class=\"local\">'Conditional systems'</a> below.)</p><p class=\"headingAnchor\" id=\"H12494409\"><span class=\"h2\">Genome editing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Notable technical advances have occurred in the ability to create knockout and knock-in animals through the use of zinc finger nucleases, transcription activator-like effector nucleases (TALENs), and the <span class=\"nowrap\">CRISPR/Cas</span> system [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/52\" class=\"abstract_t\">52</a>] (see <a href=\"topic.htm?path=genetics-glossary-of-terms\" class=\"medical medical_review\">&quot;Genetics: Glossary of terms&quot;</a>). These tools have greatly reduced the time and expense of creating such animals, and in addition, they can be used in multiple species. They are therefore widely used in zebrafish and rats.</p><p class=\"headingAnchor\" id=\"H16\"><span class=\"h2\">Conditional systems</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Some gene disruptions are lethal or lethal too early in development to allow fruitful investigation of their consequences. A variety of methods have been developed to overcome this class of problems. These are briefly described here.</p><p>Modulating transgene expression can be achieved by placing the transgene under the transcriptional control of an inducible promoter. The bacterial <a href=\"topic.htm?path=tetracycline-drug-information\" class=\"drug drug_general\">tetracycline</a> resistance operon provides such a reagent system [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/53-55\" class=\"abstract_t\">53-55</a>]. The investigator can then regulate transgene expression by titrating administration of tetracycline to the transgenic animals.</p><p>A strategy by which knockout expression can be restricted is generation of tissue-specific knockouts. This approach exploits the ability of bacteriophage P1 Cre recombinase to mediate site-specific recombination at specific short sequence elements called loxP sites [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/56\" class=\"abstract_t\">56</a>]. Two engineered mice must be produced to carry out such an experiment. One construct introduces the loxP sites into the target gene, flanking sufficient DNA so that its deletion will achieve the desired loss of target gene activity. The second construct, which can be a simple transgene, introduces a functional gene for Cre recombinase driven by a tissue-specific promoter. Mice homozygous for the target gene construct are then mated to mice harboring the tissue-specific Cre construct. Offspring receiving the construct (expected to be 50 percent of the progeny) will express Cre recombinase in a tissue-specific manner, leading to excision of the portion of the target gene flanked by the loxP sites, thus knocking out the gene in that tissue exclusively (<a href=\"image.htm?imageKey=PC%2F51486\" class=\"graphic graphic_figure graphicRef51486 \">figure 4</a>) [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/57,58\" class=\"abstract_t\">57,58</a>]. Many investigators test the biological function of the Cre transgene by mating it to any of several available &quot;reporter&quot; mice [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/59-65\" class=\"abstract_t\">59-65</a>].</p><p>Collaborative projects in North America and in Europe have developed conditional knockout alleles for thousands of genes [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/66-69\" class=\"abstract_t\">66-69</a>]. In addition to Cre reporter mice, there are also reporter mice that can identify the anatomic sites at which specific biologic pathways are active. As an example, the TOPGAL mouse expressed beta-galactosidase at sites of canonical Wnt signaling [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/70\" class=\"abstract_t\">70</a>], allowing investigators to identify the tissues in which the pathway is active.</p><p>The extraembryonic tissues are important in early developmental steps including gastrulation. One group developed a method by which early embryos can be tetraploidized, and chimeric embryos produced from tetraploidized embryos and cultured ES cells [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/71,72\" class=\"abstract_t\">71,72</a>]. Under these conditions, the tetraploid cells give rise exclusively to extraembryonic tissue, while the diploid ES cells produce all of the embryo proper. This approach allows investigators to overcome some early developmental defects, thus allowing later functions of the disrupted gene to be studied [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/73,74\" class=\"abstract_t\">73,74</a>].</p><p class=\"headingAnchor\" id=\"H17\"><span class=\"h2\">Limitations and caveats</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Characteristics of inbred strains are highly variable, so knowing the background in which a genetic construct has been studied is essential to interpreting its biology. For technical reasons, many genetically engineered mice have been generated on one of the 129 strains, which recently have been found to be more diverse than previously believed [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/75-77\" class=\"abstract_t\">75-77</a>]. Investigators routinely breed founder animals to a recipient strain, most often <span class=\"nowrap\">C57BL/6</span>. Consequently, the constructs are often studied on a poorly-defined &quot;mixed 129 X <span class=\"nowrap\">C57BL/6</span> background,&quot; without further information regarding the relative contributions of the progenitor genomes, number of generations of subsequent inbreeding, or often even the correct strain information regarding the progenitors. While unfortunate, this is the status of most of the extant literature. Efforts to improve reporting of strain background in the future are underway [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/78\" class=\"abstract_t\">78</a>].</p><p>For mice in which the construct has not been targeted to a specific locus, incorporation of the transgene may result in insertional mutagenesis, with the resulting phenotype arising not from the transgene, but from disruption of the gene into which the transgene was placed [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/79\" class=\"abstract_t\">79</a>]. The transgene's expression may also be variable according to the properties of the insertion site [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/80\" class=\"abstract_t\">80</a>].</p><p>A third complication is the usual result of multiple copies of the transgene being inserted into the genome, with consequent differences in transgene expression level. These limitations can be addressed by targeting transgenes to specific sites. One site that allows insertion of a single copy of the transgene while allowing transcription to be mediated by sequences included in the targeting vector is the HPRT locus, encoding the salvage purine utilization enzyme <span class=\"nowrap\">hypoxanthine/guanine</span> phosphoribosyl transferase [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/81,82\" class=\"abstract_t\">81,82</a>].</p><p>The major limitation of knockouts is that the allele generated is, by design, null. Therefore, while they are useful for establishing the role of the target gene in a pathway, it is not necessarily the case that mutations in the target gene account for human diseases or population variation in downstream phenotypes mediated by it. Knock-in strategies can address this by allowing study of a series of mutant alleles. By virtue of being targeted to the homologous locus, the issue of unintentional insertional mutation does not arise with knockout mice. The limitations related to strain background are significant, particularly because 129 related strains have been the source of most ES cell lines. More recent success in generating successful ES cells from other strains promises to mitigate this problem in the future [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/83,84\" class=\"abstract_t\">83,84</a>].</p><p>Another difficulty is that Cre constructs are neither perfectly efficient nor perfectly specific. Thus, &quot;tissue-specific&quot; Cre constructs are sometimes active outside the target tissue, and Cre constructs presumed to be inducible are active in the absence of the inducing substance [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/85,86\" class=\"abstract_t\">85,86</a>]. Furthermore, Cre constructs can exert inherent biological effects, including &quot;Cre toxicity&quot; and Cre-mediated suppression of tumor growth [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/87-91\" class=\"abstract_t\">87-91</a>].</p><p>The practical response to these limitations is that when transgenic mice are generated, investigators routinely study animals derived from several different founders. The minimal characterization will generally include estimation of copy number, transgene mRNA level, and transgene protein level. More detailed analysis is then conducted on one or a small number of the transgenic lines. A growing number of studies also use lineage tracing to monitor Cre activity [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/92\" class=\"abstract_t\">92</a>]; lineage tracing is emerging as a &quot;best practice&quot; [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/93\" class=\"abstract_t\">93</a>]. Appropriate and complete control groups need to be characterized in parallel. Careful investigators will also report the breeding history between founder and the analyzed animals.</p><p class=\"headingAnchor\" id=\"H18\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Laboratory mice are among the most widely used model systems in biomedical research. The key properties of inbred mice relative to genetics studies are that they are isogenic and homozygous. F1 animals, the first generation obtained from a cross between parents of different inbred strains, are isogenic but not homozygous. Isogenicity allows study of multiple animals sharing a single genotype, thus allowing better estimation of the phenotype being studied. (See <a href=\"#H2\" class=\"local\">'Inbred mice'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Recombinant inbred strains, the result of a series of brother-sister matings from an initial intercross, allow gene mapping (<a href=\"image.htm?imageKey=ALLRG%2F62083\" class=\"graphic graphic_figure graphicRef62083 \">figure 1</a>). Congenic strains, in which a specific genetic locus is transferred onto a different recipient strain (<a href=\"image.htm?imageKey=ALLRG%2F63654\" class=\"graphic graphic_figure graphicRef63654 \">figure 2</a>), allow evaluation of a gene in isolation from the effect of strain background. Chromosome substitution (CS) strains facilitate mapping of quantitative traits by moving chromosomes from a donor strain to a recipient strain. (See <a href=\"#H3\" class=\"local\">'Special breeding schemes'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The collaborative cross preserves the key advantage of isogenicity, while overcoming many of the limitations of inbred mice. This model better approximates the human genetic structure by being outbred and having a preponderance of short haplotype blocks, making this a powerful tool for relating phenotypes to specific DNA variants. (See <a href=\"#H8\" class=\"local\">'The collaborative cross'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The ability to introduce foreign DNA into pluripotent recipient cells is at the core of most genetically engineered mice. (See <a href=\"#H11\" class=\"local\">'Genetically engineered mice'</a> above.)</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Transgenic mice are those into which foreign DNA has been incorporated into the genome.</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Knockout mice depend upon successful gene transfer targeted to disrupt a specific gene to study the consequences of loss of function of the targeted gene directly in vivo.</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Knock-in technology allows examination of the effects of different mutations on the same gene.</p></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/1\" class=\"nounderline abstract_t\">Haldane JB, Waddington CH. Inbreeding and Linkage. Genetics 1931; 16:357.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/2\" class=\"nounderline abstract_t\">Broman KW. The genomes of recombinant inbred lines. Genetics 2005; 169:1133.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/3\" class=\"nounderline abstract_t\">RODERICK TH. Selection for radiation resistance in mice. Genetics 1963; 48:205.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/4\" class=\"nounderline abstract_t\">Darvasi A, Soller M. Advanced intercross lines, an experimental population for fine genetic mapping. Genetics 1995; 141:1199.</a></li><li class=\"breakAll\">Taylor BA. Recombinant inbred strains: Use in gene mapping. In: Origins of Inbred Mice, Morse HC (Ed), Academic Press, New York 1978. p.423.</li><li class=\"breakAll\">Bailey DW. Recombinant inbred strains and bilineal congenic strains. In: The Mouse in Biomedical Research, Foster HL, Small JD, Fox JG (Eds), Academic Press, New York 1981. p.223.</li><li class=\"breakAll\">Silver LM. Mouse Genetics, Oxford University Press, New York 1995.</li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/8\" class=\"nounderline abstract_t\">Blake JA, Eppig JT, Richardson JE, et al. The Mouse Genome Database (MGD): integration nexus for the laboratory mouse. Nucleic Acids Res 2001; 29:91.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/9\" class=\"nounderline abstract_t\">Plomin R, McClearn GE, Gora-Maslak G, Neiderhiser JM. Use of recombinant inbred strains to detect quantitative trait loci associated with behavior. Behav Genet 1991; 21:99.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/10\" class=\"nounderline abstract_t\">Darvasi A. Experimental strategies for the genetic dissection of complex traits in animal models. Nat Genet 1998; 18:19.</a></li><li class=\"breakAll\">Flaherty L. Congenic Strains. In: The Mouse in Biomedical Research: History, Genetics, and Wild Mice, Small JD, Fox JG (Eds), Academic Press, New York 1981. p.215.</li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/12\" class=\"nounderline abstract_t\">Visscher PM. Speed congenics: accelerated genome recovery using genetic markers. Genet Res 1999; 74:81.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/13\" class=\"nounderline abstract_t\">Wakeland E, Morel L, Achey K, et al. Speed congenics: a classic technique in the fast lane (relatively speaking). Immunol Today 1997; 18:472.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/14\" class=\"nounderline abstract_t\">Markel P, Shu P, Ebeling C, et al. Theoretical and empirical issues for marker-assisted breeding of congenic mouse strains. Nat Genet 1997; 17:280.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/15\" class=\"nounderline abstract_t\">D&eacute;mant P, Hart AA. Recombinant congenic strains--a new tool for analyzing genetic traits determined by more than one gene. Immunogenetics 1986; 24:416.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/16\" class=\"nounderline abstract_t\">van Zutphen LF, Den Bieman M, Lankhorst A, Demant P. Segregation of genes from donor strain during the production of recombinant congenic strains. Lab Anim 1991; 25:193.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/17\" class=\"nounderline abstract_t\">Moen CJ, van der Valk MA, Snoek M, et al. The recombinant congenic strains--a novel genetic tool applied to the study of colon tumor development in the mouse. Mamm Genome 1991; 1:217.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/18\" class=\"nounderline abstract_t\">Fijneman RJ, de Vries SS, Jansen RC, Demant P. Complex interactions of new quantitative trait loci, Sluc1, Sluc2, Sluc3, and Sluc4, that influence the susceptibility to lung cancer in the mouse. Nat Genet 1996; 14:465.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/19\" class=\"nounderline abstract_t\">van Wezel T, Stassen AP, Moen CJ, et al. Gene interaction and single gene effects in colon tumour susceptibility in mice. Nat Genet 1996; 14:468.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/20\" class=\"nounderline abstract_t\">Frankel WN, Schork NJ. Who's afraid of epistasis? Nat Genet 1996; 14:371.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/21\" class=\"nounderline abstract_t\">Nadeau JH, Singer JB, Matin A, Lander ES. Analysing complex genetic traits with chromosome substitution strains. Nat Genet 2000; 24:221.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/22\" class=\"nounderline abstract_t\">Hudgins CC, Steinberg RT, Klinman DM, et al. Studies of consomic mice bearing the Y chromosome of the BXSB mouse. J Immunol 1985; 134:3849.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/23\" class=\"nounderline abstract_t\">Eales BA, Nahas M, Biddle FG. Directional dominance and a developmental model for the expression of the Tda testis-determining autosomal trait of the mouse. Genome 1996; 39:520.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/24\" class=\"nounderline abstract_t\">Matin A, Collin GB, Asada Y, et al. Susceptibility to testicular germ-cell tumours in a 129.MOLF-Chr 19 chromosome substitution strain. Nat Genet 1999; 23:237.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/25\" class=\"nounderline abstract_t\">Ulbrich M, Schmidt VC, Ronsiek M, et al. Genetic modifiers that aggravate the neurological phenotype of the wobbler mouse. Neuroreport 2002; 13:535.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/26\" class=\"nounderline abstract_t\">Churchill GA, Airey DC, Allayee H, et al. The Collaborative Cross, a community resource for the genetic analysis of complex traits. Nat Genet 2004; 36:1133.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/27\" class=\"nounderline abstract_t\">Morahan G, Balmer L, Monley D. Establishment of &quot;The Gene Mine&quot;: a resource for rapid identification of complex trait genes. Mamm Genome 2008; 19:390.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/28\" class=\"nounderline abstract_t\">Chesler EJ, Miller DR, Branstetter LR, et al. The Collaborative Cross at Oak Ridge National Laboratory: developing a powerful resource for systems genetics. Mamm Genome 2008; 19:382.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/29\" class=\"nounderline abstract_t\">Iraqi FA, Churchill G, Mott R. The Collaborative Cross, developing a resource for mammalian systems genetics: a status report of the Wellcome Trust cohort. Mamm Genome 2008; 19:379.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/30\" class=\"nounderline abstract_t\">Collaborative Cross Consortium. The genome architecture of the Collaborative Cross mouse genetic reference population. Genetics 2012; 190:389.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/31\" class=\"nounderline abstract_t\">Aylor DL, Valdar W, Foulds-Mathes W, et al. Genetic analysis of complex traits in the emerging Collaborative Cross. Genome Res 2011; 21:1213.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/32\" class=\"nounderline abstract_t\">Svenson KL, Gatti DM, Valdar W, et al. High-resolution genetic mapping using the Mouse Diversity outbred population. Genetics 2012; 190:437.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/33\" class=\"nounderline abstract_t\">Acevedo-Arozena A, Wells S, Potter P, et al. ENU mutagenesis, a way forward to understand gene function. Annu Rev Genomics Hum Genet 2008; 9:49.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/34\" class=\"nounderline abstract_t\">Smits P, Bolton AD, Funari V, et al. Lethal skeletal dysplasia in mice and humans lacking the golgin GMAP-210. N Engl J Med 2010; 362:206.</a></li><li class=\"breakAll\">Ausubel FM, Brent R, Kingston RE, et al. Current Protocols in Molecular Biology. In: Current Protocols, Janssen (Ed), Greene Publishing Associates, Inc. and John Wiley &amp; Sons, Inc., New York 1996.</li><li class=\"breakAll\">Sambrook J, Fritsch EF, Maniatis T. Molecular Cloning: A Laboratory Manual, Cold Spring Laboratory Press, Cold Spring Harbor, NY 1989.</li><li class=\"breakAll\">Gene Targeting: A Practical Approach, 2nd ed, Joyner AL (Ed), Oxford University Press, New York 2000.</li><li class=\"breakAll\">Manipulating the Mouse Embryo: Laboratory Manual, Hogan B, Beddington R, Costantini F, Lacy E (Eds), Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY 1994.</li><li class=\"breakAll\">Mouse Genetics and Transgenics: A Practical Approach, Jackson IJ, Abbott CM (Eds), Oxford University Press, New York 2000.</li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/40\" class=\"nounderline abstract_t\">Bradley A, Evans M, Kaufman MH, Robertson E. Formation of germ-line chimaeras from embryo-derived teratocarcinoma cell lines. Nature 1984; 309:255.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/41\" class=\"nounderline abstract_t\">Robertson E, Bradley A, Kuehn M, Evans M. Germ-line transmission of genes introduced into cultured pluripotential cells by retroviral vector. Nature 1986; 323:445.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/42\" class=\"nounderline abstract_t\">Martin GR. Isolation of a pluripotent cell line from early mouse embryos cultured in medium conditioned by teratocarcinoma stem cells. Proc Natl Acad Sci U S A 1981; 78:7634.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/43\" class=\"nounderline abstract_t\">Hardin JD, Boast S, Mendelsohn M, et al. Transgenes encoding both type I and type IV c-abl proteins rescue the lethality of c-abl mutant mice. Oncogene 1996; 12:2669.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/44\" class=\"nounderline abstract_t\">Kalajzic I, Kalajzic Z, Kaliterna M, et al. Use of type I collagen green fluorescent protein transgenes to identify subpopulations of cells at different stages of the osteoblast lineage. J Bone Miner Res 2002; 17:15.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/45\" class=\"nounderline abstract_t\">Chai Y, Jiang X, Ito Y, et al. Fate of the mammalian cranial neural crest during tooth and mandibular morphogenesis. Development 2000; 127:1671.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/46\" class=\"nounderline abstract_t\">Snippert HJ, van der Flier LG, Sato T, et al. Intestinal crypt homeostasis results from neutral competition between symmetrically dividing Lgr5 stem cells. Cell 2010; 143:134.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/47\" class=\"nounderline abstract_t\">Khillan JS, Olsen AS, Kontusaari S, et al. Transgenic mice that express a mini-gene version of the human gene for type I procollagen (COL1A1) develop a phenotype resembling a lethal form of osteogenesis imperfecta. J Biol Chem 1991; 266:23373.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/48\" class=\"nounderline abstract_t\">Taketo M, Schroeder AC, Mobraaten LE, et al. FVB/N: an inbred mouse strain preferable for transgenic analyses. Proc Natl Acad Sci U S A 1991; 88:2065.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/49\" class=\"nounderline abstract_t\">Mansour SL, Thomas KR, Capecchi MR. Disruption of the proto-oncogene int-2 in mouse embryo-derived stem cells: a general strategy for targeting mutations to non-selectable genes. Nature 1988; 336:348.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/50\" class=\"nounderline abstract_t\">Adams DJ, Quail MA, Cox T, et al. A genome-wide, end-sequenced 129Sv BAC library resource for targeting vector construction. Genomics 2005; 86:753.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/51\" class=\"nounderline abstract_t\">Liu P, Jenkins NA, Copeland NG. A highly efficient recombineering-based method for generating conditional knockout mutations. Genome Res 2003; 13:476.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/52\" class=\"nounderline abstract_t\">Maggio I, Gon&ccedil;alves MA. Genome editing at the crossroads of delivery, specificity, and fidelity. Trends Biotechnol 2015; 33:280.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/53\" class=\"nounderline abstract_t\">Fedorov LM, Tyrsin OY, Krenn V, et al. Tet-system for the regulation of gene expression during embryonic development. Transgenic Res 2001; 10:247.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/54\" class=\"nounderline abstract_t\">Bohl D, Heard JM. Modulation of erythropoietin delivery from engineered muscles in mice. Hum Gene Ther 1997; 8:195.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/55\" class=\"nounderline abstract_t\">Paulus W, Baur I, Boyce FM, et al. Self-contained, tetracycline-regulated retroviral vector system for gene delivery to mammalian cells. J Virol 1996; 70:62.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/56\" class=\"nounderline abstract_t\">Hoess RH, Ziese M, Sternberg N. P1 site-specific recombination: nucleotide sequence of the recombining sites. Proc Natl Acad Sci U S A 1982; 79:3398.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/57\" class=\"nounderline abstract_t\">Rajewsky K, Gu H, K&uuml;hn R, et al. Conditional gene targeting. J Clin Invest 1996; 98:600.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/58\" class=\"nounderline abstract_t\">Rossant J, McMahon A. &quot;Cre&quot;-ating mouse mutants-a meeting review on conditional mouse genetics. Genes Dev 1999; 13:142.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/59\" class=\"nounderline abstract_t\">Soriano P. Generalized lacZ expression with the ROSA26 Cre reporter strain. Nat Genet 1999; 21:70.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/60\" class=\"nounderline abstract_t\">Lobe CG, Koop KE, Kreppner W, et al. Z/AP, a double reporter for cre-mediated recombination. Dev Biol 1999; 208:281.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/61\" class=\"nounderline abstract_t\">Mao X, Fujiwara Y, Orkin SH. Improved reporter strain for monitoring Cre recombinase-mediated DNA excisions in mice. Proc Natl Acad Sci U S A 1999; 96:5037.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/62\" class=\"nounderline abstract_t\">Kawamoto S, Niwa H, Tashiro F, et al. A novel reporter mouse strain that expresses enhanced green fluorescent protein upon Cre-mediated recombination. FEBS Lett 2000; 470:263.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/63\" class=\"nounderline abstract_t\">Novak A, Guo C, Yang W, et al. Z/EG, a double reporter mouse line that expresses enhanced green fluorescent protein upon Cre-mediated excision. Genesis 2000; 28:147.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/64\" class=\"nounderline abstract_t\">Mao X, Fujiwara Y, Chapdelaine A, et al. Activation of EGFP expression by Cre-mediated excision in a new ROSA26 reporter mouse strain. Blood 2001; 97:324.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/65\" class=\"nounderline abstract_t\">Srinivas S, Watanabe T, Lin CS, et al. Cre reporter strains produced by targeted insertion of EYFP and ECFP into the ROSA26 locus. BMC Dev Biol 2001; 1:4.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/66\" class=\"nounderline abstract_t\">Schn&uuml;tgen F, De-Zolt S, Van Sloun P, et al. Genomewide production of multipurpose alleles for the functional analysis of the mouse genome. Proc Natl Acad Sci U S A 2005; 102:7221.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/67\" class=\"nounderline abstract_t\">Schn&uuml;tgen F. Generation of multipurpose alleles for the functional analysis of the mouse genome. Brief Funct Genomic Proteomic 2006; 5:15.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/68\" class=\"nounderline abstract_t\">Floss T, Schn&uuml;tgen F. Conditional gene trapping using the FLEx system. Methods Mol Biol 2008; 435:127.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/69\" class=\"nounderline abstract_t\">Friedel RH, Seisenberger C, Kaloff C, Wurst W. EUCOMM--the European conditional mouse mutagenesis program. Brief Funct Genomic Proteomic 2007; 6:180.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/70\" class=\"nounderline abstract_t\">DasGupta R, Fuchs E. Multiple roles for activated LEF/TCF transcription complexes during hair follicle development and differentiation. Development 1999; 126:4557.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/71\" class=\"nounderline abstract_t\">Nagy A, Rossant J, Nagy R, et al. Derivation of completely cell culture-derived mice from early-passage embryonic stem cells. Proc Natl Acad Sci U S A 1993; 90:8424.</a></li><li class=\"breakAll\">Nagy A, Rossant J. Production of completely ES cell-derived fetuses. In: Gene Targeting: A Practical Approach, Joyner AL (Ed), Oxford University Press, New York 1993. p.147.</li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/73\" class=\"nounderline abstract_t\">Duncan SA, Nagy A, Chan W. Murine gastrulation requires HNF-4 regulated gene expression in the visceral endoderm: tetraploid rescue of Hnf-4(-/-) embryos. Development 1997; 124:279.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/74\" class=\"nounderline abstract_t\">Li J, Ning G, Duncan SA. Mammalian hepatocyte differentiation requires the transcription factor HNF-4alpha. Genes Dev 2000; 14:464.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/75\" class=\"nounderline abstract_t\">Threadgill DW, Yee D, Matin A, et al. Genealogy of the 129 inbred strains: 129/SvJ is a contaminated inbred strain. Mamm Genome 1997; 8:390.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/76\" class=\"nounderline abstract_t\">Simpson EM, Linder CC, Sargent EE, et al. Genetic variation among 129 substrains and its importance for targeted mutagenesis in mice. Nat Genet 1997; 16:19.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/77\" class=\"nounderline abstract_t\">Threadgill DW, Matin A, Yee D, et al. SSLPs to map genetic differences between the 129 inbred strains and closed-colony, random-bred CD-1 mice. Mamm Genome 1997; 8:441.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/78\" class=\"nounderline abstract_t\">Festing MF, Simpson EM, Davisson MT, Mobraaten LE. Revised nomenclature for strain 129 mice. Mamm Genome 1999; 10:836.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/79\" class=\"nounderline abstract_t\">Tutois S, Salaun J, Mattei MG, Gu&eacute;net JL. Tg (9 HSA-MYC), a homozygous lethal insertion in the mouse. Mamm Genome 1991; 1:184.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/80\" class=\"nounderline abstract_t\">DeLoia JA, Solter D. A transgene insertional mutation at an imprinted locus in the mouse genome. Dev Suppl 1990; :73.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/81\" class=\"nounderline abstract_t\">Bronson SK, Plaehn EG, Kluckman KD, et al. Single-copy transgenic mice with chosen-site integration. Proc Natl Acad Sci U S A 1996; 93:9067.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/82\" class=\"nounderline abstract_t\">Misra RP, Bronson SK, Xiao Q, et al. Generation of single-copy transgenic mouse embryos directly from ES cells by tetraploid embryo complementation. BMC Biotechnol 2001; 1:12.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/83\" class=\"nounderline abstract_t\">Kitani H, Takagi N, Atsumi T, et al. Isolation of a germline-transmissible embryonic stem (ES) cell line from C3H/He mice. Zoolog Sci 1996; 13:865.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/84\" class=\"nounderline abstract_t\">Schuster-Gossler K, Lee AW, Lerner CP, et al. Use of coisogenic host blastocysts for efficient establishment of germline chimeras with C57BL/6J ES cell lines. Biotechniques 2001; 31:1022.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/85\" class=\"nounderline abstract_t\">Jeffery E, Berry R, Church CD, et al. Characterization of Cre recombinase models for the study of adipose tissue. Adipocyte 2014; 3:206.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/86\" class=\"nounderline abstract_t\">Kristianto J, Johnson MG, Zastrow RK, et al. Spontaneous recombinase activity of Cre-ERT2 in vivo. Transgenic Res 2017; 26:411.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/87\" class=\"nounderline abstract_t\">Schmidt-Supprian M, Rajewsky K. Vagaries of conditional gene targeting. Nat Immunol 2007; 8:665.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/88\" class=\"nounderline abstract_t\">Loonstra A, Vooijs M, Beverloo HB, et al. Growth inhibition and DNA damage induced by Cre recombinase in mammalian cells. Proc Natl Acad Sci U S A 2001; 98:9209.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/89\" class=\"nounderline abstract_t\">Jeannotte L, Aubin J, Bourque S, et al. Unsuspected effects of a lung-specific Cre deleter mouse line. Genesis 2011; 49:152.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/90\" class=\"nounderline abstract_t\">Thanos A, Morizane Y, Murakami Y, et al. Evidence for baseline retinal pigment epithelium pathology in the Trp1-Cre mouse. Am J Pathol 2012; 180:1917.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/91\" class=\"nounderline abstract_t\">Li Y, Choi PS, Casey SC, Felsher DW. Activation of Cre recombinase alone can induce complete tumor regression. PLoS One 2014; 9:e107589.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/92\" class=\"nounderline abstract_t\">Kretzschmar K, Watt FM. Lineage tracing. Cell 2012; 148:33.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice/abstract/93\" class=\"nounderline abstract_t\">Heffner CS, Herbert Pratt C, Babiuk RP, et al. Supporting conditional mouse mutagenesis with a comprehensive cre characterization resource. Nat Commun 2012; 3:1218.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 2894 Version 17.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H18\"><span>SUMMARY</span></a></li><li><a href=\"#H1\" id=\"outline-link-H1\">INTRODUCTION</a></li><li><a href=\"#H2\" id=\"outline-link-H2\">INBRED MICE</a></li><li><a href=\"#H3\" id=\"outline-link-H3\">SPECIAL BREEDING SCHEMES</a><ul><li><a href=\"#H4\" id=\"outline-link-H4\">Recombinant inbred strains</a></li><li><a href=\"#H5\" id=\"outline-link-H5\">Congenic strains</a></li><li><a href=\"#H6\" id=\"outline-link-H6\">Recombinant congenic strains</a></li><li><a href=\"#H7\" id=\"outline-link-H7\">Chromosome substitution strains</a></li><li><a href=\"#H8\" id=\"outline-link-H8\">The collaborative cross</a></li></ul></li><li><a href=\"#H9\" id=\"outline-link-H9\">OUTBRED MICE</a></li><li><a href=\"#H10\" id=\"outline-link-H10\">LARGE SCALE MUTAGENESIS</a></li><li><a href=\"#H11\" id=\"outline-link-H11\">GENETICALLY ENGINEERED MICE</a><ul><li><a href=\"#H12\" id=\"outline-link-H12\">Technical underpinnings</a></li><li><a href=\"#H13\" id=\"outline-link-H13\">Transgenic mice</a></li><li><a href=\"#H14\" id=\"outline-link-H14\">Knockout mice</a></li><li><a href=\"#H15\" id=\"outline-link-H15\">Knock-in mice</a></li><li><a href=\"#H12494409\" id=\"outline-link-H12494409\">Genome editing</a></li><li><a href=\"#H16\" id=\"outline-link-H16\">Conditional systems</a></li><li><a href=\"#H17\" id=\"outline-link-H17\">Limitations and caveats</a></li></ul></li><li><a href=\"#H18\" id=\"outline-link-H18\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>GRAPHICS <a href=\"#\" id=\"viewAllGraphicsLink\">View All</a></h2><div id=\"outlineGraphics\"><ul><li><div id=\"PC/2894|ALG\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"ALGORITHMS\">ALGORITHMS</a></div><ul><li><a href=\"image.htm?imageKey=PC/75129\" class=\"graphic graphic_algorithm\">- Congenic breeding scheme</a></li></ul></li><li><div id=\"PC/2894|FIG\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"FIGURES\">FIGURES</a></div><ul><li><a href=\"image.htm?imageKey=ALLRG/62083\" class=\"graphic graphic_figure\">- Construction RI strains</a></li><li><a href=\"image.htm?imageKey=ALLRG/63654\" class=\"graphic graphic_figure\">- Creation congenic strain</a></li><li><a href=\"image.htm?imageKey=PC/79203\" class=\"graphic graphic_figure\">- Generating DNA knockout mice</a></li><li><a href=\"image.htm?imageKey=PC/51486\" class=\"graphic graphic_figure\">- Tissue specific knockout const</a></li></ul></li><li><div id=\"PC/2894|PIC\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"PICTURES\">PICTURES</a></div><ul><li><a href=\"image.htm?imageKey=PC/77166\" class=\"graphic graphic_picture\">- Microinjection into fert egg</a></li></ul></li></ul></div></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=genetic-association-studies-principles-and-applications\" class=\"medical medical_review\">Genetic association studies: Principles and applications</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetics-glossary-of-terms\" class=\"medical medical_review\">Genetics: Glossary of terms</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-model-systems\" class=\"medical medical_review\">Tools for genetics and genomics: Model systems</a></li></ul></div></div>","javascript":null}