var data={"title":"Tools for genetics and genomics: Polymerase chain reaction","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Tools for genetics and genomics: Polymerase chain reaction</div><dl id=\"topicContributors\"><dt><span> </span>Author:</dt><dd><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/contributors\" class=\"contributor contributor_credentials\">Benjamin A Raby, MD, MPH</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/contributors\" class=\"contributor contributor_credentials\">Anne Slavotinek, MBBS, PhD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/contributors\" class=\"contributor contributor_credentials\">Jennifer S Tirnauer, MD</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Aug 09, 2016.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H8006600\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The polymerase chain reaction (PCR) is the basis of many modern molecular biology and molecular genetics techniques. In just a few hours, PCR can amplify a single DNA molecule a million-fold [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/1\" class=\"abstract_t\">1</a>]. The greatly amplified target DNA is subsequently analyzed via other techniques.</p><p>Since its first publication in 1985 [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/2\" class=\"abstract_t\">2</a>], the impact of PCR on biomedical research has been immense. This technology allows large quantities of rare sequences to be synthesized, cloned, and analyzed with high reliability and minimum effort. The award of the 1993 Nobel Prize in Chemistry to Kary B. Mullis for inventing the technique recognized the importance of PCR-based methods [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/3\" class=\"abstract_t\">3</a>].</p><p>PCR allows for rapid and highly-specific amplification of DNA fragments. The method is relatively inexpensive and commonly performed in most molecular laboratories. Pieces of DNA from about 50 base pairs (bp) to over 10 kilobases (kb) can be amplified, even from vanishingly small amounts of starting genomic DNA. The two most important principles underlying PCR are:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Complementarity-driven binding of single DNA strands to form a duplex</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Template-driven, semi-conservative synthesis of DNA, by DNA polymerases</p><p/><p>These two principles are discussed separately in detail. (See <a href=\"topic.htm?path=principles-of-molecular-genetics\" class=\"medical medical_review\">&quot;Principles of molecular genetics&quot;</a>.)</p><p>This topic presents a brief overview of PCR as well as a discussion of some applications of PCR and quantitative PCR. Other molecular genetic tools are discussed separately.</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>(See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Cytogenetics and molecular genetics&quot;</a>.)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>(See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-gene-expression-profiling\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Gene expression profiling&quot;</a>.)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>(See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Specially bred and genetically engineered mice&quot;</a>.)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>(See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-model-systems\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Model systems&quot;</a>.)</p><p/><p class=\"headingAnchor\" id=\"H79835171\"><span class=\"h1\">THE PCR PROCESS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Two technical components underlie the polymerase chain reaction (PCR) process: the existence of thermostable DNA polymerases purified or cloned from microorganisms living in hot springs and the ability to readily synthesize specific oligonucleotide primers of 20 to 30 residues.</p><p>PCR is a single tube reaction. Each reaction is customized to amplify one specific genomic sequence. This specificity is dictated by the pair of oligonucleotide primers (short DNA sequences) designed to hybridize to either strand of a target sequence.</p><p>The standard PCR reaction mixture consists of a large excess of oligonucleotide primer pairs, a template DNA (typically genomic DNA), free deoxynucleotide triphosphates (DNA bases), reaction buffer, and thermostable DNA polymerase (the enzyme that drives the PCR reaction). </p><p>Thermostable DNA polymerases can withstand heating to 95&deg;C with minimal loss of activity and function optimally near 70&deg;C [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/1\" class=\"abstract_t\">1</a>]. The standard enzyme currently used in most PCR is derived from the bacterium Thermus (also Thermophilus) aquaticus (Taq). The error rate (per nucleotide per cycle) for standard Taq is about 1 in 10<sup>5</sup> nucleotides. High-fidelity polymerases with in vitro proofreading ability, such as those derived from the archaebacterium <em>Pyrococcus furiosus</em> (Pfu), have error rates that are approximately 10 times lower than Taq and are used for applications that mandate more stringent DNA synthesis [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/4\" class=\"abstract_t\">4</a>].</p><p>The PCR consists of three stages that are repeated 30 to 40 times (<a href=\"image.htm?imageKey=PC%2F64383\" class=\"graphic graphic_figure graphicRef64383 \">figure 1</a>): &#160;</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Denaturing</strong> &mdash; The mixture is heated to 95&deg;C to allow the double-stranded template DNA to denature into single strands. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Annealing</strong> &mdash; The mixture is cooled to a temperature just below the predicted primer pair melting temperatures, resulting in primer binding to the single-stranded DNA templates, followed by binding of DNA polymerase to the 3&rsquo; end of the primers. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>Elongation</strong> &mdash; The temperature is then raised to a polymerase activation temperature (~70 to 72&deg;C) to initiate chain elongation. The polymerase catalyzes the addition of free nucleotides to the 3&rsquo; ends of the primers. Bases are added to complement the template sequence (A with T, C with G). Elongation continues for about one minute and is terminated by cycling back to step one of the reaction: heating the temperature to 95&deg;C, resulting in strand separation. </p><p/><p>The number of potential targets for primer annealing has been doubled after one cycle of PCR, as both the original substrate DNA and the newly-synthesized strands are available for primer binding. With each subsequent cycle, the number of amplified targets (defined at both ends by the reaction primers) is doubled, resulting in exponential amplification of the target sequence. It is possible to start with as little as one template DNA molecule and generate tens of millions of copies. If the reaction starts with two copies, then N cycles will theoretically yield 2<sup>N</sup> copies.</p><p>To amplify the sequence of interest, single-stranded oligonucleotide primers (which are typically 20 nucleotides in length) must be carefully designed and synthesized. Highly-specific amplification can be achieved if the sequence combination of the primer pairs is present only once in the genome. Computer programs that facilitate primer design and calculate annealing temperatures are freely distributed online [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/5\" class=\"abstract_t\">5</a>].</p><p>After the reaction is completed, the amplified product (called an amplicon) can be visualized by gel electrophoresis, during which the negatively charged DNA fragments migrate toward the positive electrode through an agarose gel. The pores of the gel depend upon its agarose percentage and can be compared to a (molecular) sieve. Thus, migration speed of the DNA fragments through the gel depends on both DNA fragment size and the applied voltage. After electrophoresis and addition of a fluorescent molecule that binds to DNA and forms a complex with nucleic acids, exposure of the gel to ultraviolet light demonstrates the amplified fragment as a single fluorescing band. The gel can be photographed for a permanent record of the amplification process.</p><p>The advantages of the PCR technique are the following:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Rapidity &mdash; The amplification reaction takes approximately three hours. Additional time (several hours) is required to first extract DNA from cells and then perform gel electrophoresis after completion of PCR.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Extreme sensitivity &mdash; PCR can amplify the DNA from a single cell and thus is sufficiently sensitive to be used in preimplantation genetic diagnosis in embryos.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Robust &mdash; Degraded DNA can frequently be successfully amplified.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Specificity &mdash; The technique also permits the detection of small nucleotide mutations (see below), as well as trinucleotide repeat expansions.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Inexpensive and simple.</p><p/><p>There are a number of disadvantages to the PCR technique:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Because of its extreme sensitivity, contamination of the sample with small amounts of extraneous DNA may result in a spurious or false-positive finding. To help decrease the incidence of this problem, diagnostic PCR laboratories are often subdivided into separate designated pre- and post-amplification areas in different rooms.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>To design appropriate PCR primers, the nucleotide sequence information of the region of interest must be known.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The ideal length of a sequence to be amplified (PCR amplicon) is several hundred nucleotides. Although longer length PCR methods are available, the analysis of very large regions is more difficult and requires different techniques. Similarly, large mutations, such as duplications or deletions that encompass the amplified region, cannot be detected because only the unaffected allele will be amplified.</p><p/><p class=\"headingAnchor\" id=\"H8006607\"><span class=\"h1\">UNIQUE GENOMIC SEQUENCES</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Although the human genome sequence is very long (3 billion bases), relatively short DNA sequences suffice to specify a unique sequence in the genome. The expected count of a specified sequence of length N bases can be expressed by the following formula, based on probability theory:</p><div class=\"formulaContainer\"><div class=\"formula\"><p> &#160; &#160;(3 X 10<sup>9</sup>)<sup> </sup>x <span class=\"nowrap\">1/4<sup>N</sup></span> &#160; </p></div></div><p>where <span class=\"nowrap\">1/4<sup>N</sup></span> is the probability of a given sequence of length N bases and the constant 3 X 10<sup>9</sup> is the number of bases in the human genome.</p><p>Thus, a PCR primer 20 nucleotides in length would be estimated to occur by chance at a frequency of 0.003 per genome, and a primer 25 nucleotides in length would occur at a frequency of 0.000003 per genome. Thus, these estimates demonstrate that a specific 20-residue sequence is likely to be unique in the human genome. Furthermore, PCR requires two primers to colocalize with a specific orientation to one another in a relatively small genomic region. Therefore, the probability of amplifying more than one region is exceedingly small. </p><p>Sequence alignment algorithms can be useful to align designed primer pairs to the known genome sequence, to verify if a primer pair is unique in the genome, to check if any polymorphisms underlie the primers, and to calculate the exact length of the expected amplicon [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/6\" class=\"abstract_t\">6</a>].</p><p class=\"headingAnchor\" id=\"H8006628\"><span class=\"h1\">PCR COMPARED TO OTHER AMPLIFICATION METHODS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The exponential amplification of polymerase chain reactions (PCR) can be contrasted with other types of nucleic acid amplification that are widely used in molecular biology. Two examples of other amplification methods are whole genome amplification and RNA amplification.</p><p class=\"headingAnchor\" id=\"H3509399\"><span class=\"h2\">Whole genome amplification</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Whole genome amplification (WGA) non-specifically amplifies the entire genome and is used when sample quantities are too small for multiple PCR reactions or to meet the desired application's requirements. Random, very short primers are used, along with Phi29 polymerase, a highly-active polymerase capable of incorporating over 70,000 nucleotides per priming event [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/7\" class=\"abstract_t\">7</a>]. In order to limit amplification bias (uneven amplification of different genomic sequences or alleles), it is best to amplify the template DNA only a few 1000-fold. </p><p>WGA is distinct from standard PCR in that it is non-specific (all genomic sequences are targeted, not only one site), and it generates fragments that are tens of thousands of bases in length, compared to 250 to 1000 bases that are typical for PCR. Often the fragments generated by WGA are short, thus precluding long-range PCR or applications that require long template strands.</p><p class=\"headingAnchor\" id=\"H3509458\"><span class=\"h2\">RNA amplification</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Linear amplification of RNA is the precursor to most gene expression microarray experiments. Linear amplification is preferred over exponential amplification (ie, classical PCR) in this setting because occasional imbalances in sequence amplification rates (that result from preferential reaction kinetics and favor amplification of some regions over others) can bias subsequent quantification [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/8\" class=\"abstract_t\">8</a>]. The general method for global amplification of mRNA is based upon the Eberwine protocol [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/9\" class=\"abstract_t\">9</a>], whereby single-stranded RNA is the template, the reaction is primed by a poly-T oligonucleotide primer that binds the 3&rsquo; poly-A tail common to most mRNA sequences, and the reaction is catalyzed by a reverse transcriptase which generates complementary DNA strands. Following second strand synthesis with DNA polymerase, and subsequent sample purification, T7 polymerase is used to drive in vitro transcription (IVT), producing multiple copies of amplified RNA (aRNA) from each dsDNA. By incorporating labeled nucleotides in the in vitro transcription step, amplification products to be used as probes in expression analysis can also be prepared. Amplification achieved is typically in the range of 10- to 100-fold, but can vary depending on the amount of input RNA.</p><p class=\"headingAnchor\" id=\"H8006635\"><span class=\"h1\">CLINICAL APPLICATIONS OF PCR</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Though the applications of polymerase chain reactions (PCR) are innumerable, its clinical utility is greatest in the context of genotyping and sequencing for diagnostic and predictive testing. </p><p class=\"headingAnchor\" id=\"H8006642\"><span class=\"h2\">Genotyping and sequencing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Genotyping is the process of characterizing an individual&rsquo;s genotype (the combination of alleles) at a particular genomic locus (location). Most types of genetic variation can be characterized using PCR techniques, including microsatellite and single nucleotide polymorphism (SNP) markers, <span class=\"nowrap\">insertion/deletion</span> variants (indels), and some structural variants, such as copy-number variants. (See <a href=\"topic.htm?path=overview-of-genetic-variation\" class=\"medical medical_review\">&quot;Overview of genetic variation&quot;</a>.) </p><p class=\"headingAnchor\" id=\"H37955591\"><span class=\"h3\">SNP genotyping</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>PCR is the basis of the most commonly used single-nucleotide polymorphism (SNP) genotyping methods, including probe-hybridization based assays (such as the Taqman SNP assay) [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/10\" class=\"abstract_t\">10</a>], allele-specific PCR [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/11,12\" class=\"abstract_t\">11,12</a>], and minisequencing assays, such as those implemented in genome-wide SNP microarrays [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/13\" class=\"abstract_t\">13</a>]. These and other methods have been adopted for clinical use in CLIA-certified laboratories. </p><p>DNA sequencing is the process of characterizing the nucleotide (base-pair) sequence of a specific DNA sequence. Clinically, sequencing is used to identify the pathogenic mutations in individuals with genetic disorders caused by rare variants at a specific locus. An example is resequencing of the <em>BRCA1</em> locus in individuals at risk for familial breast cancer. Current clinical sequencing is PCR-based, with each assay focused on a specific exon or gene region. Next-generation sequencing platforms that are essentially PCR-free enable high-throughput characterization of billions of nucleotides simultaneously. (See <a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">&quot;Next-generation DNA sequencing (NGS): Principles and clinical applications&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H8006686\"><span class=\"h3\">Detection of rare sequences</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>PCR technology also allows detection of rare sequences in a population of DNA or RNA molecules. This application is particularly useful in searching for DNA rearrangements in the setting of neoplasia. An example is the discovery that a Herpes simplex&ndash;related virus is involved in the pathogenesis of Kaposi's sarcoma [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/14-16\" class=\"abstract_t\">14-16</a>]. </p><p>Rare sequence detection by PCR is used routinely in obstetrics for preimplantation testing and non-invasive prenatal testing (NIPT). Preimplantation testing (whereby established embryos are screened for suspected genetic mutations prior to uterine implantation) is demanding because PCR must be performed from a single cell, removed from the very early embryo. The major concerns in single-cell PCR are preventing allele dropout and contamination with maternal or other outside DNA [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/17\" class=\"abstract_t\">17</a>]. NIPT relies on detection of free fetal DNA (ffDNA) in maternal blood; since ffDNA makes up only a small percentage of the circulating DNA, pains must be taken to differentiate fetal from maternal sequences [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/18\" class=\"abstract_t\">18</a>]. (See <a href=\"topic.htm?path=prenatal-screening-for-common-aneuploidies-using-cell-free-dna\" class=\"medical medical_review\">&quot;Prenatal screening for common aneuploidies using cell-free DNA&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H8006693\"><span class=\"h2\">Quantifying the amount of a nucleic acid sequence</span></p><p class=\"headingAnchor\" id=\"H986053159\"><span class=\"h3\">RT-PCR</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Real-time quantitative PCR (RT-PCR; qPCR) is a highly-sensitive method for quantifying the absolute or relative amount of a specific nucleic acid sequence in which the accumulation of PCR products over time is measured directly, without post-PCR modifications. Quantitation is estimated from the cycle number required for the reaction to generate sufficient fluorescence to cross a set threshold. This method is thus reliant on performing multiple rounds of amplification [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/19\" class=\"abstract_t\">19</a>]. Common applications of qPCR include gene expression profiling, quantification of viral load, and determination of copy number. </p><p>qPCR is performed by quantifying the amount of amplified product with each round of PCR cycling. In addition to the two primers that are necessary for successful PCR amplification, this method frequently applies a fluorescent, non-extendible probe that hybridizes to the target sequence between the primer pair.</p><p>This probe contains a fluorogenic reporter dye at its 5' end and a quencher at the 3' end. The quencher blocks fluorescence emission as long as it is in close proximity to the reporter. As the primer extends downstream during amplification, however, the exonuclease activity of the DNA polymerase cleaves away the hybridized probe and removes the quencher. An increase in the fluorescence signal ensues (<a href=\"image.htm?imageKey=PC%2F81663\" class=\"graphic graphic_figure graphicRef81663 \">figure 2</a>).</p><p>Due to the exponential nature of the PCR, the fluorescence signal increases proportionally to the amount of generated PCR product until a plateau is reached. Quantification is accomplished by comparing the cycle number at which the patient sample reaches a predetermined level of fluorescence to a standardized curve of a control sample, thus deriving copy number at the start of the reaction (<a href=\"image.htm?imageKey=HEME%2F71655\" class=\"graphic graphic_figure graphicRef71655 \">figure 3</a>). This method is rapidly becoming the method of choice for monitoring residual disease in patients receiving chemotherapy <span class=\"nowrap\">and/or</span> hematopoietic cell transplantation for hematologic malignancies [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/20,21\" class=\"abstract_t\">20,21</a>]. (See <a href=\"topic.htm?path=genetic-abnormalities-in-hematologic-and-lymphoid-malignancies\" class=\"medical medical_review\">&quot;Genetic abnormalities in hematologic and lymphoid malignancies&quot;</a>.)</p><p>Advantages of this method include the following:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Extreme sensitivity and a wide dynamic range</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Time-efficient, as the method requires no post-PCR processing of samples</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Automated high-throughput instruments (such as the ABI PRISM 7700 Sequence Detection System) are available with interpretation-supporting software</p><p/><p class=\"headingAnchor\" id=\"H2654862010\"><span class=\"h3\">Digital PCR (dPCR)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Digital PCR (dPCR), also known as digital drop PCR, is an adaptation of traditional PCR for the purposes of sample quantification without the need for multiple PCR cycles. The basic principle of dPCR is that a sample is separated (using complex microfluidics) into discrete aliquots (packets), each containing a very small number of nucleic acid sequences from the patient (on average, 1). PCR is then performed in each packet, resulting in a binary result in which the target nucleic acid of interest is either present or absent (in binary terms, either positive [1] or negative [0]). Subsequent modeling (assuming a Poisson distribution) enables estimation of the starting amount of target nucleic acid without the reliance on multiple amplification cycles as required in qPCR. The measured quantity in dPCR is thus considered an absolute measure, rather than the relative measure of traditional PCR (compared to the threshold value of a standard). dPCR is being used in several clinical settings, including mutation detection and the detection of infectious agents [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/22,23\" class=\"abstract_t\">22,23</a>]. </p><p class=\"headingAnchor\" id=\"H8006714\"><span class=\"h3\">Gene expression profiles</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Specific gene transcripts can be measured by qPCR from RNA samples that have been converted to cDNA by reverse transcription. Typically, relative quantification is used to calculate fold-change among a set of samples. For example, inflammation-related genes can be assayed by Taqman qPCR, comparing normal tissue with colon cancer samples [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/24\" class=\"abstract_t\">24</a>]. qPCR expression profiling is used to validate measurements of specific candidate genes identified through genome-wide expression microarray studies. </p><p class=\"headingAnchor\" id=\"H8006721\"><span class=\"h3\">Clinical measurement of viral RNA or DNA</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>PCR of RNA isolated from blood is a standard tool in monitoring the viral load in HIV-infected patients [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/25,26\" class=\"abstract_t\">25,26</a>]. Several available commercial real-time PCR assays provide absolute quantitation of HIV-1 RNA copies per mL of plasma [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/27\" class=\"abstract_t\">27</a>]. Key considerations for HIV RNA testing are throughput, automation, accuracy, and dynamic range. PCR quantification has also been applied to other RNA viruses causing chronic infection, such as hepatitis C virus [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/28,29\" class=\"abstract_t\">28,29</a>]. (See <a href=\"topic.htm?path=techniques-and-interpretation-of-hiv-1-rna-quantitation\" class=\"medical medical_review\">&quot;Techniques and interpretation of HIV-1 RNA quantitation&quot;</a> and <a href=\"topic.htm?path=diagnosis-and-evaluation-of-chronic-hepatitis-c-virus-infection\" class=\"medical medical_review\">&quot;Diagnosis and evaluation of chronic hepatitis C virus infection&quot;</a>.) </p><p>The 2009 outbreak of swine-origin influenza A (H1N1) prompted rapid development of diagnostic real-time, RT-PCR assays [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/30\" class=\"abstract_t\">30</a>]. The assays developed by the Centers for Disease Control consist of four sets of primers and Taqman probes for universal detection of type A influenza, universal detection of swine influenza A, specific detection of swine H1 influenza, and the positive reference control gene, RnaseP [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/31\" class=\"abstract_t\">31</a>].</p><p>Another area in which qPCR is becoming useful is in the quantitation of Epstein-Barr viral load in the diagnosis and monitoring of patients with post-transplant lymphoproliferative disease [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/32\" class=\"abstract_t\">32</a>]. (See <a href=\"topic.htm?path=epidemiology-clinical-manifestations-and-diagnosis-of-post-transplant-lymphoproliferative-disorders#H17288449\" class=\"medical medical_review\">&quot;Epidemiology, clinical manifestations, and diagnosis of post-transplant lymphoproliferative disorders&quot;, section on 'Measurement of EBV viral load'</a>.)</p><p>Monitoring of hepatitis B treatment response by quantifying viral DNA in serum, with real-time PCR, is another important clinical application [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/33\" class=\"abstract_t\">33</a>]. (See <a href=\"topic.htm?path=hepatitis-b-virus-screening-and-diagnosis#H7\" class=\"medical medical_review\">&quot;Hepatitis B virus: Screening and diagnosis&quot;, section on 'Serum HBV DNA assays'</a>.) &#160;</p><p class=\"headingAnchor\" id=\"H8006742\"><span class=\"h1\">CRITICAL EVALUATION OF DATA</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The ease of performing polymerase chain reactions (PCR) has led to wide dissemination of the methodology. However, reports of PCR analyses should be reviewed for validity of the results.</p><p>Every PCR experiment requires a minimum of two technical controls (positive and negative controls), in addition to biological controls specific to the research question being addressed. The technical controls should demonstrate that amplification occurs when it should (positive control) and that it does not occur when it should not (negative control). The positive control template should be prepared in the same manner as the test sample, with the positive control DNA typically extracted using the same extraction protocol and starting material as the sample. The negative control (also referred to as a no-template control) involves using all components of the PCR mixture except that water is added in lieu of the DNA template. </p><p>The quality of the test sample DNA is critical, as highly-fragmented DNA can result in allelic dropout (selective amplification of only one allele at a heterozygous locus) or false-negative results. Impurities in the template DNA sample could inhibit amplification. This is of particular concern in qPCR, since inhibitors will lower reaction efficiency.</p><p class=\"headingAnchor\" id=\"H15008227\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Polymerase chain reaction (PCR) allows for rapid and highly-specific amplification of DNA fragments. Two technical components underlie the PCR process: the existence of thermostable DNA polymerases purified or cloned from microorganisms living in hot springs and the ability to readily synthesize specific oligonucleotide primers of 20 to 30 residues. The PCR consists of three stages, multiply repeated: denaturing, annealing, and elongation. After the reaction is completed, the amplified product (called an amplicon) can be visualized by gel electrophoresis. Relatively short DNA primer sequences suffice to specify a unique sequence in the genome. (See <a href=\"#H79835171\" class=\"local\">'The PCR process'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The exponential amplification of PCR can be contrasted with other types of nucleic acid amplification that are widely used in molecular biology. Whole genome amplification (WGA) non-specifically amplifies the entire genome and is used when sample quantities are too small for PCR to meet the desired application's requirements. Linear amplification of RNA is the precursor to most gene expression microarray experiments. (See <a href=\"#H8006628\" class=\"local\">'PCR compared to other amplification methods'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Most types of genetic variation can be characterized using PCR techniques, including microsatellite and single nucleotide polymorphism (SNP) markers, <span class=\"nowrap\">insertion/deletion</span> variants (indels), and some structural variants such as copy-number variants. (See <a href=\"#H37955591\" class=\"local\">'SNP genotyping'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>PCR technology allows detection of rare sequences in a population of DNA or RNA molecules. This application is particularly useful in searching for DNA rearrangements in the setting of neoplasia. Rare sequence detection by PCR also is used routinely in obstetrics for preimplantation testing and non-invasive prenatal testing (NIPT). (See <a href=\"#H8006686\" class=\"local\">'Detection of rare sequences'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Real-time quantitative PCR (qPCR) and digital PCR (dPCR) can be used to quantify the absolute or relative amount of a specific nucleic acid sequence. This method is often used for monitoring residual disease in patients receiving chemotherapy <span class=\"nowrap\">and/or</span> hematopoietic cell transplantation for hematologic malignancies. (See <a href=\"#H8006693\" class=\"local\">'Quantifying the amount of a nucleic acid sequence'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>PCR of RNA isolated from blood is a standard tool in monitoring the viral load in HIV-infected patients. qPCR also has been used for assays of H1N1, Epstein-Barr viral load monitoring post-transplant patients, and monitoring response to therapy for hepatitis B. (See <a href=\"#H8006721\" class=\"local\">'Clinical measurement of viral RNA or DNA'</a> above.)</p></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/1\" class=\"nounderline abstract_t\">Templeton NS. The polymerase chain reaction. History, methods, and applications. Diagn Mol Pathol 1992; 1:58.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/2\" class=\"nounderline abstract_t\">Saiki RK, Scharf S, Faloona F, et al. Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia. Science 1985; 230:1350.</a></li><li class=\"breakAll\">Rabinow P. Making PCR: a story of biotechnology, University of Chicago Press, Chicago 1996.</li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/4\" class=\"nounderline abstract_t\">Eckert KA, Kunkel TA. DNA polymerase fidelity and the polymerase chain reaction. PCR Methods Appl 1991; 1:17.</a></li><li class=\"breakAll\">Rozen S, Skaletsky HJ. Primer3 on the WWW for general users and for biologist programmers. In: Bioinformatics Methods and Protocols: Methods in Molecular Biology, Krawetz S, Misener S (Eds), Humana Press, Totowa, NJ 2000. p.365.</li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/6\" class=\"nounderline abstract_t\">Zweig AS, Karolchik D, Kuhn RM, et al. UCSC genome browser tutorial. Genomics 2008; 92:75.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/7\" class=\"nounderline abstract_t\">Lasken RS. Genomic DNA amplification by the multiple displacement amplification (MDA) method. Biochem Soc Trans 2009; 37:450.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/8\" class=\"nounderline abstract_t\">Ginsberg SD. RNA amplification strategies for small sample populations. Methods 2005; 37:229.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/9\" class=\"nounderline abstract_t\">Nygaard V, Hovig E. Options available for profiling small samples: a review of sample amplification technology when combined with microarray profiling. Nucleic Acids Res 2006; 34:996.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/10\" class=\"nounderline abstract_t\">Livak KJ, Flood SJ, Marmaro J, et al. Oligonucleotides with fluorescent dyes at opposite ends provide a quenched probe system useful for detecting PCR product and nucleic acid hybridization. PCR Methods Appl 1995; 4:357.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/11\" class=\"nounderline abstract_t\">Papp AC, Pinsonneault JK, Cooke G, Sad&eacute;e W. Single nucleotide polymorphism genotyping using allele-specific PCR and fluorescence melting curves. Biotechniques 2003; 34:1068.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/12\" class=\"nounderline abstract_t\">Hurd CM, Cavanagh G, Schuh A, et al. Genotyping for platelet-specific antigens: techniques for the detection of single nucleotide polymorphisms. Vox Sang 2002; 83:1.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/13\" class=\"nounderline abstract_t\">Matsuzaki H, Dong S, Loi H, et al. Genotyping over 100,000 SNPs on a pair of oligonucleotide arrays. Nat Methods 2004; 1:109.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/14\" class=\"nounderline abstract_t\">Chang Y, Cesarman E, Pessin MS, et al. Identification of herpesvirus-like DNA sequences in AIDS-associated Kaposi's sarcoma. Science 1994; 266:1865.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/15\" class=\"nounderline abstract_t\">Moore PS, Chang Y. Detection of herpesvirus-like DNA sequences in Kaposi's sarcoma in patients with and those without HIV infection. N Engl J Med 1995; 332:1181.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/16\" class=\"nounderline abstract_t\">Mesri EA, Cesarman E, Arvanitakis L, et al. Human herpesvirus-8/Kaposi's sarcoma-associated herpesvirus is a new transmissible virus that infects B cells. J Exp Med 1996; 183:2385.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/17\" class=\"nounderline abstract_t\">Spits C, Sermon K. PGD for monogenic disorders: aspects of molecular biology. Prenat Diagn 2009; 29:50.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/18\" class=\"nounderline abstract_t\">Maddocks DG, Alberry MS, Attilakos G, et al. The SAFE project: towards non-invasive prenatal diagnosis. Biochem Soc Trans 2009; 37:460.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/19\" class=\"nounderline abstract_t\">Sykes PJ, Neoh SH, Brisco MJ, et al. Quantitation of targets for PCR by use of limiting dilution. Biotechniques 1992; 13:444.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/20\" class=\"nounderline abstract_t\">Olsson K, Gerard CJ, Zehnder J, et al. Real-time t(11;14) and t(14;18) PCR assays provide sensitive and quantitative assessments of minimal residual disease (MRD). Leukemia 1999; 13:1833.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/21\" class=\"nounderline abstract_t\">Martin S, Fischer C, Free M, et al. LightCycler-based quantitative real-time PCR monitoring of patients with follicular lymphoma receiving rituximab in combination with conventional or high-dose cytotoxic chemotherapy. Eur J Haematol 2005; 74:282.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/22\" class=\"nounderline abstract_t\">Vogelstein B, Kinzler KW. Digital PCR. Proc Natl Acad Sci U S A 1999; 96:9236.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/23\" class=\"nounderline abstract_t\">Vynck M, Trypsteen W, Thas O, et al. The Future of Digital Polymerase Chain Reaction in Virology. Mol Diagn Ther 2016; 20:437.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/24\" class=\"nounderline abstract_t\">Erreni M, Bianchi P, Laghi L, et al. Expression of chemokines and chemokine receptors in human colon cancer. Methods Enzymol 2009; 460:105.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/25\" class=\"nounderline abstract_t\">Clarke JR, McClure MO. HIV-1 viral load testing. J Infect 1999; 38:141.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/26\" class=\"nounderline abstract_t\">Churchill D, Weber J. Measurement of plasma HIV viral load--clinical applications. J Infect 1999; 38:147.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/27\" class=\"nounderline abstract_t\">Scott LE, Noble LD, Moloi J, et al. Evaluation of the Abbott m2000 RealTime human immunodeficiency virus type 1 (HIV-1) assay for HIV load monitoring in South Africa compared to the Roche Cobas AmpliPrep-Cobas Amplicor, Roche Cobas AmpliPrep-Cobas TaqMan HIV-1, and BioMerieux NucliSENS EasyQ HIV-1 assays. J Clin Microbiol 2009; 47:2209.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/28\" class=\"nounderline abstract_t\">Takeuchi T, Katsume A, Tanaka T, et al. Real-time detection system for quantification of hepatitis C virus genome. Gastroenterology 1999; 116:636.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/29\" class=\"nounderline abstract_t\">Martell M, G&oacute;mez J, Esteban JI, et al. High-throughput real-time reverse transcription-PCR quantitation of hepatitis C virus RNA. J Clin Microbiol 1999; 37:327.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/30\" class=\"nounderline abstract_t\">Perez-Padilla R, de la Rosa-Zamboni D, Ponce de Leon S, et al. Pneumonia and respiratory failure from swine-origin influenza A (H1N1) in Mexico. N Engl J Med 2009; 361:680.</a></li><li class=\"breakAll\">CDC protocol of realtime RTPCR for influenza A (H1N1). Geneva: World Health Organization, April 2009.  Available at: www.who.int/csr/resources/publications/swineflu/CDCRealtimeRTPCR_SwineH1Assay-2009_20090430.pdf (Accessed on June 23, 2009).</li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/32\" class=\"nounderline abstract_t\">Wagner HJ, Wessel M, Jabs W, et al. Patients at risk for development of posttransplant lymphoproliferative disorder: plasma versus peripheral blood mononuclear cells as material for quantification of Epstein-Barr viral load by using real-time quantitative polymerase chain reaction. Transplantation 2001; 72:1012.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-polymerase-chain-reaction/abstract/33\" class=\"nounderline abstract_t\">Andersson KL, Chung RT. Monitoring during and after antiviral therapy for hepatitis B. Hepatology 2009; 49:S166.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 2896 Version 18.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H15008227\"><span>SUMMARY</span></a></li><li><a href=\"#H8006600\" id=\"outline-link-H8006600\">INTRODUCTION</a></li><li><a href=\"#H79835171\" id=\"outline-link-H79835171\">THE PCR PROCESS</a></li><li><a href=\"#H8006607\" id=\"outline-link-H8006607\">UNIQUE GENOMIC SEQUENCES</a></li><li><a href=\"#H8006628\" id=\"outline-link-H8006628\">PCR COMPARED TO OTHER AMPLIFICATION METHODS</a><ul><li><a href=\"#H3509399\" id=\"outline-link-H3509399\">Whole genome amplification</a></li><li><a href=\"#H3509458\" id=\"outline-link-H3509458\">RNA amplification</a></li></ul></li><li><a href=\"#H8006635\" id=\"outline-link-H8006635\">CLINICAL APPLICATIONS OF PCR</a><ul><li><a href=\"#H8006642\" id=\"outline-link-H8006642\">Genotyping and sequencing</a><ul><li><a href=\"#H37955591\" id=\"outline-link-H37955591\">- SNP genotyping</a></li><li><a href=\"#H8006686\" id=\"outline-link-H8006686\">- Detection of rare sequences</a></li></ul></li><li><a href=\"#H8006693\" id=\"outline-link-H8006693\">Quantifying the amount of a nucleic acid sequence</a><ul><li><a href=\"#H986053159\" id=\"outline-link-H986053159\">- RT-PCR</a></li><li><a href=\"#H2654862010\" id=\"outline-link-H2654862010\">- Digital PCR (dPCR)</a></li><li><a href=\"#H8006714\" id=\"outline-link-H8006714\">- Gene expression profiles</a></li><li><a href=\"#H8006721\" id=\"outline-link-H8006721\">- Clinical measurement of viral RNA or DNA</a></li></ul></li></ul></li><li><a href=\"#H8006742\" id=\"outline-link-H8006742\">CRITICAL EVALUATION OF DATA</a></li><li><a href=\"#H15008227\" id=\"outline-link-H15008227\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>GRAPHICS <a href=\"#\" id=\"viewAllGraphicsLink\">View All</a></h2><div id=\"outlineGraphics\"><ul><li><div id=\"PC/2896|FIG\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"FIGURES\">FIGURES</a></div><ul><li><a href=\"image.htm?imageKey=PC/64383\" class=\"graphic graphic_figure\">- PCR technique</a></li><li><a href=\"image.htm?imageKey=PC/81663\" class=\"graphic graphic_figure\">- Quantitative real time PCR</a></li><li><a href=\"image.htm?imageKey=HEME/71655\" class=\"graphic graphic_figure\">- Quantitative PCR</a></li></ul></li></ul></div></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=diagnosis-and-evaluation-of-chronic-hepatitis-c-virus-infection\" class=\"medical medical_review\">Diagnosis and evaluation of chronic hepatitis C virus infection</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=epidemiology-clinical-manifestations-and-diagnosis-of-post-transplant-lymphoproliferative-disorders\" class=\"medical medical_review\">Epidemiology, clinical manifestations, and diagnosis of post-transplant lymphoproliferative disorders</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetic-abnormalities-in-hematologic-and-lymphoid-malignancies\" class=\"medical medical_review\">Genetic abnormalities in hematologic and lymphoid malignancies</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=hepatitis-b-virus-screening-and-diagnosis\" class=\"medical medical_review\">Hepatitis B virus: Screening and diagnosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">Next-generation DNA sequencing (NGS): Principles and clinical applications</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=overview-of-genetic-variation\" class=\"medical medical_review\">Overview of genetic variation</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=prenatal-screening-for-common-aneuploidies-using-cell-free-dna\" class=\"medical medical_review\">Prenatal screening for common aneuploidies using cell-free DNA</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=principles-of-molecular-genetics\" class=\"medical medical_review\">Principles of molecular genetics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=techniques-and-interpretation-of-hiv-1-rna-quantitation\" class=\"medical medical_review\">Techniques and interpretation of HIV-1 RNA quantitation</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">Tools for genetics and genomics: Cytogenetics and molecular genetics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-gene-expression-profiling\" class=\"medical medical_review\">Tools for genetics and genomics: Gene expression profiling</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-model-systems\" class=\"medical medical_review\">Tools for genetics and genomics: Model systems</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-specially-bred-and-genetically-engineered-mice\" class=\"medical medical_review\">Tools for genetics and genomics: Specially bred and genetically engineered mice</a></li></ul></div></div>","javascript":null}