var data={"title":"Tools for genetics and genomics: Gene expression profiling","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Tools for genetics and genomics: Gene expression profiling</div><dl id=\"topicContributors\"><dt><span> </span>Authors:</dt><dd><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/contributors\" class=\"contributor contributor_credentials\">Avrum Spira, MD, MSc</a></dd><dd><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/contributors\" class=\"contributor contributor_credentials\">Katrina Steiling, MD, MSc</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/contributors\" class=\"contributor contributor_credentials\">Benjamin A Raby, MD, MPH</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/contributors\" class=\"contributor contributor_credentials\">Jennifer S Tirnauer, MD</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Sep 01, 2017.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H3544017\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The genetic basis for disease is determined by the inheritance of genes containing specific sequences of deoxyribonucleic acid (DNA). The phenotypic expression of these genes, through the synthesis of specific proteins, involves interaction with environmental signals that trigger activation of particular genes.</p><p>According to the central dogma of biology, ribonucleic acid (RNA) is transcribed from a DNA template; messenger RNA (mRNA) is then translated into protein (<a href=\"image.htm?imageKey=PC%2F54914\" class=\"graphic graphic_figure graphicRef54914 \">figure 1</a>). Transcription and translation underlie gene expression.</p><p>Approximately 3 to 5 percent of genes are active in a particular cell, even though all cells have the same information contained in their DNA. Most of the genome is selectively repressed, a property that is governed by the regulation of gene expression, mostly at the level of transcription (ie, the production of messenger RNA from the DNA). In response to a cellular perturbation, changes in gene expression take place that result in the expression of hundreds of gene products and the suppression of others. This molecular heterogeneity can affect when and how a disease presents clinically in an individual with genetic predisposition to a condition and how individuals with a given disease will respond to specific treatments.</p><p>Analyses of gene expression can be clinically useful for disease classification, diagnosis, prognosis, and tailoring treatment to underlying genetic determinants of pharmacologic response.</p><p>This topic will focus on the role of mRNA in the cell, platforms for profiling mRNA expression, the challenges in interpreting the data from these analyses, and the emerging clinical applications of gene expression measurements. An overview of molecular genetics in clinical oncology is presented separately. (See <a href=\"topic.htm?path=principles-of-molecular-genetics\" class=\"medical medical_review\">&quot;Principles of molecular genetics&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H95557489\"><span class=\"h1\">RNA IN CELL FUNCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>There are several classes of ribonucleic acid (RNA). Messenger RNA (mRNA) is the RNA that is translated into protein. Noncoding RNAs are not translated into protein and serve other functions in the cell. Classes of noncoding RNAs include transfer RNAs (tRNAs), ribosomal RNAs (rRNAs), small nuclear RNAs (snRNAs), small nucleolar RNAs (snoRNAs), microRNAs, Piwi-interacting RNAs (piRNAs), and large intergenic noncoding RNAs (lincRNAs) [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/1-5\" class=\"abstract_t\">1-5</a>].</p><p>Messenger RNA accounts for about 1 percent of the total RNA in a cell [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/6\" class=\"abstract_t\">6</a>] and is transcribed from approximately 20,000 to 25,000 protein-coding genes in the human genome [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/7\" class=\"abstract_t\">7</a>]. After mRNA is transcribed from deoxyribonucleic acid (DNA), it typically undergoes further modifications including the addition of a methyl-guanosine cap (5' cap), the addition of a series of adenines to the 3' end of RNA (poly-A tail), and the splicing out of introns (<a href=\"image.htm?imageKey=PC%2F70605\" class=\"graphic graphic_figure graphicRef70605 \">figure 2</a>) [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/1\" class=\"abstract_t\">1</a>]. mRNA is then transported from the nucleus to the cytoplasm where it is translated into protein. Messenger RNA serves as a transient intermediate between DNA and protein, and is degraded in minutes to hours [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/1\" class=\"abstract_t\">1</a>].</p><p>MicroRNAs (miRNAs) are small endogenous noncoding RNAs (length 18 to 24 nucleotides) that regulate gene expression by binding to the untranslated regions of messenger RNAs (mRNAs) and inducing mRNA degradation or inhibiting protein translation, in turn reducing gene expression. Profiling of miRNAs can be done using similar methods to those used for mRNAs. A searchable database of characterized miRNAs is available (<a href=\"http://www.mirbase.org/&amp;token=l79bgTJaoAltP1TadK9WINS0IMz5QkFUfmj8j30GNPQ=&amp;TOPIC_ID=14602\" target=\"_blank\" class=\"external\">www.mirbase.org/</a>).</p><p class=\"headingAnchor\" id=\"H3544024\"><span class=\"h1\">MEASURING GENE EXPRESSION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Since messenger ribonucleic acid (mRNA) represents the functional bridge between deoxyribonucleic acid (DNA) and protein, alterations in mRNA may serve as markers for the activation or inhibition of a particular gene.</p><p>The challenge in measuring ribonucleic acid (RNA) relates to the susceptibility of RNAs to degradation by ribonucleases (RNases). Methods of RNA detection take advantage of the single stranded structure of RNA and its complementarity to the DNA from which it was transcribed.</p><p class=\"headingAnchor\" id=\"H3544040\"><span class=\"h2\">Expression profiling of single genes or small gene panels</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Prior to the development of microarray technology, available methods of measuring gene expression included:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Northern blot (see <a href=\"#H3544047\" class=\"local\">'Northern blot'</a> below)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Ribonuclease protection assay (see <a href=\"#H3544054\" class=\"local\">'Ribonuclease protection assay'</a> below)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In-situ hybridization (see <a href=\"#H3544061\" class=\"local\">'In-situ hybridization'</a> below)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Reverse-transcription quantitative polymerase chain reaction (RT-PCR) (see <a href=\"#H9716963\" class=\"local\">'Real-time reverse transcription polymerase chain reaction'</a> below)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Spotted cDNA arrays (see <a href=\"#H3544068\" class=\"local\">'Spotted cDNA arrays'</a> below)</p><p/><p>A comparison of these methods is summarized in a table (<a href=\"image.htm?imageKey=PC%2F76106\" class=\"graphic graphic_table graphicRef76106 \">table 1</a>).</p><p class=\"headingAnchor\" id=\"H3544047\"><span class=\"h3\">Northern blot</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Northern blots allow the determination of both the presence of an RNA molecule and its size [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/8\" class=\"abstract_t\">8</a>]. RNA molecules from a patient sample are first separated based on size using gel electrophoresis. The size-separated RNA molecules are then transferred and cross-linked to a nylon membrane. The RNA of interest is detected by incubating the membrane with a labeled single-stranded DNA probe that is complimentary to this RNA. Probe bound to the RNA of interest can then be detected using autoradiography or chemiluminescence.</p><p class=\"headingAnchor\" id=\"H3544054\"><span class=\"h3\">Ribonuclease protection assay</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Whereas the Northern blot uses complimentary DNA probes, the ribonuclease protection assay (RPA) uses antisense RNA probes [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/6\" class=\"abstract_t\">6</a>]. These so-called riboprobes are single-stranded radiolabeled RNA molecules complimentary to the RNA of interest. The riboprobe is incubated with the RNA from the sample and binds to complementary RNA to form double-stranded RNA complexes. Incubating the mixture with ribonucleases degrades unbound single-stranded RNA from both the sample and excess probe. The remaining double stranded RNA complexes are size-separated by electrophoresis, and detected by autoradiography.</p><p class=\"headingAnchor\" id=\"H3544061\"><span class=\"h3\">In-situ hybridization</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>In-situ hybridization (ISH) uses a nucleic acid probe to detect any other nucleic acids in a tissue section. ISH can localize the RNA of interest at the anatomic or cellular level. The tissue section is fixed to preserve tissue morphology and nucleic acid integrity [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/9,10\" class=\"abstract_t\">9,10</a>]. The sample is then treated with proteases to eliminate proteins bound to the RNA of interest [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/9,10\" class=\"abstract_t\">9,10</a>]. A labeled probe is hybridized to the sample and detected using autoradiography or chemiluminescence [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/10\" class=\"abstract_t\">10</a>]. In-situ hybridization using a chemiluminescence-labeled probe is also called fluorescence in situ hybridization (FISH). The use of FISH to detect gene mutations is discussed separately. (See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics#H15538631\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Cytogenetics and molecular genetics&quot;, section on 'Fluorescence in situ hybridization'</a>.)</p><p class=\"headingAnchor\" id=\"H9716963\"><span class=\"h3\">Real-time reverse transcription polymerase chain reaction</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Real-time reverse transcription polymerase chain reaction (RT-PCR) is a relatively simple approach that can be used to assay small or large numbers of genes from a single sample [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/11\" class=\"abstract_t\">11</a>]. After isolating RNA from a sample, complementary DNAs (cDNAs) are synthesized by reverse transcription with an RNA-dependent DNA polymerase. This cDNA mixture is then combined with a DNA-dependent DNA polymerase and fluorescently-labeled oligonucleotide primers [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/12\" class=\"abstract_t\">12</a>]. These primers are short sequences of nucleotides complementary to a portion of the cDNA and allow amplification. Fluorescence increases as the cDNA of interest is amplified with PCR. The fluorescence intensity is monitored and the total number of PCR cycles is counted [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/6\" class=\"abstract_t\">6</a>].</p><p>The point at which the PCR cycler can distinguish fluorescence related to gene amplification from background is the cycle threshold, and this number can be used to estimate the relative starting quantity of the RNA of interest [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/11\" class=\"abstract_t\">11</a>]. Careful primer selection is required to prevent amplification of related genes [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/6\" class=\"abstract_t\">6</a>]. (See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-polymerase-chain-reaction\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Polymerase chain reaction&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H3544068\"><span class=\"h3\">Spotted cDNA arrays</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Unlike Northern blotting, RPA, or ISH, spotted cDNA arrays are capable of testing the relative expression levels, between two conditions, of several hundred genes. With increased knowledge of which sequences are expressed from the genome, it became possible to create cDNA probes targeting the expressed DNA sequences from which RNA is transcribed.</p><p>cDNA probes are amplified using PCR and spotted onto a glass slide [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/13\" class=\"abstract_t\">13</a>]. RNA is then isolated from two samples representing different conditions. mRNA from each sample is isolated, and labeled with one of two fluorescent dyes (green or red) [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/14\" class=\"abstract_t\">14</a>]. The samples are then mixed together and co-hybridized to the cDNA probes on the glass slide [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/13\" class=\"abstract_t\">13</a>]. This approach directly compares gene expression in the first condition to the second condition, and allows the detection of as many genes as there are probes on the array. However, reproducibility is limited across arrays because of the need to manually spot probes on slides.</p><p class=\"headingAnchor\" id=\"H3544075\"><span class=\"h2\">Genome-wide gene expression profiling</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Platforms for profiling gene expression take advantage of increased knowledge of the sequence of the human genome and require smaller quantities of starting RNA. Current platforms for profiling gene expression include:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Oligonucleotide arrays (microarrays)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Transcriptome sequencing</p><p/><p>A comparison of these methods is summarized in a table (<a href=\"image.htm?imageKey=PC%2F76106\" class=\"graphic graphic_table graphicRef76106 \">table 1</a>).</p><p class=\"headingAnchor\" id=\"H95557895\"><span class=\"h3\">Oligonucleotide arrays (microarrays)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Oligonucleotide arrays operate on a similar principle to spotted cDNA arrays, but differ in how they are produced. Rather than spotting probes onto a glass slide, the short probes are synthesized directly on the slide [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/15,16\" class=\"abstract_t\">15,16</a>]. Depending on the commercial manufacturer, probes vary from ~20 to 60 base pairs in length. Several types of arrays are commercially available.</p><p>Sample preparation begins with the isolation of RNA from the tissue of interest, resulting in an extraction that contains all of the genes transcribed in the tissue at the time the RNA is isolated. The RNA is then reverse-transcribed into cDNA and amplified using polymerase chain reaction (PCR) technique. Finally, a <a href=\"topic.htm?path=biotin-vitamin-h-pediatric-drug-information\" class=\"drug drug_pediatric\">biotin</a> label is incorporated through an in vitro transcription process, which converts cDNA into labeled cRNA.</p><p>A single sample of the labeled cRNA is applied to each array. Hybridization occurs between the labeled cRNA from the sample and complementary probes on the array. This is followed by binding to an avidin-conjugated fluorophore, and a washing step that removes any unbound material. The fluorophore is excited by a laser scanner coupled to a computer that captures the image fluorophores linked to hybridized target molecules on the array, thus enabling the detection of the expression of thousands of genes simultaneously.</p><p>In general, the greater the amount of mRNA from a particular gene (ie, the higher the expression of that gene), the more fluorescently-labeled material corresponding to that gene will bind to complementary probes on the array. Background fluorescence or nonspecific binding may limit detection of lowly expressed transcripts. Probe-based detection for gene expression limits analysis to genes that are known.</p><p class=\"headingAnchor\" id=\"H3544096\"><span class=\"h3\">Transcriptome sequencing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>An alternative for measuring gene expression is the direct sequencing and quantification of RNA molecules. This method of measuring gene expression has also been referred to as &quot;RNA-seq,&quot; &quot;massively parallel sequencing,&quot; &quot;next generation sequencing,&quot; or &quot;deep sequencing,&quot; and several commercial platforms are available. The details of each system vary. In general, the sample is prepared so that many sequencing reactions occur simultaneously on a glass slide, and yields millions of RNA sequence reads obtained by laser scanning [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/17\" class=\"abstract_t\">17</a>].</p><p>This technique allows improved detection of low abundance transcripts, as well as detection of novel transcripts and polymorphisms within a transcript's sequence. Advances in sample processing techniques also allow for transcriptome sequencing of individual cells [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/18-21\" class=\"abstract_t\">18-21</a>], and preservation of the identity of the sense and antisense strands [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/22\" class=\"abstract_t\">22</a>].</p><p class=\"headingAnchor\" id=\"H3544103\"><span class=\"h2\">Genome-wide gene expression analysis and interpretation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Microarrays can assay large numbers of genes with relatively high throughput. They are currently the platform most often used for gene expression profiling of clinical specimens. Typically, investigators involved with microarray experiments are interested in comparing gene expression across different conditions [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/23\" class=\"abstract_t\">23</a>]. While there are many approaches to the data analysis in order to accomplish this goal, there are generally several analytical steps that must first be taken (<a href=\"image.htm?imageKey=PC%2F71360\" class=\"graphic graphic_figure graphicRef71360 \">figure 3</a>).</p><p>There are four general considerations in approaching microarray data analysis and interpretation:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Preprocessing of raw data</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Data storage and analysis</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Multiple comparison problem</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Biological interpretation</p><p/><p class=\"headingAnchor\" id=\"H3544110\"><span class=\"h3\">Preprocessing of raw data</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Preprocessing prepares the raw data for statistical analysis. Preprocessing steps include quantification of expression levels, and quality assessment of the raw data. The expression levels of genes are quantified differently for microarrays and transcriptome sequencing. For microarrays, a process called normalization adjusts measured fluorescence intensities so that they are comparable across different experiments. Quality assessment eliminates low-quality microarrays, poorly aligning sequencing reads, or outlier measurements. Preprocessing prepares the data for statistical analysis.</p><p class=\"headingAnchor\" id=\"H3544117\"><span class=\"h4\">Quantification of expression</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>For microarray data analysis, each microarray can be considered a separate experiment that contains slightly different amounts of starting RNA and different labeling efficiencies [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/23\" class=\"abstract_t\">23</a>]. Data normalization adjusts the fluorescence intensities representing the amount of RNA bound to each probe so that these intensities are comparable across different arrays. There are several methods for normalizing microarray data, including:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Scaling &ndash; Adjusts intensities by a constant factor so that the average expression level across microarrays is similar.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Quantile normalization &ndash; Adjusts the distribution of intensities across microarrays. This is accomplished by ranking the probe intensities from highest to lowest for each array. A numerical value is assigned to represent this intensity on an individual array based on the behavior across all arrays and the rank of that probe on the individual array.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Locally weighted scatterplot smoothing (LOWESS) &ndash; Adjusts the brightness or darkness of different fluorescent labels for two-color array experiments.</p><p/><p>For transcriptome sequencing, each sample generates millions of sequencing reads which are used to estimate expression levels of each gene or isoform. First, high quality sequencing reads are then aligned to the reference genome. This step is performed using one of many available sequence aligners [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/24\" class=\"abstract_t\">24</a>]. Next, expression levels of each gene or isoform are calculated. This is frequently done by counting the reads aligned to a particular gene or isoform are counted, and then normalizing for the length of the gene. This yields reads per kilobase per million reads (RPKM) [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/24\" class=\"abstract_t\">24</a>].</p><p class=\"headingAnchor\" id=\"H3544124\"><span class=\"h4\">Quality assessment</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Quality assessment occurs both before and after data normalization.</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Pre-normalization quality assessment evaluates the quality of the raw data before preprocessing. For microarrays, the array itself is inspected to ensure there are no bubbles, scratches, or other artifacts on the array. Some commercial arrays also contain controls inserted during sample processing (&quot;spike-in&quot; controls) to ensure that all steps leading to the hybridization were successful. For transcriptome sequencing, each base pair call and individual sequencing read is considered a separate experiment that must be quality controlled [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/24\" class=\"abstract_t\">24</a>]. This is performed with tools such as FastQC or NGSQC. Sequencing reads may be then &quot;trimmed&quot; to remove the leading or tailing adapter sequences added during sample processing, or to remove lower quality base pairs at the ends of each sequencing read [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/20,24\" class=\"abstract_t\">20,24</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Post-normalization quality assessment evaluate processed data from a microarray or transcriptome sequencing sample relative to others in the experiment. This helps to identify outlier samples or differences in batches of microarrays or sequencing. Samples identified as significantly different from others can be adjusted statistically or excluded from the analysis.</p><p/><p class=\"headingAnchor\" id=\"H3544131\"><span class=\"h4\">Data transformation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Many common statistical procedures assume a normal and continuous distribution of data. Gene expression levels from microarrays or transcriptome sequencing can be mathematically transformed, often using a logarithmic scale, so that they become normally distributed. Transcriptome sequencing data, which is comprised of read counts rather than continuous numerical values, can be filtered to include only higher read counts which may approximate continuous data. Preprocessing can also include filtering out low quality probe sets or genes with low variability across all samples in the experiment.</p><p class=\"headingAnchor\" id=\"H95557425\"><span class=\"h3\">Data storage and analysis</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Microarray and transcriptome sequencing experiments require computational tools to store raw data, analyze gene expression, and ensure uniformity across different laboratories.</p><p class=\"headingAnchor\" id=\"H3544145\"><span class=\"h4\">Data storage</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The fluorescence intensities generated by scanning an oligonucleotide array or sequencing flow cell with a laser scanner results in an image file. Most scientific journals specify that raw data be made publicly available as a requirement for publication [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/25\" class=\"abstract_t\">25</a>]. A typical microarray raw data file, called a CEL file, is 0.1 to 1 gigabytes (GB) per array [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/17\" class=\"abstract_t\">17</a>]. A typical sequencing texted-based raw data file, called a FASTQ file, is approximately 1 to 5 GB per sample. Thus, these experiments generate a large amount of data that must be stored. Beyond the storage of raw data, data files also include the clinical variables associated with each sample and the normalized, quality assessed preprocessed expression levels for each array. This is often accomplished with the use of a database capable of storing and integrating both gene expression data and clinical variables.</p><p class=\"headingAnchor\" id=\"H3544152\"><span class=\"h4\">Data analysis</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>There are several possible levels of data analysis, ranging from simple statistical tests that can be performed with commercial software packages, to advanced analyses and the development of novel algorithms. Advanced analyses and novel algorithms are implemented with a variety of programming languages, such as Perl and Python, and computational software, such as R [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/26\" class=\"abstract_t\">26</a>] and Matlab [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/27\" class=\"abstract_t\">27</a>]. The flexibility to write, modify and share algorithms using these tools makes them particularly well suited for gene expression data analysis.</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Differential expression &ndash; One of the most common analyses performed on gene expression data is to determine which genes are altered in one condition as compared to another. This can be accomplished by performing a t-test, ANOVA, or linear model for continuous data, or binomial models for count data.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Class prediction &ndash; In this type of analysis, samples from two conditions are split into a training set and a test set. A list of genes that distinguishes the two conditions is derived from the training set of samples, and the accuracy of this gene expression signature is assessed on the test set of samples.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Class discovery &ndash; Genome wide gene expression data can be used to explore novel molecular phenotypes. By taking the most variable genes across all samples regardless of their clinical phenotype, it can be determined which samples are most closely correlated with each other based on gene expression alone. Samples that share similar patterns of gene expression may represent previously unrecognized subtypes of the disease.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Network analysis &ndash; The number of genes assayed by microarrays and transcriptome sequencing allows the entire dataset to be harnessed to make new predictions about how genes might interact. One approach, the Context Likelihood Relatedness (CLR) algorithm, uses a mathematical representation of the amount of information that one gene's expression gives about any other gene's expression [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/28\" class=\"abstract_t\">28</a>]. This can then be represented as a network of predicted interactions between genes. Genome-wide gene expression data can also be integrated with other data types, such as DNA methylation, proteomics, and metabolomics [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/20\" class=\"abstract_t\">20</a>].</p><p/><p>In addition to these general types of analyses of gene expression data, transcriptome sequencing also allows for more advanced analyses, such as discovery of novel transcripts or isoforms, detection of alternative splicing, de novo reconstruction of the transcriptome [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/20\" class=\"abstract_t\">20</a>].</p><p class=\"headingAnchor\" id=\"H3544159\"><span class=\"h3\">The multiple comparison problem</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Statistical analyses of several thousand genes pose unique problems in the interpretation of the statistical results because of the large number of tests performed. This is because every statistical test has a small possibility of leading to the conclusion that an association is present when no such association actually exists, and when thousands of genes are tested with a microarray, an unacceptably high number of false-positive associations may be produced. An overview of statistical principles relevant to the multiple comparison problem is presented separately. (See <a href=\"topic.htm?path=proof-p-values-and-hypothesis-testing\" class=\"medical medical_review\">&quot;Proof, p-values, and hypothesis testing&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H3544180\"><span class=\"h3\">Biological interpretation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The final step in microarray data analysis is to interpret the results in a biologically meaningful context. Making biological sense of a microarray-derived gene list is one of the more challenging aspects of the analysis. While there are many strategies for accomplishing this goal, two broad approaches are discussed below. Additional studies are often required to validate biological predictions that are made from the microarray data.</p><p class=\"headingAnchor\" id=\"H3544187\"><span class=\"h4\">Comparison with other genome-wide gene expression datasets</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>In one study in 2005, RNA was distributed to four laboratories for analysis by 12 different microarray platforms [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/29\" class=\"abstract_t\">29</a>]. Good reproducibility was found within a laboratory, but there was decreased reproducibility across laboratories [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/29\" class=\"abstract_t\">29</a>]. Reproducibility improved with the use of standardized protocols for RNA labeling, hybridization, microarray processing, data acquisition and data normalization.</p><p>Several tools exist for comparing gene expression datasets, including large databases containing gene expression data to look for a shared gene expression signature [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/30,31\" class=\"abstract_t\">30,31</a>], alternative gene probes [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/32\" class=\"abstract_t\">32</a>], and analytic tools that incorporate phenotypic associations [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/33-35\" class=\"abstract_t\">33-35</a>].</p><p class=\"headingAnchor\" id=\"H3544194\"><span class=\"h4\">Enrichment ranking</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Gene set enrichment analysis (GSEA) is a method by which gene expression data is ranked by association with phenotypes, and is used as a means to identify biologically-relevant pathways [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/33,34\" class=\"abstract_t\">33,34</a>]. Other techniques provide other mechanisms to enrich pathways or functional categories [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/36\" class=\"abstract_t\">36</a>] or to visualize previously published interactions between genes of interest [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/37\" class=\"abstract_t\">37</a>]. Data visualization using heat maps, which organize samples by columns and genes by rows according to similarity in gene expression, are also useful for determining which groups of genes or samples share similar patterns of expression. Gene set variation analysis (GSVA) uses a similar approach to identify pathway enrichment in a gene-expression dataset [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/38\" class=\"abstract_t\">38</a>].</p><p class=\"headingAnchor\" id=\"H3544201\"><span class=\"h1\">OVERVIEW OF CLINICAL APPLICATIONS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Gene expression profiling within clinical specimens has the potential to be used for disease screening, diagnosis, prognostication, and optimizing treatment regimens. As the platforms for measuring gene expression continue to evolve, personalized approaches to the diagnosis and treatment of complex human disease will increasingly find their place in routine clinical practice.</p><p>The promise of using gene expression profiling to identify subjects at risk for disease has yet to be fully reached. However, in certain circumstances this tool has been incorporated into routine clinical evaluation. As an example, the Oncotype Dx assay is used to guide the evaluation and management of subsets of patients with breast cancer. (See <a href=\"topic.htm?path=prognostic-and-predictive-factors-in-early-nonmetastatic-breast-cancer#H103670818\" class=\"medical medical_review\">&quot;Prognostic and predictive factors in early, nonmetastatic breast cancer&quot;, section on 'Gene expression profiles'</a> and <a href=\"topic.htm?path=prognostic-and-predictive-factors-in-metastatic-breast-cancer#H3119003\" class=\"medical medical_review\">&quot;Prognostic and predictive factors in metastatic breast cancer&quot;, section on 'Predictive factors'</a>.)</p><p class=\"headingAnchor\" id=\"H19682771\"><span class=\"h2\">Diagnosis</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Examples of the use of gene expression profiling to target selected patients for more conservative monitoring and less invasive diagnostic testing include the following:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Routine endomyocardial biopsy surveillance is used for cardiac transplant patients to detect the presence of acute cellular rejection. A study of gene expression profiling of ribonucleic acid (RNA) from peripheral blood mononuclear cells in heart transplant recipients revealed that an 11 gene expression profile could predict the presence of rejection [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/39\" class=\"abstract_t\">39</a>]. Use of this gene expression profile has the potential to decrease the number of transplant patients who would need to undergo invasive myocardial biopsy to confirm the diagnosis of rejection. (See <a href=\"topic.htm?path=acute-cardiac-allograft-rejection-diagnosis#H17\" class=\"medical medical_review\">&quot;Acute cardiac allograft rejection: Diagnosis&quot;, section on 'Gene expression profiling'</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Smokers suspected of having lung cancer on the basis of an abnormal chest computed tomography (CT) scan often need to undergo invasive diagnostic procedures, beyond bronchoscopy, to achieve a final diagnosis. A gene expression signature resulting from gene expression profiling of histologically normal airway epithelium obtained during bronchoscopy was capable of distinguishing between smokers with and without lung cancer [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/40\" class=\"abstract_t\">40</a>]. Combining results of this gene expression biomarker with clinical variables, in an integrated clinico-genomic model, improved the discriminatory potential to predict lung cancer [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/41\" class=\"abstract_t\">41</a>]. Refinement of this signature using an independent dataset and application in a prospective multicenter validation trial resulted in a diagnostic biomarker with a 91 percent negative predictive value in patients with an intermediate pretest probability of lung cancer [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/42,43\" class=\"abstract_t\">42,43</a>]. Gene expression profiling may help to stratify patients with an abnormal chest CT scan who should undergo invasive diagnostic testing for a potential lung cancer and those for whom imaging surveillance would be appropriate. (See <a href=\"topic.htm?path=screening-for-lung-cancer\" class=\"medical medical_review\">&quot;Screening for lung cancer&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Thyroid nodules are frequently evaluated using fine-needle aspirate biopsies, but this approach sometimes yields indeterminate results and requirement for thyroid surgery to achieve a definitive diagnosis. A diagnostic biomarker resulting from gene expression profiling of indeterminate thyroid nodule aspirates demonstrated a 92 percent sensitivity and high negative predictive value, suggesting that patients with indeterminate results from a fine needle aspirate of a thyroid nodule can be monitored less invasively, potentially avoiding unnecessary surgery [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/44\" class=\"abstract_t\">44</a>].</p><p/><p class=\"headingAnchor\" id=\"H3544236\"><span class=\"h2\">Prognosis</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The most advanced application of gene expression profiling is in predicting outcome from disease. The risk of certain therapies might be outweighed by the potential benefit for patients at high risk for relapse or with a poor prognosis, whereas the risks might outweigh the benefits for patients with a relatively good prognosis. Gene expression profiling has been helpful in targeting appropriate therapy for patients with acute leukemia, prostate cancer, colon cancer, breast cancer, lung carcinoma, and lymphoma.</p><p>MiRNAs have also been found to be dysregulated in a number of solid tumors and hematologic malignancies [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/45-57\" class=\"abstract_t\">45-57</a>]. (See <a href=\"#H95557489\" class=\"local\">'RNA in cell function'</a> above.)</p><p>A 2012 systematic review of available studies examining associations between miRNA profiling and cancer prognosis found associations between certain miRNAs and poor outcomes, including decreased overall survival but noted several potential sources of bias [<a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/58\" class=\"abstract_t\">58</a>]. Further work will be needed to validate the use of these findings in clinical applications.</p><p class=\"headingAnchor\" id=\"H3544264\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>There are a variety of methods for measuring ribonucleic acid (RNA) to evaluate gene expression. These methods differ in their requirements for the amount of starting RNA, their sensitivity to detect the RNA of interest, and the computational requirements for data analysis. (See <a href=\"#H3544024\" class=\"local\">'Measuring gene expression'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>While oligonucleotide array profiling and RNA sequencing are commonly used biomarker discovery platforms, there are numerous challenges and pitfalls in the analysis and interpretation of the large volume of data generated. (See <a href=\"#H3544103\" class=\"local\">'Genome-wide gene expression analysis and interpretation'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Gene expression profiling is rapidly emerging as a novel approach for the diagnosis and prognosis of complex human disease. However, a number of important barriers remain, including validation of these biomarkers in prospective multi-center studies to demonstrate their reproducibility and accuracy across multiple sites and operators. (See <a href=\"#H3544201\" class=\"local\">'Overview of clinical applications'</a> above.)</p></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li class=\"breakAll\">Brown TA. Genomes 3, 3rd ed, Garland Science, 2007.</li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/2\" class=\"nounderline abstract_t\">Lee RC, Feinbaum RL, Ambros V. The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14. Cell 1993; 75:843.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/3\" class=\"nounderline abstract_t\">Girard A, Sachidanandam R, Hannon GJ, Carmell MA. A germline-specific class of small RNAs binds mammalian Piwi proteins. Nature 2006; 442:199.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/4\" class=\"nounderline abstract_t\">Aravin A, Gaidatzis D, Pfeffer S, et al. A novel class of small RNAs bind to MILI protein in mouse testes. Nature 2006; 442:203.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/5\" class=\"nounderline abstract_t\">Khalil AM, Guttman M, Huarte M, et al. Many human large intergenic noncoding RNAs associate with chromatin-modifying complexes and affect gene expression. Proc Natl Acad Sci U S A 2009; 106:11667.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/6\" class=\"nounderline abstract_t\">Dvor&aacute;k Z, Pascussi JM, Modriansk&yacute; M. Approaches to messenger RNA detection - comparison of methods. Biomed Pap Med Fac Univ Palacky Olomouc Czech Repub 2003; 147:131.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/7\" class=\"nounderline abstract_t\">International Human Genome Sequencing Consortium. Finishing the euchromatic sequence of the human genome. Nature 2004; 431:931.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/8\" class=\"nounderline abstract_t\">Alwine JC, Kemp DJ, Stark GR. Method for detection of specific RNAs in agarose gels by transfer to diazobenzyloxymethyl-paper and hybridization with DNA probes. Proc Natl Acad Sci U S A 1977; 74:5350.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/9\" class=\"nounderline abstract_t\">Gall JG, Pardue ML. Formation and detection of RNA-DNA hybrid molecules in cytological preparations. Proc Natl Acad Sci U S A 1969; 63:378.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/10\" class=\"nounderline abstract_t\">Jin L, Lloyd RV. In situ hybridization: methods and applications. J Clin Lab Anal 1997; 11:2.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/11\" class=\"nounderline abstract_t\">Nolan T, Hands RE, Bustin SA. Quantification of mRNA using real-time RT-PCR. Nat Protoc 2006; 1:1559.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/12\" class=\"nounderline abstract_t\">VanGuilder HD, Vrana KE, Freeman WM. Twenty-five years of quantitative PCR for gene expression analysis. Biotechniques 2008; 44:619.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/13\" class=\"nounderline abstract_t\">Schena M, Shalon D, Davis RW, Brown PO. Quantitative monitoring of gene expression patterns with a complementary DNA microarray. Science 1995; 270:467.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/14\" class=\"nounderline abstract_t\">Churchill GA. Fundamentals of experimental design for cDNA microarrays. Nat Genet 2002; 32 Suppl:490.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/15\" class=\"nounderline abstract_t\">Pease AC, Solas D, Sullivan EJ, et al. Light-generated oligonucleotide arrays for rapid DNA sequence analysis. Proc Natl Acad Sci U S A 1994; 91:5022.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/16\" class=\"nounderline abstract_t\">Nuwaysir EF, Huang W, Albert TJ, et al. Gene expression analysis using oligonucleotide arrays produced by maskless photolithography. Genome Res 2002; 12:1749.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/17\" class=\"nounderline abstract_t\">Wilhelm BT, Landry JR. RNA-Seq-quantitative measurement of expression through massively parallel RNA-sequencing. Methods 2009; 48:249.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/18\" class=\"nounderline abstract_t\">Gr&uuml;n D, van Oudenaarden A. Design and Analysis of Single-Cell Sequencing Experiments. Cell 2015; 163:799.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/19\" class=\"nounderline abstract_t\">Liu S, Trapnell C. Single-cell transcriptome sequencing: recent advances and remaining challenges. F1000Res 2016; 5.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/20\" class=\"nounderline abstract_t\">Conesa A, Madrigal P, Tarazona S, et al. A survey of best practices for RNA-seq data analysis. Genome Biol 2016; 17:13.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/21\" class=\"nounderline abstract_t\">Stegle O, Teichmann SA, Marioni JC. Computational and analytical challenges in single-cell transcriptomics. Nat Rev Genet 2015; 16:133.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/22\" class=\"nounderline abstract_t\">Ozsolak F, Milos PM. RNA sequencing: advances, challenges and opportunities. Nat Rev Genet 2011; 12:87.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/23\" class=\"nounderline abstract_t\">Quackenbush J. Microarray data normalization and transformation. Nat Genet 2002; 32 Suppl:496.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/24\" class=\"nounderline abstract_t\">Yang IS, Kim S. Analysis of Whole Transcriptome Sequencing Data: Workflow and Software. Genomics Inform 2015; 13:119.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/25\" class=\"nounderline abstract_t\">Brazma A, Hingamp P, Quackenbush J, et al. Minimum information about a microarray experiment (MIAME)-toward standards for microarray data. Nat Genet 2001; 29:365.</a></li><li class=\"breakAll\">R Development Core Team. R: A Language and Environment for Statistical Computing 2009. R Foundation for Statistical Computing. Available at: www.R-project.org (Accessed on December 14, 2009).</li><li class=\"breakAll\">The Mathworks I. The MathWorks - MATLAB and Simulink for Technical Computing 2009. Available at: www.mathworks.com (Accessed on December 14, 2009).</li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/28\" class=\"nounderline abstract_t\">Faith JJ, Hayete B, Thaden JT, et al. Large-scale mapping and validation of Escherichia coli transcriptional regulation from a compendium of expression profiles. PLoS Biol 2007; 5:e8.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/29\" class=\"nounderline abstract_t\">Bammler T, Beyer RP, Bhattacharya S, et al. Standardizing global gene expression analysis between laboratories and across platforms. Nat Methods 2005; 2:351.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/30\" class=\"nounderline abstract_t\">Barrett T, Troup DB, Wilhite SE, et al. NCBI GEO: archive for high-throughput functional genomic data. Nucleic Acids Res 2009; 37:D885.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/31\" class=\"nounderline abstract_t\">Gower AC, Spira A, Lenburg ME. Discovering biological connections between experimental conditions based on common patterns of differential gene expression. BMC Bioinformatics 2011; 12:381.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/32\" class=\"nounderline abstract_t\">Dai M, Wang P, Boyd AD, et al. Evolving gene/transcript definitions significantly alter the interpretation of GeneChip data. Nucleic Acids Res 2005; 33:e175.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/33\" class=\"nounderline abstract_t\">Subramanian A, Tamayo P, Mootha VK, et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A 2005; 102:15545.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/34\" class=\"nounderline abstract_t\">Mootha VK, Lindgren CM, Eriksson KF, et al. PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. Nat Genet 2003; 34:267.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/35\" class=\"nounderline abstract_t\">Lamb J, Crawford ED, Peck D, et al. The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease. Science 2006; 313:1929.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/36\" class=\"nounderline abstract_t\">Dennis G Jr, Sherman BT, Hosack DA, et al. DAVID: Database for Annotation, Visualization, and Integrated Discovery. Genome Biol 2003; 4:P3.</a></li><li class=\"breakAll\">Ingenuity Systems. Ingenuity Pathway Analysis Software 2009. Available at: www.ingenuity.com (Accessed on December 14, 2009).</li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/38\" class=\"nounderline abstract_t\">H&auml;nzelmann S, Castelo R, Guinney J. GSVA: gene set variation analysis for microarray and RNA-seq data. BMC Bioinformatics 2013; 14:7.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/39\" class=\"nounderline abstract_t\">Deng MC, Eisen HJ, Mehra MR, et al. Noninvasive discrimination of rejection in cardiac allograft recipients using gene expression profiling. Am J Transplant 2006; 6:150.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/40\" class=\"nounderline abstract_t\">Spira A, Beane JE, Shah V, et al. Airway epithelial gene expression in the diagnostic evaluation of smokers with suspect lung cancer. Nat Med 2007; 13:361.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/41\" class=\"nounderline abstract_t\">Beane J, Sebastiani P, Whitfield TH, et al. A prediction model for lung cancer diagnosis that integrates genomic and clinical features. Cancer Prev Res (Phila) 2008; 1:56.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/42\" class=\"nounderline abstract_t\">Whitney DH, Elashoff MR, Porta-Smith K, et al. Derivation of a bronchial genomic classifier for lung cancer in a prospective study of patients undergoing diagnostic bronchoscopy. BMC Med Genomics 2015; 8:18.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/43\" class=\"nounderline abstract_t\">Silvestri GA, Vachani A, Whitney D, et al. A Bronchial Genomic Classifier for the Diagnostic Evaluation of Lung Cancer. N Engl J Med 2015; 373:243.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/44\" class=\"nounderline abstract_t\">Alexander EK, Kennedy GC, Baloch ZW, et al. Preoperative diagnosis of benign thyroid nodules with indeterminate cytology. N Engl J Med 2012; 367:705.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/45\" class=\"nounderline abstract_t\">Calin GA, Sevignani C, Dumitru CD, et al. Human microRNA genes are frequently located at fragile sites and genomic regions involved in cancers. Proc Natl Acad Sci U S A 2004; 101:2999.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/46\" class=\"nounderline abstract_t\">Lovat F, Valeri N, Croce CM. MicroRNAs in the pathogenesis of cancer. Semin Oncol 2011; 38:724.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/47\" class=\"nounderline abstract_t\">Esquela-Kerscher A, Slack FJ. Oncomirs - microRNAs with a role in cancer. Nat Rev Cancer 2006; 6:259.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/48\" class=\"nounderline abstract_t\">Calin GA, Dumitru CD, Shimizu M, et al. Frequent deletions and down-regulation of micro- RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia. Proc Natl Acad Sci U S A 2002; 99:15524.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/49\" class=\"nounderline abstract_t\">Calin GA, Ferracin M, Cimmino A, et al. A MicroRNA signature associated with prognosis and progression in chronic lymphocytic leukemia. N Engl J Med 2005; 353:1793.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/50\" class=\"nounderline abstract_t\">Garzon R, Volinia S, Liu CG, et al. MicroRNA signatures associated with cytogenetics and prognosis in acute myeloid leukemia. Blood 2008; 111:3183.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/51\" class=\"nounderline abstract_t\">Yanaihara N, Caplen N, Bowman E, et al. Unique microRNA molecular profiles in lung cancer diagnosis and prognosis. Cancer Cell 2006; 9:189.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/52\" class=\"nounderline abstract_t\">Yu SL, Chen HY, Chang GC, et al. MicroRNA signature predicts survival and relapse in lung cancer. Cancer Cell 2008; 13:48.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/53\" class=\"nounderline abstract_t\">Raponi M, Dossey L, Jatkoe T, et al. MicroRNA classifiers for predicting prognosis of squamous cell lung cancer. Cancer Res 2009; 69:5776.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/54\" class=\"nounderline abstract_t\">Fanini F, Vannini I, Amadori D, Fabbri M. Clinical implications of microRNAs in lung cancer. Semin Oncol 2011; 38:776.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/55\" class=\"nounderline abstract_t\">Boeri M, Pastorino U, Sozzi G. Role of microRNAs in lung cancer: microRNA signatures in cancer prognosis. Cancer J 2012; 18:268.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/56\" class=\"nounderline abstract_t\">Casta&ntilde;eda CA, Agullo-Ortu&ntilde;o MT, Fresno Vara JA, et al. Implication of miRNA in the diagnosis and treatment of breast cancer. Expert Rev Anticancer Ther 2011; 11:1265.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/57\" class=\"nounderline abstract_t\">Sandhu S, Garzon R. Potential applications of microRNAs in cancer diagnosis, prognosis, and treatment. Semin Oncol 2011; 38:781.</a></li><li><a href=\"https://www.uptodate.com/contents/tools-for-genetics-and-genomics-gene-expression-profiling/abstract/58\" class=\"nounderline abstract_t\">Nair VS, Maeda LS, Ioannidis JP. Clinical outcome prediction by microRNAs in human cancer: a systematic review. J Natl Cancer Inst 2012; 104:528.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 14602 Version 20.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H3544264\"><span>SUMMARY</span></a></li><li><a href=\"#H3544017\" id=\"outline-link-H3544017\">INTRODUCTION</a></li><li><a href=\"#H95557489\" id=\"outline-link-H95557489\">RNA IN CELL FUNCTION</a></li><li><a href=\"#H3544024\" id=\"outline-link-H3544024\">MEASURING GENE EXPRESSION</a><ul><li><a href=\"#H3544040\" id=\"outline-link-H3544040\">Expression profiling of single genes or small gene panels</a><ul><li><a href=\"#H3544047\" id=\"outline-link-H3544047\">- Northern blot</a></li><li><a href=\"#H3544054\" id=\"outline-link-H3544054\">- Ribonuclease protection assay</a></li><li><a href=\"#H3544061\" id=\"outline-link-H3544061\">- In-situ hybridization</a></li><li><a href=\"#H9716963\" id=\"outline-link-H9716963\">- Real-time reverse transcription polymerase chain reaction</a></li><li><a href=\"#H3544068\" id=\"outline-link-H3544068\">- Spotted cDNA arrays</a></li></ul></li><li><a href=\"#H3544075\" id=\"outline-link-H3544075\">Genome-wide gene expression profiling</a><ul><li><a href=\"#H95557895\" id=\"outline-link-H95557895\">- Oligonucleotide arrays (microarrays)</a></li><li><a href=\"#H3544096\" id=\"outline-link-H3544096\">- Transcriptome sequencing</a></li></ul></li><li><a href=\"#H3544103\" id=\"outline-link-H3544103\">Genome-wide gene expression analysis and interpretation</a><ul><li><a href=\"#H3544110\" id=\"outline-link-H3544110\">- Preprocessing of raw data</a><ul><li><a href=\"#H3544117\" id=\"outline-link-H3544117\">Quantification of expression</a></li><li><a href=\"#H3544124\" id=\"outline-link-H3544124\">Quality assessment</a></li><li><a href=\"#H3544131\" id=\"outline-link-H3544131\">Data transformation</a></li></ul></li><li><a href=\"#H95557425\" id=\"outline-link-H95557425\">- Data storage and analysis</a><ul><li><a href=\"#H3544145\" id=\"outline-link-H3544145\">Data storage</a></li><li><a href=\"#H3544152\" id=\"outline-link-H3544152\">Data analysis</a></li></ul></li><li><a href=\"#H3544159\" id=\"outline-link-H3544159\">- The multiple comparison problem</a></li><li><a href=\"#H3544180\" id=\"outline-link-H3544180\">- Biological interpretation</a><ul><li><a href=\"#H3544187\" id=\"outline-link-H3544187\">Comparison with other genome-wide gene expression datasets</a></li><li><a href=\"#H3544194\" id=\"outline-link-H3544194\">Enrichment ranking</a></li></ul></li></ul></li></ul></li><li><a href=\"#H3544201\" id=\"outline-link-H3544201\">OVERVIEW OF CLINICAL APPLICATIONS</a><ul><li><a href=\"#H19682771\" id=\"outline-link-H19682771\">Diagnosis</a></li><li><a href=\"#H3544236\" id=\"outline-link-H3544236\">Prognosis</a></li></ul></li><li><a href=\"#H3544264\" id=\"outline-link-H3544264\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>GRAPHICS <a href=\"#\" id=\"viewAllGraphicsLink\">View All</a></h2><div id=\"outlineGraphics\"><ul><li><div id=\"PC/14602|FIG\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"FIGURES\">FIGURES</a></div><ul><li><a href=\"image.htm?imageKey=PC/54914\" class=\"graphic graphic_figure\">- Central dogma molecular biology</a></li><li><a href=\"image.htm?imageKey=PC/70605\" class=\"graphic graphic_figure\">- mRNA processing</a></li><li><a href=\"image.htm?imageKey=PC/71360\" class=\"graphic graphic_figure\">- Analysis strategy for genome-wide gene expression studies</a></li></ul></li><li><div id=\"PC/14602|TAB\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"TABLES\">TABLES</a></div><ul><li><a href=\"image.htm?imageKey=PC/76106\" class=\"graphic graphic_table\">- Methods to quantify gene expression</a></li></ul></li></ul></div></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=acute-cardiac-allograft-rejection-diagnosis\" class=\"medical medical_review\">Acute cardiac allograft rejection: Diagnosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=principles-of-molecular-genetics\" class=\"medical medical_review\">Principles of molecular genetics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=prognostic-and-predictive-factors-in-early-nonmetastatic-breast-cancer\" class=\"medical medical_review\">Prognostic and predictive factors in early, nonmetastatic breast cancer</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=prognostic-and-predictive-factors-in-metastatic-breast-cancer\" class=\"medical medical_review\">Prognostic and predictive factors in metastatic breast cancer</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=proof-p-values-and-hypothesis-testing\" class=\"medical medical_review\">Proof, p-values, and hypothesis testing</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=screening-for-lung-cancer\" class=\"medical medical_review\">Screening for lung cancer</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">Tools for genetics and genomics: Cytogenetics and molecular genetics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-polymerase-chain-reaction\" class=\"medical medical_review\">Tools for genetics and genomics: Polymerase chain reaction</a></li></ul></div></div>","javascript":null}