var data={"title":"Use of chromosomal microarray in obstetrics","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Use of chromosomal microarray in obstetrics</div><dl id=\"topicContributors\"><dt><span> </span>Author:</dt><dd><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/contributors\" class=\"contributor contributor_credentials\">David T Miller, MD, PhD</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/contributors\" class=\"contributor contributor_credentials\">Louise Wilkins-Haug, MD, PhD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/contributors\" class=\"contributor contributor_credentials\">Vanessa A Barss, MD, FACOG</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Jan 09, 2018.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H18810847\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Prenatal diagnosis and postnatal evaluation of pregnancy loss often involves cytogenetic analysis of amniocytes, chorionic villi, or fetal cells. Although conventional Giemsa(G)-banding of metaphase chromosomes detects aneuploidies and large structural changes (eg, balanced or unbalanced translocations, inversions), this approach also has limitations: it does not consistently identify submicroscopic (ie, smaller than what is visible under a light microscope) genomic defects (&lt;3 to 10 million base pairs [Mb]) and requires cell culture, which takes a minimum of seven days to obtain an adequate number of dividing cells. The use of fluorescence in situ hybridization (FISH) reduces the time to obtain a result and is less labor intensive, but can only detect a limited number of pre-specified targets. </p><p>Chromosomal microarray (CMA) is an array-based molecular cytogenic technique that can overcome some limitations of a karyotype, and is particularly useful for its ability to detect submicroscopic gains and losses on every chromosome (<a href=\"image.htm?imageKey=PC%2F58888\" class=\"graphic graphic_figure graphicRef58888 \">figure 1</a>) and to provide results from products of conception when cells do not grow in culture. This technique compares the genomic content (DNA) of a patient (target) with that of a normal control individual (or individuals) and detects gains and losses (duplications and deletions) ranging in size from very large, including aneuploidy of entire chromosomes, to very small (typically as small as about 200,000 base pairs or 0.2 Mb). Gains and losses may correspond to structural changes such as unbalanced translocations), but CMA cannot detect the physical location of the extra material (as in a translocation) nor can it detect structural chromosome changes that do not result in deletions or duplications (eg, balanced translocations or balanced inversions). Different laboratories perform CMA using different technology platforms and with different array design and content [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/1-7\" class=\"abstract_t\">1-7</a>]. </p><p>This topic will discuss the use of CMA in obstetrics. Basic principles of genetics are reviewed separately, and include:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>(See <a href=\"topic.htm?path=basic-principles-of-genetic-disease\" class=\"medical medical_review\">&quot;Basic principles of genetic disease&quot;</a>.)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>(See <a href=\"topic.htm?path=principles-of-molecular-genetics\" class=\"medical medical_review\">&quot;Principles of molecular genetics&quot;</a>.)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>(See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-gene-expression-profiling\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Gene expression profiling&quot;</a>.)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>(See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Cytogenetics and molecular genetics&quot;</a>.)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>(See <a href=\"topic.htm?path=genomic-disorders-an-overview#H3390507\" class=\"medical medical_review\">&quot;Genomic disorders: An overview&quot;, section on 'Array comparative genomic hybridization'</a>.)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>(See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-gene-expression-profiling#H3544103\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Gene expression profiling&quot;, section on 'Genome-wide gene expression analysis and interpretation'</a>.)</p><p/><p class=\"headingAnchor\" id=\"H763272546\"><span class=\"h1\">PROCEDURES</span></p><p class=\"headingAnchor\" id=\"H4160558480\"><span class=\"h2\">Specimen</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Prenatal CMA is performed on DNA from uncultured amniocytes (amniotic fluid), chorionic villus cells, or cord blood [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/8\" class=\"abstract_t\">8</a>]. </p><p>CMA is not able to be performed on cell-free DNA in maternal blood during noninvasive prenatal testing because there is no amplification step so the signal from the fetus would be &quot;drowned out&quot; by the maternal DNA. High read depth next-generation sequencing (NGS) is being studied to detect gains and losses in the range of 7 to 10 Mb. However, recurrent deletion syndromes such as Williams syndrome and Prader-Willi syndrome are smaller than the reliable detection limit of noninvasive prenatal testing [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/9\" class=\"abstract_t\">9</a>].</p><p class=\"headingAnchor\" id=\"H1997296363\"><span class=\"h2\">Types of microarray</span></p><p class=\"headingAnchor\" id=\"H2857363707\"><span class=\"h3\">Targeted versus whole genome</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Prenatal CMA typically assesses the whole genome or may be targeted to evaluate regions of the genome known or highly suspected to cause specific phenotypes when deleted or duplicated. It has become the standard practice to include single nucleotide polymorphism (SNP) probes to allow the detection of triploidy and some cases of uniparental disomy (UPD). (See <a href=\"#H4180752565\" class=\"local\">'Platform'</a> below.)</p><p>Compared to whole genome CMA, targeted CMA lowers the chance of identifying variants of uncertain significance since the targets are associated with known phenotypes. However, targeted CMA also potentially results in a lower diagnostic yield as unique genomic events that could be clinically significant may be missed.</p><p>Whole genome CMA may result in an increased frequency of challenging counseling issues because copy number variants of unknown significance are more often detected. In addition, copy number variants related to late-onset diseases may be identified in genomic regions other than those specifically targeted (so-called secondary findings [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/10\" class=\"abstract_t\">10</a>]), thus the additional information provided by whole genome CMA can have implications for both the parent and offspring. For example, a prenatal diagnostic procedure to detect Down syndrome may exclude Down syndrome, but identify an inherited BRCA 1 or 2 deletion. Thus, although CMA may be used as a diagnostic test to identify a specific disorder in a patient at risk for that disorder, it can also act as a screening test for a broad range of other genetic abnormalities, blurring the distinction between prenatal diagnosis and prenatal screening. In clinical practice, the detection of late-onset conditions is quite rare.</p><p>ACOG recommends not routinely assessing whole genome or whole exome sequencing for prenatal diagnosis as there are insufficient data of clinical utility to support routine use [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/11\" class=\"abstract_t\">11</a>]. Whole-exome sequencing is a reasonable option in select cases, such as recurrent or lethal fetal anomalies in which a thorough evaluation has been non-informative. Before testing, these patients should receive extensive counseling by an obstetrician-gynecologist or other health care provider with genetics expertise who is familiar with the new technology and its limitations. For example, patients should be informed that prenatal exome and genome sequencing would result in an even higher possibility of variants of uncertain significance <span class=\"nowrap\">and/or</span> incidental findings compared with CMA. </p><p class=\"headingAnchor\" id=\"H4180752565\"><span class=\"h3\">Platform</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Regardless of &quot;targeted&quot; or &quot;whole genome&quot; coverage, essentially all clinical laboratories are now offering an array platform capable of detecting both copy number changes and stretches of homozygosity, a phenomenon that may indicate uniparental disomy. As such, current arrays have a combination of copy number probes (oligonucleotides for comparative genomic hybridization) to detect copy gains and losses and SNP probes&nbsp;to detect similarity in single nucleotide sequences (homozygosity). This &quot;hybrid&quot; approach is now standard for prenatal microarray analysis. Interspersing SNP probes along with the copy number probes has the additional advantage that prenatal CMA including SNP probes can recognize triploidy as effectively as a G-banded karyotype.</p><p>The combination of CMA plus SNP detects runs of homozygosity between the maternal and paternal copy of each chromosome, thus enabling diagnosis of triploidy as well as uniparental disomy and consanguinity. The addition of SNP probes also improves detection of low levels of mosaicism. A typical 20-cell G-banded karyotype detects mosaicism at approximately 14 percent of cells or more, while most current clinical CMA with SNP platforms typically detect mosaicism at comparable levels to G-banding (approximately 10 to 15 percent). If there is suspicion of even lower level mosaicism on CMA with SNP or G-banding, or even postnatally, a cytogenetics laboratory can be asked to count additional cells to improve the detection of mosaicism below 10 percent of cells. Very low-level mosaicism, however, can present a challenge in counseling as it generally results in a milder phenotypic abnormality than the same but non-mosaic genomic imbalance. </p><p class=\"headingAnchor\" id=\"H12423065\"><span class=\"h1\">BENEFITS AND LIMITATIONS OF CMA</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>CMA with SNP has advantages and disadvantages compared with conventional G-banding (<a href=\"image.htm?imageKey=OBGYN%2F109837\" class=\"graphic graphic_table graphicRef109837 \">table 1</a>).</p><p class=\"headingAnchor\" id=\"H18811338\"><span class=\"h2\">Benefits</span></p><p class=\"headingAnchor\" id=\"H18811391\"><span class=\"h3\">Higher diagnostic yield</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Although the resolution of the array is dependent upon the type of array used and the average spacing of the probes on the array, the minimum resolution is 50,000 to 200,000 base pairs compared with 3 to 10 million base pairs for G-banding. The ability of CMA to detect extremely small changes in the genome results in a higher rate of detection of abnormalities compared with conventional G-banding [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/12-16\" class=\"abstract_t\">12-16</a>]. </p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A 2013 systematic review of four large studies calculated the utility of prenatal microarrays when the conventional G-band karyotype was normal and found copy number changes of clinical significance were noted in [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/17\" class=\"abstract_t\">17</a>]:</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>2.4 percent <span class=\"nowrap\">(295/12,362)</span> of cases from all prenatal ascertainment groups reported in four large studies</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>6.5 percent <span class=\"nowrap\">(201/3090)</span> of cases with an abnormal ultrasound examination in these four studies</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>1.0 percent <span class=\"nowrap\">(50/5108)</span> of cases performed because of advanced maternal age </p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>1.1 percent <span class=\"nowrap\">(44/4164)</span> of cases performed because of parental anxiety, history of chromosome abnormality, abnormal serum screening result, and &ldquo;other&rdquo;</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In a 2014 systematic review that combined data from 18 studies (n = 2220 fetuses), 3.1 to 7.9 percent of fetuses with a structural ultrasound anomaly restricted to one anatomical system and a normal karyotype had a submicroscopic copy number variant that explained its phenotype and provided information for fetal prognosis [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/18\" class=\"abstract_t\">18</a>].</p><p/><p class=\"headingAnchor\" id=\"H2514550684\"><span class=\"h3\">No requirement for cell culture</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>CMA can be performed directly on uncultured cells, which shortens the time required to obtain results and increases diagnostic yield when only non-dividing cells are available for analysis, such as in some stillborns.</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In a prospective study of 532 stillborn fetuses that compared diagnostic yield with CMA versus traditional karyotype, CMA resulted in a positive finding: 42 percent more often than karyotype in all stillbirths, 35 percent more often in antepartum stillbirths, and 54 percent more often in stillbirths with anomalies [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/13\" class=\"abstract_t\">13</a>]. The actual yields of positive results for CMA and karyotype were: all stillbirths 8.3 versus 5.8 percent (P = 0.007), antepartum stillbirths 8.8 versus 6.5 percent (P = 0.02), and stillbirths with congenital anomalies 29.9 versus 19.4 percent, (P = 0.008). Importantly, CMA was successful (from a technical standpoint) more often than karyotype (87.4 versus 70.5 percent, P&lt;0.001), since karyotype requires cell culture, which is often unsuccessful in this setting.</p><p/><p class=\"headingAnchor\" id=\"H18811433\"><span class=\"h3\">Faster turnaround time</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>G-banding requires cell culture to obtain sufficient DNA from dividing cells, which results in a turnaround time of one to two weeks. CMA can be performed directly on high quality DNA extracted from isolated cells; results are typically available within one week (although in the best circumstances results can be available in as little as one to two days), since time for culture is not required. CMA is also more amenable to automation in the laboratory, which provides objective interpretation and decreases labor costs.</p><p class=\"headingAnchor\" id=\"H2325619208\"><span class=\"h3\">Detection of consanguinity and other familial relationships</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>CMA including SNP probes is not standard but can detect consanguinity or other close familial relationships. If CMA is also performed on the purported parents, nonpaternity can be identified.</p><p class=\"headingAnchor\" id=\"H18812072\"><span class=\"h2\">Limitations</span></p><p class=\"headingAnchor\" id=\"H18812086\"><span class=\"h3\">Inability to detect balanced structural rearrangements</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>In contrast to G-banding,<strong> </strong>CMA is unable to detect balanced structural rearrangements (balanced translocations and inversions) because there is no change in copy number. Although this is a disadvantage of the technique, it is not always of clinical significance because:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The likelihood that a truly balanced rearrangement would interrupt a gene is low. Even those that have breakpoints within a gene are not necessarily pathogenic. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>When the parent and fetus have the same balanced rearrangement, it is generally considered benign if the parent is unaffected. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Although there is greater concern that a de novo balanced rearrangement may be clinically significant (ie, associated with an abnormal phenotype), de novo balanced rearrangements are rare and represent a small proportion of the abnormalities detected by G-banding.</p><p/><p class=\"bulletIndent1\">In one study including 269,371 analyses, the rate of prenatally diagnosed de novo balanced chromosome rearrangements in amniocytes, chorionic villus samples, and fetal blood samples was 0.09, 0.08 and 0.05 percent, respectively [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/19\" class=\"abstract_t\">19</a>]. In another study of 377,357 amniocenteses, de novo reciprocal translocations were detected in approximately 0.05 percent and a balanced Robertsonian translocation in 0.01 percent [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/20\" class=\"abstract_t\">20</a>].</p><p/><p>De novo cytogenetic rearrangements that appear balanced by G-banding may have submicroscopic genomic differences in the number of copies of one or more sections of DNA. These copy number variants may be known to be benign or associated with phenotypic consequences, or their significance may be unknown. Although most apparently balanced reciprocal translocations in phenotypically normal individuals do not contain genomic imbalance and are benign [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/21\" class=\"abstract_t\">21</a>], there is a 6 percent probability that an apparently balanced translocation identified on prenatal G-banding has phenotypic consequences [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/22\" class=\"abstract_t\">22</a>]. As a result, the overall ability to predict abnormalities (eg, developmental delay, malformation) is more strongly correlated with findings on CMA than G-banding [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/23-25\" class=\"abstract_t\">23-25</a>].</p><p class=\"headingAnchor\" id=\"H70778136\"><span class=\"h3\">Inability to detect disorders associated with single gene point mutations</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>This includes autosomal-recessive and autosomal-dominant disorders such as inborn errors of metabolism, skeletal dysplasias, and many other syndromes.</p><p class=\"headingAnchor\" id=\"H2388440353\"><span class=\"h3\">Inability to detect all cases of uniparental disomy</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>CMA with SNP can detect runs of homozygosity, which may indicate isodisomy, a subtype pf uniparental disomy (UPD). However, UPD is equally likely to occur through heterodisomy (non-identical stretches of chromosome inherited from the same parent) and these would not be detected. If an imprinting disorder is suspected, relying on CMA with SNP for detection of UPD results in a high rate of false-negative results (approximately 50 percent). Note that a G-banded karyoptype will not detect UPD.</p><p class=\"headingAnchor\" id=\"H3735402877\"><span class=\"h3\">Other nondetectable events</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Copy number probe coverage on a targeted array is more densely clustered in the &quot;known disease regions&quot; caused by copy number changes, and coverage is less dense elsewhere. Therefore, outside the disease regions, only larger copy number changes would be detected (approaching the size visible on a karyotype). Likewise, SNP probe coverage for a targeted array may be limited to chromosomes known to cause disease through imprinting, such as chromosome 11 for Beckwith-Wiedemann syndrome or chromosome 15 for Prader-Willi syndrome. Due to limited probe coverage on other chromosomes, the targeted array cannot reliably detect consanguinity.</p><p class=\"headingAnchor\" id=\"H1897120\"><span class=\"h3\">Detection of variants of uncertain significance</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Copy number variants of uncertain significance (VUS) are copy number changes that have not been reported and thus have an unknown phenotype. These small changes are not detectable by G-banding, but are identified by CMA in 1 to 2 percent of cases [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/12,13,26,27\" class=\"abstract_t\">12,13,26,27</a>]. The additional information provided by CMA can be challenging to interpret, particularly if a phenotypically normal parent carries the same change. In the absence of clear prognostic information, parents may find it difficult to make a decision about continuing the pregnancy. (See <a href=\"topic.htm?path=overview-of-genetic-variation#H14\" class=\"medical medical_review\">&quot;Overview of genetic variation&quot;, section on 'Copy number variations (CNVs)'</a>.)</p><p>In a large prospective study performed from 2008 to mid-2011 [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/12\" class=\"abstract_t\">12</a>], CMA detected VUS in 3.4 percent (130 of 3822) of all cases that were normal by karyotype. Although this degree of uncertainty is concerning, the authors pointed out that collective knowledge in this field continues to improve, thereby decreasing the frequency of VUS. If the VUS identified in this study were reanalyzed in 2012, the overall VUS rate would be approximately 1.5 percent (56 of 3822). Some prenatal diagnostic centers have started to periodically compare VUS detected in past patients against updated information on phenotypes associated with copy number variants and provide updates to the clinical reports for clinicians.</p><p>Detection of VUS has raised the question of whether it is ethically justifiable to withhold these test results from the patient [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/28\" class=\"abstract_t\">28</a>]. VUS should become less problematic in the future as the International Standards for Cytogenomic Arrays (ISCA) is creating a database of array results and associated phenotypes for the National Institutes of Health (NIH). </p><p class=\"headingAnchor\" id=\"H2963146976\"><span class=\"h3\">Cost</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>CMA is now approximately twice the cost of a conventional karyotype, but also has increased diagnostic yield. As indicated by the recent ACOG and SMFM opinions, either G-banding or CMA is reasonable for patients undergoing invasive prenatal diagnosis who have a structurally normal, viable fetus, but CMA is preferred for evaluation of a fetus with structural abnormalities.</p><p class=\"headingAnchor\" id=\"H16315619\"><span class=\"h3\">Variable clinical sensitivity</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Clinical sensitivity refers to the proportion of patients who have the clinical symptoms of a disorder and are detectable by the testing method. This limitation is true of all genetic tests as there is no single test to detect all types of genetic aberration. Some genetic conditions are primarily caused by a change in genomic copy number (eg, a microdeletion or microduplication syndrome). An example of a relatively common genomic disorder is velocardiofacial syndrome, also known as 22q11.2 deletion syndrome. Among individuals with <span class=\"nowrap\">signs/symptoms</span> of this condition, a microdeletion will be detected by chromosomal microarray (CMA) in over 90 percent. When the microdeletion is not detected, the patient usually does not have all of the symptoms and may have another diagnosis that is similar. In contrast, neurofibromatosis type-1 (NF1) can be caused by a genomic deletion that includes the NF1 gene, but less than 5 percent of NF1 patients have a deletion; most of these patients have a sequence change in the gene that is undetectable by CMA. If a patient has a genetic condition in which a subset of cases are caused by sequence changes, then other testing should be considered either in place of, or in addition to, CMA.</p><p class=\"headingAnchor\" id=\"H1897463\"><span class=\"h1\">CLINICAL INDICATIONS FOR CMA IN OBSTETRICS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Clinical guidelines for use of CMA are evolving as more data on the benefits and limitations of this technique are published [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/11,29-31\" class=\"abstract_t\">11,29-31</a>]. </p><p>The American College of Obstetrics and Gynecology (ACOG) and Society for Maternal-Fetal Medicine (SMFM) recommend use of CMA, and state that CMA typically can replace the need for a karyotype, when genetic analysis is desired because of [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/11,32\" class=\"abstract_t\">11,32</a>]: </p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Fetal congenital anomalies</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Intrauterine fetal death or stillbirth</p><p/><p>The ACOG opinion concluded that either G-banding or CMA is reasonable for patients undergoing invasive prenatal diagnosis who have a structurally normal, viable fetus. Some clinicians have advocated microarray as a first-line test whenever fetal chromosomal analysis is planned [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/33\" class=\"abstract_t\">33</a>].</p><p>In the setting of a fetus with structural abnormalities, some providers order fluorescence in situ hybridization (FISH) for chromosomes 13, 18, 21, X, and Y as a first-line test. This FISH panel will typically produce a result in 24 to 48 hours, which may be desirable in some cases since typical CMA can take approximately one week. However, if the difference in turnaround times is not important to patient management, then FISH has no benefit. Essentially all clinical labs performing prenatal CMA are using a SNP platform and thus will be able to detect triploidy, so there is no increase in diagnostic yield with a FISH panel.</p><p>Although not addressed by guidelines, CMA may be indicated in:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Prenatal assessment of fetuses with growth restriction. A systematic review of 10 studies on fetuses with growth restriction reported an additional yield of CMA after normal karyotype of 10 percent in those with structural abnormalities and 4 percent in those without structural abnormalities [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/34\" class=\"abstract_t\">34</a>].</p><p/><p>In the postnatal setting, American College of Medical Genetics (ACMG) guidelines recommend CMA as the first-tier test for patients with [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/35\" class=\"abstract_t\">35</a>]:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Unexplained developmental <span class=\"nowrap\">delay/intellectual</span> disability (see <a href=\"topic.htm?path=intellectual-disability-in-children-evaluation-for-a-cause\" class=\"medical medical_review\">&quot;Intellectual disability in children: Evaluation for a cause&quot;</a>)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Autism spectrum disorders (see <a href=\"topic.htm?path=autism-spectrum-disorder-diagnosis\" class=\"medical medical_review\">&quot;Autism spectrum disorder: Diagnosis&quot;</a>)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Multiple congenital anomalies not explained by a specific syndrome (see <a href=\"topic.htm?path=birth-defects-approach-to-evaluation\" class=\"medical medical_review\">&quot;Birth defects: Approach to evaluation&quot;</a>)</p><p/><p>Also, if a live-born child is considered a candidate for CMA testing, and had prenatal CMA testing, postnatal CMA may be indicated since prenatal CMA may have been targeted or of lower resolution [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/32\" class=\"abstract_t\">32</a>]</p><p class=\"headingAnchor\" id=\"H12423086\"><span class=\"h2\">Prenatal ultrasound showing fetal structural abnormalities</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Fetal structural abnormalities detectable by ultrasound should raise suspicion of a chromosomal abnormality in the fetus. In a study that karyotyped 2086 fetuses after ultrasonographic examination revealed fetal malformations, growth restriction, or both, chromosomal abnormalities by G-banding were detected in 301 cases (14 percent), and were more common among fetuses with multisystem malformations (29 percent) than among those with isolated defects (2 percent) [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/36\" class=\"abstract_t\">36</a>]. </p><p>Among fetuses presenting with an ultrasound structural abnormality and a normal G-banded karyotype, a 2013 meta-analysis reported that CMA was abnormal in 7.2 percent (95% CI 5.4-9.8; 7 large cohort studies published 2011 to 2013) when the referral indication was structural malformation on ultrasound [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/37\" class=\"abstract_t\">37</a>]. This number is higher than the 1 to 3 percent previously thought to have a genetic imbalance (eg, microdeletion, microduplication) on CMA [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/38-40\" class=\"abstract_t\">38-40</a>], but the lower prevalence was based on earlier studies. The increased yield reported by the meta-analysis included data from recent studies and was likely due to increased coverage of CMA probes since the detection of abnormalities on the array increases as the density of the array increases, as well as when there are multiple anomalies present. Postnatal studies of neonates with structural anatomical abnormalities confirm these findings [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/41,42\" class=\"abstract_t\">41,42</a>].</p><p>In fetuses with congenital heart disease, a systematic review concluded performing CMA had value even if conventional G-banding and 22q11 microdeletion analysis are normal [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/43\" class=\"abstract_t\">43</a>]. In cases of isolated congenital heart disease with normal karyotype and 22q11 microdeletion analysis by FISH, the yield of additional pathogenic copy number variants detected by CMA was approximately 3 percent. The incremental yield was 7 percent in fetuses with both cardiac and noncardiac anomalies.</p><p class=\"headingAnchor\" id=\"H12423093\"><span class=\"h2\">Fetal demise</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>CMA is useful in evaluation of stillbirth (pregnancy loss &ge;20 weeks of gestation) because both chromosomal abnormalities and culture failure are common in these cases Chromosomal abnormalities identifiable by G-banded karyotype are identified in about 5 percent of stillborn fetuses in the absence of an anatomical malformation and 35 to 40 percent of stillborns when structural abnormalities are present [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/22,44\" class=\"abstract_t\">22,44</a>]. Culture failure is common when the fetus has died, and thus prevents the accurate diagnosis of a karyotypic abnormality in these cases, CMA can overcome this difficulty.</p><p>As discussed above, in a study of samples from 532 stillbirths, SNP oligonucleotide microarray analysis was more likely to yield results than karyotype analysis (87.4 versus 70.5 percent, P&lt;0.001), provided better detection of genetic abnormalities (aneuploidy or pathogenic copy number variants, 8.3 versus 5.8 percent; P = 0.007), and identified more genetic abnormalities among the 443 antepartum stillbirths (8.8 versus 6.5 percent, P = 0.02) and the 67 stillbirths with congenital anomalies (29.9 versus 19.4 percent, P = 0.008) [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/13,45\" class=\"abstract_t\">13,45</a>]. A copy-number VUS was reported in 2.6 percent (12 of 465) of the samples.</p><p>CMA may also have some advantages over G-banded karyotyping for diagnostic evaluation of miscarriages (pregnancy loss before 20 weeks). In a 2013 systematic review, CMA and karyotyping results agreed in 86.0 percent of cases (95% CI 77.0-96.0), CMA detected 13 percent (95% CI 8.0-21.0) additional chromosome abnormalities over conventional full karyotyping but conventional full karyotyping detected 3 percent (95% CI 1.0-10.0) additional abnormalities over CMA [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/46\" class=\"abstract_t\">46</a>]. The incidence of a variant of unknown significance was 2 percent (95% CI 1.0-10.0). However, the clinical value of CMA over G-banding in these cases is less clear than in the evaluation of stillbirth because CMA may miss polyploidy, which is the cause of 10 to 20 percent of miscarriages, and there is a lack of information about potential relationships between small deletions, duplications, and other variants detected by CMA and risk of miscarriage. A previously unreported CNV is found in 30 to 50 percent of miscarriages, requiring further parental evaluation; only 6 percent of miscarriages reveal structural chromosomal abnormalities requiring parental follow-up [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/47\" class=\"abstract_t\">47</a>].</p><p class=\"headingAnchor\" id=\"H1897951\"><span class=\"h2\">Marker chromosome</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A marker chromosome refers to small chromosome fragments that may be observed with a G-banded karyotype. Because of the small size of the marker, the chromosome of origin cannot always be determined. CMA is often able to identify the chromosomal origin of a marker chromosome and thus allow prognostic counseling based on its origin and gene content. (See <a href=\"topic.htm?path=congenital-cytogenetic-abnormalities#H11375661\" class=\"medical medical_review\">&quot;Congenital cytogenetic abnormalities&quot;, section on 'Supernumerary marker chromosome'</a>.)</p><p class=\"headingAnchor\" id=\"H1897960\"><span class=\"h1\">POSSIBLE INDICATIONS FOR PRENATAL CMA</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Prenatal CMA may be useful after a normal G-band karyotype or instead of conventional karyotyping in the following situations, but data are limited and the diagnostic yield of CMA in these settings is not well-described:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Intrauterine growth restriction (IUGR). </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Increased nuchal lucency. (See <a href=\"topic.htm?path=cystic-hygroma-and-increased-nuchal-translucency\" class=\"medical medical_review\">&quot;Cystic hygroma and increased nuchal translucency&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Abnormal maternal serum screening showing an increased risk for Down syndrome. (See <a href=\"topic.htm?path=down-syndrome-overview-of-prenatal-screening\" class=\"medical medical_review\">&quot;Down syndrome: Overview of prenatal screening&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>One or more soft markers for aneuploidy on ultrasound. (See <a href=\"topic.htm?path=sonographic-findings-associated-with-fetal-aneuploidy\" class=\"medical medical_review\">&quot;Sonographic findings associated with fetal aneuploidy&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Oligohydramnios or polyhydramnios.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Positive cell-free DNA screen for a microdeletion -- Noninvasive prenatal screening (NIPS) by next-generation sequencing of cell-free DNA in maternal plasma is used for primary or secondary screening for common aneuploidies (trisomies 21, 18, 13, and sex chromosome aneuploidies). Commercial companies have begun to offer expanded panels including screening for common microdeletion syndromes such as 22q11.2 deletion (DiGeorge syndrome) without reporting the genomic coordinates or whether the deletion is maternal or fetal [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/48\" class=\"abstract_t\">48</a>].&nbsp;If microdeletion NIPS is performed, diagnostic prenatal or postnatal testing with chromosomal microarray and appropriate parental studies to determine precise genomic coordinates and inheritance should follow a positive microdeletion NIPS result.</p><p/><p class=\"headingAnchor\" id=\"H12423100\"><span class=\"h1\">PATIENT COUNSELING</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Couples considering CMA testing should receive pre-test and post-test genetic counseling that is nondirective and clearly explains that testing is voluntary. Nondirective information enables the parents to balance the risks, benefits, and limitations of this approach to prenatal diagnosis. Genetic counseling should involve a provider with specific expertise in the area of prenatal genetic testing, usually a genetic counselor, medical geneticist, or provider with relevant experience, given the range of potential results from CMA and the variable clinical effects of particular genomic imbalance disorders.</p><p>Pre-test counseling should provide the following information [<a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/49\" class=\"abstract_t\">49</a>]:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Risks associated with prenatal diagnosis (eg, procedure-related fetal loss).</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Potential psychological implications of prenatal diagnosis (eg, uncertainty, anxiety, need for consideration of pregnancy termination or other intervention). In particular, the patient should understand that the severity of disease cannot be predicted by prenatal genetic studies, genetic studies may detect the gene for a disease that never manifests clinically, and genes for adult onset diseases in themselves may be identified. This can be particularly problematic for copy number changes associated with risk of neurocognitive or neuropsychiatric symptoms where the degree of symptom severity can be quite variable, even within the same family. Depending on the type of microarray, additional concerns regarding the identification of consanguinity and non-paternity exist.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Implications of having a child with a genomic disorder.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Advantages and disadvantages of prenatal CMA.</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Detection rate for chromosomal abnormalities is higher than with G-banding. </p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Younger women have a similar risk of CMA abnormality in offspring as older women: The frequency of abnormal copy number variants in offspring does not increase with maternal age, in contrast to autosomal and sex chromosome aneuploidy.</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Possibility and clinical implications of an undetected balanced rearrangement. </p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Possibility and clinical implications of detecting a variant of unknown significance.</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Possibility of detecting a carrier state for genetic disease within the parent. </p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Possibility of identifying <span class=\"nowrap\">non-paternity/consanguinity</span>. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Information about the length of time necessary to obtain CMA results.</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Results may be available one week after biopsy with direct cell preparation (eg, chorionic villus biopsy, biopsy of fetal tissue, amniocytes).</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>If amniocytes are cultured initially for G-banding and then CMA is performed, results may be available three weeks after amniocentesis. </p><p/><p>Post-test counseling is best provided through consultation with a genetic counselor or a medical geneticist, particularly after a diagnosis of a genomic imbalance disorder, to provide the following information:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><span class=\"nowrap\">Normal/negative</span> results do not rule out the presence of a genetic disorder, because many disorders are not caused by microdeletions or microduplications. Other genetic testing may be warranted based on the clinical presentation.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><span class=\"nowrap\">Abnormal/positive</span> results allow a better understanding of the etiology of clinical features present in the fetus, but not necessarily an ability to predict the medical or developmental outcome of a child born with that genomic disorder. The couple should be told about:</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>The spectrum of medical and intellectual issues associated with the identified genomic disorder. </p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>The difficulty in predicting phenotype due to variable penetrance and expressivity.</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>How mutations arise (de novo versus inherited.</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Recurrence risks for future pregnancies. </p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Information about pregnancy termination. (See <a href=\"topic.htm?path=first-trimester-pregnancy-termination-uterine-aspiration\" class=\"medical medical_review\">&quot;First-trimester pregnancy termination: Uterine aspiration&quot;</a> and <a href=\"topic.htm?path=overview-of-second-trimester-pregnancy-termination\" class=\"medical medical_review\">&quot;Overview of second-trimester pregnancy termination&quot;</a>.)</p><p/><p class=\"headingAnchor\" id=\"H12423107\"><span class=\"h1\">SUMMARY AND RECOMMENDATIONS</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Prenatal chromosomal microarray (CMA) analysis is usually performed on DNA from uncultured amniocytes (amniotic fluid) or chorionic villus cells. (See <a href=\"#H4160558480\" class=\"local\">'Specimen'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Prenatal CMA provides most of the information derived from conventional G-banded chromosome analysis (ie, <span class=\"nowrap\">presence/absence</span> of aneuploidy, unbalanced structural changes, and triploidy), and also detects submicroscopic deletions and duplications (ie, copy number variants). (See <a href=\"#H18810847\" class=\"local\">'Introduction'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Prenatal CMA routinely includes copy number plus single nucleotide polymorphism (SNP) probes, facilitating detection of additional genomic events. The inclusion of SNP probes allows detection of runs of homozygosity between two copies of the same chromosome, thus enabling diagnosis of some cases of uniparental disomy and consanguinity. CMA with SNP can detect both triploidy and low levels of mosaicism as reliably as a karyotype. (See <a href=\"#H4180752565\" class=\"local\">'Platform'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Prenatal CMA may assess the whole genome or may be targeted to evaluate regions of the genome known or highly suspected to cause specific phenotypes when deleted or duplicated. (See <a href=\"#H2857363707\" class=\"local\">'Targeted versus whole genome'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Balanced rearrangements are not detectable by CMA; however, these represent a smaller number of cases as compared to those with genomic imbalance. (See <a href=\"#H18812086\" class=\"local\">'Inability to detect balanced structural rearrangements'</a> above.) </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Advantages of chromosomal microarray (CMA) over conventional G-banding in the prenatal setting include higher diagnostic yield, faster turnaround time, and ability to perform the test without requiring cell culture. (See <a href=\"#H18811338\" class=\"local\">'Benefits'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>CMA detects variants of unknown significance in about 1 to 2 percent of cases. Counseling patients with these findings can be difficult. (See <a href=\"#H1897120\" class=\"local\">'Detection of variants of uncertain significance'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In the obstetrical setting, CMA is a preferred option for further evaluation of fetuses with structural abnormalities, after fetal demise (particularly when chromosomal analysis is desired but G-banding is not possible due to failure of cell culture), and when a marker chromosome is identified. It is equivalent to G-banding when the primary goal is to detect aneuploidy, as in Down syndrome screening. Some clinicians have advocated microarray as a first-line test whenever fetal chromosomal analysis is planned. (See <a href=\"#H1897463\" class=\"local\">'Clinical indications for CMA in obstetrics'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Genetic counseling by a qualified provider should always be offered before and after prenatal CMA. Pre-test counseling should include a discussion of the medical and psychological risks and the advantages and disadvantages of this approach compared with conventional G-banding. The genetic principles of variants of uncertain significance and variable expressivity and penetrance should be explained. Post-test counseling should include interpretation of the findings and an explanation of possible follow-up studies. (See <a href=\"#H12423100\" class=\"local\">'Patient counseling'</a> above.)</p><p/></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/1\" class=\"nounderline abstract_t\">Rauch A, R&uuml;schendorf F, Huang J, et al. Molecular karyotyping using an SNP array for genomewide genotyping. J Med Genet 2004; 41:916.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/2\" class=\"nounderline abstract_t\">de Vries BB, Pfundt R, Leisink M, et al. Diagnostic genome profiling in mental retardation. Am J Hum Genet 2005; 77:606.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/3\" class=\"nounderline abstract_t\">Hochstenbach R, Ploos van Amstel HK, Poot M. Microarray-based genome investigation: molecular karyotyping or segmental aneuploidy profiling? Eur J Hum Genet 2006; 14:262.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/4\" class=\"nounderline abstract_t\">Vermeesch JR, Rauch A. Reply to Hochstenbach et al. 'Molecular karyotyping'. Eur J Hum Genet 2006; 14:1063.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/5\" class=\"nounderline abstract_t\">Vermeesch JR, Fiegler H, de Leeuw N, et al. Guidelines for molecular karyotyping in constitutional genetic diagnosis. Eur J Hum Genet 2007; 15:1105.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/6\" class=\"nounderline abstract_t\">Hoyer J, Dreweke A, Becker C, et al. Molecular karyotyping in patients with mental retardation using 100K single-nucleotide polymorphism arrays. J Med Genet 2007; 44:629.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/7\" class=\"nounderline abstract_t\">Wou K, Levy B, Wapner RJ. Chromosomal Microarrays for the Prenatal Detection of Microdeletions and Microduplications. Clin Lab Med 2016; 36:261.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/8\" class=\"nounderline abstract_t\">Brady PD, Vermeesch JR. Genomic microarrays: a technology overview. Prenat Diagn 2012; 32:336.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/9\" class=\"nounderline abstract_t\">Lo KK, Karampetsou E, Boustred C, et al. Limited Clinical Utility of Non-invasive Prenatal Testing for Subchromosomal Abnormalities. Am J Hum Genet 2016; 98:34.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/10\" class=\"nounderline abstract_t\">Kalia SS, Adelman K, Bale SJ, et al. Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2016 update (ACMG SF v2.0): a policy statement of the American College of Medical Genetics and Genomics. Genet Med 2017; 19:249.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/11\" class=\"nounderline abstract_t\">Committee on Genetics and the Society for Maternal-Fetal Medicine. Committee Opinion No.682: Microarrays and Next-Generation Sequencing Technology: The Use of Advanced Genetic Diagnostic Tools in Obstetrics and Gynecology. Obstet Gynecol 2016; 128:e262.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/12\" class=\"nounderline abstract_t\">Wapner RJ, Martin CL, Levy B, et al. Chromosomal microarray versus karyotyping for prenatal diagnosis. N Engl J Med 2012; 367:2175.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/13\" class=\"nounderline abstract_t\">Reddy UM, Page GP, Saade GR, et al. Karyotype versus microarray testing for genetic abnormalities after stillbirth. N Engl J Med 2012; 367:2185.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/14\" class=\"nounderline abstract_t\">Shaffer LG, Dabell MP, Fisher AJ, et al. Experience with microarray-based comparative genomic hybridization for prenatal diagnosis in over 5000 pregnancies. Prenat Diagn 2012; 32:976.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/15\" class=\"nounderline abstract_t\">Shaffer LG, Rosenfeld JA, Dabell MP, et al. Detection rates of clinically significant genomic alterations by microarray analysis for specific anomalies detected by ultrasound. Prenat Diagn 2012; 32:986.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/16\" class=\"nounderline abstract_t\">Scott F, Murphy K, Carey L, et al. Prenatal diagnosis using combined quantitative fluorescent polymerase chain reaction and array comparative genomic hybridization analysis as a first-line test: results from over 1000 consecutive cases. Ultrasound Obstet Gynecol 2013; 41:500.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/17\" class=\"nounderline abstract_t\">Callaway JL, Shaffer LG, Chitty LS, et al. The clinical utility of microarray technologies applied to prenatal cytogenetics in the presence of a normal conventional karyotype: a review of the literature. Prenat Diagn 2013; 33:1119.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/18\" class=\"nounderline abstract_t\">de Wit MC, Srebniak MI, Govaerts LC, et al. Additional value of prenatal genomic array testing in fetuses with isolated structural ultrasound abnormalities and a normal karyotype: a systematic review of the literature. Ultrasound Obstet Gynecol 2014; 43:139.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/19\" class=\"nounderline abstract_t\">Giardino D, Corti C, Ballarati L, et al. De novo balanced chromosome rearrangements in prenatal diagnosis. Prenat Diagn 2009; 29:257.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/20\" class=\"nounderline abstract_t\">Warburton D. De novo balanced chromosome rearrangements and extra marker chromosomes identified at prenatal diagnosis: clinical significance and distribution of breakpoints. Am J Hum Genet 1991; 49:995.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/21\" class=\"nounderline abstract_t\">Baptista J, Prigmore E, Gribble SM, et al. Molecular cytogenetic analyses of breakpoints in apparently balanced reciprocal translocations carried by phenotypically normal individuals. Eur J Hum Genet 2005; 13:1205.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/22\" class=\"nounderline abstract_t\">Savage MS, Mourad MJ, Wapner RJ. Evolving applications of microarray analysis in prenatal diagnosis. Curr Opin Obstet Gynecol 2011; 23:103.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/23\" class=\"nounderline abstract_t\">Gribble SM, Prigmore E, Burford DC, et al. The complex nature of constitutional de novo apparently balanced translocations in patients presenting with abnormal phenotypes. J Med Genet 2005; 42:8.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/24\" class=\"nounderline abstract_t\">De Gregori M, Ciccone R, Magini P, et al. Cryptic deletions are a common finding in &quot;balanced&quot; reciprocal and complex chromosome rearrangements: a study of 59 patients. J Med Genet 2007; 44:750.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/25\" class=\"nounderline abstract_t\">Feenstra I, Hanemaaijer N, Sikkema-Raddatz B, et al. Balanced into array: genome-wide array analysis in 54 patients with an apparently balanced de novo chromosome rearrangement and a meta-analysis. Eur J Hum Genet 2011; 19:1152.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/26\" class=\"nounderline abstract_t\">Hillman SC, Pretlove S, Coomarasamy A, et al. Additional information from array comparative genomic hybridization technology over conventional karyotyping in prenatal diagnosis: a systematic review and meta-analysis. Ultrasound Obstet Gynecol 2011; 37:6.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/27\" class=\"nounderline abstract_t\">Breman A, Pursley AN, Hixson P, et al. Prenatal chromosomal microarray analysis in a diagnostic laboratory; experience with &gt;1000 cases and review of the literature. Prenat Diagn 2012; 32:351.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/28\" class=\"nounderline abstract_t\">McGillivray G, Rosenfeld JA, McKinlay Gardner RJ, Gillam LH. Genetic counselling and ethical issues with chromosome microarray analysis in prenatal testing. Prenat Diagn 2012; 32:389.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/29\" class=\"nounderline abstract_t\">Lee CN, Lin SY, Lin CH, et al. Clinical utility of array comparative genomic hybridisation for prenatal diagnosis: a cohort study of 3171 pregnancies. BJOG 2012; 119:614.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/30\" class=\"nounderline abstract_t\">Duncan A, Langlois S, SOGC Genetics Committee, CCMG Prenatal Diagnosis Committee. Use of array genomic hybridization technology in prenatal diagnosis in Canada. J Obstet Gynaecol Can 2011; 33:1256.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/31\" class=\"nounderline abstract_t\">Novelli A, Grati FR, Ballarati L, et al. Microarray application in prenatal diagnosis: a position statement from the cytogenetics working group of the Italian Society of Human Genetics (SIGU), November 2011. Ultrasound Obstet Gynecol 2012; 39:384.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/32\" class=\"nounderline abstract_t\">Society for Maternal-Fetal Medicine (SMFM). Electronic address: pubs@smfm.org, Dugoff L, Norton ME, Kuller JA. The use of chromosomal microarray for&nbsp;prenatal&nbsp;diagnosis. Am J Obstet Gynecol 2016; 215:B2.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/33\" class=\"nounderline abstract_t\">Fiorentino F, Napoletano S, Caiazzo F, et al. Chromosomal microarray analysis as a first-line test in pregnancies with a priori low risk for the detection of submicroscopic chromosomal abnormalities. Eur J Hum Genet 2013; 21:725.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/34\" class=\"nounderline abstract_t\">Borrell A, Grande M, Pauta M, et al. Chromosomal Microarray Analysis in Fetuses with Growth Restriction and Normal Karyotype: A Systematic Review and Meta-Analysis. Fetal Diagn Ther 2017.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/35\" class=\"nounderline abstract_t\">Manning M, Hudgins L, Professional Practice and Guidelines Committee. Array-based technology and recommendations for utilization in medical genetics practice for detection of chromosomal abnormalities. Genet Med 2010; 12:742.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/36\" class=\"nounderline abstract_t\">Nicolaides KH, Snijders RJ, Gosden CM, et al. Ultrasonographically detectable markers of fetal chromosomal abnormalities. Lancet 1992; 340:704.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/37\" class=\"nounderline abstract_t\">Hillman SC, McMullan DJ, Hall G, et al. Use of prenatal chromosomal microarray: prospective cohort study and systematic review and meta-analysis. Ultrasound Obstet Gynecol 2013; 41:610.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/38\" class=\"nounderline abstract_t\">Sahoo T, Cheung SW, Ward P, et al. Prenatal diagnosis of chromosomal abnormalities using array-based comparative genomic hybridization. Genet Med 2006; 8:719.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/39\" class=\"nounderline abstract_t\">Shaffer LG, Coppinger J, Alliman S, et al. Comparison of microarray-based detection rates for cytogenetic abnormalities in prenatal and neonatal specimens. Prenat Diagn 2008; 28:789.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/40\" class=\"nounderline abstract_t\">Kleeman L, Bianchi DW, Shaffer LG, et al. Use of array comparative genomic hybridization for prenatal diagnosis of fetuses with sonographic anomalies and normal metaphase karyotype. Prenat Diagn 2009; 29:1213.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/41\" class=\"nounderline abstract_t\">Lu XY, Phung MT, Shaw CA, et al. Genomic imbalances in neonates with birth defects: high detection rates by using chromosomal microarray analysis. Pediatrics 2008; 122:1310.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/42\" class=\"nounderline abstract_t\">Goemaere N, Douben H, Van Opstal D, et al. The use of comparative genomic hybridization and fluorescent in situ hybridization in postmortem pathology investigation of congenital malformations. Pediatr Dev Pathol 2010; 13:85.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/43\" class=\"nounderline abstract_t\">Jansen FA, Blumenfeld YJ, Fisher A, et al. Array comparative genomic hybridization and fetal congenital heart defects: a systematic review and meta-analysis. Ultrasound Obstet Gynecol 2015; 45:27.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/44\" class=\"nounderline abstract_t\">Warburton D, Susser M, Stein Z, Kline J. Genetic and epidemiologic investigation of spontaneous abortion: relevance to clinical practice. Birth Defects Orig Artic Ser 1979; 15:127.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/45\" class=\"nounderline abstract_t\">Reddy UM, Page GP, Saade GR. The role of DNA microarrays in the evaluation of fetal death. Prenat Diagn 2012; 32:371.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/46\" class=\"nounderline abstract_t\">Dhillon RK, Hillman SC, Morris RK, et al. Additional information from chromosomal microarray analysis (CMA) over conventional karyotyping when diagnosing chromosomal abnormalities in miscarriage: a systematic review and meta-analysis. BJOG 2014; 121:11.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/47\" class=\"nounderline abstract_t\">Rajcan-Separovic E. Chromosome microarrays in human reproduction. Hum Reprod Update 2012; 18:555.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/48\" class=\"nounderline abstract_t\">Yatsenko SA, Peters DG, Saller DN, et al. Maternal cell-free DNA-based screening for fetal microdeletion and the importance of careful diagnostic follow-up. Genet Med 2015; 17:836.</a></li><li><a href=\"https://www.uptodate.com/contents/use-of-chromosomal-microarray-in-obstetrics/abstract/49\" class=\"nounderline abstract_t\">Wapner RJ, Driscoll DA, Simpson JL. Integration of microarray technology into prenatal diagnosis: counselling issues generated during the NICHD clinical trial. Prenat Diagn 2012; 32:396.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 16683 Version 23.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H12423107\"><span>SUMMARY &amp; RECOMMENDATIONS</span></a></li><li><a href=\"#H18810847\" id=\"outline-link-H18810847\">INTRODUCTION</a></li><li><a href=\"#H763272546\" id=\"outline-link-H763272546\">PROCEDURES</a><ul><li><a href=\"#H4160558480\" id=\"outline-link-H4160558480\">Specimen</a></li><li><a href=\"#H1997296363\" id=\"outline-link-H1997296363\">Types of microarray</a><ul><li><a href=\"#H2857363707\" id=\"outline-link-H2857363707\">- Targeted versus whole genome</a></li><li><a href=\"#H4180752565\" id=\"outline-link-H4180752565\">- Platform</a></li></ul></li></ul></li><li><a href=\"#H12423065\" id=\"outline-link-H12423065\">BENEFITS AND LIMITATIONS OF CMA</a><ul><li><a href=\"#H18811338\" id=\"outline-link-H18811338\">Benefits</a><ul><li><a href=\"#H18811391\" id=\"outline-link-H18811391\">- Higher diagnostic yield</a></li><li><a href=\"#H2514550684\" id=\"outline-link-H2514550684\">- No requirement for cell culture</a></li><li><a href=\"#H18811433\" id=\"outline-link-H18811433\">- Faster turnaround time</a></li><li><a href=\"#H2325619208\" id=\"outline-link-H2325619208\">- Detection of consanguinity and other familial relationships</a></li></ul></li><li><a href=\"#H18812072\" id=\"outline-link-H18812072\">Limitations</a><ul><li><a href=\"#H18812086\" id=\"outline-link-H18812086\">- Inability to detect balanced structural rearrangements</a></li><li><a href=\"#H70778136\" id=\"outline-link-H70778136\">- Inability to detect disorders associated with single gene point mutations</a></li><li><a href=\"#H2388440353\" id=\"outline-link-H2388440353\">- Inability to detect all cases of uniparental disomy</a></li><li><a href=\"#H3735402877\" id=\"outline-link-H3735402877\">- Other nondetectable events</a></li><li><a href=\"#H1897120\" id=\"outline-link-H1897120\">- Detection of variants of uncertain significance</a></li><li><a href=\"#H2963146976\" id=\"outline-link-H2963146976\">- Cost</a></li><li><a href=\"#H16315619\" id=\"outline-link-H16315619\">- Variable clinical sensitivity</a></li></ul></li></ul></li><li><a href=\"#H1897463\" id=\"outline-link-H1897463\">CLINICAL INDICATIONS FOR CMA IN OBSTETRICS</a><ul><li><a href=\"#H12423086\" id=\"outline-link-H12423086\">Prenatal ultrasound showing fetal structural abnormalities</a></li><li><a href=\"#H12423093\" id=\"outline-link-H12423093\">Fetal demise</a></li><li><a href=\"#H1897951\" id=\"outline-link-H1897951\">Marker chromosome</a></li></ul></li><li><a href=\"#H1897960\" id=\"outline-link-H1897960\">POSSIBLE INDICATIONS FOR PRENATAL CMA</a></li><li><a href=\"#H12423100\" id=\"outline-link-H12423100\">PATIENT COUNSELING</a></li><li><a href=\"#H12423107\" id=\"outline-link-H12423107\">SUMMARY AND RECOMMENDATIONS</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>GRAPHICS <a href=\"#\" id=\"viewAllGraphicsLink\">View All</a></h2><div id=\"outlineGraphics\"><ul><li><div id=\"OBGYN/16683|FIG\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"FIGURES\">FIGURES</a></div><ul><li><a href=\"image.htm?imageKey=PC/58888\" class=\"graphic graphic_figure\">- Schematic CGH for microarray setup</a></li></ul></li><li><div id=\"OBGYN/16683|TAB\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"TABLES\">TABLES</a></div><ul><li><a href=\"image.htm?imageKey=OBGYN/109837\" class=\"graphic graphic_table\">- Comparison of prenatal genetic testing methods</a></li></ul></li></ul></div></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=autism-spectrum-disorder-diagnosis\" class=\"medical medical_review\">Autism spectrum disorder: Diagnosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=basic-principles-of-genetic-disease\" class=\"medical medical_review\">Basic principles of genetic disease</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=birth-defects-approach-to-evaluation\" class=\"medical medical_review\">Birth defects: Approach to evaluation</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=congenital-cytogenetic-abnormalities\" class=\"medical medical_review\">Congenital cytogenetic abnormalities</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=cystic-hygroma-and-increased-nuchal-translucency\" class=\"medical medical_review\">Cystic hygroma and increased nuchal translucency</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=down-syndrome-overview-of-prenatal-screening\" class=\"medical medical_review\">Down syndrome: Overview of prenatal screening</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=first-trimester-pregnancy-termination-uterine-aspiration\" class=\"medical medical_review\">First-trimester pregnancy termination: Uterine aspiration</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genomic-disorders-an-overview\" class=\"medical medical_review\">Genomic disorders: An overview</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=intellectual-disability-in-children-evaluation-for-a-cause\" class=\"medical medical_review\">Intellectual disability in children: Evaluation for a cause</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=overview-of-genetic-variation\" class=\"medical medical_review\">Overview of genetic variation</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=overview-of-second-trimester-pregnancy-termination\" class=\"medical medical_review\">Overview of second-trimester pregnancy termination</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=principles-of-molecular-genetics\" class=\"medical medical_review\">Principles of molecular genetics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=sonographic-findings-associated-with-fetal-aneuploidy\" class=\"medical medical_review\">Sonographic findings associated with fetal aneuploidy</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">Tools for genetics and genomics: Cytogenetics and molecular genetics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-gene-expression-profiling\" class=\"medical medical_review\">Tools for genetics and genomics: Gene expression profiling</a></li></ul></div></div>","javascript":null}