var data={"title":"Cholera: Microbiology and pathogenesis","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Cholera: Microbiology and pathogenesis</div><dl id=\"topicContributors\"><dt><span> </span>Authors:</dt><dd><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/contributors\" class=\"contributor contributor_credentials\">Regina LaRocque, MD, MPH</a></dd><dd><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/contributors\" class=\"contributor contributor_credentials\">Jason B Harris, MD, MPH</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/contributors\" class=\"contributor contributor_credentials\">Stephen B Calderwood, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/contributors\" class=\"contributor contributor_credentials\">Allyson Bloom, MD</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Oct 06, 2016.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H1\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Cholera is a rapidly dehydrating diarrheal disease caused by a toxin-producing bacteria, <em>Vibrio cholerae. </em></p><p>The etiologic agent and pathogenesis of infection with toxigenic <em>V. cholerae</em> is reviewed here. The clinical approach to patients with cholera is discussed separately. (See <a href=\"topic.htm?path=cholera-clinical-features-diagnosis-treatment-and-prevention\" class=\"medical medical_review\">&quot;Cholera: Clinical features, diagnosis, treatment, and prevention&quot;</a>.)</p><p>Infections due to other strains of <em>V. cholerae</em> that do not cause epidemic cholera, are also discussed elsewhere. (See <a href=\"topic.htm?path=infections-due-to-non-o1-o139-vibrio-cholerae\" class=\"medical medical_review\">&quot;Infections due to non-O1/O139 Vibrio cholerae&quot;</a>.) </p><p class=\"headingAnchor\" id=\"H2795158396\"><span class=\"h1\">MICROBIOLOGY</span></p><p class=\"headingAnchor\" id=\"H468406280\"><span class=\"h2\">Etiologic agent</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span><em>V. cholerae</em> is a distinctive, comma-shaped gram-negative rod. Organisms are highly motile and possess a single polar flagellum. <em>V. cholerae</em> is salt-tolerant, requiring NaCl for growth (halophilic) and exists naturally in aquatic environments. While in aquatic environments, <em>V. cholerae</em> may enter a viable but non-culturable form [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/1\" class=\"abstract_t\">1</a>]. However, <em>V. cholerae</em> is readily grown from clinical specimens, including stool and rectal swabs, and can be identified in microbiology laboratories using selective media and biochemical tests. (See <a href=\"topic.htm?path=cholera-clinical-features-diagnosis-treatment-and-prevention#H938291637\" class=\"medical medical_review\">&quot;Cholera: Clinical features, diagnosis, treatment, and prevention&quot;, section on 'Diagnosis'</a>.) </p><p>Only cholera toxin-producing (toxigenic) strains of <em>V. cholerae </em>are associated with cholera. While some environmental <em>V. cholerae</em> are toxigenic and capable of causing cholera, most environmental <em>V. cholerae </em>isolates are not toxigenic. Toxigenic strains harbor a filamentous bacteriophage (CTX&Phi;) which encodes cholera toxin [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/2\" class=\"abstract_t\">2</a>]. (See <a href=\"#H1523432749\" class=\"local\">'Genomic features'</a> below.)</p><p class=\"headingAnchor\" id=\"H2754921559\"><span class=\"h2\">Classification</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span><em>V. cholerae </em>is a diverse species and includes pathogenic and non-pathogenic variants. The organism can be classified serologically based on differences in the structure of the O antigen of its lipopolysaccharide. More than 200 serogroups of <em>V. cholerae</em> have been reported. However, only serogroups O1 and O139 have been associated with large scale epidemics of cholera [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/3\" class=\"abstract_t\">3</a>]. Other serogroups are generally grouped together as <em>V. cholerae</em> &quot;non-O1, non-O139.&quot; Toxin-producing <span class=\"nowrap\">non-O1/O139</span> <em>V. cholerae</em> strains have been associated with isolated cases or small outbreaks of dehydrating diarrhea, whereas non-toxin-producing <span class=\"nowrap\">non-O1/O139</span> strains have been associated with sporadic gastroenteritis and sepsis. (See <a href=\"topic.htm?path=infections-due-to-non-o1-o139-vibrio-cholerae\" class=\"medical medical_review\">&quot;Infections due to non-O1/O139 Vibrio cholerae&quot;</a>.) </p><p class=\"headingAnchor\" id=\"H2695256845\"><span class=\"h3\">Serogroup O1</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span><em>V. cholerae </em>O1 is the cause of the current global pandemic of cholera and is divided into two major serotypes, Inaba and Ogawa, which differ by the presence of a single methyl group on the Ogawa O-specific polysaccharide antigen. The change between serotypes Ogawa and Inaba arises from mutations in the WbeT methyltransferase [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/4\" class=\"abstract_t\">4</a>].</p><p><em>V. cholerae</em> O1 is also divided into two biotypes, El Tor and classical, which are differentiated by biochemical distinctions and susceptibility to specific bacteriophages. Previous <em>V. cholerae</em> pandemics were caused by the classical biotype, but this is now thought to be extinct. In conventional microbiologic tests, classical strains of <em>V. cholerae</em> O1 are sensitive to <a href=\"topic.htm?path=polymyxin-b-drug-information\" class=\"drug drug_general\">polymyxin B</a>, have a negative Voges-Proskauer test (which detects the metabolic product acetoin), and do not agglutinate chicken red blood cells.</p><p class=\"headingAnchor\" id=\"H3090572465\"><span class=\"h3\">Serogroup O139</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span><em>V. cholerae</em> O139 emerged in 1992 and was a major cause of epidemic cholera for a decade, but is no longer a major cause of cholera. The epidemic O139 strain arose via the acquisition of a single genomic island, the rfb region, which included the genes required for synthesis of the O139 antigen but the organism was otherwise identical to previously circulating <em>V. cholerae</em> O1 strains [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/5,6\" class=\"abstract_t\">5,6</a>].</p><p class=\"headingAnchor\" id=\"H1523432749\"><span class=\"h2\">Genomic features</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The genome of <em>V. cholerae</em> and other <em>Vibrio</em> species contains two chromosomes. This feature is distinct from many other bacteria. The first <em>V. cholerae</em> reference genome was derived from a strain, N16961, a <em>V. cholerae</em> O1 El Tor strain that was isolated from a cholera patient in Bangladesh in 1971 [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/7\" class=\"abstract_t\">7</a>]. <em>V. cholerae</em> can evolve rapidly through acquisition of mobile genetic elements. Notable features of the <em>V. cholerae</em> genome include:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>CTX&Phi; &ndash; </strong>Toxin-producing strains of <em>V. cholerae</em> contain the bacteriophage CTX&Phi;. (A lysogenic bacteriophage is a virus that infects bacteria<em> </em>and inserts its genetic material into the host bacterium's genome.) This bacteriophage can be transmitted between bacteria and includes the genes that encode cholera toxin [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/2\" class=\"abstract_t\">2</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong>VPI &ndash; </strong>The <em>Vibrio</em> Pathogenicity Island (VPI) [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/8\" class=\"abstract_t\">8</a>] contains the genes that encode the toxin coregulated pilus (TCP), a specialized bacterial structure in pathogenic strains that allows colonization of the human intestine [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/9\" class=\"abstract_t\">9</a>]. The production of TCP is coordinated with production of cholera toxin [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/10\" class=\"abstract_t\">10</a>]. TCP is also the receptor for the bacteriophage CTX&Phi; [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/2\" class=\"abstract_t\">2</a>]. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong><span class=\"nowrap\">SXT/R391</span> ICE</strong> &ndash; The <span class=\"nowrap\">SXT/R391</span> integrative and conjugative element (ICE) is an important part of the genome that allows <em>V. cholerae</em> to acquire certain types of foreign DNA and has allowed the bacteria to acquire antibiotic resistance phenotypes [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/11\" class=\"abstract_t\">11</a>]. It was first identified in <em>V. cholerae</em> O139 [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/11\" class=\"abstract_t\">11</a>]. Subsequently, this horizontally-acquired region was found in pandemic <em>V. cholerae </em>O1 El Tor. </p><p/><p class=\"headingAnchor\" id=\"H1816296437\"><span class=\"h2\">Molecular epidemiology</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The use of whole genome sequencing and single nucleotide polymorphisms (SNPs) as high resolution markers has allowed for a better understanding of the evolution of <em>V. cholerae</em>. Over the last two centuries, there have been seven pandemics of <em>V. cholerae</em> [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/3\" class=\"abstract_t\">3</a>]<em>.</em> <em>V. cholerae</em> O1 El Tor is the cause of the current, seventh pandemic. A study of 154 representative isolates confirmed that this pandemic represents the expansion of a single bacterial lineage with a most recent common ancestral clone estimated to date to 1952 [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/5\" class=\"abstract_t\">5</a>]. This analysis also showed that the current global cholera pandemic has actually been comprised of three independent waves of transmission, each coinciding with a major acquisition of new genetic material, which conferred increased environmental stability, drug resistance, or virulence to the previously circulating strains of <em>V. cholerae </em>[<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/5\" class=\"abstract_t\">5</a>]: </p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The current cholera pandemic began with the emergence of <em>V. cholerae</em> O1 biotype El Tor, which replaced the previously circulating classical biotype, leading to the apparent extinction of the classical biotype [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/5\" class=\"abstract_t\">5</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A second wave occurred when the original pandemic<em> V. cholerae</em> O1 El Tor strains were replaced by <em>V. cholerae </em>O1 isolates containing the <span class=\"nowrap\">SXT/R391</span> antibiotic resistance element. <em>V. cholerae</em> O139 strains arose from and are included in this second wave.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The third wave, in which the most recent common ancestral clone is estimated to have originated in 1988, was associated with acquisition of a cholera toxin variant that shared characteristics of a cholera toxin sequence previously observed in the now extinct classical biotype.</p><p/><p>Each wave of transmission originated in South Asia before spreading to distant locations throughout Asia, Africa, South America, and the Caribbean [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/5\" class=\"abstract_t\">5</a>]. </p><p>Similar genomic methods have also been applied to assess the origins of major cholera epidemics. One analysis demonstrated that the origin of cholera in Haiti could be traced to the introduction of a single strain that had been most recently been circulating in South Asia, counter to the idea that the epidemic had originated from a local environmental strain [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/12\" class=\"abstract_t\">12</a>]. Additional analyses have since confirmed that isolates from Haiti are virtually identical to those from Nepal [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/13\" class=\"abstract_t\">13</a>], a finding consistent with epidemiologic evidence linking the first cases in Haiti to an area contaminated with raw sewage originating from Nepalese stabilization forces at a United Nations base [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/13\" class=\"abstract_t\">13</a>]. </p><p class=\"headingAnchor\" id=\"H2476010745\"><span class=\"h1\">PATHOGENESIS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span><em>V. cholerae </em>may cause asymptomatic or mild infection. However, <em>V. cholerae</em> may also cause a uniquely severe secretory diarrhea that can be fatal in less than 12 hours [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/14\" class=\"abstract_t\">14</a>]. The infectious dose of <em>V. cholerae</em> required to cause cholera is thought to be relatively high; an inoculum of 10<sup>8</sup> <em>V. cholerae</em> or higher generally results in severe infection in healthy North American volunteers<sup> </sup>[<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/15,16\" class=\"abstract_t\">15,16</a>]. </p><p>In order for infection to occur, <em>V. cholerae</em> must survive the acidic environment of the stomach and then form colonies of bacteria on the surface of the small intestine. Organisms attached to the small intestine produce cholera toxin (CT), which is responsible for the profound secretory diarrhea that is the hallmark of cholera. As little as 5 micrograms of ingested cholera toxin may mimic a cholera-like illness [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/17\" class=\"abstract_t\">17</a>]. (See <a href=\"#H209077641\" class=\"local\">'Colonization'</a> below and <a href=\"#H3284679111\" class=\"local\">'Cholera toxin'</a> below.)</p><p>Previous exposure and immunity, nutrition, genetic factors, and interaction with other microbes may influence human susceptibility to cholera. (See <a href=\"#H4004052879\" class=\"local\">'Host susceptibility'</a> below and <a href=\"#H505715236\" class=\"local\">'Interactions with other microbes'</a> below.)</p><p class=\"headingAnchor\" id=\"H209077641\"><span class=\"h2\">Colonization</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>After ingestion of <em>V. cholerae </em>and passage through the upper gastrointestinal tract, the organisms colonize the small intestine. An important factor for colonization is the toxin-coregulated pilus (TCP), which is a rope-like structure produced by toxigenic <em>V. cholerae </em>[<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/9\" class=\"abstract_t\">9</a>]. TCP also leads to bacterial aggregation and likely allows the bacteria to resist killing by bile [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/18\" class=\"abstract_t\">18</a>]. Bacterial motility is another important factor for colonization, and a hallmark of <em>V. cholerae</em> is the rapid propulsion of the bacterial by a single flagellum which enables it<em> </em>to swim through mucous and arrive at the mucosal surface [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/19\" class=\"abstract_t\">19</a>]. </p><p class=\"headingAnchor\" id=\"H3284679111\"><span class=\"h2\">Cholera toxin</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>CT is the main virulence factor of pathogenic strains of <em>V. cholerae,</em> and is the primary cause of the characteristic profuse watery diarrhea of cholera. CT is an AB<sub>5</sub> protein toxin that consists of a single A subunit (CtxA) non-covalently attached to a donut-shaped ring of five B subunits (CtxB). Once it colonizes the epithelial surface, <em>V. cholerae </em>secretes CT. Post-translational modification by a protease nicks the CtxA subunit, yielding two CtxA portions that remain connected by a single disulfide bond [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/20\" class=\"abstract_t\">20</a>]. The CtxB pentamer attaches with high affinity to the apical surface of small intestinal epithelial cells via the monosialoganglioside GM<sub>1 </sub>receptor [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/21\" class=\"abstract_t\">21</a>]. The entire CT then undergoes endocytosis via a retrograde transport pathway and travels to the endoplasmic reticulum, where CtxA is cleaved into two polypeptide chains by reduction of a disulfide bond to yield CtxA<sub>1</sub> and CtxA<sub>2 </sub>peptides [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/22\" class=\"abstract_t\">22</a>]. The enzymatically active CtxA<sub>1</sub> is translocated to the cytosol, where it activates the G-protein regulated adenylyl cyclase [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/23\" class=\"abstract_t\">23</a>]. This increases concentrations of intracellular cyclic AMP, resulting in secretion of chloride via apical cystic fibrosis transmembrane conductance regulator (CFTR) channels and inhibition of sodium chloride absorption [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/24\" class=\"abstract_t\">24</a>]. The ultimate result is massive fluid secretion into the small intestine, resulting in large amounts of fluid losses with high concentrations of sodium, chloride, bicarbonate, and potassium [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/25\" class=\"abstract_t\">25</a>]. </p><p class=\"headingAnchor\" id=\"H3449103022\"><span class=\"h2\">Regulation of virulence</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Toxigenic <em>V. cholerae</em> coordinate the expression of virulence genes through a regulatory network known as the ToxR regulon [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/26\" class=\"abstract_t\">26</a>]. This network responds to and integrates complex environmental signals, which can include amino acid levels, bacterial cell density (through quorum sensing pathways), and other mechanisms. Downstream components of the ToxR regulon are controlled by ToxT, a transcriptional activator that directly activates transcription of genes encoding CT, TCP, and other virulence factors of toxigenic <em>V. cholerae </em>[<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/26\" class=\"abstract_t\">26</a>]<em>.</em></p><p class=\"headingAnchor\" id=\"H2917349817\"><span class=\"h2\">Hyperinfectivity</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span><em>V. cholerae</em> shed by humans with cholera is more than a hundred times more infective in mice compared with organisms grown in vitro [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/27,28\" class=\"abstract_t\">27,28</a>].<em> </em>This hyperinfectivity is transient and is lost in less than 24 hours after the bacteria is shed into the environment [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/29\" class=\"abstract_t\">29</a>]. However, epidemiologic models suggest that person-to-person spread by hyperinfective <em>V. cholerae</em> contributes to the explosive [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/30\" class=\"abstract_t\">30</a>] nature of cholera epidemics. Studies of <em>V. cholerae</em> shed in human stool suggest that the downregulation of chemotaxis genes may contribute to this hyperinfective phenotype [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/31\" class=\"abstract_t\">31</a>].</p><p class=\"headingAnchor\" id=\"H4004052879\"><span class=\"h2\">Host susceptibility</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Most risk factors for cholera reflect the underlying mode of transmission via contaminated food and water. However, a number of specific host factors associated with a higher risk of <em>V. cholerae </em>infection and cholera have been identified and reflect the biological interaction between the host and pathogen. </p><p>A major determinant of host susceptibility to cholera is immunity acquired through previous infection or cholera vaccination (see <a href=\"topic.htm?path=cholera-clinical-features-diagnosis-treatment-and-prevention#H930101666\" class=\"medical medical_review\">&quot;Cholera: Clinical features, diagnosis, treatment, and prevention&quot;, section on 'Vaccines'</a>). Both human challenge and longitudinal studies in endemic areas demonstrate that a single episode of cholera protects against subsequent infection [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/32,33\" class=\"abstract_t\">32,33</a>]. However, this protection is serogroup specific, and the level and duration of protection following <em>V. cholerae</em> O1 infection varies depending on the biotype and serotype of the organism [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/34\" class=\"abstract_t\">34</a>].</p><p>Individuals with blood group O are more likely to develop severe cholera than individuals with other blood groups [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/35,36\" class=\"abstract_t\">35,36</a>]. The mechanism for this is uncertain. In one cohort study from Bangladesh, blood group O was actually associated with a lower risk for <em>V. cholerae</em> infection overall despite the risk for higher severity of illness once infected, suggesting a susceptibility to the effects of cholera toxin rather than an increased susceptibility to colonization [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/36,37\" class=\"abstract_t\">36,37</a>].</p><p>Other genetic variation are also likely to affect the risk of developing cholera. As an example, a candidate gene study demonstrated that variants in the innate immunity protein BPIFB1 (also known as LPLUNC1) are associated with susceptibility to cholera [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/38\" class=\"abstract_t\">38</a>]. A genome-wide study of a population from a cholera-endemic region in Bangladesh found a number of genetic regions, including genes related to innate immunity and encoding potassium channels, to be associated with cholera<sup> </sup>[<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/39\" class=\"abstract_t\">39</a>]. </p><p>Other host factors also modulate susceptibility to infection. Ingested bacteria must survive the gastric acid barrier of the stomach in order to reach the small intestine. <em>V. cholerae</em> are sensitive to gastric acid, and hypochlorhydria lowers the infectious dose needed to cause infection [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/40\" class=\"abstract_t\">40</a>]. Among nutritional factors, retinol deficiency is associated with an increased risk of cholera, and breastfeeding has been consistently shown to be protective against cholera [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/41,42\" class=\"abstract_t\">41,42</a>].</p><p class=\"headingAnchor\" id=\"H505715236\"><span class=\"h2\">Interactions with other microbes</span></p><p class=\"headingAnchor\" id=\"H375671532\"><span class=\"h3\">Gut microbiota</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Metagenomic analysis during and after cholera demonstrates that the intestinal microbiota is drastically altered by cholera, and then undergoes a reproducible pattern of successive repopulation [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/43,44\" class=\"abstract_t\">43,44</a>]. This repopulation may mirror the accumulation of bacterial taxa of the gut microbiota in childhood [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/44\" class=\"abstract_t\">44</a>]. The pattern of microbial repopulation after cholera may be affected by changing enteric oxygen and carbohydrate levels after diarrheal illness [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/43\" class=\"abstract_t\">43</a>].</p><p class=\"headingAnchor\" id=\"H43640651\"><span class=\"h3\">Lytic bacteriophages</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Lytic bacteriophages are viruses that kill bacteria. A number of lytic phages that attack toxigenic <em>V. cholerae</em> have been identified [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/45\" class=\"abstract_t\">45</a>]. Evidence from environmental studies suggests that increasing amounts of lytic phages in the environment may coincide with the termination of cholera epidemics [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/46,47\" class=\"abstract_t\">46,47</a>]. Furthermore, the simultaneous ingestion of toxigenic <em>V. cholerae</em> and lytic phage has been shown to exert a strong selective pressure for the bacteria to develop mutations that may make it less virulent in order to resist killing by the bacteriophage [<a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/48\" class=\"abstract_t\">48</a>].</p><p class=\"headingAnchor\" id=\"H2807496372\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><em>Vibrio cholerae</em> is a facultative gram-negative rod found in aquatic environments. Only cholera toxin-producing (toxigenic) strains of <em>V. cholerae </em>are capable of causing cholera, a rapidly dehydrating diarrheal disease. (See <a href=\"#H468406280\" class=\"local\">'Etiologic agent'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><em>V. cholerae</em> is classified serologically, with more than 200 reported serogroups. Only serogroups O1 and O139 have been associated with large scale epidemics of cholera. <em>V. cholerae </em>O1 can be further classified by serotype or biotype. <em>V. cholerae</em> O1 biotype El tor is the cause of the current global pandemic of cholera. (See <a href=\"#H2754921559\" class=\"local\">'Classification'</a> above and <a href=\"#H1816296437\" class=\"local\">'Molecular epidemiology'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Horizontal gene transfer is the primary mechanism responsible for the evolution of pathogenic <em>V. cholerae</em>. (See <a href=\"#H1523432749\" class=\"local\">'Genomic features'</a> above and <a href=\"#H1816296437\" class=\"local\">'Molecular epidemiology'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In order for infection to occur, <em>V. cholerae</em> must survive the acidic environment of the stomach, colonize the small intestine, and produce cholera toxin, an AB<sub>5</sub> bacterial toxin that is the primary cause of the characteristic profuse watery diarrhea of cholera. (See <a href=\"#H2476010745\" class=\"local\">'Pathogenesis'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Previous exposure and immunity, nutrition, genetic factors, and interaction with other microbes may influence human susceptibility to cholera. (See <a href=\"#H4004052879\" class=\"local\">'Host susceptibility'</a> above and <a href=\"#H505715236\" class=\"local\">'Interactions with other microbes'</a> above.)</p><p/><p><strong>ACKNOWLEDGMENT</strong> &mdash; The editorial staff at UpToDate would like to acknowledge Joan R Butterton, MD, who contributed to an earlier version of this topic review.</p></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/1\" class=\"nounderline abstract_t\">Alam M, Sultana M, Nair GB, et al. Viable but nonculturable Vibrio cholerae O1 in biofilms in the aquatic environment and their role in cholera transmission. Proc Natl Acad Sci U S A 2007; 104:17801.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/2\" class=\"nounderline abstract_t\">Waldor MK, Mekalanos JJ. Lysogenic conversion by a filamentous phage encoding cholera toxin. Science 1996; 272:1910.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/3\" class=\"nounderline abstract_t\">Harris JB, LaRocque RC, Qadri F, et al. Cholera. Lancet 2012; 379:2466.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/4\" class=\"nounderline abstract_t\">Lebens M, Karlsson SL, K&auml;llg&aring;rd S, et al. Construction of novel vaccine strains of Vibrio cholerae co-expressing the Inaba and Ogawa serotype antigens. Vaccine 2011; 29:7505.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/5\" class=\"nounderline abstract_t\">Mutreja A, Kim DW, Thomson NR, et al. Evidence for several waves of global transmission in the seventh cholera pandemic. Nature 2011; 477:462.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/6\" class=\"nounderline abstract_t\">Waldor MK, Mekalanos JJ. Emergence of a new cholera pandemic: molecular analysis of virulence determinants in Vibrio cholerae O139 and development of a live vaccine prototype. J Infect Dis 1994; 170:278.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/7\" class=\"nounderline abstract_t\">Heidelberg JF, Eisen JA, Nelson WC, et al. DNA sequence of both chromosomes of the cholera pathogen Vibrio cholerae. Nature 2000; 406:477.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/8\" class=\"nounderline abstract_t\">Karaolis DK, Johnson JA, Bailey CC, et al. A Vibrio cholerae pathogenicity island associated with epidemic and pandemic strains. Proc Natl Acad Sci U S A 1998; 95:3134.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/9\" class=\"nounderline abstract_t\">Herrington DA, Hall RH, Losonsky G, et al. Toxin, toxin-coregulated pili, and the toxR regulon are essential for Vibrio cholerae pathogenesis in humans. J Exp Med 1988; 168:1487.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/10\" class=\"nounderline abstract_t\">Taylor RK, Miller VL, Furlong DB, Mekalanos JJ. Use of phoA gene fusions to identify a pilus colonization factor coordinately regulated with cholera toxin. Proc Natl Acad Sci U S A 1987; 84:2833.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/11\" class=\"nounderline abstract_t\">Waldor MK, Tsch&auml;pe H, Mekalanos JJ. A new type of conjugative transposon encodes resistance to sulfamethoxazole, trimethoprim, and streptomycin in Vibrio cholerae O139. J Bacteriol 1996; 178:4157.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/12\" class=\"nounderline abstract_t\">Chin CS, Sorenson J, Harris JB, et al. The origin of the Haitian cholera outbreak strain. N Engl J Med 2011; 364:33.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/13\" class=\"nounderline abstract_t\">Eppinger M, Pearson T, Koenig SS, et al. Genomic epidemiology of the Haitian cholera outbreak: a single introduction followed by rapid, extensive, and continued spread characterized the onset of the epidemic. MBio 2014; 5:e01721.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/14\" class=\"nounderline abstract_t\">Centers for Disease Control and Prevention (CDC). Update: outbreak of cholera ---Haiti, 2010. MMWR Morb Mortal Wkly Rep 2010; 59:1586.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/15\" class=\"nounderline abstract_t\">Levine MM, Kaper JB, Herrington D, et al. Safety, immunogenicity, and efficacy of recombinant live oral cholera vaccines, CVD 103 and CVD 103-HgR. Lancet 1988; 2:467.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/16\" class=\"nounderline abstract_t\">Nelson EJ, Harris JB, Morris JG Jr, et al. Cholera transmission: the host, pathogen and bacteriophage dynamic. Nat Rev Microbiol 2009; 7:693.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/17\" class=\"nounderline abstract_t\">Levine MM, Kaper JB, Black RE, Clements ML. New knowledge on pathogenesis of bacterial enteric infections as applied to vaccine development. Microbiol Rev 1983; 47:510.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/18\" class=\"nounderline abstract_t\">Krebs SJ, Taylor RK. Protection and attachment of Vibrio cholerae mediated by the toxin-coregulated pilus in the infant mouse model. J Bacteriol 2011; 193:5260.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/19\" class=\"nounderline abstract_t\">Bishop AL, Schild S, Patimalla B, et al. Mucosal immunization with Vibrio cholerae outer membrane vesicles provides maternal protection mediated by antilipopolysaccharide antibodies that inhibit bacterial motility. Infect Immun 2010; 78:4402.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/20\" class=\"nounderline abstract_t\">Gill DM. The arrangement of subunits in cholera toxin. Biochemistry 1976; 15:1242.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/21\" class=\"nounderline abstract_t\">Holmgren J, L&ouml;nnroth I, M&aring;nsson J, Svennerholm L. Interaction of cholera toxin and membrane GM1 ganglioside of small intestine. Proc Natl Acad Sci U S A 1975; 72:2520.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/22\" class=\"nounderline abstract_t\">Feng Y, Jadhav AP, Rodighiero C, et al. Retrograde transport of cholera toxin from the plasma membrane to the endoplasmic reticulum requires the trans-Golgi network but not the Golgi apparatus in Exo2-treated cells. EMBO Rep 2004; 5:596.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/23\" class=\"nounderline abstract_t\">Gill DM, Meren R. ADP-ribosylation of membrane proteins catalyzed by cholera toxin: basis of the activation of adenylate cyclase. Proc Natl Acad Sci U S A 1978; 75:3050.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/24\" class=\"nounderline abstract_t\">Ma T, Thiagarajah JR, Yang H, et al. Thiazolidinone CFTR inhibitor identified by high-throughput screening blocks cholera toxin-induced intestinal fluid secretion. J Clin Invest 2002; 110:1651.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/25\" class=\"nounderline abstract_t\">Molla AM, Rahman M, Sarker SA, et al. Stool electrolyte content and purging rates in diarrhea caused by rotavirus, enterotoxigenic E. coli, and V. cholerae in children. J Pediatr 1981; 98:835.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/26\" class=\"nounderline abstract_t\">DiRita VJ, Parsot C, Jander G, Mekalanos JJ. Regulatory cascade controls virulence in Vibrio cholerae. Proc Natl Acad Sci U S A 1991; 88:5403.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/27\" class=\"nounderline abstract_t\">Merrell DS, Butler SM, Qadri F, et al. Host-induced epidemic spread of the cholera bacterium. Nature 2002; 417:642.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/28\" class=\"nounderline abstract_t\">Merrell DS, Camilli A. The cadA gene of Vibrio cholerae is induced during infection and plays a role in acid tolerance. Mol Microbiol 1999; 34:836.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/29\" class=\"nounderline abstract_t\">Nelson EJ, Chowdhury A, Flynn J, et al. Transmission of Vibrio cholerae is antagonized by lytic phage and entry into the aquatic environment. PLoS Pathog 2008; 4:e1000187.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/30\" class=\"nounderline abstract_t\">Hartley DM, Morris JG Jr, Smith DL. Hyperinfectivity: a critical element in the ability of V. cholerae to cause epidemics? PLoS Med 2006; 3:e7.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/31\" class=\"nounderline abstract_t\">Butler SM, Nelson EJ, Chowdhury N, et al. Cholera stool bacteria repress chemotaxis to increase infectivity. Mol Microbiol 2006; 60:417.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/32\" class=\"nounderline abstract_t\">Levine MM, Black RE, Clements ML, et al. Duration of infection-derived immunity to cholera. J Infect Dis 1981; 143:818.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/33\" class=\"nounderline abstract_t\">Glass RI, Becker S, Huq MI, et al. Endemic cholera in rural Bangladesh, 1966-1980. Am J Epidemiol 1982; 116:959.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/34\" class=\"nounderline abstract_t\">Ali M, Emch M, Park JK, et al. Natural cholera infection-derived immunity in an endemic setting. J Infect Dis 2011; 204:912.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/35\" class=\"nounderline abstract_t\">Glass RI, Holmgren J, Haley CE, et al. Predisposition for cholera of individuals with O blood group. Possible evolutionary significance. Am J Epidemiol 1985; 121:791.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/36\" class=\"nounderline abstract_t\">Harris JB, Khan AI, LaRocque RC, et al. Blood group, immunity, and risk of infection with Vibrio cholerae in an area of endemicity. Infect Immun 2005; 73:7422.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/37\" class=\"nounderline abstract_t\">Kuhlmann FM, Santhanam S, Kumar P, et al. Blood Group O-Dependent Cellular Responses to Cholera Toxin: Parallel Clinical and Epidemiological Links to Severe Cholera. Am J Trop Med Hyg 2016; 95:440.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/38\" class=\"nounderline abstract_t\">Larocque RC, Sabeti P, Duggal P, et al. A variant in long palate, lung and nasal epithelium clone 1 is associated with cholera in a Bangladeshi population. Genes Immun 2009; 10:267.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/39\" class=\"nounderline abstract_t\">Karlsson EK, Harris JB, Tabrizi S, et al. Natural selection in a bangladeshi population from the cholera-endemic ganges river delta. Sci Transl Med 2013; 5:192ra86.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/40\" class=\"nounderline abstract_t\">Bavishi C, Dupont HL. Systematic review: the use of proton pump inhibitors and increased susceptibility to enteric infection. Aliment Pharmacol Ther 2011; 34:1269.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/41\" class=\"nounderline abstract_t\">Glass RI, Svennerholm AM, Stoll BJ, et al. Protection against cholera in breast-fed children by antibodies in breast milk. N Engl J Med 1983; 308:1389.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/42\" class=\"nounderline abstract_t\">Harris JB, LaRocque RC, Chowdhury F, et al. Susceptibility to Vibrio cholerae infection in a cohort of household contacts of patients with cholera in Bangladesh. PLoS Negl Trop Dis 2008; 2:e221.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/43\" class=\"nounderline abstract_t\">David LA, Weil A, Ryan ET, et al. Gut microbial succession follows acute secretory diarrhea in humans. MBio 2015; 6:e00381.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/44\" class=\"nounderline abstract_t\">Hsiao A, Ahmed AM, Subramanian S, et al. Members of the human gut microbiota involved in recovery from Vibrio cholerae infection. Nature 2014; 515:423.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/45\" class=\"nounderline abstract_t\">Seed KD, Bodi KL, Kropinski AM, et al. Evidence of a dominant lineage of Vibrio cholerae-specific lytic bacteriophages shed by cholera patients over a 10-year period in Dhaka, Bangladesh. MBio 2011; 2:e00334.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/46\" class=\"nounderline abstract_t\">Faruque SM, Islam MJ, Ahmad QS, et al. Self-limiting nature of seasonal cholera epidemics: Role of host-mediated amplification of phage. Proc Natl Acad Sci U S A 2005; 102:6119.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/47\" class=\"nounderline abstract_t\">Faruque SM, Naser IB, Islam MJ, et al. Seasonal epidemics of cholera inversely correlate with the prevalence of environmental cholera phages. Proc Natl Acad Sci U S A 2005; 102:1702.</a></li><li><a href=\"https://www.uptodate.com/contents/cholera-microbiology-and-pathogenesis/abstract/48\" class=\"nounderline abstract_t\">Seed KD, Yen M, Shapiro BJ, et al. Evolutionary consequences of intra-patient phage predation on microbial populations. Elife 2014; 3:e03497.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 2682 Version 13.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H2807496372\"><span>SUMMARY</span></a></li><li><a href=\"#H1\" id=\"outline-link-H1\">INTRODUCTION</a></li><li><a href=\"#H2795158396\" id=\"outline-link-H2795158396\">MICROBIOLOGY</a><ul><li><a href=\"#H468406280\" id=\"outline-link-H468406280\">Etiologic agent</a></li><li><a href=\"#H2754921559\" id=\"outline-link-H2754921559\">Classification</a><ul><li><a href=\"#H2695256845\" id=\"outline-link-H2695256845\">- Serogroup O1</a></li><li><a href=\"#H3090572465\" id=\"outline-link-H3090572465\">- Serogroup O139</a></li></ul></li><li><a href=\"#H1523432749\" id=\"outline-link-H1523432749\">Genomic features</a></li><li><a href=\"#H1816296437\" id=\"outline-link-H1816296437\">Molecular epidemiology</a></li></ul></li><li><a href=\"#H2476010745\" id=\"outline-link-H2476010745\">PATHOGENESIS</a><ul><li><a href=\"#H209077641\" id=\"outline-link-H209077641\">Colonization</a></li><li><a href=\"#H3284679111\" id=\"outline-link-H3284679111\">Cholera toxin</a></li><li><a href=\"#H3449103022\" id=\"outline-link-H3449103022\">Regulation of virulence</a></li><li><a href=\"#H2917349817\" id=\"outline-link-H2917349817\">Hyperinfectivity</a></li><li><a href=\"#H4004052879\" id=\"outline-link-H4004052879\">Host susceptibility</a></li><li><a href=\"#H505715236\" id=\"outline-link-H505715236\">Interactions with other microbes</a><ul><li><a href=\"#H375671532\" id=\"outline-link-H375671532\">- Gut microbiota</a></li><li><a href=\"#H43640651\" id=\"outline-link-H43640651\">- Lytic bacteriophages</a></li></ul></li></ul></li><li><a href=\"#H2807496372\" id=\"outline-link-H2807496372\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=cholera-clinical-features-diagnosis-treatment-and-prevention\" class=\"medical medical_review\">Cholera: Clinical features, diagnosis, treatment, and prevention</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=infections-due-to-non-o1-o139-vibrio-cholerae\" class=\"medical medical_review\">Infections due to non-O1/O139 Vibrio cholerae</a></li></ul></div></div>","javascript":null}