var data={"title":"Chromosomal translocations, deletions, and inversions","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Chromosomal translocations, deletions, and inversions</div><dl id=\"topicContributors\"><dt><span> </span>Authors:</dt><dd><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/contributors\" class=\"contributor contributor_credentials\">Iris Schrijver, MD</a></dd><dd><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/contributors\" class=\"contributor contributor_credentials\">James L Zehnder, MD</a></dd><dd><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/contributors\" class=\"contributor contributor_credentials\">Athena M Cherry, PhD</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/contributors\" class=\"contributor contributor_credentials\">Richard A Larson, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/contributors\" class=\"contributor contributor_credentials\">Jennifer S Tirnauer, MD</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Oct 18, 2017.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H1\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>In this review, we will describe the most common structural chromosomal anomalies, give examples of disease processes resulting from these germline or somatic alterations, and discuss the mechanisms underlying these disorders. Due to the important role of chromosomal abnormalities in the pathogenesis of many hematological disorders, a general review of these specific aberrations is presented separately, as is a review of cytogenetic and molecular genetic tools used to characterize these abnormalities. (See <a href=\"topic.htm?path=general-aspects-of-cytogenetic-analysis-in-hematologic-malignancies\" class=\"medical medical_review\">&quot;General aspects of cytogenetic analysis in hematologic malignancies&quot;</a> and <a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Cytogenetics and molecular genetics&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H2\"><span class=\"h1\">OVERVIEW</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Chromosomal aberrations are due to either numerical abnormalities or structural defects. The normal diploid number of chromosomes in humans is 46. There are 23 pairs of chromosomes with 22 pairs of autosomes and two sex chromosomes, the X and the Y. Human females have two X chromosomes (46,XX), while males have one X and one Y chromosome (46,XY).</p><p class=\"headingAnchor\" id=\"H3\"><span class=\"h2\">Abnormality of chromosome number</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A numerical abnormality in a cell's chromosomal endowment may be caused by the gain of one or more complete haploid chromosome sets (polyploid karyotype). An example is the triploid chromosomal number (eg, 69,XXY) in a partial hydatiform mole. (See <a href=\"topic.htm?path=gestational-trophoblastic-disease-pathology#H8\" class=\"medical medical_review\">&quot;Gestational trophoblastic disease: Pathology&quot;, section on 'Genetics'</a>.)</p><p>More commonly, there is a selective gain or loss of an individual chromosome (aneuploidy). As an example, trisomy 21, which causes Down syndrome, is characterized by the gain of one additional copy of chromosome 21. All mutations that change the total number of chromosomes are considered to be <strong>genome mutations</strong>.</p><p class=\"headingAnchor\" id=\"H4\"><span class=\"h2\">Abnormality of chromosomal structure</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Structural chromosomal anomalies, also called <strong>chromosomal mutations</strong>, comprise those changes that are due to one or more breaks in a chromosome. Following a break, the separated fragments are likely to participate in chromosomal rearrangements. Structural chromosomal changes can result in a displacement of chromosomal regions without any loss or duplication of genetic material (ie, balanced rearrangements) or they may be unbalanced:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Balanced rearrangements are frequently inherited and are not commonly associated with phenotypic abnormalities. Two exceptions to this rule include a breakpoint that directly disrupts a gene, and the displacement of chromosomal material from an X chromosome to an autosome or vice versa.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Unbalanced rearrangements result in a partial trisomy or monosomy and are a frequent cause of congenital abnormalities or developmental delay.</p><p/><p>On a submicroscopic level, <strong>gene mutations</strong> such as single base-pair substitutions, insertions, deletions, duplications, nucleotide repeat expansions and inversions are usually too small to be identified by standard karyotypic analysis, and require DNA analysis for their detection. These submicroscopic mutations can be as small as a change in a single base pair, or as large as megabase stretches of DNA sequence in very large genes. (See <a href=\"topic.htm?path=overview-of-genetic-variation\" class=\"medical medical_review\">&quot;Overview of genetic variation&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H5\"><span class=\"h1\">MECHANISMS OF CHROMOSOMAL BREAKAGE</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Chromosomal breakage is caused by double strand DNA breaks of endogenous or exogenous origin. Endogenous breaks are associated with DNA replication, recombination, transcription, and repair processes, all of which require temporary double strand DNA breaks. Permanent endogenous breaks occur in programmed cell death due to aging of the cell, cell damage, or regulation of cell numbers. Exogenous causes include exposure to irradiation and certain chemical substances, as well as other types of DNA damage.</p><p>The repair of DNA breaks occurs with high fidelity as long as the damaged area can be removed and replaced by a DNA sequence that is copied from an undamaged template. Double strand DNA breaks, however, need repair across the damaged area on both strands and are more susceptible to errors. By altering gene structure or expression patterns, such mutations can have pathogenic consequences, including malignant transformation.</p><p>Chromosomal breakage is also a requirement for subsequent translocations, deletions, inversions, and other chromosomal rearrangements [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/1\" class=\"abstract_t\">1</a>]. These will be discussed in the following sections.</p><p class=\"headingAnchor\" id=\"H6\"><span class=\"h1\">TRANSLOCATIONS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Most individuals with a balanced translocation are phenotypically normal. However, early studies demonstrated a significantly higher incidence of &quot;balanced&quot; rearrangements in institutionalized individuals [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/2\" class=\"abstract_t\">2</a>]. It is theorized that, in such instances, the translocation directly disrupts a gene (by &ldquo;position effect&rdquo;) or elicits the loss of a small part of the chromosome.</p><p>There are three types of chromosomal translocations: reciprocal, Robertsonian, and insertional (non-reciprocal).</p><p class=\"headingAnchor\" id=\"H7\"><span class=\"h2\">Reciprocal translocations</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Congenital reciprocal translocations have an incidence of 0.2 percent and originate from the breakage of two non-homologous chromosomes that interchange their separated parts. Rarely, more than two chromosomes are involved in a reciprocal exchange of chromosome segments. When this occurs, the translocation is considered to be complex.</p><p>Normally, genetic material is not lost in a reciprocal translocation. During meiosis, the normal and translocated chromosomes pair by forming a quadrivalent structure, from which one normal, one balanced, and four unbalanced products could be generated (<a href=\"image.htm?imageKey=PC%2F58692\" class=\"graphic graphic_figure graphicRef58692 \">figure 1</a>). The unbalanced products can be subdivided into two products with correct and two products with incorrect centromere segregation. The latter situation, in which the gamete contains two identical centromeres, is very rare and less compatible with a viable outcome. While the theoretical expectation is that 50 percent of the gametes produced are abnormal or unbalanced, the empirical general risk of having abnormal liveborn offspring rarely exceeds 15 percent.</p><p>Reciprocal translocations also frequently occur as a somatic mutation in hematologic malignancies and other neoplasms. The reciprocal translocation in one cell either juxtaposes two genes and induces upregulation of one of these genes, or causes oncogene activation by creation of a novel fusion gene. In both scenarios, clonal expansion of the affected cell directly leads to tumorigenesis. (See <a href=\"topic.htm?path=genetic-abnormalities-in-hematologic-and-lymphoid-malignancies\" class=\"medical medical_review\">&quot;Genetic abnormalities in hematologic and lymphoid malignancies&quot;</a>.)</p><p>The first situation is exemplified by the translocation of an oncogene to a gene that either encodes an immunoglobulin heavy or light chain, or a T-cell receptor. These genes naturally undergo extensive rearrangements to ensure the necessary diversity in cell-mediated or antibody responses. This process, however, increases the risk of erroneous interchromosomal rearrangements. An example is t(14;18), which is present in nearly 80 percent of patients with follicular B-cell lymphoma. In this translocation, the BCL2 gene on chromosome 18 is adjoined to the immunoglobulin heavy chain gene on chromosome 14, which is constitutively expressed and causes overexpression of BCL2 [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/3\" class=\"abstract_t\">3</a>].</p><p>In the second circumstance, the breakpoints occur within two genes and a hybrid gene is created. Expression of the chimeric protein can change cell signaling pathways due to the altered activity of transcription factors. One of many examples is t(9;22) which causes fusion of the BCR gene on chromosome 22 and the ABL1 gene on chromosome 9 (<a href=\"image.htm?imageKey=HEME%2F72417\" class=\"graphic graphic_figure graphicRef72417 \">figure 2</a>). This hybrid BCR-ABL1 product is present in all patients with chronic myeloid leukemia [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/4\" class=\"abstract_t\">4</a>]. (See <a href=\"topic.htm?path=genetic-abnormalities-in-hematologic-and-lymphoid-malignancies\" class=\"medical medical_review\">&quot;Genetic abnormalities in hematologic and lymphoid malignancies&quot;</a> and <a href=\"topic.htm?path=molecular-genetics-of-chronic-myeloid-leukemia\" class=\"medical medical_review\">&quot;Molecular genetics of chronic myeloid leukemia&quot;</a>.)</p><p>Although it is currently unknown which cellular processes determine that chromosome breaks will result in a reciprocal translocation, some pathogenic mechanisms of juxtaposed and chimeric genes have been elucidated:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The BCL2 gene encodes proteins that are oncogenic by prevention of programmed cell death rather than by promoting cell proliferation,. B-cells with t(14;18) escape apoptosis and are thus predisposed to the accumulation of additional mutations.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The CCND1 gene on chromosome 11, which is juxtaposed to the immunoglobulin heavy chain locus on chromosome 14 in many patients with mantle cell lymphoma, encodes the cell cycle regulator cyclin D1. The t(11;14) directly interferes with normal cell cycle events [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/5\" class=\"abstract_t\">5</a>].</p><p/><p>These examples illustrate the functional diversity of genes that undergo translocation to the immunoglobulin or T-cell receptor genes.</p><p>However, the genes most frequently involved in oncogenic translocations are transcription factors. Their dysregulation gives rise to inappropriate target gene activation in one or more cellular pathways [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/6\" class=\"abstract_t\">6</a>]. An example is the widely studied BCR-ABL1 fusion gene, which encodes an oncoprotein that is involved in the activation of multiple signal transduction cascades by constitutive expression of the ABL1 oncogene. (See <a href=\"topic.htm?path=cellular-and-molecular-biology-of-chronic-myeloid-leukemia\" class=\"medical medical_review\">&quot;Cellular and molecular biology of chronic myeloid leukemia&quot;</a>.)</p><p>While cell proliferation and survival become independent of the cytokines that normally regulate the differentiation as well as proliferation of the hematopoietic cells, phosphorylation of several substrates activates signal transduction pathways including RAS, JUN kinase, MYC and STAT [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/7\" class=\"abstract_t\">7</a>]. The induction of additional transcription factors may upregulate or reduce the expression of a potentially large number of genes. Finally, the BCR-ABL1 fusion protein may also influence the cell cycle directly [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/8\" class=\"abstract_t\">8</a>].</p><p class=\"headingAnchor\" id=\"H8\"><span class=\"h2\">Robertsonian translocations</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The acrocentric human chromosomes 13, 14, 15, 21 and 22 have very short p-arms that contain only chromosomal satellites and the genetic code for ribosomal RNAs. Their centromeres, therefore, are very close to one end of the chromosome. When the long arms of two acrocentric chromosomes merge by translocation, their short arms are lost and, depending upon the location of the breakpoints, a dicentric or monocentric fusion-chromosome is created.</p><p>Loss of the short arms of acrocentric chromosomes does not have phenotypic consequences, because the lost sections do not contain unique genetic sequences. The only consequence is a balanced karyotype with 45 instead of 46 chromosomes. This is identified in approximately 1 in 1000 individuals.</p><p>However, unbalanced gametes of heterozygous carriers are common and give rise to a monosomic or trisomic fetus. Most monosomies and trisomies are lethal and spontaneously abort early in the pregnancy. Surviving fetuses with trisomy 21, however, can be viable and affected with Down syndrome [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/9\" class=\"abstract_t\">9</a>], although about 85 percent of trisomy 21 conceptuses will not make it to term. When this syndrome is caused by a parental Robertsonian translocation, the carrier parent typically has a fusion of the long arms of chromosomes 14 and 21 (<a href=\"image.htm?imageKey=HEME%2F81746\" class=\"graphic graphic_figure graphicRef81746 \">figure 3</a>). One of the three viable gametes will be balanced, one will be normal, and one will contain the fused chromosome [der(14;21)] as well as the unaffected chromosome 21. Normal fertilization of this gamete results in a fetus with trisomy 21. The empiric risk for a child with Down syndrome is 10 to 15 percent for a carrier mother, but only 2 percent if the father carries the Robertsonian translocation.</p><p class=\"headingAnchor\" id=\"H9\"><span class=\"h2\">Insertional translocations</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Insertional translocations, by definition, are not reciprocal and are caused by integration of a segment from a &quot;donor&quot; chromosome that has incurred two breakpoints into a recipient chromosome that contains only one breakpoint. The resulting zygotes may be normal, balanced, partially trisomic, or partially monosomic. The phenotypic outcome of this anomaly, once considered rare, strongly depends upon the location of the breakpoints and on the nature of the translocated sequence.</p><p>With the advent of array comparative genomic hybridization (aCGH, also known as chromosomal microarray assay), it appears that insertional translocations (IT) are not as rare as previously believed. In 2000, the estimated incidence of microscopically visible ITs was <span class=\"nowrap\">1/80,000,</span> whereas more recent studies have shown them to be more common with one estimate of <span class=\"nowrap\">1/500</span> [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/10-14\" class=\"abstract_t\">10-14</a>].</p><p class=\"headingAnchor\" id=\"H10\"><span class=\"h1\">DELETIONS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Deletion of chromosomal material may or may not be detected by standard chromosomal banding techniques (ie, macrodeletions and microdeletions, respectively). (See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics#H9\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Cytogenetics and molecular genetics&quot;, section on 'Genotyping new mutations'</a>.)</p><p class=\"headingAnchor\" id=\"H11\"><span class=\"h2\">Macrodeletions</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>To microscopically see a chromosomal deletion by chromosome banding, the deletion must span at least three to five megabases (Mb). Considering that the average gene density approximates one gene per 50 kilobases (kb), it is not surprising that the loss of such a large region is likely to lead to phenotypic manifestations. Perhaps the most well-known example is Cri-du-chat syndrome, which is due to a terminal deletion of the short arm of chromosome 5 (<a href=\"image.htm?imageKey=HEME%2F66560\" class=\"graphic graphic_figure graphicRef66560 \">figure 4</a>). Congenital abnormalities in this disorder include cat-like neonatal crying, microcephaly with dysmorphic facial features, cardiac defects, hypotonia, and severe mental retardation. Even though the size of the deleted fragment differs from patient to patient, chromosomal bands 5p15.2 and 5p15.3 are always included. These bands, therefore, encompass the critical region for this phenotype [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/15\" class=\"abstract_t\">15</a>].</p><p>Another example of a variable size of the deleted fragment has been described in patients with multiple myeloma. In 106 patients, all but eight had a deletion of the long arm of chromosome 13 which included the 13q14 region (<a href=\"image.htm?imageKey=HEME%2F64638\" class=\"graphic graphic_figure graphicRef64638 \">figure 5</a>) [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/16\" class=\"abstract_t\">16</a>].</p><p class=\"headingAnchor\" id=\"H12\"><span class=\"h2\">Microdeletions</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Cytogenetic developments, such as high resolution banding techniques and fluorescence in situ hybridization (FISH), have enabled the detection of relatively small (&quot;micro&quot;) deletions. If the FISH probe is designed to localize to the region that is deleted on one allele, only one double signal will be present in the (pro)metaphase chromosomal evaluation. (See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Cytogenetics and molecular genetics&quot;</a>.)</p><p>One example of a microdeletion syndrome is Smith-Magenis syndrome, with an incidence of 1 in 25,000 and characterized by a microdeletion of chromosome band 17p11.2. Phenotypic features include intellectual disability, a deep and hoarse voice, self-abusive behavior, insomnia, short stature and brachydactyly. While most deletions span about 5 Mb and are cytogenetically discernible near the centromere, the remainder must be diagnosed with more refined diagnostic methods, such as FISH (<a href=\"image.htm?imageKey=HEME%2F67417\" class=\"graphic graphic_diagnosticimage graphicRef67417 \">image 1</a>) [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/17\" class=\"abstract_t\">17</a>]. The interstitial deletion in this syndrome is facilitated by the presence of regions of low-copy repeats that flank the common breakpoint regions [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/18\" class=\"abstract_t\">18</a>]. These segments are highly homologous, which makes them prone to unequal crossing over in meiosis I. During unequal crossing over, two non-sister chromatids recombine at an incorrect location due to inappropriate alignment of highly homologous DNA sequences. Thus, one of the chromatids acquires a duplication of the intervening sequence, while the corresponding segment is deleted on the other chromatid (<a href=\"image.htm?imageKey=HEME%2F63004\" class=\"graphic graphic_figure graphicRef63004 \">figure 6</a>). This is called non-allelic homologous recombination (NAHR), and is the basis of most <span class=\"nowrap\">microdeletion/microduplication</span> syndromes. (See <a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-1-to-11\" class=\"medical medical_review\">&quot;Microdeletion syndromes (chromosomes 1 to 11)&quot;</a> and <a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-12-to-22\" class=\"medical medical_review\">&quot;Microdeletion syndromes (chromosomes 12 to 22)&quot;</a>.)</p><p>Another example of a microdeletion syndrome, which is the consequence of NAHR, is Williams syndrome (or Williams-Beuren syndrome, WBS). Phenotypic features include cardiovascular disease (most frequently supravalvular aortic stenosis), distinctive faces with full lips and periorbital fullness, and an unusual and intellectual disability with relative cognitive strength in verbal abilities. Over 99 percent of individuals with Williams syndrome have a contiguous gene deletion which encompasses the elastin (ELN) gene at 7q11.23. Approximately 95 percent of individuals affected with WBS have a 1.55 Mb microdeletion, while the other 5 percent have a larger 1.84 Mb microdeletion. (See <a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-1-to-11\" class=\"medical medical_review\">&quot;Microdeletion syndromes (chromosomes 1 to 11)&quot;</a>.)</p><p>Homologous recombination during mitosis is both rare and abnormal. Nevertheless, unequal crossing over can also occur in mitotic division and leads to unequal sister chromatid exchange. The basic mechanism of misalignment between repeat sequences is the same as in meiosis. This abnormal event in mitosis, however, results in a somatic mutation and can predispose to cancer if the deleted sequence contains (or is part of) a tumor suppressor gene.</p><p>A subset of patients with hypereosinophilic syndrome have a novel FIP1L1-PDGFRA fusion due to a large interstitial deletion on chromosome 4q12. This results in a fusion protein with constitutive tyrosine kinase activity, eosinophilic proliferation and end-organ damage. Patients with this fusion frequently respond to the tyrosine kinase inhibitor, <a href=\"topic.htm?path=imatinib-drug-information\" class=\"drug drug_general\">imatinib</a> mesylate. Diagnosis of this fusion protein can be made by demonstrating the loss of the CHIC2 gene within the deleted region by FISH, or by PCR demonstrating the presence of a fusion transcript [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/19,20\" class=\"abstract_t\">19,20</a>]. (See <a href=\"topic.htm?path=hypereosinophilic-syndromes-clinical-manifestations-pathophysiology-and-diagnosis#H7\" class=\"medical medical_review\">&quot;Hypereosinophilic syndromes: Clinical manifestations, pathophysiology, and diagnosis&quot;, section on 'Myeloid HES variants'</a>.)</p><p class=\"headingAnchor\" id=\"H13\"><span class=\"h1\">INVERSIONS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Chromosomal inversions are characterized by two breaks on the same chromosome, rotation of the intervening segment by 180 degrees, followed by chromosomal reintegration of the intervening segment in an &quot;upside down&quot; position. Inversions can be subclassified based on the location of their breaks. Breakpoints on just one chromosome arm define paracentric inversions, whereas pericentric inversions result from breaks on both sides of the centromere. Inversion carriers are typically phenotypically normal. However, inversion carriers may have reproductive issues. At meiosis the inverted chromosome must pair with the non-inverted, normal chromosome forming a loop. If crossing-over occurs, unbalanced or abnormal gametes may result.</p><p class=\"headingAnchor\" id=\"H14\"><span class=\"h2\">Paracentric inversions</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Paracentric inversions (<a href=\"image.htm?imageKey=HEME%2F63995\" class=\"graphic graphic_figure graphicRef63995 \">figure 7</a>) are relatively rare, but have been observed in all 23 autosomes and both sex chromosomes [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/21\" class=\"abstract_t\">21</a>]. The chromosomes in individuals with a paracentric inversion are more likely to be ascertained because of infertility or repeated miscarriages by the carrier, than because of phenotypic abnormalities in the offspring. The risk of an affected child is generally very small, because of one of the following mechanisms:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The transmitted chromosome is normal</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The transmitted chromosome is inverted and compatible with life, as in the parent</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>After recombination in the inverted region, the transmitted chromosome becomes acentric or dicentric</p><p/><p>Gametes with acentric fragments or dicentric chromosomes are not typically viable. In general, inherited paracentric inversions are innocuous. However, there are rare reported cases of abnormal offspring in individuals with paracentric inversions.</p><p class=\"headingAnchor\" id=\"H15\"><span class=\"h2\">Pericentric inversions</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A pericentric inversion (<a href=\"image.htm?imageKey=HEME%2F51808\" class=\"graphic graphic_figure graphicRef51808 \">figure 8</a>) not only causes changes in the banding pattern of the affected chromosome, but may change the position of the centromere as well. Telomeric to the inverted segment, recombinant chromosomes will be partially duplicated or deleted [<a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/22\" class=\"abstract_t\">22</a>]. Larger segments are more susceptible to crossing over in meiosis I, since generally at least one cross-over takes place per chromosome arm. Hence, larger pericentric inversions are associated with an increased risk of phenotypic anomalies in offspring because the duplicated or deleted segments will be smaller and more likely compatible with life.</p><p>In cells with malignant transformation due to somatic mutation, reciprocal translocations and deletions are much more commonly identified than inversions. As a class, inversions are encountered in only 2 percent of all chromosome aberrations seen in cancer patients who have one or multiple chromosome anomalies.</p><p class=\"headingAnchor\" id=\"H6203102\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Chromosomal aberrations are due to either numerical abnormalities or structural defects. Polyploidy is the gain of one or more complete set of haploid chromosomes (eg, 69,XXY in a partial hydatiform mole). Aneuploidy, gain or loss of an individual chromosome, is more common. Trisomy 21, which causes Down syndrome, is characterized by the gain of one additional copy of chromosome 21. Polyploidy and aneuploidy represent <strong>genome mutations</strong>. (See <a href=\"#H3\" class=\"local\">'Abnormality of chromosome number'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Structural chromosomal anomalies, or <strong>chromosomal mutations</strong>, are due to one or more breaks in a chromosome. Structural chromosomal changes can be balanced (without any loss or gain of genetic material, and typically phenotypically normal) or unbalanced (with loss and gain of genetic material, and typically phenotypically abnormal). (See <a href=\"#H4\" class=\"local\">'Abnormality of chromosomal structure'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>There are three types of chromosomal translocations: reciprocal, Robertsonian, and insertional (non-reciprocal). Congenital reciprocal translocations can result in gametes with unbalanced genetic material, although most viable offspring carry normal or balanced chromosomes. Reciprocal translocations frequently occur as a somatic mutation in hematologic and other malignancies. Transcription factor genes are most frequently involved in oncogenic translocation. (See <a href=\"#H7\" class=\"local\">'Reciprocal translocations'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Robertsonian translocations involve acrocentric chromosome with short p-arms that can be lost in translocation of the long arms, resulting in a balanced karyotype with 45, rather than 46, chromosomes and a normal phenotype. However, unbalanced gametes of heterozygous carriers are common and give rise to a monosomic or trisomic fetus. Trisomy 21 can be caused by a parent with a Robertsonian translocation. (See <a href=\"#H8\" class=\"local\">'Robertsonian translocations'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Insertional translocations are caused by integration of a segment from a &quot;donor&quot; chromosome that has incurred two breakpoints into a recipient chromosome that contains only one breakpoint. The resulting zygotes may be normal, balanced, partially trisomic, or partially monosomic. (See <a href=\"#H9\" class=\"local\">'Insertional translocations'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Macrodeletions, which can be seen microscopically, must span at least three to five megabases (Mb) and therefore are likely to have phenotypic manifestations. Examples are found in patients with Cri-du-chat syndrome and some patients with multiple myeloma. (See <a href=\"#H11\" class=\"local\">'Macrodeletions'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Smaller deletions can be detected by high-resolution banding techniques and fluorescence in situ hybridization (FISH), as well as by array comparative genomic hybridization (aCGH). Most microdeletion or microduplication syndromes result from non-allelic homologous recombination (NAHR), in which highly homologous repeat DNA sequences lead to unequal crossing over and erroneous recombination in meiosis I. Similarly, unequal crossing over in mitotic division can result in a somatic mutation which can predispose to cancer if the deleted sequence contains a tumor suppressor gene. (See <a href=\"#H12\" class=\"local\">'Microdeletions'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Inversions, in which a segment of the chromosome is rotated 180 degrees, can be paracentric, involving one chromosome arm, or pericentric, involving both arms of the chromosome. Chromosomes in individuals with inversions are likely to be recognized at the time of evaluation for infertility or repeated miscarriages by the carrier, or found incidentally during prenatal diagnosis. A pericentric inversion causes changes in the banding pattern of the affected chromosome, and may change the position of the centromere. Larger pericentric inversions are associated with an increased risk of offspring with phenotypic anomalies because the duplicated or deleted segments will be smaller and more likely compatible with life. (See <a href=\"#H13\" class=\"local\">'Inversions'</a> above.)</p></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/1\" class=\"nounderline abstract_t\">Vamvakas S, Vock EH, Lutz WK. On the role of DNA double-strand breaks in toxicity and carcinogenesis. Crit Rev Toxicol 1997; 27:155.</a></li><li class=\"breakAll\">Warburton, D. De novo structural rearrangements: implications in prenatal diagnosis. In: Clinical Genetics: Problems in Diagnosis and Counseling, Wiley, AM, Carter, TP, Kelly, S, Porter (Eds), Academic Press, New York 1982. p.63.</li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/3\" class=\"nounderline abstract_t\">Rezuke WN, Abernathy EC, Tsongalis GJ. Molecular diagnosis of B- and T-cell lymphomas: fundamental principles and clinical applications. Clin Chem 1997; 43:1814.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/4\" class=\"nounderline abstract_t\">Melo JV. BCR-ABL gene variants. Baillieres Clin Haematol 1997; 10:203.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/5\" class=\"nounderline abstract_t\">Griesser H. Applied molecular genetics in the diagnosis of malignant non-Hodgkin's lymphoma. Diagn Mol Pathol 1993; 2:177.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/6\" class=\"nounderline abstract_t\">Mitelman F, Johansson B, Mertens F. The impact of translocations and gene fusions on cancer causation. Nat Rev Cancer 2007; 7:233.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/7\" class=\"nounderline abstract_t\">Sawyers CL. Chronic myeloid leukemia. N Engl J Med 1999; 340:1330.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/8\" class=\"nounderline abstract_t\">Warmuth M, Danhauser-Riedl S, Hallek M. Molecular pathogenesis of chronic myeloid leukemia: implications for new therapeutic strategies. Ann Hematol 1999; 78:49.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/9\" class=\"nounderline abstract_t\">Mutton D, Alberman E, Hook EB. Cytogenetic and epidemiological findings in Down syndrome, England and Wales 1989 to 1993. National Down Syndrome Cytogenetic Register and the Association of Clinical Cytogeneticists. J Med Genet 1996; 33:387.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/10\" class=\"nounderline abstract_t\">Kang SH, Shaw C, Ou Z, et al. Insertional translocation detected using FISH confirmation of array-comparative genomic hybridization (aCGH) results. Am J Med Genet A 2010; 152A:1111.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/11\" class=\"nounderline abstract_t\">de Ravel T, Aerssens P, Vermeesch JR, Fryns JP. Trisomy of chromosome 16p13.3 due to an unbalanced insertional translocation into chromosome 22p13. Eur J Med Genet 2005; 48:355.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/12\" class=\"nounderline abstract_t\">Shinawi M, Cheung SW. The array CGH and its clinical applications. Drug Discov Today 2008; 13:760.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/13\" class=\"nounderline abstract_t\">Bi W, Sapir T, Shchelochkov OA, et al. Increased LIS1 expression affects human and mouse brain development. Nat Genet 2009; 41:168.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/14\" class=\"nounderline abstract_t\">Carvalho CM, Zhang F, Liu P, et al. Complex rearrangements in patients with duplications of MECP2 can occur by fork stalling and template switching. Hum Mol Genet 2009; 18:2188.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/15\" class=\"nounderline abstract_t\">Marinescu RC, Johnson EI, Dykens EM, et al. No relationship between the size of the deletion and the level of developmental delay in cri-du-chat syndrome. Am J Med Genet 1999; 86:66.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/16\" class=\"nounderline abstract_t\">Shaughnessy J, Tian E, Sawyer J, et al. High incidence of chromosome 13 deletion in multiple myeloma detected by multiprobe interphase FISH. Blood 2000; 96:1505.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/17\" class=\"nounderline abstract_t\">Juyal RC, Greenberg F, Mengden GA, et al. Smith-Magenis syndrome deletion: a case with equivocal cytogenetic findings resolved by fluorescence in situ hybridization. Am J Med Genet 1995; 58:286.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/18\" class=\"nounderline abstract_t\">Chen KS, Manian P, Koeuth T, et al. Homologous recombination of a flanking repeat gene cluster is a mechanism for a common contiguous gene deletion syndrome. Nat Genet 1997; 17:154.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/19\" class=\"nounderline abstract_t\">Cools J, DeAngelo DJ, Gotlib J, et al. A tyrosine kinase created by fusion of the PDGFRA and FIP1L1 genes as a therapeutic target of imatinib in idiopathic hypereosinophilic syndrome. N Engl J Med 2003; 348:1201.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/20\" class=\"nounderline abstract_t\">Gotlib J. Eosinophilic myeloid disorders: new classification and novel therapeutic strategies. Curr Opin Hematol 2010; 17:117.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/21\" class=\"nounderline abstract_t\">Pettenati MJ, Rao PN, Phelan MC, et al. Paracentric inversions in humans: a review of 446 paracentric inversions with presentation of 120 new cases. Am J Med Genet 1995; 55:171.</a></li><li><a href=\"https://www.uptodate.com/contents/chromosomal-translocations-deletions-and-inversions/abstract/22\" class=\"nounderline abstract_t\">Kaiser P. Pericentric inversions. Problems and significance for clinical genetics. Hum Genet 1984; 68:1.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 4490 Version 17.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H6203102\"><span>SUMMARY</span></a></li><li><a href=\"#H1\" id=\"outline-link-H1\">INTRODUCTION</a></li><li><a href=\"#H2\" id=\"outline-link-H2\">OVERVIEW</a><ul><li><a href=\"#H3\" id=\"outline-link-H3\">Abnormality of chromosome number</a></li><li><a href=\"#H4\" id=\"outline-link-H4\">Abnormality of chromosomal structure</a></li></ul></li><li><a href=\"#H5\" id=\"outline-link-H5\">MECHANISMS OF CHROMOSOMAL BREAKAGE</a></li><li><a href=\"#H6\" id=\"outline-link-H6\">TRANSLOCATIONS</a><ul><li><a href=\"#H7\" id=\"outline-link-H7\">Reciprocal translocations</a></li><li><a href=\"#H8\" id=\"outline-link-H8\">Robertsonian translocations</a></li><li><a href=\"#H9\" id=\"outline-link-H9\">Insertional translocations</a></li></ul></li><li><a href=\"#H10\" id=\"outline-link-H10\">DELETIONS</a><ul><li><a href=\"#H11\" id=\"outline-link-H11\">Macrodeletions</a></li><li><a href=\"#H12\" id=\"outline-link-H12\">Microdeletions</a></li></ul></li><li><a href=\"#H13\" id=\"outline-link-H13\">INVERSIONS</a><ul><li><a href=\"#H14\" id=\"outline-link-H14\">Paracentric inversions</a></li><li><a href=\"#H15\" id=\"outline-link-H15\">Pericentric inversions</a></li></ul></li><li><a href=\"#H6203102\" id=\"outline-link-H6203102\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>GRAPHICS <a href=\"#\" id=\"viewAllGraphicsLink\">View All</a></h2><div id=\"outlineGraphics\"><ul><li><div id=\"PC/4490|DiagnosticImage\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"DIAGNOSTIC IMAGES\">DIAGNOSTIC IMAGES</a></div><ul><li><a href=\"image.htm?imageKey=HEME/67417\" class=\"graphic graphic_diagnosticimage\">- Smith Magenis syndrome</a></li></ul></li><li><div id=\"PC/4490|FIG\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"FIGURES\">FIGURES</a></div><ul><li><a href=\"image.htm?imageKey=PC/58692\" class=\"graphic graphic_figure\">- Reciprocal translocations</a></li><li><a href=\"image.htm?imageKey=HEME/72417\" class=\"graphic graphic_figure\">- Reciprocal translocation CML</a></li><li><a href=\"image.htm?imageKey=HEME/81746\" class=\"graphic graphic_figure\">- Robertsonian translocation</a></li><li><a href=\"image.htm?imageKey=HEME/66560\" class=\"graphic graphic_figure\">- Cri du chat syndrome</a></li><li><a href=\"image.htm?imageKey=HEME/64638\" class=\"graphic graphic_figure\">- Chromosome 13 in myeloma</a></li><li><a href=\"image.htm?imageKey=HEME/63004\" class=\"graphic graphic_figure\">- Unequal crossing over</a></li><li><a href=\"image.htm?imageKey=HEME/63995\" class=\"graphic graphic_figure\">- Paracentric inversion</a></li><li><a href=\"image.htm?imageKey=HEME/51808\" class=\"graphic graphic_figure\">- Pericentric inversion</a></li></ul></li></ul></div></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=cellular-and-molecular-biology-of-chronic-myeloid-leukemia\" class=\"medical medical_review\">Cellular and molecular biology of chronic myeloid leukemia</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=general-aspects-of-cytogenetic-analysis-in-hematologic-malignancies\" class=\"medical medical_review\">General aspects of cytogenetic analysis in hematologic malignancies</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetic-abnormalities-in-hematologic-and-lymphoid-malignancies\" class=\"medical medical_review\">Genetic abnormalities in hematologic and lymphoid malignancies</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=gestational-trophoblastic-disease-pathology\" class=\"medical medical_review\">Gestational trophoblastic disease: Pathology</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=hypereosinophilic-syndromes-clinical-manifestations-pathophysiology-and-diagnosis\" class=\"medical medical_review\">Hypereosinophilic syndromes: Clinical manifestations, pathophysiology, and diagnosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-1-to-11\" class=\"medical medical_review\">Microdeletion syndromes (chromosomes 1 to 11)</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-12-to-22\" class=\"medical medical_review\">Microdeletion syndromes (chromosomes 12 to 22)</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=molecular-genetics-of-chronic-myeloid-leukemia\" class=\"medical medical_review\">Molecular genetics of chronic myeloid leukemia</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=overview-of-genetic-variation\" class=\"medical medical_review\">Overview of genetic variation</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">Tools for genetics and genomics: Cytogenetics and molecular genetics</a></li></ul></div></div>","javascript":null}