var data={"title":"Cytogenetics and molecular genetics in acute lymphoblastic leukemia","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Cytogenetics and molecular genetics in acute lymphoblastic leukemia</div><dl id=\"topicContributors\"><dt><span> </span>Authors:</dt><dd><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/contributors\" class=\"contributor contributor_credentials\">Yanming Zhang, MD</a></dd><dd><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/contributors\" class=\"contributor contributor_credentials\">Michelle M Le Beau, PhD</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/contributors\" class=\"contributor contributor_credentials\">Richard A Larson, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/contributors\" class=\"contributor contributor_credentials\">Alan G Rosmarin, MD</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Feb 02, 2018.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H1\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The most useful prognostic indicators in acute lymphoblastic leukemia (ALL) are age, white blood cell count, immunophenotype, minimal residual disease detection, and karyotype. A number of recurring cytogenetic abnormalities are associated with distinct immunologic phenotypes of ALL and characteristic outcomes [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/1-5\" class=\"abstract_t\">1-5</a>]. These abnormalities will be reviewed here. An overview of cytogenetic analysis in hematologic malignancies, including definitions, methods of detection, and the genetic consequences of chromosomal translocations, is presented separately, as is the use of minimal residual disease assessment in patients with ALL. (See <a href=\"topic.htm?path=general-aspects-of-cytogenetic-analysis-in-hematologic-malignancies\" class=\"medical medical_review\">&quot;General aspects of cytogenetic analysis in hematologic malignancies&quot;</a> and <a href=\"topic.htm?path=detection-of-minimal-residual-disease-in-acute-lymphoblastic-leukemia\" class=\"medical medical_review\">&quot;Detection of minimal residual disease in acute lymphoblastic leukemia&quot;</a> and <a href=\"topic.htm?path=clinical-use-of-minimal-residual-disease-detection-in-acute-lymphoblastic-leukemia\" class=\"medical medical_review\">&quot;Clinical use of minimal residual disease detection in acute lymphoblastic leukemia&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H2\"><span class=\"h1\">B CELL ACUTE LYMPHOBLASTIC LEUKEMIA</span></p><p class=\"headingAnchor\" id=\"H3\"><span class=\"h2\">Cytogenetic abnormalities</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Cytogenetic analysis and molecular cytogenetic studies, such as fluorescence in situ hybridization (FISH), reveal recurring chromosome abnormalities in approximately 80 percent of ALL, including numerical and structural changes, such as translocations, inversions, or deletions [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/1,2,6-8\" class=\"abstract_t\">1,2,6-8</a>]. These cytogenetic abnormalities are incorporated in the World Health Organization (WHO) classification of acute leukemia (<a href=\"image.htm?imageKey=HEME%2F110239\" class=\"graphic graphic_table graphicRef110239 \">table 1</a>) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/5\" class=\"abstract_t\">5</a>].</p><p>There are substantial differences between children and adults with ALL in the frequencies of some recurring abnormalities, including the following:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The t(9;22) is observed in about 2 to 5 percent of children [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/9-11\" class=\"abstract_t\">9-11</a>] compared with about 30 percent of adults [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/6,8,12\" class=\"abstract_t\">6,8,12</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The t(12;21), which is detectable only by FISH or polymerase chain reaction (PCR) analysis, is observed in about 25 percent of children with B-lineage leukemia [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/7,13\" class=\"abstract_t\">7,13</a>], compared with about 3 percent of adults.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A hyperdiploid karyotype is found in 30 to 40 percent of children compared with 2 to 10 percent of adults [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/7,11\" class=\"abstract_t\">7,11</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The t(4;11) is present in up to 60 percent of infants younger than 12 months, but is rarely observed in adult ALL patients [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/7,14\" class=\"abstract_t\">7,14</a>].</p><p/><p>While less well understood, ALL that develops in adolescents and young adults (AYA, ages 16 to 21 years) appears to have a biology and prognosis distinct from that of younger children and older adults [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/15,16\" class=\"abstract_t\">15,16</a>]. While the best age cutoff to define this population is not known, some clinical trials define AYA up to age 39 years. AYA appear to have a better prognosis when treated with pediatric regimens. This is discussed in more detail separately. (See <a href=\"topic.htm?path=induction-therapy-for-philadelphia-chromosome-negative-acute-lymphoblastic-leukemia-in-adults#H23\" class=\"medical medical_review\">&quot;Induction therapy for Philadelphia chromosome negative acute lymphoblastic leukemia in adults&quot;, section on 'Young adults/adolescents'</a>.) &#160; </p><p>As will be described below, certain translocations, such as t(4;11) and t(9;22), are associated with treatment failure even when using intensive chemotherapy; in comparison, the t(12;21), t(1;19), and hyperdiploidy (50 to 60 chromosomes) are associated with a more favorable outcome [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/7,9,10,12-14,17,18\" class=\"abstract_t\">7,9,10,12-14,17,18</a>].</p><p class=\"headingAnchor\" id=\"H1280995514\"><span class=\"h3\">12;21 translocation in precursor B ALL</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A t(12;21)(p13.2;q22.1) is the most common genetic lesion in childhood ALL, occurring in 15 to 25 percent of children with B-lineage disease, specifically precursor B ALL [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/19-22\" class=\"abstract_t\">19-22</a>]. Most affected children are between the ages of 1 and 10 [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/19,22,23\" class=\"abstract_t\">19,22,23</a>]. This translocation is less common in adults, accounting for about 3 to 4 percent of patients with ALL [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/23,24\" class=\"abstract_t\">23,24</a>], and is not seen in T cell ALL.</p><p>The t(12;21) is not easily detected by cytogenetic analysis because of the similarity in size and banding patterns of 12p and 21q [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/19\" class=\"abstract_t\">19</a>]. As a result, RT-PCR or FISH analysis is required to detect this rearrangement [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/20\" class=\"abstract_t\">20</a>]. (See <a href=\"topic.htm?path=general-aspects-of-cytogenetic-analysis-in-hematologic-malignancies\" class=\"medical medical_review\">&quot;General aspects of cytogenetic analysis in hematologic malignancies&quot;</a>.)</p><p>The t(12;21) fuses the <em>ETV6</em> (<em>TEL</em>) gene at 12p13.2 with the <em><span class=\"nowrap\">RUNX1/AML1</em></span> gene at 21q22.1, resulting in the production of a fusion protein [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/25\" class=\"abstract_t\">25</a>].</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The RUNX1 protein (also known as core binding factor alpha-2, CBFA2) is a member of a family of transcription factors with homology to the pair-rule Drosophila gene, runt. RUNX1 heterodimerizes with another protein, core binding factor beta (CBFB, also known as PEBP2 beta), to form a transcription factor.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The ETV6 protein functions as a transcriptional repressor [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/26\" class=\"abstract_t\">26</a>]. It contains two domains: a helix-loop-helix (HLH) protein dimerization domain and an ETS DNA binding domain.</p><p/><p>The t(12;21) results in fusion of the 5' HLH domain of <em>ETV6</em> with the 3' DNA-binding and transactivation domain of <em>RUNX1</em> [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/25\" class=\"abstract_t\">25</a>]. ETV6-RUNX1 inhibits transactivation of gene expression by the normal RUNX1 protein, an effect that requires the HLH domain of ETV6. In many patients with ALL and the t(12;21), the <em>ETV6</em> allele on the other chromosome 12 homolog is deleted [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/27\" class=\"abstract_t\">27</a>], which has no effect on the favorable prognostic significance. Alterations of both <em>ETV6</em> alleles in leukemic cells suggest that <em>ETV6</em> may be a tumor suppressor gene.</p><p>The <em>ETV6</em> gene is also involved in several other recurring translocations in the leukemias and myeloproliferative disorders, including the t(5;12)(q32;p13.2) in chronic myelomonocytic leukemia. In this translocation, <em>ETV6</em> is fused to the platelet-derived growth factor receptor (<em>PDGFR</em>)-beta gene [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/28\" class=\"abstract_t\">28</a>]. Direct evidence supporting a pathogenetic role for this fusion protein comes from a transgenic mouse model in which expression of the <em>ETV6-PDGFRB</em> fusion mRNA produces a myeloproliferative disorder that shares features with chronic myelomonocytic leukemia [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/29\" class=\"abstract_t\">29</a>].</p><p>The <em>ETV6</em>-<em>RUNX1</em> fusion gene can be detected in B lineage progenitor cells during normal prenatal hematopoiesis in approximately 1 percent of developing fetuses, but only progresses to frank ALL in 1 percent of these cases [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/30\" class=\"abstract_t\">30</a>]. <em>ETV6</em>-<em>RUNX1</em> appears to induce a preleukemic stage, but additional cooperating genetic mutations are required for the conversion into acute leukemia [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/30-32\" class=\"abstract_t\">30-32</a>].</p><p>Patients with ALL and the t(12;21) generally have a favorable prognosis [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/4,13,19-22\" class=\"abstract_t\">4,13,19-22</a>]. One series, for example, evaluated 188 patients with precursor B ALL, 48 of whom (26 percent) had a rearrangement of the <em>ETV6</em> gene [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/21\" class=\"abstract_t\">21</a>]. Patients with the <em>ETV6</em> rearrangement had a much higher five-year event-free survival than those without the rearrangement (91 versus 65 percent). In the <span class=\"nowrap\">MRC/ECOG</span> study, the event-free and overall survival rates at five years of 368 childhood ALL patients with the t(12;21) were 89 and 96 percent, respectively [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/13\" class=\"abstract_t\">13</a>]. In another report, none of 22 patients with the <em><span class=\"nowrap\">ETV6/RUNX1</em></span> fusion relapsed compared with 16 of 54 (30 percent) without the <em>ETV6-RUNX1</em> fusion [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/23\" class=\"abstract_t\">23</a>].</p><p>Many of these patients would fall into a high-risk group using standard risk factors. Thus, the presence of the t(12;21) may identify patients within the &quot;high-risk&quot; group who would benefit from less toxic and less intensive therapy [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/21\" class=\"abstract_t\">21</a>]. However, within the group of children with ALL and the t(12;21), those with the CpG island methylator phenotype (CIMP) may be at higher risk of relapse [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/33\" class=\"abstract_t\">33</a>].</p><p class=\"headingAnchor\" id=\"H4\"><span class=\"h3\">8;14 translocation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A reciprocal translocation involving the long arms of chromosomes 8 and 14, designated t(8;14)(q24.2;q32.3), is present in a high proportion of Burkitt cell tumors of both African and non-African origin. The same translocation occurs in patients with ALL with FAB-L3-type leukemia cells, suggesting that Burkitt lymphoma and most B cell ALL of the L3 type are different manifestations of the same disease. Variant translocations have been described in some patients with Burkitt lymphoma and B cell ALL, including t(2;8)(p12;q24.2) and t(8;22)(q24.2;q11.2). (See <a href=\"topic.htm?path=pathobiology-of-burkitt-lymphoma#H7\" class=\"medical medical_review\">&quot;Pathobiology of Burkitt lymphoma&quot;, section on 'Chromosomal translocations'</a>.)</p><p>In a series of 1522 adult patients with ALL, the t(8;14) and its variants were detected in 16 (1 percent) patients [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/11\" class=\"abstract_t\">11</a>]. Eight of these patients had a mature B cell immunophenotype, whereas the remaining cases showed a common or pre-B (CD10-positive) phenotype. These patients had a high incidence of central nervous system involvement at diagnosis and a poorer prognosis than any other group of patients classified by chromosomal abnormalities. In a series of 21 adults with a t(8;14), all of whom had B cell ALL; the complete remission rate was 62 percent, but the median event-free survival was only two months [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/2\" class=\"abstract_t\">2</a>].</p><p class=\"headingAnchor\" id=\"H5\"><span class=\"h3\">4;11 translocation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Translocations involving 11q23.3 are observed in 5 to 7 percent of patients with ALL [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/2,4,8,11\" class=\"abstract_t\">2,4,8,11</a>]. The most common is t(4;11)(q21.3;q23.3); the t(11;19)(q23.3;p13.3) is second in frequency. However, about 50 percent of patients with the latter rearrangement have AML, usually acute monoblastic leukemia (AML-M5). (See <a href=\"topic.htm?path=cytogenetics-in-acute-myeloid-leukemia\" class=\"medical medical_review\">&quot;Cytogenetics in acute myeloid leukemia&quot;</a>.)</p><p>Patients with the t(4;11) and ALL have a number of characteristic clinical findings [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/2,4,13,14,18,34-38\" class=\"abstract_t\">2,4,13,14,18,34-38</a>]:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>High leukocyte counts (median 183,000 <span class=\"nowrap\">cells/microL)</span></p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>An immature (progenitor B cell) immunophenotype (CD10-, CD19+, HLA-DR+)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>B cell lineage</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Frequent co-expression of myeloid antigens (CD15+, CDw65+)</p><p/><p>In most cases of t(4;11) acute leukemia, the blasts are lymphoid in appearance [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/2,4,7\" class=\"abstract_t\">2,4,7</a>]. A notable finding is that the leukemia cells in about 50 percent of patients with the t(4;11) express myeloid enzymes, such as myeloperoxidase or nonspecific esterase (a granulocyte-monocyte marker) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/14,39\" class=\"abstract_t\">14,39</a>]. Thus, occasional blasts appear monocytic and, in some patients, lymphoid and monocytoid blasts occur in approximately equal proportion [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/14\" class=\"abstract_t\">14</a>]. These leukemic cells are generally terminal deoxynucleotidyl transferase (Tdt) positive and usually express pan-B cell antigens; in comparison, T cell markers are absent.</p><p>The t(4;11) fuses the <em>KMT2A</em> (previously called mixed-lineage, leukemia [<em>MLL</em>]) gene on chromosome 11 band q23.3 to the <em>AFF1</em> (AF4) gene on chromosome 4 band q21.3, forming a <em>KMT2A</em>-<em>AFF1</em> fusion gene, which is transcribed into a chimeric mRNA [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/40\" class=\"abstract_t\">40</a>]. Reverse transcriptase-polymerase chain reaction (RT-PCR) techniques have been developed for this transcript, which have been used for monitoring of residual disease in this ALL variant [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/40\" class=\"abstract_t\">40</a>]. (See <a href=\"topic.htm?path=detection-of-minimal-residual-disease-in-acute-lymphoblastic-leukemia\" class=\"medical medical_review\">&quot;Detection of minimal residual disease in acute lymphoblastic leukemia&quot;</a>.)</p><p>ALL with a t(4;11) has a poor prognosis in both adults and children. Typical findings include a remission rate of 75 percent, but a median event-free survival of only seven months in adults [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/2,8,14,41\" class=\"abstract_t\">2,8,14,41</a>], and a complete remission rate of 88 percent but a median survival of 10 months in children [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/4,13,34\" class=\"abstract_t\">4,13,34</a>]. In a group of 35 infant ALL patients with t(4;11), the five-year event-free survival was only 29 percent [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/14\" class=\"abstract_t\">14</a>].</p><p class=\"headingAnchor\" id=\"H351299836\"><span class=\"h3\">11q23.3 translocation in infants</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The majority (60 to 80 percent) of infants with ALL have translocations involving 11q23.3 [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/4,36-38,42,43\" class=\"abstract_t\">4,36-38,42,43</a>]. These translocations involve the <em>KMT2A </em>gene [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/37,38,42,44,45\" class=\"abstract_t\">37,38,42,44,45</a>]. The KMT2A protein is a histone methyltransferase that assembles in protein complexes that regulate gene transcription via chromatin remodeling. Target genes of the KMT2A transcriptional regulatory complexes include the <em>HOX</em> genes and the <em>IKZF1</em> (IKAROS) gene, which have a critical role in development as well as hematopoiesis [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/45\" class=\"abstract_t\">45</a>]. IKZF1 is a DNA-binding protein that is required for normal lymphocyte development. Mutant mice that express only dominant-negative IKZF1 isoforms develop ALL at three to six months of age; similar isoforms have been identified in infants with ALL and <em>KMT2A</em> translocations [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/46\" class=\"abstract_t\">46</a>]. (See <a href=\"topic.htm?path=cytogenetics-in-acute-myeloid-leukemia\" class=\"medical medical_review\">&quot;Cytogenetics in acute myeloid leukemia&quot;</a> and <a href=\"#H12\" class=\"local\">'Gene expression'</a> below.)</p><p>Leukemias in infants with <em>KMT2A</em> translocations have a characteristic and distinctive gene expression profile, suggesting that they constitute a distinct disease entity, characterized by a high rate of early treatment failure and a very poor outcome [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/37,38,47,48\" class=\"abstract_t\">37,38,47,48</a>]. In one series, for example, the estimated event-free survival in infants with ALL and a <em>KMT2A</em> rearrangement was 19 percent at three years compared with 46 percent for patients with germline <em>KMT2A</em> [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/37\" class=\"abstract_t\">37</a>]. In another series of 1725 childhood ALL patients, the event-free and overall survival rates at five years were 50 and 60 percent, respectively, compared with 89 and 96 percent of patients with a t(12;21) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/13\" class=\"abstract_t\">13</a>]. Among infants with <em>KMT2A</em> rearrangements, the t(4;11) may confer a particularly poor prognosis; as a result, intensified therapy, including transplantation, may be preferred in these patients [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/4,13,38\" class=\"abstract_t\">4,13,38</a>].</p><p class=\"headingAnchor\" id=\"H6\"><span class=\"h3\">9;22 translocation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The t(9;22)(q34.1;q11.2) resulting in the Philadelphia (Ph) chromosome is observed in about 5 percent of children [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/9,10\" class=\"abstract_t\">9,10</a>] compared with about 30 percent of adults overall [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/4,6,8,12\" class=\"abstract_t\">4,6,8,12</a>] and 50 percent of adults with B-lineage ALL [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/12\" class=\"abstract_t\">12</a>]. The normal primitive stem cells, rather than committed lymphoid progenitors, appear to be the target for leukemic transformation [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/49\" class=\"abstract_t\">49</a>]. The fusion product of the Ph chromosome (the BCR-ABL1 protein), which is also seen in chronic myeloid leukemia (CML), is discussed elsewhere. (See <a href=\"topic.htm?path=molecular-genetics-of-chronic-myeloid-leukemia\" class=\"medical medical_review\">&quot;Molecular genetics of chronic myeloid leukemia&quot;</a>.)</p><p>The Ph chromosome is the most frequent rearrangement in adult ALL. In one series of 141 consecutive adults with ALL, for example, 36 (26 percent) had a t(9;22); all of these occurred among the 109 patients with precursor B ALL [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/50\" class=\"abstract_t\">50</a>]. However, some patients have had both B cell and myeloid markers [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/9,12,51\" class=\"abstract_t\">9,12,51</a>]. Because of the high frequency of the <em>BCR-ABL1</em> rearrangement and its dire clinical consequences, molecular studies using RT-PCR or FISH should be routinely performed at diagnosis when clinical suspicion is high and cytogenetic analysis is nondiagnostic [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/52\" class=\"abstract_t\">52</a>].</p><p>In a large series of 1522 adult ALL patients from the UK MRC and ECOG clinical trial, 267 (19 percent) patients had the t(9;22) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/11\" class=\"abstract_t\">11</a>]. In this study, as well as in one reported by CALGB [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/53\" class=\"abstract_t\">53</a>], approximately 60 to 70 percent of patients had abnormalities in addition to the Ph chromosome; gain of a Ph chromosome, monosomy 7, +8, +X, and del(9p) are among the commonly associated secondary abnormalities. However, the survival significance of these secondary chromosome abnormalities is controversial [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/10,11,53\" class=\"abstract_t\">10,11,53</a>]. Interestingly, these abnormalities differ from those that evolve during the accelerated and blast phases of CML. Another difference from CML is that approximately 70 percent of patients with Ph chromosome positive ALL have a chromosomally normal cell line in the bone marrow at diagnosis, a rare finding in untreated CML.</p><p>Molecular studies of Ph chromosome positive ALL have revealed two distinct subgroups [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/54,55\" class=\"abstract_t\">54,55</a>]:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In approximately 30 to 50 percent of adults, the molecular rearrangement is identical to that seen in CML. The breaks occur within the <em>ABL1</em> gene and within the major bcr region (called M-bcr) of the <em>BCR</em> gene, producing a chimeric gene that encodes a 210 kilodalton fusion protein. Many of these patients have persistence of metaphase cells with the Ph chromosome after chemotherapy-induced hematologic remissions. They also have the Ph chromosome in myeloid cells and in myeloid colonies grown in vitro, suggesting that they represent cases of CML presenting in lymphoid blast crisis after an unrecognized chronic phase.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In the remaining patients, the breakpoint occurs upstream of bcr but still within the <em>BCR</em> gene (called m-bcr), giving rise to smaller fusion proteins (185 to 190 kilodaltons) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/56-58\" class=\"abstract_t\">56-58</a>]. Most of these patients lack the additional cytogenetic abnormalities typical of CML blast crisis, do not have the Ph chromosome in myeloid cells and colonies, and become Ph chromosome negative in hematologic remission. These observations suggest that the Ph translocation occurred in a cell that is more restricted in its differentiation, perhaps a committed B-lymphoid progenitor.</p><p/><p>Both the 210 and 185 kilodalton fusion proteins are involved in constitutive signaling via the RAS pathway of signal transduction and promote leukemogenesis [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/59-62\" class=\"abstract_t\">59-62</a>]. In a mouse model, animals transplanted with cells transfected with p185 <em>BCR-ABL1</em> develop tumors more rapidly than those transplanted with p210 <em>BCR-ABL1</em> [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/59\" class=\"abstract_t\">59</a>]. This could explain why patients with Ph chromosome positive ALL are biologically and clinically different from those with CML even though both disorders have the same chromosomal abnormality. The observation that patients in whom the Ph chromosome is restricted to lymphoblasts, which is more common with the m-bcr breakpoint, have shorter survivals than those with multilineage involvement (median 8 versus 47 months) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/54,55\" class=\"abstract_t\">54,55</a>] is compatible with the mouse studies showing more aggressive disease with p185 BCR-ABL1.</p><p>Ph chromosome positivity is a poor prognostic sign in ALL. Early chemotherapy trials reported no survivors at five years in adults with this abnormality [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/63,64\" class=\"abstract_t\">63,64</a>]. The initial response is better with more intensive regimens, but the likelihood of remaining in remission is still much lower than in those without this translocation [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/8\" class=\"abstract_t\">8</a>]. In a prospective study conducted by the Cancer and Leukemia Group B (CALGB), patients with ALL who were Ph chromosome positive, when compared with those who were Ph chromosome negative, had a similar complete remission rate (71 versus 77 percent), but a shorter median duration of remission (10 versus 18 months) and survival (11 versus 22 months) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/12\" class=\"abstract_t\">12</a>]. In a later review of clinical trials from CALGB, the likelihood of remaining in remission was much lower in patients who were Ph chromosome positive. In the <span class=\"nowrap\">MRC/ECOG</span> clinical trial, the five-year event-free and overall survival rates in Ph positive ALL patients were 16 and 22 percent, respectively, in comparison with 36 and 41 percent in those without the t(9;22) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/11\" class=\"abstract_t\">11</a>].</p><p>Tyrosine kinase (TK) inhibitors (eg, <a href=\"topic.htm?path=imatinib-drug-information\" class=\"drug drug_general\">imatinib</a>, <a href=\"topic.htm?path=dasatinib-drug-information\" class=\"drug drug_general\">dasatinib</a>, <a href=\"topic.htm?path=nilotinib-drug-information\" class=\"drug drug_general\">nilotinib</a>) that target the BCR-ABL1 fusion protein have been used in combination with chemotherapy in the initial treatment of Ph positive ALL. The addition of a TK inhibitor results in significantly higher rates of complete remission and overall survival. This is discussed in more detail separately. (See <a href=\"topic.htm?path=induction-therapy-for-philadelphia-chromosome-negative-acute-lymphoblastic-leukemia-in-adults\" class=\"medical medical_review\">&quot;Induction therapy for Philadelphia chromosome negative acute lymphoblastic leukemia in adults&quot;</a> and <a href=\"topic.htm?path=induction-therapy-for-philadelphia-chromosome-negative-acute-lymphoblastic-leukemia-in-adults#H19\" class=\"medical medical_review\">&quot;Induction therapy for Philadelphia chromosome negative acute lymphoblastic leukemia in adults&quot;, section on 'Philadelphia chromosome positive ALL'</a>.)</p><p class=\"headingAnchor\" id=\"H7\"><span class=\"h3\">1;19 translocation in pre-B cell ALL</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Pre-B cell ALL is a distinct subtype of ALL that can be distinguished from null-cell and common B cell ALL by the presence of cytoplasmic immunoglobulin mu-chain (Cmu) expression. The pre-B leukemias have rearrangements of the immunoglobulin heavy chain genes and occasionally of the light chain genes. The t(1;19) occurs in approximately 30 percent of patients with pre-B cell childhood ALL [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/65\" class=\"abstract_t\">65</a>]. Although most common in pre-B cell ALL, the t(1;19) can also occur in other B lineage cells, including null cell and mature B cell ALL [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/50,65\" class=\"abstract_t\">50,65</a>].</p><p>Children with pre-B ALL and this translocation tend to have a white blood cell count of <span class=\"nowrap\">&gt;20,000/microL</span> and a characteristic surface antigen profile that is CD19+, CD10+ (CALLA+), CD22+, CD34-, and <span class=\"nowrap\">CD20+/-</span> [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/66,67\" class=\"abstract_t\">66,67</a>]. In comparison, the t(1;19) is much less common in ALL in adults [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/8,50\" class=\"abstract_t\">8,50</a>].</p><p>The t(1;19) occurs in two forms: a reciprocal translocation, t(1;19)(q23.3;p13.3) or, more often, an unbalanced form characterized by two normal chromosome 1 and 19 homologs, and a rearranged chromosome 19, der(19)t(1;19)(q23.3;p13.3) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/67\" class=\"abstract_t\">67</a>]. The presenting clinical and laboratory features and event-free survival appear to be the same for these two subtypes [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/67,68\" class=\"abstract_t\">67,68</a>].</p><p>The t(1;19) involves the <em>TCF3</em> (<em>E2A</em>) gene at 19p13.3; this gene encodes two transcription factors (E12 and E47) that bind to enhancer elements in the <em>IGK</em> gene and to the regulatory elements of other genes [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/69,70\" class=\"abstract_t\">69,70</a>]. In the translocation, the <em>TCF3</em> gene is juxtaposed with <em>PBX1</em>, a homeobox (<em>HOX</em>) gene on chromosome 1. <em>HOX</em> genes encode DNA-binding transcription factors that regulate developmental processes and hematopoiesis. TCF3-PBX1 fusion mRNAs are formed and code for chimeric proteins that consist of the transcriptional activation domain of <span class=\"nowrap\">E12/E47</span> and the DNA-binding and protein dimerization domains of PBX1 [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/69,70\" class=\"abstract_t\">69,70</a>]. <em>TCF3</em> is expressed in all tissues; the normal <em>PBX1</em> gene is expressed in many fetal and adult tissues, but not in lymphoid cells.</p><p>How the TCF3-PBX1 fusion protein might promote leukemogenesis is not well understood. The following possible mechanisms for transcriptional activation have been described:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The fusion protein may directly activate gene transcription of <em>PBX1-HOX</em>-complex target genes.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The fusion protein may bind DNA cooperatively with a specific HOX protein; the potent transactivation domain of TCF3 may then activate gene transcription.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The PBX1-HOX complexes formed with other ubiquitously expressed PBX family members may normally inhibit transcription; in comparison, <span class=\"nowrap\">TCF3/PBX1/HOX</span> complexes may induce the transcription of genes that are normally repressed in lymphoid cells.</p><p/><p>With each of these scenarios, the fusion protein may lead to the transactivation of a number of genes that are not normally expressed in lymphoid tissues [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/71,72\" class=\"abstract_t\">71,72</a>]. As an example, a novel <em>WNT</em> gene is normally expressed in peripheral lymphoid organs but not in bone marrow. However, high levels of <em>WNT16</em> transcripts are present in bone marrow and cell lines from patients with pre-B ALL who have the <em>TCF3</em>-<em>PBX1</em> fusion gene; furthermore, expression of <em>WNT16 </em>is reduced by inhibition of <em>TCF3-PBX1</em> expression [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/71\" class=\"abstract_t\">71</a>].</p><p>In the past, patients with the t(1;19) typically had early treatment failure, suggesting that this translocation may distinguish a subgroup of patients who have a poor prognosis [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/67,68,73\" class=\"abstract_t\">67,68,73</a>]. In one large study, the adverse outcome of patients with the t(1;19) remained significant even after adjustment for recognized adverse clinical features, indicating that it is an independent risk factor [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/68\" class=\"abstract_t\">68</a>]. However, studies have shown that this adverse prognosis can be overcome by more intensive chemotherapy. In the <span class=\"nowrap\">MRC/ECOG</span> study, the event-free and overall survival rates at five-years were 80 and 84 percent, respectively, which is comparable to those (84 and 93 percent) with a hyperdiploid clone [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/13\" class=\"abstract_t\">13</a>]. In fact, t(1;19), t(12;21) and hyperdiploidy are all associated with a favorable prognosis [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/4\" class=\"abstract_t\">4</a>].</p><p class=\"headingAnchor\" id=\"H2516253092\"><span class=\"h3\">Intrachromosomal amplification of chromosome 21 (iAMP21)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>FISH analysis using the <em><span class=\"nowrap\">RUNX1/AML1</em></span> and <em><span class=\"nowrap\">ETV6/TEL</em></span> gene probes reveals multiple signals for the <em>RUNX1</em> gene in approximately 3 to 5 percent of older pediatric patients with B cell ALL. By cytogenetic analysis, various structural abnormalities of the long arm of chromosome 21 are noted, including an increase in the length of 21q, consistent with gene amplification [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/74\" class=\"abstract_t\">74</a>]. Genomic microarray studies revealed complex genomic imbalances of 21q, with multiple gains of most of the long arm, along with deletion of the subtelomeric region [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/75\" class=\"abstract_t\">75</a>]. Detailed analysis revealed that breakage-fusion-bridge cycles occurred first, followed by chromothripsis, leading to the formation of the tandem duplication(s) of 21q [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/76\" class=\"abstract_t\">76</a>]. <em>RUNX1</em> translocations or mutations were not observed. In a series of 530 pediatric patients with B cell ALL with iAMP21 from the United Kingdom, secondary chromosome abnormalities were common, including gain of the X chromosome, loss of chromosome 7 or abnormalities of 7q, del(9p), del(12p), <span class=\"nowrap\">del(13q/<em>RB1</em>),</span> deletion of <em>IKZF1</em>, and the <em>P2YB8-CRLF2</em> fusion [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/74\" class=\"abstract_t\">74</a>]. iAMP21 was associated with a dismal outcome, with a high risk of both early and late relapse, when patients were treated using standard risk regimens. The poor prognosis can be overcome by intensive treatment using high-risk regimens [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/74,77\" class=\"abstract_t\">74,77</a>].</p><p class=\"headingAnchor\" id=\"H2233889466\"><span class=\"h3\">9p abnormalities in ALL</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Approximately 5 to 10 percent of childhood and adult ALL cases show a deletion of the short arm of chromosome 9. It can occur as a single abnormality in both children and adult patients. However, it often exists as a part of a complex clone with other numerical and structural abnormalities, in particular with a del(12p) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/78\" class=\"abstract_t\">78</a>]. Loss of material from 9p can be in various forms, such as del(9p), add(9p), der(9)t(V;9)(V;p), an isochromosome of 9q, or a dicentric chromosome with 9q.</p><p>In one study, microarray analysis and genomic DNA sequencing identified a mono-allelic deletion of the <em>PAX5</em> gene at 9p13.2 in 28 percent of ALL patients with a cryptic or a large deletion involving 9p [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/79\" class=\"abstract_t\">79</a>]. Deletion or mutation of the <em>PAX5</em> gene results in reduced level of the PAX5 protein or the generation of hypomorphic alleles, thereby leading primarily to loss of function [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/80\" class=\"abstract_t\">80</a>].</p><p>Somatically acquired mutations in the Janus kinase (<em>JAK2</em>) gene on chromosome band 9p24 has been reported in 18 percent of patients with Down syndrome associated ALL, but is not commonly found in ALL patients without Down syndrome or in patients with Down syndrome associated acute megakaryoblastic leukemia [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/81-83\" class=\"abstract_t\">81-83</a>]. This <em>JAK2</em> mutation (R683) differs from the V617 mutation commonly seen in myeloproliferative neoplasms.</p><p>Deletion of 9p is an unfavorable risk factor associated with a high rate of relapse in precursor-B ALL in children [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/84\" class=\"abstract_t\">84</a>]. This is more pronounced in standard-risk patients defined as those one to nine years old with WBC count below <span class=\"nowrap\">50,000/microL</span> [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/78\" class=\"abstract_t\">78</a>].</p><p>In a study of 381 adult patients with precursor-B ALL, 8 percent had abnormalities in 9p [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/85\" class=\"abstract_t\">85</a>]. This group showed an overall survival that was significantly shorter than those with normal karyotypes, and similar to those with poor prognosis <span class=\"nowrap\">t(9;22)/<em>BCR-ABL1</em>-positive</span> ALL.</p><p class=\"headingAnchor\" id=\"H9\"><span class=\"h3\">Hyperdiploidy</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Some patients with ALL have a gain of many chromosomes in the leukemic cells with relatively few structural abnormalities. Two subgroups have been described: a group with 1 to 4 extra chromosomes (total 47 to 50) and a more common group that usually has 51 to 60 chromosomes but may have up to 65 chromosomes. Chromosome 21 is gained most frequently (100 percent of cases), and multiple additional copies of this chromosome are often seen [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/4,86-88\" class=\"abstract_t\">4,86-88</a>]. Other frequent additional chromosomes include the X chromosome, and chromosomes 4, 6, 10, 14, 17, and 18 [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/86-88\" class=\"abstract_t\">86-88</a>].</p><p>Structural rearrangements occur in the leukemic cells in as many of 25 percent of patients with hyperdiploid ALL cells [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/88\" class=\"abstract_t\">88</a>]. The most common are duplication of 1q (14 percent) and an isochromosome of 17q (5 percent). Chromosomal translocations occur in approximately 5 to 10 percent, including t(9;22) and t(1;19).</p><p>Patients who have hyperdiploidy with more than 50 chromosomes often have all of the clinical features that indicate a good prognosis. These include age between one and nine years, low white blood cell count (median <span class=\"nowrap\">6700/microL),</span> and favorable immunophenotype (early pre-B or pre-B) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/4,13\" class=\"abstract_t\">4,13</a>]. It has been suggested that this is not a uniform group, as the prognosis of children with 51 to 55 chromosomes may differ from those with 56 to 65 chromosomes.</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In a series of 168 patients, the 105 patients with 51 to 55 chromosomes had a significantly lower event-free survival at five years (72 versus 86 percent) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/88\" class=\"abstract_t\">88</a>]. Adverse prognosis features within the 51 to 55 chromosome group included a higher frequency of isochromosome 17 and a white blood cell count above 50,000 <span class=\"nowrap\">cells/microL</span>. Structural abnormalities did not appear to influence event-free survival, except for a poor prognosis associated with the t(9;22). </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In another series of 541 children with hyperdiploid ALL, a higher modal number of chromosomes (MNC) was associated with improved prognosis [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/89\" class=\"abstract_t\">89</a>]. Estimated event-free survival rates at six years were 80, 89, and 99 percent for cases with MNC of 51 to 53, 54 to 57, and 58 to 66, respectively. On multivariate analysis, MNC remained a strong predictor of outcome.</p><p/><p>Together, these studies demonstrate improved outcomes for children with hyperdiploid ALL, especially for those with a MNC of 58 to 66.</p><p class=\"headingAnchor\" id=\"H10\"><span class=\"h3\">Hypodiploidy</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Approximately 5 to 6 percent of ALL patients, independent of age, have clonal loss of various chromosomes, resulting in a hypodiploid clone with fewer than 46 chromosomes [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/90\" class=\"abstract_t\">90</a>]. Three distinct subgroups of hypodiploidy in ALL have been identified:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Near-haploidy (23 to 29 chromosomes)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Low hypodiploidy (33 to 39 chromosomes)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>High hypodiploidy (42 to 45 chromosomes)</p><p/><p>Near-haploidy is exclusively observed in children (median age of 7 years), whereas low hypodiploidy is noted in relatively older patients (median age of 15 years) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/5,90-92\" class=\"abstract_t\">5,90-92</a>]. In cytogenetic analysis, the near-haploid clone is often accompanied by a doubling of the chromosome complement (tetraploidization) resulting in a hyperdiploid population with a gain of two copies of certain chromosomes, which is distinguishable from the more common hyperdiploid clone with a single gain of certain chromosomes. More detailed genetic studies in 124 cases of hypodiploid ALL, including whole-genome and exome sequencing of 40 cases, identified differences in the genetic alterations among cases with near-haploidy and low hypodiploidy [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/91\" class=\"abstract_t\">91</a>]. Near-haploid cases demonstrated a higher frequency of alterations involving tyrosine kinase signaling and Ras signaling (71 percent) and <em>IKZF3</em> (13 percent), whereas low hypodiploidy cases demonstrated alterations in <em>TP53</em> (91 percent), <em>IKZF2</em> (53 percent), and <em>RB1</em> (41 percent). Both groups demonstrated activation of RAS signalling and PI3K signaling pathways.</p><p>ALL patients with hypodiploid clones generally have a poor prognosis, especially patients with near-haploid and low-hypodiploid clones. As an example, one study showed that the three-year event-free-survival of children with near-haploid and low-hypodiploid clones was 29 percent, whereas it was 66 percent for those with high hypodiploid clones (42 to 45 chromosomes) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/90\" class=\"abstract_t\">90</a>]. In this regard, it is critical to avoid confusing the doubled (tetraploid) populations of a hypodiploid clone with a true hyperdiploid population, as the prognosis is considerably different between these cytogenetic groups. DNA index analysis and FISH tests can help clarify such cases.</p><p class=\"headingAnchor\" id=\"H580099884\"><span class=\"h2\">Genetic abnormalities with normal karyotype</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Some cases of B cell ALL have &ldquo;sub-karyotypic&rdquo; genetic abnormalities, in which aberrant gene expression or novel fusion genes are detected without associated chromosomal abnormalities.</p><p class=\"headingAnchor\" id=\"H12\"><span class=\"h3\">Gene expression</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Evaluation of gene expression through gene expression profiles, expression analyses of individual genes (eg, <em>IKZF1</em>), and analysis of the epigenetic regulation of microRNAs (eg, methylation) appear to have prognostic value in patients with ALL [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/79,93-100\" class=\"abstract_t\">79,93-100</a>]; the clinical value of such studies is yet to be determined.</p><p>Genomewide analyses of patients with ALL have identified deletions or loss of function mutations in genes that regulate lymphocyte development. As an example, one study evaluated single-nucleotide polymorphism arrays and genomic DNA sequencing in 242 pediatric patients with ALL [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/79\" class=\"abstract_t\">79</a>]. Forty percent of B-ALL cases were found to have deletion, amplification, point mutations, or structural rearrangement in genes regulating B lymphocyte development and differentiation. Of these, the two most frequent abnormalities were in <em>PAX5</em> and <em>IKZF1</em> (in approximately 32 and 28 percent of patients, respectively). (See <a href=\"#H2233889466\" class=\"local\">'9p abnormalities in ALL'</a> above.)</p><p><em>IKZF1</em> (IKAROS family zinc finger 1) located at 7p13 encodes the IKAROS zinc finger binding protein that is required for normal lymphoid development [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/93\" class=\"abstract_t\">93</a>]. Splicing abnormalities of IKZF1 may be found in over 80 percent of patients with Philadelphia chromosome (Ph) positive ALL and certain isoforms may be associated with <a href=\"topic.htm?path=imatinib-drug-information\" class=\"drug drug_general\">imatinib</a> resistance in these patients [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/94,95\" class=\"abstract_t\">94,95</a>]. However, abnormalities of IKZF1 may be associated with a poor outcome independent of Ph status [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/101,102\" class=\"abstract_t\">101,102</a>]. As an example, a DNA copy number analysis of 221 patients with Ph negative high risk B-ALL (training cohort) and 258 unselected patients with B-ALL (validation cohort) reported that patients with <em>IKZF1</em> deletions had an increased frequency of relapse at 5 and 10 years and resistance to chemotherapy [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/96\" class=\"abstract_t\">96</a>]. The poor prognosis associated with <em>IKZF1</em> abnormalities was independent of Ph status.</p><p class=\"headingAnchor\" id=\"H13\"><span class=\"h3\">Cell cycle regulatory genes</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Abnormalities in cell cycle regulatory genes are also present in ALL and may affect prognosis.</p><p>In a series of 42 patients, 85 percent had such abnormalities with 33 percent having more than one [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/103\" class=\"abstract_t\">103</a>]:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><em>RB1</em>-Retinoblastoma gene (51 percent)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><em>p16</em> gene (also called <em>CDKN2A</em> and multiple tumor suppressor 1) (41 percent)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><em>TP53</em> gene (26 percent)</p><p/><p>Patients with more than one abnormality had a significantly shorter median survival (8 versus 25 months).</p><p>In a second study in 70 patients, abnormalities (deletion, methylation) of the <em>CDKN2B</em> (p15) and <em>CDKN2A</em> (p16) genes were noted in 69 and 39 percent, respectively [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/104\" class=\"abstract_t\">104</a>]. On multivariate analysis, absence of <em>CDKN2B</em> gene alterations was a favorable prognostic factor for longer disease-free survival.</p><p>In a third study in 251 consecutive patients with ALL, the hypermethylation profile of cancer-related genes was an independent prognostic factor in predicting disease-free and overall survival [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/105\" class=\"abstract_t\">105</a>].</p><p>Using next-generation sequencing techniques, <em>TP53</em> gene mutations may be observed in 16 percent of ALL overall with higher rates in certain patient subsets [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/106\" class=\"abstract_t\">106</a>]. The highest rates are seen in patients with low hypodiploidy (92 percent), <em>MYC</em> translocations (63 percent), and complex karyotype (23 percent). In addition, the frequency of <em>TP53</em> <span class=\"nowrap\">mutations/deletions</span> increases with age, and mutations of <em>TP53</em> are higher in B-ALL, including Burkitt leukemia, than in T-ALL (21 versus 8 percent). Clinical outcomes were considerably worse if both <em>TP53</em> genes were abnormal (eg, both mutated or one by mutation and one by chromosomal deletion, so-called &quot;double hits&quot; of <em>TP53</em>).</p><p class=\"headingAnchor\" id=\"H73697009\"><span class=\"h3\">BCR-ABL1-like B cell ALL</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>BCR-ABL1-like B cell ALL (also called Ph-like ALL) shares a similar gene expression profile with B cell ALL with the <span class=\"nowrap\">t(9;22)/<em>BCR-ABL1</em></span> fusion, but lacks the <em>BCR-ABL1</em> fusion or t(9;22) by cytogenetic, FISH, or molecular analysis [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/5,99,107\" class=\"abstract_t\">5,99,107</a>]. This subtype of ALL is detected in 10 to 13 percent of pediatric patients, 21 percent in adolescents, and up to 33 percent of adults with B-ALL, and is resistant to standard chemotherapy and associated with a poor prognosis [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/15,101,108-110\" class=\"abstract_t\">15,101,108-110</a>]. BCR-ABL1-like B cell ALL is mutually exclusive of the well-defined recurring chromosome abnormalities in B-ALL, such as the t(12;21), t(4;11) and other 11q23.<span class=\"nowrap\">3/<em>KMT2A</em></span> translocations, t(1;19), and a hyperdiploid karyotype. </p><p>Diagnosis requires the combined analysis of flow cytometry, cytogenetic and FISH patterns, and molecular mutation and deletion studies. Detection of BCR-ABL1-like B cell ALL is important for prognostic stratification and will become increasingly important for selection of targeted treatment strategies. </p><p>When compared with other cases of precursor B cell ALL, this subgroup has:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Higher median leukocyte count at presentation </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Higher median levels of minimal residual disease at the end of induction therapy</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Inferior estimated event free survival at five years &ndash; Children (58 versus 84 percent), adolescents (41 versus 83 percent), and young adults (24 versus 63 percent)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Inferior estimated overall survival at five years &ndash; Children (73 versus 92 percent), adolescents (66 versus 93 percent), and young adults (26 versus 75 percent)</p><p/><p>Larger studies that have characterized the genetic lesions in BCR-ABL1-like B cell ALL include the following:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Genomic profiling identified BCR-ABL1-like ALL in 264 of 1725 children, adolescents, or young adults with precursor B cell ALL enrolled on clinical trials [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/15\" class=\"abstract_t\">15</a>]. Analysis identified alternative mechanisms of activated kinase signalling in 91 percent of cases, including ABL1-class fusions, rearrangements of <em>EPOR</em> or <em>JAK2</em>, rearrangements of <em>CRLF2</em>, and other changes that affect <span class=\"nowrap\">JAK/STAT</span> or RAS pathways.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>BCR-ABL1-like ALL was identified in 20 percent of nearly 1400 consecutively diagnosed patients with childhood B-ALL enrolled in Children's Oncology Group trials [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/111\" class=\"abstract_t\">111</a>]. Genetic findings included overexpression and rearrangement of <em>CRLF2</em> in 44 percent, with concomitant genomic alterations activating the JAK-STAT pathway in half of those children. Other molecular lesions involved fusions of ABL-class genes, <em>EPOR </em>rearrangements, <em>JAK2</em> fusions, alterations of other JAK-STAT signaling genes, and mutations involving the RAS pathway.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>BCR-ABL1-like B-ALL is also seen in adult patients. Among 798 adult patients with B-ALL, this entity was identified in 28 percent of patients of age 21 to 39, 20 percent of patients of age 40 to 59, and 24 percent of patients of age 60 to 89 [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/109\" class=\"abstract_t\">109</a>]. Kinase-activating alterations were present in 88 percent of these cases, including <em>CRLF2</em> rearrangement in just over half. Similarly, in another study of 148 adult patients with B-ALL, 49 (33 percent) patients showed BCR-ABL1-like gene expression features, with 61 percent of patients involving <em>CRLF2</em> overexpression and rearrangement [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/110\" class=\"abstract_t\">110</a>]. These patients, particularly those with <em>CRFL2</em> rearrangements, had significantly worse overall survival and event free survival compared with other patients with B-ALL and no BCR-ABL1-like abnormalities.</p><p/><p>Genes that are mutated in BCR-ABL1-like B-ALL generally encode proteins involved in B cell development, proliferation and differentiation, cell cycle regulation, and cell signaling. These mutations result in constitutive kinase activation and signaling via activation of the ABL1 and JAK-STAT pathways, which are sensitive to tyrosine kinase inhibitors or JAK kinase inhibitors [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/15,112,113\" class=\"abstract_t\">15,112,113</a>]. Prospective studies are needed to determine the efficacy of these agents in this patient population. </p><p>In approximately 50 percent of patients with the BCR-ABL1-like B cell ALL, the cytokine receptor-like factor 2 (<em>CRLF2</em>) gene within the pseudo-autosomal region 1 (PAR1) of the short arm of the X or Y chromosome is involved in a cryptic translocation with the <em>IGH</em> gene at 14q32.3 (ie, t(X;14) or t(Y;14)), or is fused with the <em>P2RY8</em> gene via small interstitial deletions within the PAR1 locus [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/15,83,101\" class=\"abstract_t\">15,83,101</a>]. Both chromosome translocations and the interstitial deletion lead to <em>CRLF2</em> overexpression due to juxtaposition with the enhancer of <em>IGH</em> or the promoter of the <em>P2RY8</em> gene. Rarely, gain-of-function point mutations of the <em>CRLF2</em> gene or gain of extra copies of the <em>CRLF2</em> locus can result in its overexpression. In approximately 30 to 55 percent of these patients, mutations in <em>JAK2 </em>or<em> JAK1 </em>or in<em> IL7R, FLT3, SH2B3, </em>or<em> NRAS </em>are detected, indicative of the cooperation of these kinase-activating lesions with <em>CRLF2</em> rearrangement in leukemogenesis [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/15,108\" class=\"abstract_t\">15,108</a>]. In addition, deletions involving part or all of the <em>IKZF1</em> locus, the <em>PAX5</em> locus, and the <em>EBF1</em> gene are frequently detected in these patients. Almost all patients with Down syndrome-related B cell ALL will have a deletion of <em>IKZF1</em> [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/83,108\" class=\"abstract_t\">83,108</a>].</p><p>In the remaining BCR-ABL1-like B cell ALL cases with no <em>CRLF2</em> overexpression, other tyrosine kinase targets, such as <em>ABL1, ABL2, JAK2, EPOR, </em>and<em> PDGFRB, </em>are activated via various translocations and fusions, such as the <em>NUP214-ABL1, BCR-JAK2, STRB3-JAK2, IGH-EPOR, </em>and<em> EBF1-PDGFRB</em> fusions [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/15,113\" class=\"abstract_t\">15,113</a>]. In addition, mutations of <em>FLT3, CREBBP, IL7R, </em>or <em>SH2B3</em> (<em>LNK</em>) occur frequently in these patients.</p><p class=\"headingAnchor\" id=\"H4205269779\"><span class=\"h3\">Recurring gene fusions</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Up to 30 percent of B-ALLs are not defined by current WHO genetic subgroups (ie, they do not have recurring chromosome abnormalities, kinase activating gene fusions, or Ph-like B-ALL). Next generation sequencing of RNA and other genomic studies have revealed several recurring genetic abnormalities in this category of B-ALL in adolescent and young adult patients. Examples include:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong><em>MEF2D</em></strong><em>-</em><strong><em>BCL9</em></strong><strong> fusion</strong> - The myocyte enhancer factor 2D (<em>MEF2D</em>) gene encodes a transcription factor that participates in neuronal development and myogenesis. Fusion of <em>MEF2D</em> with <em>BCL9</em> (or one of a few other genes) was detected in 5 to 7 percent of patients with B-ALL, mostly in adolescents [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/114,115\" class=\"abstract_t\">114,115</a>]. B-ALL with the <em>MEF2D-BCL9</em> fusion is characterized by a distinct immunophenotype, similar to that of ALL with the t(1;19), markedly high expression of <em>HDAC9</em>, resistance to chemotherapy, early relapse, and a poor outcome. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong><em>DUX4</em></strong><em>-</em><strong><em>ERG</em></strong><strong> or </strong><strong><em>DUX</em></strong><em>-</em><strong><em>IGH</em></strong><strong> fusion</strong> - The double homeobox 4 gene (<em>DUX4</em>) is located near the telomere of the long arm of chromosome 4, and encodes the transcriptional activator PITX1. <em>DUX4-IGH</em> or <em>DUX4-ERG</em> fusions were detected in about 4 to 7 percent of pediatric and young patients with B-ALL [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/116-119\" class=\"abstract_t\">116-119</a>]. The <em>DUX4-IGH</em> translocation leads to overexpression of an altered DUX4 protein with an aberrant carboxy terminus, which induces B cell leukemia in mice. Nearly all patients with <em>DUX4</em> aberrations also show deletion of <em>ERG</em>, indicative of the cooperation of <em>DUX4</em> and <em>ERG</em> in mediating leukemogenesis. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><strong><em>EP300</em></strong><em>-</em><strong><em>ZNF384</em></strong><strong> and</strong> <strong><em>CREBBP</em></strong>-<strong><em>ZNF384</em></strong><strong> fusions</strong> - The zinc finger 384 protein (ZNF384) regulates expression of genes encoding extracellular matrix proteins. An <em>EP300-ZNF384</em> or <em>CREBBP-ZNF384</em> fusion is detected in 7 to 12 percent of adolescent and young adult patients with B-ALL; the disease is CD10-negative, and is characterized by high expression of <em>GATA3</em>, <em>CEBPA</em>, and <em>CEBPB</em>, upregulated JAK-STAT signaling, and a relatively low frequency of mutations or deletions of B cell development-related genes, such as <em>IKZF1</em>, <em>PAX5</em>, <em>RUNX1</em>, <em>ETV6</em>, or cell cycle regulators <em><span class=\"nowrap\">CDKN2A/CDKN2B</em></span> [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/119,120\" class=\"abstract_t\">119,120</a>].</p><p/><p class=\"headingAnchor\" id=\"H14\"><span class=\"h1\">T CELL ACUTE LYMPHOBLASTIC LEUKEMIA</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Patients with T cell ALL are most often young males. They have a number of unfavorable presenting characteristics, including a mediastinal tumor mass, high white blood cell count <span class=\"nowrap\">(&ge;50,000/microL),</span> and leukemic cells in the cerebrospinal fluid [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/121-123\" class=\"abstract_t\">121-123</a>]. These characteristics are also associated with T cell lymphoblastic lymphoma, a related T cell malignancy. (See <a href=\"topic.htm?path=clinical-manifestations-pathologic-features-and-diagnosis-of-precursor-t-cell-acute-lymphoblastic-leukemia-lymphoma\" class=\"medical medical_review\">&quot;Clinical manifestations, pathologic features, and diagnosis of precursor T cell acute lymphoblastic leukemia/lymphoma&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H15\"><span class=\"h2\">Genetic abnormalities</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Among children with T-ALL, approximately 60 percent have an abnormal karyotype [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/122\" class=\"abstract_t\">122</a>]. These patients have a distinct pattern of recurring karyotypic abnormalities involving both T cell receptor (<em>TCR</em>) and non-<em>TCR</em> gene loci [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/121,122,124-126\" class=\"abstract_t\">121,122,124-126</a>].</p><p class=\"headingAnchor\" id=\"H16\"><span class=\"h3\">TCR gene rearrangements</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The most common rearrangements involve the proximal bands of chromosome 14 (14q11.2) and two regions of chromosome 7 (7q34 and 7p14). These rearrangements occur at the locations of the T cell receptor (<em>TCR</em>) genes [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/127,128\" class=\"abstract_t\">127,128</a>]:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>TCR <span class=\"nowrap\">alpha/delta</span> chain at 14q11.2 (<em>TRA</em><span class=\"nowrap\">/<em>TRD</em>)</span></p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>TCR beta chain at 7q34 (<em>TRB</em>)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>TCR gamma chain at 7p14 (<em>TRG</em>)</p><p/><p>With few exceptions, the involved gene on the partner chromosome encodes a cell cycle inhibitor or a transcription factor whose expression is deregulated or activated as a result of the rearrangement [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/121,125,129\" class=\"abstract_t\">121,125,129</a>].</p><p>Oncogenes typically involved in <em>TCR</em> gene rearrangements may be overexpressed by other mechanisms. As an example, two children with X-linked severe combined immunodeficiency who received gene therapy using a provirus developed a T cell acute leukemia-like syndrome [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/126\" class=\"abstract_t\">126</a>]. The provirus was integrated within the <em>LMO2</em> gene locus (11p13), which encodes one of the oncogenes frequently involved in <em>TCR</em> gene rearrangements (eg, t(11;14), t(7;11)). (See <a href=\"topic.htm?path=hematopoietic-cell-transplantation-for-primary-immunodeficiency\" class=\"medical medical_review\">&quot;Hematopoietic cell transplantation for primary immunodeficiency&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H17\"><span class=\"h3\">Non-TCR loci</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A variety of karyotypic <span class=\"nowrap\">and/or</span> mutational abnormalities can occur in T cell ALL that do not involve the T cell receptor (TCR) loci, including [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/122\" class=\"abstract_t\">122</a>]:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>del(6q)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>t(11q23.3) (<em>KMT2A</em> gene)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>t(14q32.3)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Trisomy 8</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><span class=\"nowrap\">t(10;11)/<em>PICALM-MLLT10</em></span> fusion [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/130\" class=\"abstract_t\">130</a>]</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Gain-of-function <em>NOTCH1</em> mutations or loss-of-function <em>FBXW7</em> mutations [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/131-135\" class=\"abstract_t\">131-135</a>]</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span><em>JAK1</em> or <em>JAK3</em> mutations [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/136,137\" class=\"abstract_t\">136,137</a>]</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Overexpression of the <em>TLX1</em> (<em>HOX11</em>) oncogenic transcription factor [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/138-140\" class=\"abstract_t\">138-140</a>]</p><p/><p>NOTCH1 signalling is necessary for commitment to the T cell lineage and for the proliferation of T cell progenitors in the thymus. It has been estimated that activating mutations of <em>NOTCH1</em> are present in more than 50 percent of patients with T cell ALL [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/131\" class=\"abstract_t\">131</a>]. Although the precise mechanisms by which NOTCH1 causes T cell ALL are not yet fully elucidated, studies suggest interaction between NOTCH1 and components of the polycomb repressive complex 2 (PRC2), a highly conserved complex that influences stem cell renewal by epigenetic repression of genes involved in cell fate decisions [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/141\" class=\"abstract_t\">141</a>]. Deletion of the <em>NOTCH1</em> promoter can also result in activation of the <em>Notch1</em> gene from cryptic initiation sites [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/142,143\" class=\"abstract_t\">142,143</a>]. Spontaneous deletion of the <em>NOTCH1</em> locus is seen in approximately 75 percent of <em>IKZF1</em>-deficient T cell ALL cases [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/142\" class=\"abstract_t\">142</a>]. Patients with T cell ALL demonstrating a <em>NOTCH1</em> mutation or a related mutation in <em>FBXW7</em> may represent a subset with superior event-free and overall survival rates when compared with other patients with T cell ALL [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/135,144-146\" class=\"abstract_t\">135,144-146</a>].</p><p>MicroRNAs (miRNAs) are a novel class of small noncoding RNAs that modulate the expression of genes at the post-transcriptional level, and are involved in cancer, apoptosis, and cell metabolism. They may have particular relevance to T cell ALL. Expression profiles in human T-ALL have identified a small number of abundantly expressed miRNAs that target a set of tumor suppressor genes (<em>IKZF1, PTEN, BIM, NF1, FBXW7, </em>and<em> PHF</em>) and promote T-ALL in mice [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/147\" class=\"abstract_t\">147</a>]. Specifically, <em>mir-223</em> is differentially upregulated in T cell ALL and appears to promote NOTCH1-driven leukemia.</p><p>Recurrent mutations of loci located on the X chromosome have been identified in patients with T cell ALL and offered as a possible explanation for the increased incidence of this disease in males. As an example, inactivating mutations or deletions in the plant homeodomain finger 6 (<em>PHF6</em>) locus located on the X chromosome are found in 16 percent and 38 percent of primary T cell ALL in children and adults, respectively [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/148\" class=\"abstract_t\">148</a>]. Loss of <em>PHF6</em> is associated with aberrant expression of the <em>TLX1</em> (<em>HOX11</em>) transcription factor oncogene. Similarly, a small percentage of cases will demonstrate mutations in the ribosomal gene <em>RPL10</em> located on the X chromosome [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/149\" class=\"abstract_t\">149</a>]. However, mutations in many of these loci (eg, <em>PHF6, RPL10</em>) cannot be used to explain the difference in incidence between males and females because they are located in regions that undergo X inactivation in females. In contrast, at least part of the sex predilection may be explained by mutations in <em>UTX</em> (also known as <em>KDM6A</em>), a tumor suppressor located in a region of chromosome X that escapes X inactivation in females [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/150,151\" class=\"abstract_t\">150,151</a>]. (See <a href=\"topic.htm?path=clinical-manifestations-pathologic-features-and-diagnosis-of-precursor-t-cell-acute-lymphoblastic-leukemia-lymphoma#H732703\" class=\"medical medical_review\">&quot;Clinical manifestations, pathologic features, and diagnosis of precursor T cell acute lymphoblastic leukemia/lymphoma&quot;, section on 'Epidemiologic data'</a>.)</p><p>Up to 15 percent of T cell ALL lack expression of CD1a and CD8 and have weak expression of CD5 [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/152\" class=\"abstract_t\">152</a>]. This subtype, termed &ldquo;early T cell precursor ALL&rdquo;, has a significantly higher rate of treatment failure. Whole genome sequencing of tumor cells from 12 patients with early T cell precursor ALL demonstrated a high frequency of somatic mutations involving the pathways of normal hematopoietic development, RAS signalling, and histone modification, similar to those more frequently involved in acute myeloid leukemias [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/153\" class=\"abstract_t\">153</a>]. &#160; &#160;</p><p class=\"headingAnchor\" id=\"H18\"><span class=\"h2\">Prognosis</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Both children and adults with T cell ALL are generally treated with high-risk protocols and often have favorable outcomes [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/122,154\" class=\"abstract_t\">122,154</a>]. At least in children, the type or presence of karyotypic abnormalities does not appear to affect prognosis, as patients with any of the above translocations have an outcome similar to those with normal karyotype [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/122\" class=\"abstract_t\">122</a>]. One study found that T cell ALL with low expression of both <em>ERG</em> and <em>BAALC</em> genes by RT-PCR is associated with significantly lower relapse rates (17 versus 60 percent) and higher four-year overall survival (69 versus 30 percent) [<a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/155\" class=\"abstract_t\">155</a>]. How to apply this information clinically is unclear at this time.</p><p class=\"headingAnchor\" id=\"PATIENT_INFORMATION\"><span class=\"h1\">INFORMATION FOR PATIENTS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>UpToDate offers two types of patient education materials, &ldquo;The Basics&rdquo; and &ldquo;Beyond the Basics.&rdquo; The Basics patient education pieces are written in plain language, at the 5<sup>th</sup> to 6<sup>th</sup> grade reading level, and they answer the four or five key questions a patient might have about a given condition. These articles are best for patients who want a general overview and who prefer short, easy-to-read materials. Beyond the Basics patient education pieces are longer, more sophisticated, and more detailed. These articles are written at the 10<sup>th</sup> to 12<sup>th</sup> grade reading level and are best for patients who want in-depth information and are comfortable with some medical jargon.</p><p>Here are the patient education articles that are relevant to this topic. We encourage you to print or e-mail these topics to your patients. (You can also locate patient education articles on a variety of subjects by searching on &ldquo;patient info&rdquo; and the keyword(s) of interest.) </p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Beyond the Basics topics (see <a href=\"topic.htm?path=acute-lymphoblastic-leukemia-all-treatment-in-adults-beyond-the-basics\" class=\"medical medical_patient\">&quot;Patient education: Acute lymphoblastic leukemia (ALL) treatment in adults (Beyond the Basics)&quot;</a>)</p><p/><p class=\"headingAnchor\" id=\"H1460654043\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Cytogenetic analysis and molecular cytogenetic studies, such as fluorescence in situ hybridization (FISH), reveal recurring chromosome abnormalities in approximately 80 percent of B cell acute lymphoblastic leukemia (ALL), including numerical and structural changes, such as translocations, inversions, or deletions. There are substantial differences between children and adults with ALL in the frequencies of recurring abnormalities. (See <a href=\"#H2\" class=\"local\">'B cell acute lymphoblastic leukemia'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The t(12;21), resulting in the fusion <em><span class=\"nowrap\">ETV6/RUNX1</em></span> gene, is detectable by FISH or polymerase chain reaction analysis in about 25 percent of children with B-lineage ALL and 3 percent of adults with ALL. Patients with ALL and the t(12;21) generally have a favorable prognosis. (See <a href=\"#H1280995514\" class=\"local\">'12;21 translocation in precursor B ALL'</a> above.) </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The t(8;14), present in a high proportion of Burkitt cell <span class=\"nowrap\">leukemia/lymphoma,</span> occurs in approximately 1 percent of adults with ALL. This translocation is associated with lymphoblasts with deeply basophilic cytoplasm with prominent vacuoles, a high incidence of central nervous system involvement at diagnosis, and a poorer prognosis than any other group of patients classified by chromosomal abnormalities. (See <a href=\"#H4\" class=\"local\">'8;14 translocation'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The t(4;11) is present in up to 60 percent of infants with ALL younger than 12 months, but is rarely observed in adult patients with ALL. This translocation results in a <em>KMT2A</em>-<em>AFF1</em> fusion gene, and is associated with a poor prognosis in both adults and children. ALL with the t(4;11) is commonly characterized by high leukocyte counts, an immature immunophenotype, B cell lineage, and frequent co-expression of myeloid antigens. (See <a href=\"#H5\" class=\"local\">'4;11 translocation'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Translocations involving 11q23.3 are seen in 60 to 80 percent of ALL in infants, and cause abnormalities of <em>KMT2A</em>, which encodes a histone methyltransferase that regulates gene transcription via chromatin remodeling. Leukemias with <em>KMT2A</em> translocations have a characteristic and distinctive gene expression profile. ALL in infants and children with <em>KMT2A</em> translocations is associated with a high rate of early treatment failure and a very poor outcome. (See <a href=\"#H351299836\" class=\"local\">'11q23.3 translocation in infants'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The t(9;22) that produces the Philadelphia chromosome is observed in about 2 to 5 percent of children and about 30 percent of adults. This translocation is associated with a poor prognosis. Treatment of patients with the t(9;22) incorporates tyrosine kinase inhibitors that target the BCR-ABL1 protein, which is the fusion product of the Philadelphia chromosome. (See <a href=\"#H6\" class=\"local\">'9;22 translocation'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The t(1;19) occurs in approximately 30 percent of patients with pre-B cell childhood ALL and less commonly in other B lineage ALLs in children and adults. In the past, patients with the t(1;19) typically had early treatment failure. However, this adverse prognosis can be overcome by more intensive chemotherapy such that t(1;19) is now associated with a favorable prognosis. (See <a href=\"#H7\" class=\"local\">'1;19 translocation in pre-B cell ALL'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Patients who have hyperdiploidy with more than 50 chromosomes often have a good prognosis. The individual structural abnormalities do not appear to influence outcome in patients with hyperdiploidy except for the t(9;22), which is associated with a poor prognosis. (See <a href=\"#H9\" class=\"local\">'Hyperdiploidy'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Approximately 5 to 6 percent of ALL patients, independent of age, have clonal loss of various chromosomes, resulting in a hypodiploid clone with fewer than 46 chromosomes. ALL patients with hypodiploid clones generally have a poor prognosis, especially patients with near-haploid and low-hypodiploid clones. Similarly, deletion of 9p is an unfavorable risk factor associated with a high rate of relapse in precursor-B ALL in children. (See <a href=\"#H10\" class=\"local\">'Hypodiploidy'</a> above and <a href=\"#H2233889466\" class=\"local\">'9p abnormalities in ALL'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>BCR-ABL1-like (also known as Ph-like) B cell ALL has a gene expression profile that resembles B cell ALL with the <span class=\"nowrap\">t(9;22)/<em>BCR-ABL1</em></span> fusion, but the fusion oncogene is not present. This phenotype is detected in 10 to 15 percent of pediatric patients and up to 28 percent of adult patients with B cell ALL, is resistant to standard chemotherapy, and is associated with a poor prognosis. Translocations involving <em>CRLF2</em>, or <span class=\"nowrap\">mutations/rearrangements</span> of other tyrosine kinase targets (eg, <em>ABL1, ABL2, JAK2, PDGFRB</em>)<em> </em>activate ABL1, JAK-STAT, or RAS pathways that are involved in B cell development, proliferation, and differentiation. Some of these cases of ALL are sensitive to tyrosine kinase inhibitors or JAK kinase inhibitors. (See <a href=\"#H73697009\" class=\"local\">'BCR-ABL1-like B cell ALL'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Up to 30 percent of B-ALLs are not defined by existing WHO subgroups (ie, they do not have recurring chromosome abnormalities, kinase activating gene fusions, or Ph-like B-ALL). Several recurring genetic abnormalities in this category of B-ALL in adolescent and young adult patients have been identified recently, including the <em>MEF2D-BCL9</em>, <em>DUX4-ERG</em> or <em>DUX-IGH, and EP300-ZNF384</em> and <em>CREBBP</em>-<em>ZNF384</em> fusions. (See <a href=\"#H4205269779\" class=\"local\">'Recurring gene fusions'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Among children with T cell ALL, approximately 60 percent have an abnormal karyotype. These patients have a distinct pattern of recurring karyotypic abnormalities involving both T cell receptor (<em>TCR</em>) and non-<em>TCR</em> gene loci, including activating mutations of <em>NOTCH1</em> in more than 50 percent. Both children and adults with T cell ALL are generally treated with high-risk protocols and often have favorable outcomes. (See <a href=\"#H14\" class=\"local\">'T cell acute lymphoblastic leukemia'</a> above.)</p></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/1\" class=\"nounderline abstract_t\">Faderl S, Kantarjian HM, Talpaz M, Estrov Z. Clinical significance of cytogenetic abnormalities in adult acute lymphoblastic leukemia. Blood 1998; 91:3995.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/2\" class=\"nounderline abstract_t\">Cytogenetic abnormalities in adult acute lymphoblastic leukemia: correlations with hematologic findings outcome. A Collaborative Study of the Group Fran&ccedil;ais de Cytog&eacute;n&eacute;tique H&eacute;matologique. Blood 1996; 87:3135.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/3\" class=\"nounderline abstract_t\">Pui CH, Relling MV, Downing JR. Acute lymphoblastic leukemia. N Engl J Med 2004; 350:1535.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/4\" class=\"nounderline abstract_t\">Pui CH, Robison LL, Look AT. Acute lymphoblastic leukaemia. Lancet 2008; 371:1030.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/5\" class=\"nounderline abstract_t\">Arber DA, Orazi A, Hasserjian R, et al. The 2016 revision to the World Health Organization classification of myeloid neoplasms and acute leukemia. Blood 2016; 127:2391.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/6\" class=\"nounderline abstract_t\">Moorman AV, Chilton L, Wilkinson J, et al. A population-based cytogenetic study of adults with acute lymphoblastic leukemia. Blood 2010; 115:206.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/7\" class=\"nounderline abstract_t\">Harrison CJ, Moorman AV, Barber KE, et al. Interphase molecular cytogenetic screening for chromosomal abnormalities of prognostic significance in childhood acute lymphoblastic leukaemia: a UK Cancer Cytogenetics Group Study. Br J Haematol 2005; 129:520.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/8\" class=\"nounderline abstract_t\">Wetzler M, Dodge RK, Mr&oacute;zek K, et al. Prospective karyotype analysis in adult acute lymphoblastic leukemia: the cancer and leukemia Group B experience. Blood 1999; 93:3983.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/9\" class=\"nounderline abstract_t\">Ribeiro RC, Abromowitch M, Raimondi SC, et al. Clinical and biologic hallmarks of the Philadelphia chromosome in childhood acute lymphoblastic leukemia. Blood 1987; 70:948.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/10\" class=\"nounderline abstract_t\">Russo C, Carroll A, Kohler S, et al. Philadelphia chromosome and monosomy 7 in childhood acute lymphoblastic leukemia: a Pediatric Oncology Group study. Blood 1991; 77:1050.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/11\" class=\"nounderline abstract_t\">Moorman AV, Harrison CJ, Buck GA, et al. Karyotype is an independent prognostic factor in adult acute lymphoblastic leukemia (ALL): analysis of cytogenetic data from patients treated on the Medical Research Council (MRC) UKALLXII/Eastern Cooperative Oncology Group (ECOG) 2993 trial. Blood 2007; 109:3189.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/12\" class=\"nounderline abstract_t\">Westbrook CA, Hooberman AL, Spino C, et al. Clinical significance of the BCR-ABL fusion gene in adult acute lymphoblastic leukemia: a Cancer and Leukemia Group B Study (8762). Blood 1992; 80:2983.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/13\" class=\"nounderline abstract_t\">Moorman AV, Ensor HM, Richards SM, et al. Prognostic effect of chromosomal abnormalities in childhood B-cell precursor acute lymphoblastic leukaemia: results from the UK Medical Research Council ALL97/99 randomised trial. Lancet Oncol 2010; 11:429.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/14\" class=\"nounderline abstract_t\">Hilden JM, Dinndorf PA, Meerbaum SO, et al. Analysis of prognostic factors of acute lymphoblastic leukemia in infants: report on CCG 1953 from the Children's Oncology Group. Blood 2006; 108:441.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/15\" class=\"nounderline abstract_t\">Roberts KG, Li Y, Payne-Turner D, et al. Targetable kinase-activating lesions in Ph-like acute lymphoblastic leukemia. N Engl J Med 2014; 371:1005.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/16\" class=\"nounderline abstract_t\">Perez-Andreu V, Roberts KG, Xu H, et al. A genome-wide association study of susceptibility to acute lymphoblastic leukemia in adolescents and young adults. Blood 2015; 125:680.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/17\" class=\"nounderline abstract_t\">Mancini M, Scappaticci D, Cimino G, et al. A comprehensive genetic classification of adult acute lymphoblastic leukemia (ALL): analysis of the GIMEMA 0496 protocol. Blood 2005; 105:3434.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/18\" class=\"nounderline abstract_t\">Hoelzer D, Ludwig WD, Thiel E, et al. Improved outcome in adult B-cell acute lymphoblastic leukemia. Blood 1996; 87:495.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/19\" class=\"nounderline abstract_t\">Shurtleff SA, Buijs A, Behm FG, et al. TEL/AML1 fusion resulting from a cryptic t(12;21) is the most common genetic lesion in pediatric ALL and defines a subgroup of patients with an excellent prognosis. Leukemia 1995; 9:1985.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/20\" class=\"nounderline abstract_t\">Romana SP, Poirel H, Leconiat M, et al. High frequency of t(12;21) in childhood B-lineage acute lymphoblastic leukemia. Blood 1995; 86:4263.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/21\" class=\"nounderline abstract_t\">Rubnitz JE, Downing JR, Pui CH, et al. TEL gene rearrangement in acute lymphoblastic leukemia: a new genetic marker with prognostic significance. J Clin Oncol 1997; 15:1150.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/22\" class=\"nounderline abstract_t\">Borkhardt A, Cazzaniga G, Viehmann S, et al. Incidence and clinical relevance of TEL/AML1 fusion genes in children with acute lymphoblastic leukemia enrolled in the German and Italian multicenter therapy trials. Associazione Italiana Ematologia Oncologia Pediatrica and the Berlin-Frankfurt-M&uuml;nster Study Group. Blood 1997; 90:571.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/23\" class=\"nounderline abstract_t\">McLean TW, Ringold S, Neuberg D, et al. TEL/AML-1 dimerizes and is associated with a favorable outcome in childhood acute lymphoblastic leukemia. Blood 1996; 88:4252.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/24\" class=\"nounderline abstract_t\">Aguiar RC, Sohal J, van Rhee F, et al. TEL-AML1 fusion in acute lymphoblastic leukaemia of adults. M.R.C. Adult Leukaemia Working Party. Br J Haematol 1996; 95:673.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/25\" class=\"nounderline abstract_t\">Golub TR, Barker GF, Bohlander SK, et al. Fusion of the TEL gene on 12p13 to the AML1 gene on 21q22 in acute lymphoblastic leukemia. Proc Natl Acad Sci U S A 1995; 92:4917.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/26\" class=\"nounderline abstract_t\">Lopez RG, Carron C, Oury C, et al. TEL is a sequence-specific transcriptional repressor. J Biol Chem 1999; 274:30132.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/27\" class=\"nounderline abstract_t\">Stegmaier K, Pendse S, Barker GF, et al. Frequent loss of heterozygosity at the TEL gene locus in acute lymphoblastic leukemia of childhood. Blood 1995; 86:38.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/28\" class=\"nounderline abstract_t\">Golub TR, Barker GF, Lovett M, Gilliland DG. Fusion of PDGF receptor beta to a novel ets-like gene, tel, in chronic myelomonocytic leukemia with t(5;12) chromosomal translocation. Cell 1994; 77:307.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/29\" class=\"nounderline abstract_t\">Ritchie KA, Aprikyan AA, Bowen-Pope DF, et al. The Tel-PDGFRbeta fusion gene produces a chronic myeloproliferative syndrome in transgenic mice. Leukemia 1999; 13:1790.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/30\" class=\"nounderline abstract_t\">Mori H, Colman SM, Xiao Z, et al. Chromosome translocations and covert leukemic clones are generated during normal fetal development. Proc Natl Acad Sci U S A 2002; 99:8242.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/31\" class=\"nounderline abstract_t\">Greaves MF, Maia AT, Wiemels JL, Ford AM. Leukemia in twins: lessons in natural history. Blood 2003; 102:2321.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/32\" class=\"nounderline abstract_t\">Ford AM, Palmi C, Bueno C, et al. The TEL-AML1 leukemia fusion gene dysregulates the TGF-beta pathway in early B lineage progenitor cells. J Clin Invest 2009; 119:826.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/33\" class=\"nounderline abstract_t\">Roman-Gomez J, Jimenez-Velasco A, Agirre X, et al. CpG island methylator phenotype redefines the prognostic effect of t(12;21) in childhood acute lymphoblastic leukemia. Clin Cancer Res 2006; 12:4845.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/34\" class=\"nounderline abstract_t\">Rubin CM, Le Beau MM, Mick R, et al. Impact of chromosomal translocations on prognosis in childhood acute lymphoblastic leukemia. J Clin Oncol 1991; 9:2183.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/35\" class=\"nounderline abstract_t\">Pui CH, Frankel LS, Carroll AJ, et al. Clinical characteristics and treatment outcome of childhood acute lymphoblastic leukemia with the t(4;11)(q21;q23): a collaborative study of 40 cases. Blood 1991; 77:440.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/36\" class=\"nounderline abstract_t\">Chen CS, Sorensen PH, Domer PH, et al. Molecular rearrangements on chromosome 11q23 predominate in infant acute lymphoblastic leukemia and are associated with specific biologic variables and poor outcome. Blood 1993; 81:2386.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/37\" class=\"nounderline abstract_t\">Rubnitz JE, Link MP, Shuster JJ, et al. Frequency and prognostic significance of HRX rearrangements in infant acute lymphoblastic leukemia: a Pediatric Oncology Group study. Blood 1994; 84:570.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/38\" class=\"nounderline abstract_t\">Hilden JM, Frestedt JL, Moore RO, et al. Molecular analysis of infant acute lymphoblastic leukemia: MLL gene rearrangement and reverse transcriptase-polymerase chain reaction for t(4; 11)(q21; q23). Blood 1995; 86:3876.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/39\" class=\"nounderline abstract_t\">Bitter MA, Le Beau MM, Rowley JD, et al. Associations between morphology, karyotype, and clinical features in myeloid leukemias. Hum Pathol 1987; 18:211.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/40\" class=\"nounderline abstract_t\">Cimino G, Elia L, Rapanotti MC, et al. A prospective study of residual-disease monitoring of the ALL1/AF4 transcript in patients with t(4;11) acute lymphoblastic leukemia. Blood 2000; 95:96.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/41\" class=\"nounderline abstract_t\">Cimino G, Elia L, Mancini M, et al. Clinico-biologic features and treatment outcome of adult pro-B-ALL patients enrolled in the GIMEMA 0496 study: absence of the ALL1/AF4 and of the BCR/ABL fusion genes correlates with a significantly better clinical outcome. Blood 2003; 102:2014.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/42\" class=\"nounderline abstract_t\">Cimino G, Lo Coco F, Biondi A, et al. ALL-1 gene at chromosome 11q23 is consistently altered in acute leukemia of early infancy. Blood 1993; 82:544.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/43\" class=\"nounderline abstract_t\">Meyer C, Kowarz E, Hofmann J, et al. New insights to the MLL recombinome of acute leukemias. Leukemia 2009; 23:1490.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/44\" class=\"nounderline abstract_t\">Thirman MJ, Gill HJ, Burnett RC, et al. Rearrangement of the MLL gene in acute lymphoblastic and acute myeloid leukemias with 11q23 chromosomal translocations. N Engl J Med 1993; 329:909.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/45\" class=\"nounderline abstract_t\">Djabali M, Selleri L, Parry P, et al. A trithorax-like gene is interrupted by chromosome 11q23 translocations in acute leukaemias. Nat Genet 1992; 2:113.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/46\" class=\"nounderline abstract_t\">Sun L, Heerema N, Crotty L, et al. Expression of dominant-negative and mutant isoforms of the antileukemic transcription factor Ikaros in infant acute lymphoblastic leukemia. Proc Natl Acad Sci U S A 1999; 96:680.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/47\" class=\"nounderline abstract_t\">Armstrong SA, Staunton JE, Silverman LB, et al. MLL translocations specify a distinct gene expression profile that distinguishes a unique leukemia. Nat Genet 2002; 30:41.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/48\" class=\"nounderline abstract_t\">Yeoh EJ, Williams K, Patel D, et al. Expression profiling of pediatric acute lymphoblastic leukemia (ALL) blasts at diagnosis accurately predicts both the risk of relapse and of developing therapy-induced acute myeloid leukemia (abstract). Blood 2001; 98:433a.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/49\" class=\"nounderline abstract_t\">Cobaleda C, Guti&eacute;rrez-Cianca N, P&eacute;rez-Losada J, et al. A primitive hematopoietic cell is the target for the leukemic transformation in human philadelphia-positive acute lymphoblastic leukemia. Blood 2000; 95:1007.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/50\" class=\"nounderline abstract_t\">Rambaldi A, Attuati V, Bassan R, et al. Molecular diagnosis and clinical relevance of t(9;22), t(4;11) and t(1 ;19) chromosome abnormalities in a consecutive group of 141 adult patients with acute lymphoblastic leukemia. Leuk Lymphoma 1996; 21:457.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/51\" class=\"nounderline abstract_t\">Czuczman MS, Dodge RK, Stewart CC, et al. Value of immunophenotype in intensively treated adult acute lymphoblastic leukemia: cancer and leukemia Group B study 8364. Blood 1999; 93:3931.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/52\" class=\"nounderline abstract_t\">Campbell LJ, Martinow A, Michael PM, et al. Correlation of cytogenetics, BCR-ABL PCR studies and fluorescence in situ hybridisation (FISH) in adult acute lymphoblastic leukaemia. Aust N Z J Med 1999; 29:707.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/53\" class=\"nounderline abstract_t\">Wetzler M, Dodge RK, Mr&oacute;zek K, et al. Additional cytogenetic abnormalities in adults with Philadelphia chromosome-positive acute lymphoblastic leukaemia: a study of the Cancer and Leukaemia Group B. Br J Haematol 2004; 124:275.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/54\" class=\"nounderline abstract_t\">Secker-Walker LM, Craig JM. Prognostic implications of breakpoint and lineage heterogeneity in Philadelphia-positive acute lymphoblastic leukemia: a review. Leukemia 1993; 7:147.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/55\" class=\"nounderline abstract_t\">Anastasi J, Feng J, Dickstein JI, et al. Lineage involvement by BCR/ABL in Ph+ lymphoblastic leukemias: chronic myelogenous leukemia presenting in lymphoid blast vs Ph+ acute lymphoblastic leukemia. Leukemia 1996; 10:795.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/56\" class=\"nounderline abstract_t\">Hermans A, Heisterkamp N, von Linden M, et al. Unique fusion of bcr and c-abl genes in Philadelphia chromosome positive acute lymphoblastic leukemia. Cell 1987; 51:33.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/57\" class=\"nounderline abstract_t\">Fainstein E, Marcelle C, Rosner A, et al. A new fused transcript in Philadelphia chromosome positive acute lymphocytic leukaemia. Nature 1987; 330:386.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/58\" class=\"nounderline abstract_t\">Clark SS, McLaughlin J, Timmons M, et al. Expression of a distinctive BCR-ABL oncogene in Ph1-positive acute lymphocytic leukemia (ALL). Science 1988; 239:775.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/59\" class=\"nounderline abstract_t\">van Etten RA. Disease progression in a murine model of bcr/abl leukemogenesis. Leuk Lymphoma 1993; 11 Suppl 1:239.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/60\" class=\"nounderline abstract_t\">Daley GQ, Van Etten RA, Baltimore D. Induction of chronic myelogenous leukemia in mice by the P210bcr/abl gene of the Philadelphia chromosome. Science 1990; 247:824.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/61\" class=\"nounderline abstract_t\">Heisterkamp N, Jenster G, ten Hoeve J, et al. Acute leukaemia in bcr/abl transgenic mice. Nature 1990; 344:251.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/62\" class=\"nounderline abstract_t\">Castellanos A, Pintado B, Weruaga E, et al. A BCR-ABL(p190) fusion gene made by homologous recombination causes B-cell acute lymphoblastic leukemias in chimeric mice with independence of the endogenous bcr product. Blood 1997; 90:2168.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/63\" class=\"nounderline abstract_t\">Linker CA, Levitt LJ, O'Donnell M, et al. Treatment of adult acute lymphoblastic leukemia with intensive cyclical chemotherapy: a follow-up report. Blood 1991; 78:2814.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/64\" class=\"nounderline abstract_t\">Gaynor J, Chapman D, Little C, et al. A cause-specific hazard rate analysis of prognostic factors among 199 adults with acute lymphoblastic leukemia: the Memorial Hospital experience since 1969. J Clin Oncol 1988; 6:1014.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/65\" class=\"nounderline abstract_t\">Devaraj PE, Foroni L, Janossy G, et al. Expression of the E2A-PBX1 fusion transcripts in t(1;19)(q23;p13) and der(19)t(1;19) at diagnosis and in remission of acute lymphoblastic leukemia with different B lineage immunophenotypes. Leukemia 1995; 9:821.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/66\" class=\"nounderline abstract_t\">Michael PM, Levin MD, Garson OM. Translocation 1;19--a new cytogenetic abnormality in acute lymphocytic leukemia. Cancer Genet Cytogenet 1984; 12:333.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/67\" class=\"nounderline abstract_t\">Pui CH, Raimondi SC, Hancock ML, et al. Immunologic, cytogenetic, and clinical characterization of childhood acute lymphoblastic leukemia with the t(1;19) (q23; p13) or its derivative. J Clin Oncol 1994; 12:2601.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/68\" class=\"nounderline abstract_t\">Crist WM, Carroll AJ, Shuster JJ, et al. Poor prognosis of children with pre-B acute lymphoblastic leukemia is associated with the t(1;19)(q23;p13): a Pediatric Oncology Group study. Blood 1990; 76:117.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/69\" class=\"nounderline abstract_t\">Hunger SP. Chromosomal translocations involving the E2A gene in acute lymphoblastic leukemia: clinical features and molecular pathogenesis. Blood 1996; 87:1211.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/70\" class=\"nounderline abstract_t\">Kamps MP. E2A-Pbx1 induces growth, blocks differentiation, and interacts with other homeodomain proteins regulating normal differentiation. Curr Top Microbiol Immunol 1997; 220:25.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/71\" class=\"nounderline abstract_t\">McWhirter JR, Neuteboom ST, Wancewicz EV, et al. Oncogenic homeodomain transcription factor E2A-Pbx1 activates a novel WNT gene in pre-B acute lymphoblastoid leukemia. Proc Natl Acad Sci U S A 1999; 96:11464.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/72\" class=\"nounderline abstract_t\">Fu X, McGrath S, Pasillas M, et al. EB-1, a tyrosine kinase signal transduction gene, is transcriptionally activated in the t(1;19) subset of pre-B ALL, which express oncoprotein E2a-Pbx1. Oncogene 1999; 18:4920.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/73\" class=\"nounderline abstract_t\">Foa R, Vitale A, Mancini M, et al. E2A-PBX1 fusion in adult acute lymphoblastic leukaemia: biological and clinical features. Br J Haematol 2003; 120:484.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/74\" class=\"nounderline abstract_t\">Harrison CJ, Moorman AV, Schwab C, et al. An international study of intrachromosomal amplification of chromosome 21 (iAMP21): cytogenetic characterization and outcome. Leukemia 2014; 28:1015.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/75\" class=\"nounderline abstract_t\">Strefford JC, van Delft FW, Robinson HM, et al. Complex genomic alterations and gene expression in acute lymphoblastic leukemia with intrachromosomal amplification of chromosome 21. Proc Natl Acad Sci U S A 2006; 103:8167.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/76\" class=\"nounderline abstract_t\">Li Y, Schwab C, Ryan SL, et al. Constitutional and somatic rearrangement of chromosome 21 in acute lymphoblastic leukaemia. Nature 2014; 508:98.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/77\" class=\"nounderline abstract_t\">Moorman AV, Richards SM, Robinson HM, et al. Prognosis of children with acute lymphoblastic leukemia (ALL) and intrachromosomal amplification of chromosome 21 (iAMP21). Blood 2007; 109:2327.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/78\" class=\"nounderline abstract_t\">Heerema NA, Sather HN, Sensel MG, et al. Association of chromosome arm 9p abnormalities with adverse risk in childhood acute lymphoblastic leukemia: A report from the Children's Cancer Group. Blood 1999; 94:1537.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/79\" class=\"nounderline abstract_t\">Mullighan CG, Goorha S, Radtke I, et al. Genome-wide analysis of genetic alterations in acute lymphoblastic leukaemia. Nature 2007; 446:758.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/80\" class=\"nounderline abstract_t\">Coyaud E, Struski S, Prade N, et al. Wide diversity of PAX5 alterations in B-ALL: a Groupe Francophone de Cytogenetique Hematologique study. Blood 2010; 115:3089.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/81\" class=\"nounderline abstract_t\">Bercovich D, Ganmore I, Scott LM, et al. Mutations of JAK2 in acute lymphoblastic leukaemias associated with Down's syndrome. Lancet 2008; 372:1484.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/82\" class=\"nounderline abstract_t\">Gaikwad A, Rye CL, Devidas M, et al. Prevalence and clinical correlates of JAK2 mutations in Down syndrome acute lymphoblastic leukaemia. Br J Haematol 2009; 144:930.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/83\" class=\"nounderline abstract_t\">Mullighan CG, Collins-Underwood JR, Phillips LA, et al. Rearrangement of CRLF2 in B-progenitor- and Down syndrome-associated acute lymphoblastic leukemia. Nat Genet 2009; 41:1243.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/84\" class=\"nounderline abstract_t\">Kuchinskaya E, Heyman M, Nordgren A, et al. Interphase fluorescent in situ hybridization deletion analysis of the 9p21 region and prognosis in childhood acute lymphoblastic leukaemia (ALL): results from a prospective analysis of 519 Nordic patients treated according to the NOPHO-ALL 2000 protocol. Br J Haematol 2011; 152:615.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/85\" class=\"nounderline abstract_t\">Nahi H, H&auml;gglund H, Ahlgren T, et al. An investigation into whether deletions in 9p reflect prognosis in adult precursor B-cell acute lymphoblastic leukemia: a multi-center study of 381 patients. Haematologica 2008; 93:1734.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/86\" class=\"nounderline abstract_t\">Mertens F, Johansson B, Mitelman F. Dichotomy of hyperdiploid acute lymphoblastic leukemia on the basis of the distribution of gained chromosomes. Cancer Genet Cytogenet 1996; 92:8.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/87\" class=\"nounderline abstract_t\">Moorman AV, Clark R, Farrell DM, et al. Probes for hidden hyperdiploidy in acute lymphoblastic leukaemia. Genes Chromosomes Cancer 1996; 16:40.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/88\" class=\"nounderline abstract_t\">Raimondi SC, Pui CH, Hancock ML, et al. Heterogeneity of hyperdiploid (51-67) childhood acute lymphoblastic leukemia. Leukemia 1996; 10:213.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/89\" class=\"nounderline abstract_t\">Dastugue N, Suciu S, Plat G, et al. Hyperdiploidy with 58-66 chromosomes in childhood B-acute lymphoblastic leukemia is highly curable: 58951 CLG-EORTC results. Blood 2013; 121:2415.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/90\" class=\"nounderline abstract_t\">Harrison CJ, Moorman AV, Broadfield ZJ, et al. Three distinct subgroups of hypodiploidy in acute lymphoblastic leukaemia. Br J Haematol 2004; 125:552.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/91\" class=\"nounderline abstract_t\">Holmfeldt L, Wei L, Diaz-Flores E, et al. The genomic landscape of hypodiploid acute lymphoblastic leukemia. Nat Genet 2013; 45:242.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/92\" class=\"nounderline abstract_t\">M&uuml;hlbacher V, Zenger M, Schnittger S, et al. Acute lymphoblastic leukemia with low hypodiploid/near triploid karyotype is a specific clinical entity and exhibits a very high TP53 mutation frequency of 93%. Genes Chromosomes Cancer 2014; 53:524.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/93\" class=\"nounderline abstract_t\">Yoshida T, Ng SY, Zuniga-Pflucker JC, Georgopoulos K. Early hematopoietic lineage restrictions directed by Ikaros. Nat Immunol 2006; 7:382.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/94\" class=\"nounderline abstract_t\">Mullighan CG, Miller CB, Radtke I, et al. BCR-ABL1 lymphoblastic leukaemia is characterized by the deletion of Ikaros. Nature 2008; 453:110.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/95\" class=\"nounderline abstract_t\">Iacobucci I, Lonetti A, Messa F, et al. Expression of spliced oncogenic Ikaros isoforms in Philadelphia-positive acute lymphoblastic leukemia patients treated with tyrosine kinase inhibitors: implications for a new mechanism of resistance. Blood 2008; 112:3847.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/96\" class=\"nounderline abstract_t\">Mullighan CG, Su X, Zhang J, et al. Deletion of IKZF1 and prognosis in acute lymphoblastic leukemia. N Engl J Med 2009; 360:470.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/97\" class=\"nounderline abstract_t\">Roman-Gomez J, Agirre X, Jim&eacute;nez-Velasco A, et al. Epigenetic regulation of microRNAs in acute lymphoblastic leukemia. J Clin Oncol 2009; 27:1316.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/98\" class=\"nounderline abstract_t\">Tosello V, Mansour MR, Barnes K, et al. WT1 mutations in T-ALL. Blood 2009; 114:1038.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/99\" class=\"nounderline abstract_t\">Harvey RC, Mullighan CG, Wang X, et al. Identification of novel cluster groups in pediatric high-risk B-precursor acute lymphoblastic leukemia with gene expression profiling: correlation with genome-wide DNA copy number alterations, clinical characteristics, and outcome. Blood 2010; 116:4874.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/100\" class=\"nounderline abstract_t\">Moorman AV, Enshaei A, Schwab C, et al. A novel integrated cytogenetic and genomic classification refines risk stratification in pediatric acute lymphoblastic leukemia. Blood 2014; 124:1434.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/101\" class=\"nounderline abstract_t\">Moorman AV, Schwab C, Ensor HM, et al. IGH@ translocations, CRLF2 deregulation, and microdeletions in adolescents and adults with acute lymphoblastic leukemia. J Clin Oncol 2012; 30:3100.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/102\" class=\"nounderline abstract_t\">van der Veer A, Zaliova M, Mottadelli F, et al. IKZF1 status as a prognostic feature in BCR-ABL1-positive childhood ALL. Blood 2014; 123:1691.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/103\" class=\"nounderline abstract_t\">Stock W, Tsai T, Golden C, et al. Cell cycle regulatory gene abnormalities are important determinants of leukemogenesis and disease biology in adult acute lymphoblastic leukemia. Blood 2000; 95:2364.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/104\" class=\"nounderline abstract_t\">Hoshino K, Asou N, Okubo T, et al. The absence of the p15INK4B gene alterations in adult patients with precursor B-cell acute lymphoblastic leukaemia is a favourable prognostic factor. Br J Haematol 2002; 117:531.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/105\" class=\"nounderline abstract_t\">Roman-Gomez J, Jimenez-Velasco A, Castillejo JA, et al. Promoter hypermethylation of cancer-related genes: a strong independent prognostic factor in acute lymphoblastic leukemia. Blood 2004; 104:2492.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/106\" class=\"nounderline abstract_t\">Stengel A, Schnittger S, Weissmann S, et al. TP53 mutations occur in 15.7% of ALL and are associated with MYC-rearrangement, low hypodiploidy, and a poor prognosis. Blood 2014; 124:251.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/107\" class=\"nounderline abstract_t\">Russell LJ, Capasso M, Vater I, et al. Deregulated expression of cytokine receptor gene, CRLF2, is involved in lymphoid transformation in B-cell precursor acute lymphoblastic leukemia. Blood 2009; 114:2688.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/108\" class=\"nounderline abstract_t\">Hertzberg L, Vendramini E, Ganmore I, et al. Down syndrome acute lymphoblastic leukemia, a highly heterogeneous disease in which aberrant expression of CRLF2 is associated with mutated JAK2: a report from the International BFM Study Group. Blood 2010; 115:1006.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/109\" class=\"nounderline abstract_t\">Roberts KG, Gu Z, Payne-Turner D, et al. High Frequency and Poor Outcome of Philadelphia Chromosome-Like Acute Lymphoblastic Leukemia in Adults. J Clin Oncol 2017; 35:394.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/110\" class=\"nounderline abstract_t\">Jain N, Roberts KG, Jabbour E, et al. Ph-like acute lymphoblastic leukemia: a high-risk subtype in adults. Blood 2017; 129:572.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/111\" class=\"nounderline abstract_t\">Reshmi SC, Harvey RC, Roberts KG, et al. Targetable kinase gene fusions in high-risk B-ALL: a study from the Children's Oncology Group. Blood 2017; 129:3352.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/112\" class=\"nounderline abstract_t\">Roberts KG, Pei D, Campana D, et al. Outcomes of children with BCR-ABL1&ndash;like acute lymphoblastic leukemia treated with risk-directed therapy based on the levels of minimal residual disease. J Clin Oncol 2014; 32:3012.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/113\" class=\"nounderline abstract_t\">Roberts KG, Morin RD, Zhang J, et al. Genetic alterations activating kinase and cytokine receptor signaling in high-risk acute lymphoblastic leukemia. Cancer Cell 2012; 22:153.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/114\" class=\"nounderline abstract_t\">Gu Z, Churchman M, Roberts K, et al. Genomic analyses identify recurrent MEF2D fusions in acute lymphoblastic leukaemia. Nat Commun 2016; 7:13331.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/115\" class=\"nounderline abstract_t\">Suzuki K, Okuno Y, Kawashima N, et al. MEF2D-BCL9 Fusion Gene Is Associated With High-Risk Acute B-Cell Precursor Lymphoblastic Leukemia in Adolescents. J Clin Oncol 2016; 34:3451.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/116\" class=\"nounderline abstract_t\">Yasuda T, Tsuzuki S, Kawazu M, et al. Recurrent DUX4 fusions in B cell acute lymphoblastic leukemia of adolescents and young adults. Nat Genet 2016; 48:569.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/117\" class=\"nounderline abstract_t\">Zhang J, McCastlain K, Yoshihara H, et al. Deregulation of DUX4 and ERG in acute lymphoblastic leukemia. Nat Genet 2016; 48:1481.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/118\" class=\"nounderline abstract_t\">Lilljebj&ouml;rn H, Henningsson R, Hyrenius-Wittsten A, et al. Identification of ETV6-RUNX1-like and DUX4-rearranged subtypes in paediatric B-cell precursor acute lymphoblastic leukaemia. Nat Commun 2016; 7:11790.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/119\" class=\"nounderline abstract_t\">Liu YF, Wang BY, Zhang WN, et al. Genomic Profiling of Adult and Pediatric B-cell Acute Lymphoblastic Leukemia. EBioMedicine 2016; 8:173.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/120\" class=\"nounderline abstract_t\">Qian M, Zhang H, Kham SK, et al. Whole-transcriptome sequencing identifies a distinct subtype of acute lymphoblastic leukemia with predominant genomic abnormalities of EP300 and CREBBP. Genome Res 2017; 27:185.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/121\" class=\"nounderline abstract_t\">Uckun FM, Sensel MG, Sun L, et al. Biology and treatment of childhood T-lineage acute lymphoblastic leukemia. Blood 1998; 91:735.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/122\" class=\"nounderline abstract_t\">Heerema NA, Sather HN, Sensel MG, et al. Frequency and clinical significance of cytogenetic abnormalities in pediatric T-lineage acute lymphoblastic leukemia: a report from the Children's Cancer Group. J Clin Oncol 1998; 16:1270.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/123\" class=\"nounderline abstract_t\">Garand R, Vannier JP, B&eacute;n&eacute; MC, et al. Comparison of outcome, clinical, laboratory, and immunological features in 164 children and adults with T-ALL. The Groupe d'Etude Immunologique des Leuc&eacute;mies. Leukemia 1990; 4:739.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/124\" class=\"nounderline abstract_t\">Raimondi SC, Behm FG, Roberson PK, et al. Cytogenetics of childhood T-cell leukemia. Blood 1988; 72:1560.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/125\" class=\"nounderline abstract_t\">Gardie B, Cayuela JM, Martini S, Sigaux F. Genomic alterations of the p19ARF encoding exons in T-cell acute lymphoblastic leukemia. Blood 1998; 91:1016.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/126\" class=\"nounderline abstract_t\">McCormack MP, Rabbitts TH. Activation of the T-cell oncogene LMO2 after gene therapy for X-linked severe combined immunodeficiency. N Engl J Med 2004; 350:913.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/127\" class=\"nounderline abstract_t\">Erikson J, Williams DL, Finan J, et al. Locus of the alpha-chain of the T-cell receptor is split by chromosome translocation in T-cell leukemias. Science 1985; 229:784.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/128\" class=\"nounderline abstract_t\">Smith SD, Morgan R, Gemmell R, et al. Clinical and biologic characterization of T-cell neoplasias with rearrangements of chromosome 7 band q34. Blood 1988; 71:395.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/129\" class=\"nounderline abstract_t\">Burnett RC, Thirman MJ, Rowley JD, Diaz MO. Molecular analysis of the T-cell acute lymphoblastic leukemia-associated t(1;7)(p34;q34) that fuses LCK and TCRB. Blood 1994; 84:1232.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/130\" class=\"nounderline abstract_t\">Asnafi V, Radford-Weiss I, Dastugue N, et al. CALM-AF10 is a common fusion transcript in T-ALL and is specific to the TCRgammadelta lineage. Blood 2003; 102:1000.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/131\" class=\"nounderline abstract_t\">Weng AP, Ferrando AA, Lee W, et al. Activating mutations of NOTCH1 in human T cell acute lymphoblastic leukemia. Science 2004; 306:269.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/132\" class=\"nounderline abstract_t\">Strehl S, Nebral K, K&ouml;nig M, et al. ETV6-NCOA2: a novel fusion gene in acute leukemia associated with coexpression of T-lymphoid and myeloid markers and frequent NOTCH1 mutations. Clin Cancer Res 2008; 14:977.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/133\" class=\"nounderline abstract_t\">Sulis ML, Williams O, Palomero T, et al. NOTCH1 extracellular juxtamembrane expansion mutations in T-ALL. Blood 2008; 112:733.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/134\" class=\"nounderline abstract_t\">Armstrong F, Brunet de la Grange P, Gerby B, et al. NOTCH is a key regulator of human T-cell acute leukemia initiating cell activity. Blood 2009; 113:1730.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/135\" class=\"nounderline abstract_t\">Petit A, Trinquand A, Chevret S, et al. Oncogenetic mutations combined with MRD improve outcome prediction in pediatric T-cell acute lymphoblastic leukemia. Blood 2018; 131:289.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/136\" class=\"nounderline abstract_t\">Jeong EG, Kim MS, Nam HK, et al. Somatic mutations of JAK1 and JAK3 in acute leukemias and solid cancers. Clin Cancer Res 2008; 14:3716.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/137\" class=\"nounderline abstract_t\">Liu Y, Easton J, Shao Y, et al. The genomic landscape of pediatric and young adult T-lineage acute lymphoblastic leukemia. Nat Genet 2017; 49:1211.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/138\" class=\"nounderline abstract_t\">Ferrando AA, Neuberg DS, Dodge RK, et al. Prognostic importance of TLX1 (HOX11) oncogene expression in adults with T-cell acute lymphoblastic leukaemia. Lancet 2004; 363:535.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/139\" class=\"nounderline abstract_t\">De Keersmaecker K, Graux C, Odero MD, et al. Fusion of EML1 to ABL1 in T-cell acute lymphoblastic leukemia with cryptic t(9;14)(q34;q32). Blood 2005; 105:4849.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/140\" class=\"nounderline abstract_t\">Della Gatta G, Palomero T, Perez-Garcia A, et al. Reverse engineering of TLX oncogenic transcriptional networks identifies RUNX1 as tumor suppressor in T-ALL. Nat Med 2012; 18:436.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/141\" class=\"nounderline abstract_t\">Ntziachristos P, Tsirigos A, Van Vlierberghe P, et al. Genetic inactivation of the polycomb repressive complex 2 in T cell acute lymphoblastic leukemia. Nat Med 2012; 18:298.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/142\" class=\"nounderline abstract_t\">Jeannet R, Mastio J, Macias-Garcia A, et al. Oncogenic activation of the Notch1 gene by deletion of its promoter in Ikaros-deficient T-ALL. Blood 2010; 116:5443.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/143\" class=\"nounderline abstract_t\">Ashworth TD, Pear WS, Chiang MY, et al. Deletion-based mechanisms of Notch1 activation in T-ALL: key roles for RAG recombinase and a conserved internal translational start site in Notch1. Blood 2010; 116:5455.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/144\" class=\"nounderline abstract_t\">Asnafi V, Buzyn A, Le Noir S, et al. NOTCH1/FBXW7 mutation identifies a large subgroup with favorable outcome in adult T-cell acute lymphoblastic leukemia (T-ALL): a Group for Research on Adult Acute Lymphoblastic Leukemia (GRAALL) study. Blood 2009; 113:3918.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/145\" class=\"nounderline abstract_t\">Mansour MR, Sulis ML, Duke V, et al. Prognostic implications of NOTCH1 and FBXW7 mutations in adults with T-cell acute lymphoblastic leukemia treated on the MRC UKALLXII/ECOG E2993 protocol. J Clin Oncol 2009; 27:4352.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/146\" class=\"nounderline abstract_t\">Ben Abdelali R, Asnafi V, Leguay T, et al. Pediatric-inspired intensified therapy of adult T-ALL reveals the favorable outcome of NOTCH1/FBXW7 mutations, but not of low ERG/BAALC expression: a GRAALL study. Blood 2011; 118:5099.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/147\" class=\"nounderline abstract_t\">Mavrakis KJ, Van Der Meulen J, Wolfe AL, et al. A cooperative microRNA-tumor suppressor gene network in acute T-cell lymphoblastic leukemia (T-ALL). Nat Genet 2011; 43:673.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/148\" class=\"nounderline abstract_t\">Van Vlierberghe P, Palomero T, Khiabanian H, et al. PHF6 mutations in T-cell acute lymphoblastic leukemia. Nat Genet 2010; 42:338.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/149\" class=\"nounderline abstract_t\">De Keersmaecker K, Atak ZK, Li N, et al. Exome sequencing identifies mutation in CNOT3 and ribosomal genes RPL5 and RPL10 in T-cell acute lymphoblastic leukemia. Nat Genet 2013; 45:186.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/150\" class=\"nounderline abstract_t\">Van der Meulen J, Sanghvi V, Mavrakis K, et al. The H3K27me3 demethylase UTX is a gender-specific tumor suppressor in T-cell acute lymphoblastic leukemia. Blood 2015; 125:13.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/151\" class=\"nounderline abstract_t\">Ntziachristos P, Tsirigos A, Welstead GG, et al. Contrasting roles of histone 3 lysine 27 demethylases in acute lymphoblastic leukaemia. Nature 2014; 514:513.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/152\" class=\"nounderline abstract_t\">Coustan-Smith E, Mullighan CG, Onciu M, et al. Early T-cell precursor leukaemia: a subtype of very high-risk acute lymphoblastic leukaemia. Lancet Oncol 2009; 10:147.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/153\" class=\"nounderline abstract_t\">Zhang J, Ding L, Holmfeldt L, et al. The genetic basis of early T-cell precursor acute lymphoblastic leukaemia. Nature 2012; 481:157.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/154\" class=\"nounderline abstract_t\">Larson RA, Dodge RK, Burns CP, et al. A five-drug remission induction regimen with intensive consolidation for adults with acute lymphoblastic leukemia: cancer and leukemia group B study 8811. Blood 1995; 85:2025.</a></li><li><a href=\"https://www.uptodate.com/contents/cytogenetics-and-molecular-genetics-in-acute-lymphoblastic-leukemia/abstract/155\" class=\"nounderline abstract_t\">Baldus CD, Martus P, Burmeister T, et al. Low ERG and BAALC expression identifies a new subgroup of adult acute T-lymphoblastic leukemia with a highly favorable outcome. J Clin Oncol 2007; 25:3739.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 4484 Version 37.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H1460654043\"><span>SUMMARY</span></a></li><li><a href=\"#H1\" id=\"outline-link-H1\">INTRODUCTION</a></li><li><a href=\"#H2\" id=\"outline-link-H2\">B CELL ACUTE LYMPHOBLASTIC LEUKEMIA</a><ul><li><a href=\"#H3\" id=\"outline-link-H3\">Cytogenetic abnormalities</a><ul><li><a href=\"#H1280995514\" id=\"outline-link-H1280995514\">- 12;21 translocation in precursor B ALL</a></li><li><a href=\"#H4\" id=\"outline-link-H4\">- 8;14 translocation</a></li><li><a href=\"#H5\" id=\"outline-link-H5\">- 4;11 translocation</a></li><li><a href=\"#H351299836\" id=\"outline-link-H351299836\">- 11q23.3 translocation in infants</a></li><li><a href=\"#H6\" id=\"outline-link-H6\">- 9;22 translocation</a></li><li><a href=\"#H7\" id=\"outline-link-H7\">- 1;19 translocation in pre-B cell ALL</a></li><li><a href=\"#H2516253092\" id=\"outline-link-H2516253092\">- Intrachromosomal amplification of chromosome 21 (iAMP21)</a></li><li><a href=\"#H2233889466\" id=\"outline-link-H2233889466\">- 9p abnormalities in ALL</a></li><li><a href=\"#H9\" id=\"outline-link-H9\">- Hyperdiploidy</a></li><li><a href=\"#H10\" id=\"outline-link-H10\">- Hypodiploidy</a></li></ul></li><li><a href=\"#H580099884\" id=\"outline-link-H580099884\">Genetic abnormalities with normal karyotype</a><ul><li><a href=\"#H12\" id=\"outline-link-H12\">- Gene expression</a></li><li><a href=\"#H13\" id=\"outline-link-H13\">- Cell cycle regulatory genes</a></li><li><a href=\"#H73697009\" id=\"outline-link-H73697009\">- BCR-ABL1-like B cell ALL</a></li><li><a href=\"#H4205269779\" id=\"outline-link-H4205269779\">- Recurring gene fusions</a></li></ul></li></ul></li><li><a href=\"#H14\" id=\"outline-link-H14\">T CELL ACUTE LYMPHOBLASTIC LEUKEMIA</a><ul><li><a href=\"#H15\" id=\"outline-link-H15\">Genetic abnormalities</a><ul><li><a href=\"#H16\" id=\"outline-link-H16\">- TCR gene rearrangements</a></li><li><a href=\"#H17\" id=\"outline-link-H17\">- Non-TCR loci</a></li></ul></li><li><a href=\"#H18\" id=\"outline-link-H18\">Prognosis</a></li></ul></li><li><a href=\"#PATIENT_INFORMATION\" id=\"outline-link-H12776475\">INFORMATION FOR PATIENTS</a></li><li><a href=\"#H1460654043\" id=\"outline-link-H1460654043\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>GRAPHICS <a href=\"#\" id=\"viewAllGraphicsLink\">View All</a></h2><div id=\"outlineGraphics\"><ul><li><div id=\"HEME/4484|TAB\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"TABLES\">TABLES</a></div><ul><li><a href=\"image.htm?imageKey=HEME/110239\" class=\"graphic graphic_table\">- WHO classification of myeloid neoplasms and acute leukemia</a></li></ul></li></ul></div></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=clinical-manifestations-pathologic-features-and-diagnosis-of-precursor-t-cell-acute-lymphoblastic-leukemia-lymphoma\" class=\"medical medical_review\">Clinical manifestations, pathologic features, and diagnosis of precursor T cell acute lymphoblastic leukemia/lymphoma</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=clinical-use-of-minimal-residual-disease-detection-in-acute-lymphoblastic-leukemia\" class=\"medical medical_review\">Clinical use of minimal residual disease detection in acute lymphoblastic leukemia</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=cytogenetics-in-acute-myeloid-leukemia\" class=\"medical medical_review\">Cytogenetics in acute myeloid leukemia</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=detection-of-minimal-residual-disease-in-acute-lymphoblastic-leukemia\" class=\"medical medical_review\">Detection of minimal residual disease in acute lymphoblastic leukemia</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=general-aspects-of-cytogenetic-analysis-in-hematologic-malignancies\" class=\"medical medical_review\">General aspects of cytogenetic analysis in hematologic malignancies</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=hematopoietic-cell-transplantation-for-primary-immunodeficiency\" class=\"medical medical_review\">Hematopoietic cell transplantation for primary immunodeficiency</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=induction-therapy-for-philadelphia-chromosome-negative-acute-lymphoblastic-leukemia-in-adults\" class=\"medical medical_review\">Induction therapy for Philadelphia chromosome negative acute lymphoblastic leukemia in adults</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=molecular-genetics-of-chronic-myeloid-leukemia\" class=\"medical medical_review\">Molecular genetics of chronic myeloid leukemia</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=pathobiology-of-burkitt-lymphoma\" class=\"medical medical_review\">Pathobiology of Burkitt lymphoma</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=acute-lymphoblastic-leukemia-all-treatment-in-adults-beyond-the-basics\" class=\"medical medical_patient\">Patient education: Acute lymphoblastic leukemia (ALL) treatment in adults (Beyond the Basics)</a></li></ul></div></div>","javascript":null}