var data={"title":"Familial amyotrophic lateral sclerosis","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Familial amyotrophic lateral sclerosis</div><dl id=\"topicContributors\"><dt><span> </span>Authors:</dt><dd><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/contributors\" class=\"contributor contributor_credentials\">Leo McCluskey, MD, MBE</a></dd><dd><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/contributors\" class=\"contributor contributor_credentials\">Dana Falcone, MS, CGC</a></dd><dt><span> </span>Section Editors:</dt><dd><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/contributors\" class=\"contributor contributor_credentials\">Jeremy M Shefner, MD, PhD</a></dd><dd><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/contributors\" class=\"contributor contributor_credentials\">Ira N Targoff, MD</a></dd><dd><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/contributors\" class=\"contributor contributor_credentials\">Benjamin A Raby, MD, MPH</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/contributors\" class=\"contributor contributor_credentials\">April F Eichler, MD, MPH</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Oct 16, 2017.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H1\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Amyotrophic lateral sclerosis (ALS) is one specific type of the more general group of motor neuron diseases. These disorders variably affect motor neurons located in the anterior (ventral) horn regions of the spinal cord, the cranial nerve motor nuclei in the pons and medulla, and the frontal cortex. Familial amyotrophic lateral sclerosis accounts for 5 to 10 percent of all ALS cases.</p><p>ALS is a relentlessly progressive neurodegenerative disorder that causes muscle weakness, disability, and eventually death. The hallmark of ALS is the combination of upper motor neuron (UMN) and lower motor neuron (LMN) involvement. The LMN findings of weakness, atrophy, and fasciculations are a direct consequence of muscle denervation, hence the term &quot;amyotrophic.&quot; The UMN findings of hyperreflexia and spasticity result from degeneration of the lateral corticospinal tracts in the spinal cord, which are gliotic and hardened to palpation at autopsy, hence the term &quot;lateral sclerosis.&quot;</p><p>This topic review will discuss familial ALS. The epidemiology and clinical features of ALS are discussed separately. (See <a href=\"topic.htm?path=epidemiology-and-pathogenesis-of-amyotrophic-lateral-sclerosis\" class=\"medical medical_review\">&quot;Epidemiology and pathogenesis of amyotrophic lateral sclerosis&quot;</a> and <a href=\"topic.htm?path=clinical-features-of-amyotrophic-lateral-sclerosis-and-other-forms-of-motor-neuron-disease\" class=\"medical medical_review\">&quot;Clinical features of amyotrophic lateral sclerosis and other forms of motor neuron disease&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H136713437\"><span class=\"h1\">OVERVIEW</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Familial ALS accounts for approximately 5 to 10 percent of all ALS cases [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/1,2\" class=\"abstract_t\">1,2</a>], with the rest being sporadic (idiopathic) in origin. In our series of 399 cases of ALS at the University of Pennsylvania with a four-generation pedigree, 10 percent of individuals had an affected first or second-degree relative [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/2\" class=\"abstract_t\">2</a>].</p><p>Familial ALS is phenotypically and genetically heterogeneous. Although most familial ALS cases follow an autosomal dominant inheritance pattern, recessive and X-linked forms have been described. At least 15 different loci are thought to harbor ALS-causing mutations. The nomenclature of ALS1 through ALS15 arises from the order of their discovery and not from any particular clinical classification. Many of the types listed have been described only in one or two families, and genotype-phenotype correlation is stronger for some forms of genetic ALS than for others. In most individual cases, it may be difficult to determine on clinical grounds alone if ALS is familial or sporadic, especially at the onset of disease. The presence of atypical features such as young age of onset, sensory loss, and of course a positive family history of ALS, other neurodegenerative disorders, and dementia should alert the clinician to the possibility of familial ALS.</p><p class=\"headingAnchor\" id=\"H2\"><span class=\"h1\">AUTOSOMAL DOMINANT</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The majority of familial ALS cases follow an autosomal dominant inheritance pattern. In autosomal dominant inheritance, having a mutation in only one copy of a gene is sufficient for an individual to develop the condition. Typically, the copy of the gene with the mutation is inherited from a parent (as opposed to de novo mutations, which are uncommon), and most individuals with an autosomal dominant condition will have a positive family history. The children of a parent with a dominant mutation have a 50 percent chance of inheriting the copy of the gene with the mutation and developing ALS.</p><p>Autosomal dominant mutations in apparently sporadic ALS can occur due to de novo (new) mutations, reduced penetrance, or insufficient family history information. The frequency of de novo mutations in ALS is unknown. The recurrence risk for siblings and children of an apparently sporadic ALS case with an autosomal dominant mutation depends on whether or not the affected individual has an inherited or a de novo mutation. If the individual with ALS inherited the mutation from a parent, there is a 50 percent chance the individual's siblings and children will inherit the mutation and be at risk to develop ALS. In the case of a de novo mutation, the recurrence risk for siblings would be low, but the recurrence risk for children would be 50 percent.</p><p>The most common autosomal dominantly inherited forms of familial ALS are caused by mutations in the C9ORF72 gene (see <a href=\"#H1808661\" class=\"local\">'C9ORF72 gene'</a> below) and the SOD1 gene (see <a href=\"#H3\" class=\"local\">'ALS1 (SOD1 gene)'</a> below) described in the following sections.</p><p class=\"headingAnchor\" id=\"H1808661\"><span class=\"h2\">C9ORF72 gene</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Repeat expansions of the C9ORF72 gene on chromosome 9p21 are causally linked to classic ALS, frontotemporal dementia, and ALS with frontotemporal dementia. These expansions appear to be one of the most common causes of familial ALS, accounting for approximately 34 percent of cases in European populations and 2 percent of cases in Asian populations [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/3\" class=\"abstract_t\">3</a>]. In addition, C9ORF72 expansions are observed in approximately 5 percent of sporadic ALS cases in European populations.</p><p>Earlier reports described familial forms of ALS <span class=\"nowrap\">and/or</span> frontotemporal dementia (FTD) linked to chromosome 9p with adult onset [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/4-7\" class=\"abstract_t\">4-7</a>]. In these families, some individuals manifest both ALS and FTD, while others have FTD without ALS, or ALS without FTD. The dementia may precede, follow, or develop simultaneously with the onset of upper and lower motor neuron signs; it produces personality change, executive dysfunction, or language difficulties. Most of these cases of ALS and FTD linked to chromosome 9 are explained by a dominantly inherited form of familial ALS and FTD (FTDALS) caused by pathologic expansion of a noncoding GGGGCC hexanucleotide repeat in the C9ORF72 gene [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/8-10\" class=\"abstract_t\">8-10</a>].</p><p>The C9ORF72 gene encodes an uncharacterized protein that is highly conserved among species. The hexanucleotide repeat expansion seen in FTDALS cases is large, ranging from 250 to 1,600 repeats, compared with &lt;24 repeats in controls. In a population-based study from Ireland, the C9ORF72 expansion was found in 20 of 49 cases (41 percent) of FALS and 19 of 386 (5 percent) apparently sporadic cases of ALS [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/11\" class=\"abstract_t\">11</a>]. Among 191 cases with phenotype data, the repeat expansion was present in 21 (11 percent). Compared with those without the C9ORF72 repeat expansion, patients with the expansion had the following statistically significant findings:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Earlier disease onset (mean age 56.3 versus 61.3 years)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Higher frequency of comorbid FTD (50 versus 12 percent)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Shorter age-matched median survival (20 versus 26 months)</p><p/><p>Of note, the C9ORF72 repeat expansion was not found in patients with sporadic ALS who had normal cognition and no family history of neurodegenerative disease [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/11\" class=\"abstract_t\">11</a>].</p><p>Another locus for families with FTD <span class=\"nowrap\">and/or</span> ALS was previously identified on chromosome 9q [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/12\" class=\"abstract_t\">12</a>], but this finding has not been replicated [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/13\" class=\"abstract_t\">13</a>].</p><p>The cognitive features of ALS and the manifestations of FTD are reviewed separately. (See <a href=\"topic.htm?path=clinical-features-of-amyotrophic-lateral-sclerosis-and-other-forms-of-motor-neuron-disease#H13\" class=\"medical medical_review\">&quot;Clinical features of amyotrophic lateral sclerosis and other forms of motor neuron disease&quot;, section on 'Cognitive symptoms'</a> and <a href=\"topic.htm?path=frontotemporal-dementia-clinical-features-and-diagnosis\" class=\"medical medical_review\">&quot;Frontotemporal dementia: Clinical features and diagnosis&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H3\"><span class=\"h2\">ALS1 (SOD1 gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Familial ALS1 is linked to mutations of the copper-zinc (Cu,Zn) superoxide dismutase type 1 (SOD1) gene localized in chromosome 21q22 [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/14\" class=\"abstract_t\">14</a>]. Mutations in SOD1 account for approximately 15 percent of familial ALS cases in European populations and 30 percent of cases in Asian populations [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/3\" class=\"abstract_t\">3</a>]. Most are inherited in an autosomal dominant manner. Over 180 mutations of the SOD1 gene have been reported [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/15,16\" class=\"abstract_t\">15,16</a>]. The clinical features of ALS1 are variable, as illustrated by the following observations:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The A4V missense mutation is the most prevalent mutation in familial ALS1 and accounts for about 41 percent of patients with SOD1 mutations [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/17\" class=\"abstract_t\">17</a>]. It typically has a rapid progression with an average life expectancy of 1.5 years after the onset of symptoms [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/18\" class=\"abstract_t\">18</a>]. Upper motor neuron involvement is either absent or mild, but pathological abnormalities in systems other than the motor neurons are more frequent [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/19\" class=\"abstract_t\">19</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The I113T mutation [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/14\" class=\"abstract_t\">14</a>] is the second most common type, accounting for 16 percent of SOD1 mutations. This mutation has variable penetrance that can be low [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/20\" class=\"abstract_t\">20</a>]. The mean age of onset is 59 years. The clinical course is variable, ranging from quite rapid progression and death within two years to very slow progression that can span 20 years [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/20-22\" class=\"abstract_t\">20-22</a>]. Both upper and lower motor neuron findings occur. Posterior column involvement has been reported in one patient [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/23\" class=\"abstract_t\">23</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Families with the A4T mutation in the SOD1 gene have a variable onset and duration. In one large kindred, the age of onset ranged from 32 to 60 years, with a mean of 46 years [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/24\" class=\"abstract_t\">24</a>]. Weakness in the legs was the most frequent early symptom, and there was a predominance of lower motor neuron signs. Bulbar and respiratory muscle involvement occurred in all individuals. The mean time from onset of symptoms to death was 14 months. One man with onset at age 37 had a more slowly developing form and was alive but severely affected 76 months after diagnosis. With this one exception, the severity was similar to that of the A4V mutation [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/24\" class=\"abstract_t\">24</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Among 17 patients from three Japanese ALS1 families with the SOD1 H46R heterozygous mutation, the age of disease onset ranged from 32 to 60 years (average 44.3), the age of death ranged from 46 to older than 88 years (average 59 years), and the mean disease duration was greater than 12 years [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/25\" class=\"abstract_t\">25</a>]. The initial symptom was distal leg weakness in all patients. This relatively slow disease progression is in keeping with the clinical observation that bulbar involvement was minimal in these families; only 1 out of the 12 reported patients had tongue atrophy. Dementia was uniformly absent in these patients.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Another report described three members of a familial ALS1 pedigree with a rare A89V SOD1 mutation [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/26\" class=\"abstract_t\">26</a>]. This mutation is characterized by incomplete penetrance, variable age of onset, and an associated painful sensory neuropathy. In these three patients the disease onset was late in two and early in one. The early-onset patient was a 15-year-old male who presented with a one-year history of a progressive bilateral symmetric leg and arm weakness. The weakness was associated with sporadic cramps and fasciculations in the limbs and chest muscles but no bulbar involvement. On exam, he had brisk jaw reflexes, weak neck flexion and extension, bilateral weakness and atrophy of the upper and lower extremities, increased tone, hyperactive deep tendon reflexes, and a left Babinski response.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In a report involving 20 patients, the G93C mutation was associated with a purely lower motor neuron phenotype and absence of bulbar involvement [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/27\" class=\"abstract_t\">27</a>]. The mean age of onset was 46 years and disease progression was slow, with an average survival of nearly 13 years.</p><p/><p>SOD1 mutations, usually those that can have low penetrance, can be seen in sporadic ALS. The most common of these are the I113T and D90A mutations [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/20,23,28\" class=\"abstract_t\">20,23,28</a>]. In addition, rare cases of autosomal recessive familial ALS have been associated with double doses of the D90A mutation [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/29\" class=\"abstract_t\">29</a>], and autosomal dominant familial ALS has been reported in patients who are heterozygotes for D90A [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/30\" class=\"abstract_t\">30</a>].</p><p class=\"headingAnchor\" id=\"H4\"><span class=\"h2\">ALS3</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Familial ALS3 has been linked to chromosome 18q21 in a single large French kindred [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/31\" class=\"abstract_t\">31</a>]. The average age of onset was 43 years. Leg onset was most common. Both upper and lower motor neuron signs occurred. Other central nervous system pathways were not involved. The mean life expectancy was five years.</p><p class=\"headingAnchor\" id=\"H5\"><span class=\"h2\">ALS4 (SETX gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Familial ALS4 is a juvenile form with an average age of onset in the second decade [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/32,33\" class=\"abstract_t\">32,33</a>]. It has been linked to the SETX gene on chromosome 9q34 coding for senataxin, a DNA and RNA helicase. It most commonly begins with distal weakness and atrophy of the hands and feet. Proximal weakness occurs later. Upper motor neuron signs occur, but bulbar involvement is uncommon. The clinical course is extremely slow. Progression to wheelchair use does not occur until the fifth or sixth decades. Minor sensory changes can occur in older individuals.</p><p>Pathologically, the disorder is characterized by loss of upper and lower motor neurons as well as dorsal root ganglion neurons and posterior column fiber loss. Recessive mutations in SETX have been linked to inherited ataxia and peripheral neuropathy [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/34,35\" class=\"abstract_t\">34,35</a>].</p><p class=\"headingAnchor\" id=\"H6\"><span class=\"h2\">ALS6 (FUS gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Mutations in the FUS gene (fused in sarcoma gene, also known as the TLS gene) on chromosome 16 have been linked to familial ALS6 in multiple families from diverse ethnic groups [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/36-39\" class=\"abstract_t\">36-39</a>]. The phenotype is notable for onset at an average age of 45 years, with an average survival of 33 to 41 months [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/37,38\" class=\"abstract_t\">37,38</a>]. However, there is a wide variation in the duration of symptoms, with one reported patient surviving for 18 years [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/38\" class=\"abstract_t\">38</a>]. In a series of 18 patients, the onset was cervical more often than lumbar or bulbar [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/37\" class=\"abstract_t\">37</a>]. FUS mutations have also been identified in a few patients with familial ALS and frontotemporal lobe dementia or parkinsonism [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/38\" class=\"abstract_t\">38</a>]. In cases in which pathologic specimens were obtained, there was severe lower motor neuron loss in the spinal cord and to a lesser degree in the brainstem [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/37\" class=\"abstract_t\">37</a>].</p><p>FUS mutations are estimated to occur in approximately 3 percent of familial ALS cases in European populations and 6 percent of familial ALS cases in Asian populations [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/3\" class=\"abstract_t\">3</a>].</p><p>The <span class=\"nowrap\">FUS/TLS</span> protein is normally found in the nucleus, where it is involved in transcription regulation, RNA splicing, and RNA transport [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/36,37\" class=\"abstract_t\">36,37</a>]. Thus, it is functionally similar to the TARDBP gene described below. Furthermore, cellular expression studies reveal that mutant forms of FUS have aberrant localization in the cytoplasm of neurons, similar to the pathology of TARDBP in familial ALS. These findings suggest that both FUS and TARDBP share a common pathogenic mechanism related to impaired RNA metabolism. (See <a href=\"#H10\" class=\"local\">'ALS10 (TARDBP gene)'</a> below.)</p><p class=\"headingAnchor\" id=\"H7\"><span class=\"h2\">ALS7</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Familial ALS7 has been linked to chromosome 20p13 in a single family [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/40\" class=\"abstract_t\">40</a>]. The mean onset was in the sixth decade, with a mean life expectancy of three years after onset. It is characterized by both upper and lower motor neuron signs.</p><p class=\"headingAnchor\" id=\"H8\"><span class=\"h2\">ALS8 (VAPB gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Familial ALS8 was initially linked to chromosome 20q13.33 in a single Brazilian family [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/41\" class=\"abstract_t\">41</a>]. It is characterized by onset in the third to the fifth decade with manifestations that include cramps, fasciculations, and at times lower motor neuron weakness. Upper motor neuron findings are less common. Some individuals manifest sensory loss, a postural tremor, or both. The course is slowly progressive. The cause is a missense mutation in the VAPB gene [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/42\" class=\"abstract_t\">42</a>]. The same VAPB mutation has been identified in additional families with clinically heterogeneous types of motor neuron disease, including ALS8, typical severe ALS with rapid progression, and late-onset spinal muscular atrophy [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/42\" class=\"abstract_t\">42</a>].</p><p class=\"headingAnchor\" id=\"H9\"><span class=\"h2\">ALS9 (ANG gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Missense mutations in the ANG gene coding for the protein angiogenin have been identified in Irish, Scottish, and Scandinavian individuals, some with and some without a dominant family history of ALS [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/43,44\" class=\"abstract_t\">43,44</a>]. Mutations in ANG result in functional loss of angiogenic activity [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/45\" class=\"abstract_t\">45</a>]. The angiogenin protein promotes vascularization of tissue and is required for the activity of vascular endothelial growth factor (VEGF).</p><p>The clinical course associated with ANG mutations is that of progressive upper and lower motor neuron disease with a wide range of onset age and survival [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/43,44\" class=\"abstract_t\">43,44</a>]. The frequency of bulbar onset, about 50 percent, is greater than that of sporadic ALS. One patient with an ANG mutation initially presented with parkinsonism, followed five years later by manifestations of upper and lower motor neuron disease and features characteristic of frontotemporal dementia [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/46\" class=\"abstract_t\">46</a>].</p><p class=\"headingAnchor\" id=\"H10\"><span class=\"h2\">ALS10 (TARDBP gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The TAR DNA binding protein 43 (TDP-43) is a major component of ubiquinated inclusions in sporadic ALS, frontotemporal dementia (FTD) with <span class=\"nowrap\">ubiquitin-positive/tau-negative</span> inclusions, ALS with FTD, and familial ALS (with the exception of SOD1-related familial ALS1). (See <a href=\"topic.htm?path=epidemiology-and-pathogenesis-of-amyotrophic-lateral-sclerosis#H9\" class=\"medical medical_review\">&quot;Epidemiology and pathogenesis of amyotrophic lateral sclerosis&quot;, section on 'Intracellular inclusions'</a>.)</p><p>Mutations in the TARDBP gene on chromosome 1p36.22 that encodes TDP-43 are the cause of autosomal dominant familial ALS10, and may be a rare cause of sporadic ALS [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/47-51\" class=\"abstract_t\">47-51</a>]. These mutations account for approximately 4 percent of familial ALS cases in European populations and 2 percent of familial ALS cases in Asian populations [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/3\" class=\"abstract_t\">3</a>].</p><p>Although the phenotype is variable, the clinical features of ALS in patients with TARDBP mutations compared with patients who have sporadic ALS include a greater likelihood for earlier onset, upper limb onset, and slower progression [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/52\" class=\"abstract_t\">52</a>]. Some patients develop cognitive impairment or frontotemporal dementia. Other studies have identified TARDBP mutations in apparently sporadic and familial forms of frontotemporal lobe degeneration with and without motor neuron disease [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/53\" class=\"abstract_t\">53</a>].</p><p class=\"headingAnchor\" id=\"H11\"><span class=\"h2\">ALS11 (FIG4 gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Familial ALS11 is an autosomal dominant form of ALS caused by mutations in the FIG4 gene on chromosome 6q21 [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/54\" class=\"abstract_t\">54</a>].</p><p>In a study that screened DNA from 473 patients (364 with sporadic ALS and 109 with familial ALS), mutations in the FIG4 gene were identified in nine patients (2 percent), including six with sporadic ALS and three with familial ALS [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/54\" class=\"abstract_t\">54</a>]. The protein encoded by FIG4 is a phosphoinositide 5-phosphatase that regulates a signaling lipid involved with membrane vesicle trafficking.</p><p>Charcot-Marie-Tooth disease type 4J (CMT4J) is an allelic disorder caused by FIG4 mutations. The phenotype of CMT4J is notable for autosomal recessive inheritance, early childhood to adult onset, and involvement of both motor and sensory neurons. (See <a href=\"topic.htm?path=charcot-marie-tooth-disease-genetics-clinical-features-and-diagnosis#H14\" class=\"medical medical_review\">&quot;Charcot-Marie-Tooth disease: Genetics, clinical features, and diagnosis&quot;, section on 'CMT4'</a>.)</p><p class=\"headingAnchor\" id=\"H161409026\"><span class=\"h2\">ALS12 (OPTN gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Mutations within the OPTN gene are the cause of ALS12, as first identified in two Japanese families [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/55\" class=\"abstract_t\">55</a>]. Age of onset was 33 to 54 years and progression was slow. Later reports identified OPTN mutations in 1 to 2 percent of familial ALS and 0 to 3.5 percent of sporadic ALS cases [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/56,57\" class=\"abstract_t\">56,57</a>].</p><p class=\"headingAnchor\" id=\"H233524289\"><span class=\"h2\">ALS13 (ATXN2 gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Mounting evidence suggests an association between intermediate length polyglutamine expansions in the ataxin 2 gene (ATXN2) and increased risk of ALS. This issue is discussed separately. (See <a href=\"topic.htm?path=epidemiology-and-pathogenesis-of-amyotrophic-lateral-sclerosis#H5\" class=\"medical medical_review\">&quot;Epidemiology and pathogenesis of amyotrophic lateral sclerosis&quot;, section on 'Genetic susceptibility in sporadic ALS'</a>.)</p><p class=\"headingAnchor\" id=\"H246948\"><span class=\"h2\">ALS14 (VCP gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A mutation in the valosin-containing protein gene (VCP) was first identified in an Italian family as the cause of ALS14 [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/58\" class=\"abstract_t\">58</a>]. Additional VCP mutations have since been identified in other familial ALS cases, suggesting that VCP mutations are responsible for 1 to 2 percent of familial ALS cases [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/58,59\" class=\"abstract_t\">58,59</a>].</p><p>Mutations in VCP are also the cause of some but not all cases of an autosomal dominant familial syndrome characterized by inclusion body myopathy associated with Paget disease of bone and frontotemporal dementia (IBMPFD) [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/60,61\" class=\"abstract_t\">60,61</a>]. There is mounting evidence that the spectrum of IBMPFD includes familial ALS [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/59,61\" class=\"abstract_t\">59,61</a>]. The valosin-containing protein is implicated in numerous cellular functions, including pathways involved in ubiquitin-dependent protein degradation and autophagy. Mutant VCP may act by disrupting autophagy, resulting in the deposition of TDP-43 inclusions within the cytoplasm of affected cells [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/62\" class=\"abstract_t\">62</a>]. (See <a href=\"topic.htm?path=epidemiology-and-pathogenesis-of-amyotrophic-lateral-sclerosis#H9\" class=\"medical medical_review\">&quot;Epidemiology and pathogenesis of amyotrophic lateral sclerosis&quot;, section on 'Intracellular inclusions'</a>.)</p><p class=\"headingAnchor\" id=\"H2572382106\"><span class=\"h2\">ALS17 (CHMP2B gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Case reports have associated mutations in the chromatin modifying protein 2b gene (CHMP2B gene) in both sporadic and dominantly inherited disorders referred to as chromosome 3-linked frontotemporal dementia (FTD3). While predominately associated with FTD3, late-onset (seventh to eighth decades) bulbar ALS with and without FTD has also been associated with CHMP2B mutations [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/63,64\" class=\"abstract_t\">63,64</a>].</p><p class=\"headingAnchor\" id=\"H10539001\"><span class=\"h2\">ALS18 (PFN1 gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Mutations in the profilin 1 gene (PFN1) located on chromosome region 17p13.2 have been found in seven families with ALS displaying an autosomal dominant mode of inheritance [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/65\" class=\"abstract_t\">65</a>]. The phenotype is notable for limb onset of ALS in middle age (average 45 years). The profilin 1 protein regulates actin polymerization and has multiple other cellular functions. Mutant profilin 1 may play a role in ALS pathogenesis on the basis of reduced levels of bound actin, inhibition of axon growth, and a propensity to form intracellular aggregates compared with wild-type profilin 1.</p><p class=\"headingAnchor\" id=\"H233525093\"><span class=\"h2\">DAO gene</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A mutation in the D-amino acid oxidase gene (DAO) on chromosome region 12q24 has been associated with classic adult-onset familial ALS in a three-generational kindred from England [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/66\" class=\"abstract_t\">66</a>]. Features included upper and lower motor neuron involvement and bulbar signs but no cognitive impairment. The mean age at time of death was 44 years (range 42 to 55 years).</p><p>D-amino acid oxidase is a flavoprotein that catalyzes the oxidative deamination of D-serine, a coagonist in the activation of the N-methyl-D-aspartate glutamate receptor subtype. The R199W DAO mutation identified in this study causes a loss of enzyme activity, which could lead to cell death via excitotoxicity from enhanced NMDA receptor activation.</p><p class=\"headingAnchor\" id=\"H233525261\"><span class=\"h2\">NEFH gene</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Familial ALS linked to a mutation of the neurofilament heavy chain gene (NEFH) located on chromosome 22q12.1-q13.1 has been noted in a Scandinavian kindred [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/67\" class=\"abstract_t\">67</a>]. Variable features included monomelic amyotrophy, dementia, and bulbar dysfunction. One percent of sporadic ALS patients have been reported to have a mutation of the NEFH gene. Its role in the disease is uncertain.</p><p class=\"headingAnchor\" id=\"H2187961\"><span class=\"h2\">HNRNPA2B1 and HNRNPA1 genes</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The HNRNPA2B1 and HNRNPA1 genes encode for proteins that bind RNA and play an important role in regulating RNA function. Their role in ALS was first suggested by a study that combined whole-exome sequencing with linkage analysis in families with autosomal dominant presentations of multisystem proteinopathy [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/68\" class=\"abstract_t\">68</a>]. The investigators found nonsynonymous coding variants in conserved amino acid residues of the hnRNPA2B1 and hnRNPA1 proteins that cosegregated with disease. One of these families, with a mutation in hnRNPA2B1, had a rare inherited syndrome characterized by features of inclusion body myopathy (IBM) associated with Paget disease of the bone (PDB), frontotemporal dementia (FTD), and ALS (also known as <span class=\"nowrap\">IBMPFD/ALS)</span>. VCP locus mutations were excluded in these patients (see <a href=\"#H246948\" class=\"local\">'ALS14 (VCP gene)'</a> above). Subsequently, exome screening of 212 familial ALS cases in which known ALS genes were excluded identified a causative mutation within the hnRNPA1 protein in one patient [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/68\" class=\"abstract_t\">68</a>].</p><p>These mutations, which involve prion-like domains of the hnRNPA2B1 and hnRNPA1 proteins, appear to exacerbate the intrinsic tendency of these proteins to assemble into self-seeding fibrils, thereby leading to excess fibril formation and eventual neurodegeneration with cytoplasmic inclusions [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/68\" class=\"abstract_t\">68</a>]. Of interest, hnRNPA2B1 and hnRNPA1 also interact in the regulation of RNA metabolism with TDP-43, which also has been implicated in the pathogenesis of ALS and related neurodegenerative disorders. (See <a href=\"#H10\" class=\"local\">'ALS10 (TARDBP gene)'</a> above and <a href=\"topic.htm?path=epidemiology-and-pathogenesis-of-amyotrophic-lateral-sclerosis#H9\" class=\"medical medical_review\">&quot;Epidemiology and pathogenesis of amyotrophic lateral sclerosis&quot;, section on 'Intracellular inclusions'</a> and <a href=\"topic.htm?path=epidemiology-and-pathogenesis-of-amyotrophic-lateral-sclerosis#H107004749\" class=\"medical medical_review\">&quot;Epidemiology and pathogenesis of amyotrophic lateral sclerosis&quot;, section on 'Altered RNA processing'</a>.)</p><p class=\"headingAnchor\" id=\"H16\"><span class=\"h1\">AUTOSOMAL RECESSIVE</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Recessively inherited familial amyotrophic lateral sclerosis includes ALS2 and ALS5. In autosomal recessive inheritance, a mutation in both copies of a gene is necessary in order for an individual to develop the condition. Typically, parents of an affected individual do not have the condition themselves, but are obligate carriers for the mutation. The siblings of an individual with an autosomal recessive condition have the highest recurrence risk. Siblings have a 25 percent likelihood of inheriting both copies of the gene with the mutation and developing ALS, a 50 percent likelihood of being an unaffected carrier for the condition, and a 25 percent likelihood of being an unaffected non-carrier for the condition. Unless an individual&rsquo;s partner is also a carrier for the same disease gene, all children of an individual with autosomal recessive ALS will be unaffected carriers of the condition. In autosomal recessive inheritance, there may be no family history of the condition in previous generations because a single copy of a gene mutation may be passed silently through a family.</p><p>The ALS2 and SPG11 genes are inherited in an autosomal recessive pattern. Additionally, certain mutations in the SOD1 gene can be inherited in an autosomal recessive pattern. (See <a href=\"#H3\" class=\"local\">'ALS1 (SOD1 gene)'</a> above.)</p><p class=\"headingAnchor\" id=\"H17\"><span class=\"h2\">ALS2 (ALS2 gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Familial ALS2 is caused by a mutation in exons 3 or 4 of the ALS2 gene coding for Alsin, a putative GTPase regulatory protein, on chromosome 2q33 [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/69,70\" class=\"abstract_t\">69,70</a>]. The mean age of onset is in the first decade. Clinical features involve the development of upper motor neuron signs in the face and lower limbs, greater than in the upper limbs. Mild lower motor neuron findings develop slowly in the distal legs, more so than the arms. The course is markedly slow with note of difficulty walking in the fifth decade. In addition, there are variant phenotypes and allelic disorders:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>One affected patient had ALS onset by age two and progression to anarthria with wheelchair-bound status by age 18 [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/71\" class=\"abstract_t\">71</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Another report described juvenile onset of ALS associated with generalized dystonia and cerebellar signs in two consanguineous families [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/72\" class=\"abstract_t\">72</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A deletion in exon 9 of the Alsin gene is responsible for recessively inherited childhood-onset primary lateral sclerosis (PLS), at times associated with gaze paresis [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/73\" class=\"abstract_t\">73</a>].</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Frame shift mutations in various locations within the Alsin gene that produce a premature stop codon have been associated with recessively inherited infantile-onset familial spastic paraparesis [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/74\" class=\"abstract_t\">74</a>].</p><p/><p class=\"headingAnchor\" id=\"H18\"><span class=\"h2\">ALS5 (SPG11 gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Familial ALS5, the most common form of autosomal recessive familial ALS, has been linked to mutations of the SPG11 gene on chromosome 15q15-q22 [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/75,76\" class=\"abstract_t\">75,76</a>]. The onset is in the second decade and is characterized by the development of lower motor neuron signs that exceed the upper motor neuron findings. Distal involvement is usually greater than proximal, and arms are usually involved more than legs. Bulbar involvement occurs late.</p><p>Mutations involving SPG11 are also the cause of hereditary spastic paraplegia with a thin or atrophied corpus callosum [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/77\" class=\"abstract_t\">77</a>]. However, the clinical features of patients with ALS5 and SPG11 mutations differ from those associated with hereditary spastic paraplegia due to the presence of bulbar symptoms, upper motor neuron involvement, and pathologic manifestations of ALS [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/76\" class=\"abstract_t\">76</a>]. In addition, the patients with familial ALS5 have not developed the corpus callosum thinning, ocular abnormalities, cognitive deficits, or psychiatric problems that are observed in SPG11-related hereditary spastic paraplegia. (See <a href=\"topic.htm?path=disorders-affecting-the-spinal-cord#H29\" class=\"medical medical_review\">&quot;Disorders affecting the spinal cord&quot;, section on 'Hereditary spastic paraplegias'</a>.)</p><p class=\"headingAnchor\" id=\"H29081274\"><span class=\"h1\">X-LINKED DOMINANT</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>In X-linked dominant inheritance, the gene responsible for the condition is located on the X chromosome and only one copy of the gene with the mutation is necessary to cause the disease. Therefore, unlike X-linked recessive inheritance, both males and females may be affected. If a father has an X-linked dominant condition all of his daughters will inherit the disease and none of his sons will inherit the disease. If a mother has an X-linked dominant disease, there is a 50 percent likelihood that her daughters or her sons will inherit the condition.</p><p class=\"headingAnchor\" id=\"H233524747\"><span class=\"h2\">ALS15 (UBQLN2 gene)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>In unrelated families from North America and Europe, a form of dominantly-inherited ALS with reduced penetrance in women has been linked to mutations of the UBQLN2 gene located at Xp11.21 [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/78,79\" class=\"abstract_t\">78,79</a>]. In the first report, the age of onset ranged from 16 to 71 years and was earlier in males, while mean disease duration was approximately four years and was similar in males and females [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/78\" class=\"abstract_t\">78</a>]. A frontotemporal lobar type of dementia accompanied some cases of ALS.</p><p class=\"headingAnchor\" id=\"H108656424\"><span class=\"h1\">INDETERMINATE INHERITANCE</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The inheritance pattern is not yet clear for ALS associated with SQSTM1 mutations, as reported in the next section.</p><p class=\"headingAnchor\" id=\"H106791803\"><span class=\"h2\">SQSTM1 gene</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Following upon the knowledge that protein degradation pathways are implicated in ALS and frontotemporal lobe degeneration (FTD), two groups screened large cohorts of patients with ALS and FTD and found mutations in the sequestosome 1 (SQSTM1) gene [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/80,81\" class=\"abstract_t\">80,81</a>]. The SQSTM1 gene encodes for the p62 protein, which binds ubiquitin and is involved in protein degradation and autophagy. SQSTM1 mutations were noted in both familial and sporadic cases of ALS and FTD. While one study reports heterozygous inheritance consistent with dominant genetics [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/80\" class=\"abstract_t\">80</a>] the other is silent on this subject [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/81\" class=\"abstract_t\">81</a>]. The precise role of this gene in the pathogenesis of ALS is not well defined. </p><p class=\"headingAnchor\" id=\"H19\"><span class=\"h1\">GENETIC TESTING AND COUNSELING</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Genetic testing for specific ALS-causing mutations is most useful for establishing the diagnosis (particularly for juvenile or early-stage disease) and for assessing risk among family members of an affected relative with an established genetic cause (<a href=\"image.htm?imageKey=PC%2F61384\" class=\"graphic graphic_table graphicRef61384 \">table 1</a>). In contrast, genetic testing for prognostic purposes in ALS is of limited value.</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The most common use of genetic testing for ALS is to provide risk information and subsequent counseling for unaffected at-risk family members. (See <a href=\"topic.htm?path=genetic-testing#H18\" class=\"medical medical_review\">&quot;Genetic testing&quot;, section on 'Principles of genetic testing'</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In cases where the diagnosis is clear, genetic testing has little or no additional value in making the diagnosis of ALS and will not impact medical management or treatment options. In rare cases of early stage disease (ie, clinically suspected and clinically possible ALS by El Escorial criteria), with or without a family history of ALS, the discovery of an ALS related gene can change the diagnosis to clinically definite laboratory supported familial ALS. (See <a href=\"topic.htm?path=diagnosis-of-amyotrophic-lateral-sclerosis-and-other-forms-of-motor-neuron-disease#H5\" class=\"medical medical_review\">&quot;Diagnosis of amyotrophic lateral sclerosis and other forms of motor neuron disease&quot;, section on 'Diagnostic criteria'</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Genetic testing may provide limited prognostic information, such as predicting rapid progression in individuals with the A4V SOD1 mutation. However, genetic testing should not be used for this purpose because the phenotypic variability seen in familial ALS makes such predictions unreliable at the level of individual cases.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Genetic testing may play a role in making the diagnosis of Kennedy syndrome and late-onset Tay-Sachs disease, two genetic disorders that can mimic some of the clinical features of ALS. This is discussed separately. (See <a href=\"topic.htm?path=diagnosis-of-amyotrophic-lateral-sclerosis-and-other-forms-of-motor-neuron-disease#H17\" class=\"medical medical_review\">&quot;Diagnosis of amyotrophic lateral sclerosis and other forms of motor neuron disease&quot;, section on 'Genetic testing'</a> and <a href=\"topic.htm?path=diagnosis-of-amyotrophic-lateral-sclerosis-and-other-forms-of-motor-neuron-disease#H19\" class=\"medical medical_review\">&quot;Diagnosis of amyotrophic lateral sclerosis and other forms of motor neuron disease&quot;, section on 'Differential diagnosis'</a>.)</p><p/><p>A glossary of genetic terms is available in a table (<a href=\"image.htm?imageKey=PC%2F72267\" class=\"graphic graphic_table graphicRef72267 \">table 2</a>) and, in an expanded form, in a separate topic. (See <a href=\"topic.htm?path=genetics-glossary-of-terms\" class=\"medical medical_review\">&quot;Genetics: Glossary of terms&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H233524905\"><span class=\"h2\">Recurrence risk within a family</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>For familial ALS, the recurrence risk of ALS in a given family (ie, whether or not another family member will develop ALS) is estimated primarily by determining the inheritance pattern. The mode of inheritance in familial ALS can be autosomal dominant, autosomal recessive, or (rarely) X-linked. Autosomal dominant inheritance is the most common pattern of inheritance observed in hereditary ALS.</p><p>The mode of inheritance is determined by analysis of the family history and the results of genetic testing if available. Thus, a detailed pedigree covering at least three generations should be obtained that includes the affected individual's parents, siblings, children, grandparents, aunts and uncles, nieces and nephews, and first cousins. Information to be collected should include current age, health status, age at death, cause of death, medical diagnoses, and any possible symptoms of ALS that may have gone undiagnosed.</p><p>Of note, the family history may be falsely negative due to a number of factors, including failure to recognize the disorder in family members, death from other causes prior to ALS symptom onset, non-paternity, or late onset or reduced penetrance of the disease in affected family members.</p><p>With sporadic ALS, it is generally believed that the recurrence risk of ALS is not significantly higher than the risk in the general population when there is only one family member affected with ALS. The recurrence risk would increase should another family member be diagnosed with ALS. The recurrence risk may be as high as 50 percent if multiple family members are affected with ALS. However, the exact recurrence risk for ALS cannot be determined in the absence of a positive genetic test.</p><p class=\"headingAnchor\" id=\"H455427516\"><span class=\"h2\">Who should be tested?</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>While any willing individual with ALS can have testing, a genetic cause is most likely to be found in those who have a family history of autosomal dominant disease or juvenile-onset ALS with suspected autosomal recessive inheritance. Genetic testing may also be useful in individuals with familial ALS who lack a clear family history consistent with an autosomal dominant disease, a situation that may occur because of reduced penetrance or limited information about important family members. However, the likelihood of identifying a mutation in a gene associated with ALS is reduced in this setting. As noted earlier, the most common use of genetic testing for ALS is to provide risk information and subsequent counseling for unaffected at-risk family members.</p><p>In patients with a family history suggesting autosomal dominant inheritance, genetic testing can provide prognostic information. As examples, the finding of a C9ORF72 mutation (see <a href=\"#H1808661\" class=\"local\">'C9ORF72 gene'</a> above) implies a high risk for the development of comorbid frontotemporal dementia with ALS, while the finding of the A4V SOD1 mutation (see <a href=\"#H3\" class=\"local\">'ALS1 (SOD1 gene)'</a> above) augurs a poor prognosis due to rapid progression and an average life expectancy of only 1.5 years after the onset of symptoms. However, the impact of a positive genetic test on clinical care in patients with ALS is limited, since the treatment options are few and management is mainly symptomatic.</p><p>There may be clinical trials available to patients with specific ALS genotypes. An updated list of clinical trials is available online at <a href=\"http://clinicaltrials.gov/&amp;token=YbJELcAW9G3aZfZrjTwglubdGkNLI6vF+UUqouKTYUezNDuyrLCVJ8CPUNOYDvHP&amp;TOPIC_ID=5176\" target=\"_blank\" class=\"external\">ClinicalTrials.gov</a>.</p><p>Individuals with apparently sporadic ALS may also pursue genetic testing, as approximately 10 percent or more of apparently sporadic cases of ALS have a mutation in one of the genes currently associated with familial ALS, including TARDBP, C9ORF72, SOD1, ANG, FUS, OPTN, and SETX, or in one of the genes likely to be susceptibility factors for ALS, including ATXN2 and SMN1. In these cases, the lack of family history suggests either low penetrance or a de novo mutation of a gene associated with ALS. In addition, the absence of family history with apparently sporadic ALS makes it difficult to quantify the risk of ALS for unaffected family members with the same mutation. (See <a href=\"topic.htm?path=epidemiology-and-pathogenesis-of-amyotrophic-lateral-sclerosis#H5\" class=\"medical medical_review\">&quot;Epidemiology and pathogenesis of amyotrophic lateral sclerosis&quot;, section on 'Genetic susceptibility in sporadic ALS'</a>.)</p><p>Anyone diagnosed with ALS or any adult with a family history of ALS who is considering genetic testing should first meet with a genetic counselor or a physician knowledgeable about familial ALS to discuss the genetic basis of the disease as well as the medical, psychological, and familial implications. Those pursuing testing should be informed of a number of issues and limitations of genetic testing.</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The predictive value of a positive genetic test can be limited by a number of factors, including lack of population-based data on penetrance (ie, the likelihood of developing disease given inheritance of a disease-causing mutation) and variable expressivity (ie, phenotypic variations in the way the disease is expressed). Thus, the ability of the genetic test to predict phenotype (ie, the clinical validity) may be limited.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The impact of a positive genetic test for ALS for the purpose of clinical care is essentially nil, since the treatment options are limited and management is mainly symptomatic.</p><p/><p>Additional concerns that should be addressed by genetic counseling include the following:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The meaning of a positive or negative test, including the slight possibility of false positive or negative results</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Psychosocial issues related to family dynamics</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Disclosing genetic information to at-risk family members</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The possibility of genetic discrimination</p><p/><p>These issues are explored in detail separately. (See <a href=\"topic.htm?path=genetic-testing#H20\" class=\"medical medical_review\">&quot;Genetic testing&quot;, section on 'Assessing the validity of a genetic test'</a> and <a href=\"topic.htm?path=genetic-testing#H22\" class=\"medical medical_review\">&quot;Genetic testing&quot;, section on 'Ethical, legal, and psychosocial issues'</a>.)</p><p>In families where a mutation in a gene associated with ALS has not been previously identified, testing is most likely to identify pathogenic variants in the affected family members with ALS, rather than at-risk members. Therefore, it is preferable to begin searching for variants by gene sequencing in affected members, and then subsequently perform direct genotyping of identified pathogenic variants in potentially at-risk individuals. Once a disease-causing mutation has been identified, other family members may choose to be tested for the same mutation regardless of whether or not they have symptoms. (See <a href=\"#H29079546\" class=\"local\">'Unaffected family members'</a> below.)</p><p>For ethical reasons, genetic testing is not appropriate for children and adolescents at risk for adult-onset ALS. There are a number of potential risks (eg, lack of autonomy, discrimination, stigmatization, anxiety) associated with the predictive testing of minors and there is no compelling benefit because preventive treatment is unavailable [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/82\" class=\"abstract_t\">82</a>].</p><p class=\"headingAnchor\" id=\"H29079488\"><span class=\"h2\">How the test is done</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Genetic testing can be done on a blood, saliva, or tissue sample. The sample is then analyzed for mutations in the specific gene(s) of interest.</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Full gene sequencing is considered the gold standard for identifying both known and novel mutations in a gene and involves analyzing the entire coding region of the gene. Full gene sequencing will detect most mutations in the tested gene.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Targeted mutation analysis (genotyping) involves testing for a specific mutation or set of mutations in the gene of interest, and will only detect mutations in the tested region(s) of the gene, but will not identify all mutations in the gene of interest. Targeted mutation analysis is often used for predictive testing in at-risk family members to identify mutations already known to exist in affected family members (ie, as identified through sequencing). </p><p/><p class=\"headingAnchor\" id=\"H29079495\"><span class=\"h2\">Genes to test</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Clinical genetic testing for inherited (familial) forms of ALS is available to look for causative mutations in SOD1, TARDBP, FUS, FIG4, ANG, Alsin (ALS2), VAPB, OPTN, and SETX genes [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/83\" class=\"abstract_t\">83</a>]. Approximately 13 to 20 percent of familial ALS is linked to mutations in the SOD1 gene, and an even higher percentage is linked to mutations in the C9ORF72 gene. In addition, mutations in the FUS gene, ANG gene, TARDBP gene, and FIG4 gene may account for as much as 5, 2, 5, and 3 percent of familial ALS, respectively. (See <a href=\"#H3\" class=\"local\">'ALS1 (SOD1 gene)'</a> above and <a href=\"#H1808661\" class=\"local\">'C9ORF72 gene'</a> above and <a href=\"#H6\" class=\"local\">'ALS6 (FUS gene)'</a> above and <a href=\"#H9\" class=\"local\">'ALS9 (ANG gene)'</a> above and <a href=\"#H10\" class=\"local\">'ALS10 (TARDBP gene)'</a> above and <a href=\"#H11\" class=\"local\">'ALS11 (FIG4 gene)'</a> above.)</p><p>Testing is commercially available for most of these genes, but is usually pursued only if there is a clear family history of autosomal dominant disease.</p><p>Ideally, it is best to order a genetic panel for ALS that includes five of the most common genes that are available for clinical genetic testing: C9ORF72, SOD1, TARDBP, FUS, FIG4, and ANG genes. Genetic testing can be done in a step-wise manner if a full panel cannot be ordered for financial or insurance reasons. In the absence of a personal or family history of frontotemporal dementia, the highest yield test to order is SOD1, since mutations in the SOD1 gene cause approximately 13 to 20 percent of familial cases of ALS and SOD1 mutations do not cause FTD. If SOD1 is negative, the other genes may be ordered either in the order of frequency associated with familial ALS or by the presence of clinical features. As an example, if the patient has spinal onset ALS with mainly lower motor neuron signs and an absence of bulbar signs, TARDBP may be the best gene to order next. </p><p>In the setting of a personal or family history of frontotemporal dementia, testing should begin with the C9ORF72 gene, followed by TARDBP, FUS, ANG, FIG4, and SOD1 (last).</p><p>If the patient has juvenile-onset ALS, genetic testing for SPG11, SETX, and ALS2 may be appropriate. (See <a href=\"#H16\" class=\"local\">'Autosomal recessive'</a> above.)</p><p class=\"headingAnchor\" id=\"H29079502\"><span class=\"h2\">Laboratories offering testing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Information on laboratories offering genetic testing for ALS is available online at <a href=\"http://www.ncbi.nlm.nih.gov/sites/GeneTests/?db=GeneTests&amp;token=ODIqmPyH03K5IrNHL4oottxoiiuAaZ+qTOeXL3h2l620gqRBLO4W2Rov7YvDMyab2Y/b42XMkMddScEb7o5GEw==&amp;TOPIC_ID=5176\" target=\"_blank\" class=\"external\">GeneTests</a>. The <a href=\"http://www.ncbi.nlm.nih.gov/sites/GeneTests/?db=GeneTests&amp;token=ODIqmPyH03K5IrNHL4oottxoiiuAaZ+qTOeXL3h2l620gqRBLO4W2Rov7YvDMyab2Y/b42XMkMddScEb7o5GEw==&amp;TOPIC_ID=5176\" target=\"_blank\" class=\"external\">GeneTests</a> website is a publicly funded medical genetics information resource that lists available genetic tests for thousands of conditions, with a directory of certified clinical laboratories specializing in genetic testing.</p><p class=\"headingAnchor\" id=\"H29079509\"><span class=\"h2\">Interpretation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>To properly interpret genetic test results for ALS genes, it is necessary to know whether the person who is being tested has ALS and whether a mutation in the family has already been identified.</p><p class=\"headingAnchor\" id=\"H29079516\"><span class=\"h3\">Patients with ALS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>For individuals with ALS who do <strong>not</strong> already have a known familial mutation, there are three possible types of test results as follows:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A positive test result means that the individual has a disease-causing (pathogenic) mutation or mutations and that the cause of ALS observed in the family has been identified. A positive result also further defines the risk to develop ALS for other family members. Interpretation of the ALS risk to other family members depends on the mode of inheritance of the gene with the mutation(s). (See <a href=\"#H233524905\" class=\"local\">'Recurrence risk within a family'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A negative test result means that the individual does not have a mutation in the tested gene or genes. However, a negative result does not rule out entirely the possibility of a genetic cause for the following reasons:</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Not all genes may have been tested. For example, if only testing for SOD1 was ordered, a negative result would not rule out the possibility of a mutation in TARDBP, FUS, FIG4, or ANG.</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Not all base positions of a targeted gene may have been adequately tested. For sequencing, this may occur if the assay was designed to target coding regions only, ignoring functional noncoding regions (such as regulatory or intronic regions). Also, gene coverage may be incomplete due to assay failure. In these instances, the individual may harbor pathogenic variants that are simply missed. In the context of targeted genotyping (testing of a limited number of variants) one can only confidently report a negative result for those specific variants tested. For example, a negative result from genotyping of the most common SOD1 mutation in North America, A4V, only excludes that variant, but does not rule out any of the other 139 known SOD1 mutations.</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>Not all genetic causes of ALS have been identified.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>An indeterminate result means that a change was found in the genetic sequence of the gene, but the laboratory cannot yet make a clear determination as to whether it is a pathogenic mutation or a benign polymorphism. Benign polymorphisms are common variations in DNA that do not cause disease. An indeterminate result is sometimes called a variant of unknown significance.</p><p/><p>With an indeterminate result, it is suggested that the mutation be tracked through the family, which involves testing multiple affected and unaffected family members. However, tracking a mutation through a family for an adult-onset condition can be challenging; when an unaffected family member tests positive for the mutation it is not clear whether this individual will develop ALS in the future or if the mutation is benign. It can take years to determine whether or not a variant of unknown significance is pathogenic.</p><p>For individuals with a diagnosis of ALS who are part of a family with a known familial mutation, a positive test result confirms that the individual did inherit the known familial mutation and supports that mutation as the cause of ALS for the individual. A negative result means that the individual did not inherit the known familial mutation; therefore the cause of ALS for the individual is unknown. In this scenario, the individual would be considered to have sporadic ALS. This is known as a phenocopy. A second sample may be sent for genetic testing to confirm the result.</p><p class=\"headingAnchor\" id=\"H29079546\"><span class=\"h3\">Unaffected family members</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Unaffected family members considering genetic testing for ALS are often concerned about possible symptoms of ALS. In such cases, a neurologic evaluation to look for evidence of ALS and a discussion with a trained genetic counselor can help alleviate concern.</p><p class=\"headingAnchor\" id=\"H233524177\"><span class=\"h4\">Known family mutation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Testing an asymptomatic person who is at risk for inheriting a known family mutation is called presymptomatic (or predictive) testing. For presymptomatic testing, a mutation must have been first identified in the family so that the clinician knows the appropriate test to order. The possible test results are as follows:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A positive test result means that the individual has a pathogenic mutation and is at risk to develop ALS. The probability that the individual will develop symptoms of ALS will depend on factors such as penetrance and expressivity. Penetrance and expressivity information is available for some SOD1 mutations but is limited for the other ALS genes. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A negative test result means that the individual does not have a mutation in the tested gene and is not at risk for the familial form of ALS.</p><p/><p>Family members may have different opinions about whether or not to pursue presymptomatic testing, as this is a very personal decision that can be confusing and emotionally difficult. Meeting with a trained genetic counselor and a psychological evaluation are recommended to help the individual throughout the testing process; multiple counseling sessions are optimal, including predecision, pretest, and posttest counseling [<a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/84\" class=\"abstract_t\">84</a>].</p><p class=\"headingAnchor\" id=\"H233524184\"><span class=\"h4\">No known family mutation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Presymptomatic (or predictive) genetic testing for an unaffected individual with a family history of ALS but no known familial mutation is <strong>not</strong> recommended because it is difficult to determine the appropriate test to order and interpret the meaning of negative or indeterminate genetic test results.</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A negative test result would mean that the individual did <strong>not</strong> have a mutation in the tested gene or genes. However, this finding would <strong>not</strong> rule out the possibility that they have inherited a genetic form of ALS. In this situation there are three possible scenarios to explain the history of ALS observed in the family:</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>It is due to a mutation in one of the tested genes and the individual did not inherit the mutation. The patient is not at an increased risk of developing ALS.</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>It is due to an unknown gene and it is unknown if the individual inherited the mutation. The patient may be at an increased risk of developing ALS.</p><p/><p class=\"bulletIndent2\"><span class=\"glyph\">&#8226;</span>It is due to environmental factors and is not genetic.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>An indeterminate result means that a change was found in the genetic sequence of the gene, but the laboratory cannot make a clear determination as to whether it is a pathogenic mutation or a benign polymorphism. This individual may or may not develop ALS in the future.</p><p/><p class=\"headingAnchor\" id=\"H29079596\"><span class=\"h2\">Differences between clinical and research genetic testing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The purpose of clinical genetic testing is primarily for diagnosis or presymptomatic genetic testing (<a href=\"image.htm?imageKey=PC%2F61384\" class=\"graphic graphic_table graphicRef61384 \">table 1</a>). The testing is performed in an accredited laboratory. A report of the results is generated and is part of the patient's official medical record. There is a charge for clinical testing, which may or may not be covered by insurance.</p><p>The purpose of research genetic testing is to discover and to better understand the genetic factors of ALS. Patient identifiers and information remain confidential and results do not become part of the patient's official medical record. There is no fee when participating in a research study, and generally results are not returned to participants. Many individuals choose to participate in research genetic testing to help advance the ALS field for the benefit of others with or without ALS now and for future generations.</p><p class=\"headingAnchor\" id=\"H29079459\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Familial amyotrophic lateral sclerosis (ALS) is phenotypically and genetically heterogeneous. In most individual cases, it may be difficult to determine on clinical grounds alone if ALS is familial or sporadic, especially at the onset of disease. The presence of atypical features such as young age of onset, sensory loss, a positive family history of ALS, other neurodegenerative disorders, and dementia should alert the clinician to the possibility of familial ALS. (See <a href=\"#H1\" class=\"local\">'Introduction'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The majority of familial ALS cases follow an autosomal dominant inheritance pattern. (See <a href=\"#H2\" class=\"local\">'Autosomal dominant'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Repeat expansions in the C9ORF72 gene and mutations in the SOD1 gene are the most common causes of familial ALS. In addition, mutations in the FUS gene and the TARDBP gene are relatively common causes of familial ALS. (See <a href=\"#H1808661\" class=\"local\">'C9ORF72 gene'</a> above and <a href=\"#H3\" class=\"local\">'ALS1 (SOD1 gene)'</a> above and <a href=\"#H6\" class=\"local\">'ALS6 (FUS gene)'</a> above and <a href=\"#H10\" class=\"local\">'ALS10 (TARDBP gene)'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Less common forms of autosomal dominant familial ALS include ALS3, ALS4, ALS7, ALS8, ALS9, ALS11, ALS12, ALS14, ALS17, ALS18, and ALS caused by mutations of the DAO and NEFH genes. (See <a href=\"#H4\" class=\"local\">'ALS3'</a> above and <a href=\"#H5\" class=\"local\">'ALS4 (SETX gene)'</a> above and <a href=\"#H7\" class=\"local\">'ALS7'</a> above and <a href=\"#H8\" class=\"local\">'ALS8 (VAPB gene)'</a> above and <a href=\"#H9\" class=\"local\">'ALS9 (ANG gene)'</a> above and <a href=\"#H11\" class=\"local\">'ALS11 (FIG4 gene)'</a> above and <a href=\"#H161409026\" class=\"local\">'ALS12 (OPTN gene)'</a> above and <a href=\"#H246948\" class=\"local\">'ALS14 (VCP gene)'</a> above and <a href=\"#H2572382106\" class=\"local\">'ALS17 (CHMP2B gene)'</a> above and <a href=\"#H10539001\" class=\"local\">'ALS18 (PFN1 gene)'</a> above and <a href=\"#H233525093\" class=\"local\">'DAO gene'</a> above and <a href=\"#H233525261\" class=\"local\">'NEFH gene'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Autosomal recessive forms of familial ALS are associated with mutations of the ALS2 and SPG11 genes. (See <a href=\"#H16\" class=\"local\">'Autosomal recessive'</a> above and <a href=\"#H17\" class=\"local\">'ALS2 (ALS2 gene)'</a> above and <a href=\"#H18\" class=\"local\">'ALS5 (SPG11 gene)'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>An X-linked dominant form of familial ALS is associated with UBQLN2 gene mutations. (See <a href=\"#H29081274\" class=\"local\">'X-linked dominant'</a> above and <a href=\"#H233524747\" class=\"local\">'ALS15 (UBQLN2 gene)'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Clinical genetic testing for inherited (familial) forms of ALS is available to look for causative mutations in a number of genes. The most common use of genetic testing for ALS is to provide risk information and subsequent counseling for unaffected at-risk family members. To properly interpret genetic test results for ALS genes, it is necessary to know whether the person being tested has ALS and whether a mutation in the family has already been identified. The recurrence risk of ALS in a given family (ie, whether or not another family member will develop ALS) is estimated primarily by determining the inheritance pattern. (See <a href=\"#H19\" class=\"local\">'Genetic testing and counseling'</a> above.)</p><p/><p/></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/1\" class=\"nounderline abstract_t\">Byrne S, Walsh C, Lynch C, et al. Rate of familial amyotrophic lateral sclerosis: a systematic review and meta-analysis. J Neurol Neurosurg Psychiatry 2011; 82:623.</a></li><li class=\"breakAll\">Falcone D, McCluskey L. Unpublished data. 2011.</li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/3\" class=\"nounderline abstract_t\">Zou ZY, Zhou ZR, Che CH, et al. Genetic epidemiology of amyotrophic lateral sclerosis: a systematic review and meta-analysis. J Neurol Neurosurg Psychiatry 2017; 88:540.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/4\" class=\"nounderline abstract_t\">Morita M, Al-Chalabi A, Andersen PM, et al. A locus on chromosome 9p confers susceptibility to ALS and frontotemporal dementia. Neurology 2006; 66:839.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/5\" class=\"nounderline abstract_t\">Vance C, Al-Chalabi A, Ruddy D, et al. Familial amyotrophic lateral sclerosis with frontotemporal dementia is linked to a locus on chromosome 9p13.2-21.3. Brain 2006; 129:868.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/6\" class=\"nounderline abstract_t\">Valdmanis PN, Dupre N, Bouchard JP, et al. Three families with amyotrophic lateral sclerosis and frontotemporal dementia with evidence of linkage to chromosome 9p. Arch Neurol 2007; 64:240.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/7\" class=\"nounderline abstract_t\">Le Ber I, Camuzat A, Berger E, et al. Chromosome 9p-linked families with frontotemporal dementia associated with motor neuron disease. Neurology 2009; 72:1669.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/8\" class=\"nounderline abstract_t\">DeJesus-Hernandez M, Mackenzie IR, Boeve BF, et al. Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS. Neuron 2011; 72:245.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/9\" class=\"nounderline abstract_t\">Renton AE, Majounie E, Waite A, et al. A hexanucleotide repeat expansion in C9ORF72 is the cause of chromosome 9p21-linked ALS-FTD. Neuron 2011; 72:257.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/10\" class=\"nounderline abstract_t\">Gijselinck I, Van Langenhove T, van der Zee J, et al. A C9orf72 promoter repeat expansion in a Flanders-Belgian cohort with disorders of the frontotemporal lobar degeneration-amyotrophic lateral sclerosis spectrum: a gene identification study. Lancet Neurol 2012; 11:54.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/11\" class=\"nounderline abstract_t\">Byrne S, Elamin M, Bede P, et al. Cognitive and clinical characteristics of patients with amyotrophic lateral sclerosis carrying a C9orf72 repeat expansion: a population-based cohort study. Lancet Neurol 2012; 11:232.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/12\" class=\"nounderline abstract_t\">Hosler BA, Siddique T, Sapp PC, et al. Linkage of familial amyotrophic lateral sclerosis with frontotemporal dementia to chromosome 9q21-q22. JAMA 2000; 284:1664.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/13\" class=\"nounderline abstract_t\">Mackenzie IR, Rademakers R. The molecular genetics and neuropathology of frontotemporal lobar degeneration: recent developments. Neurogenetics 2007; 8:237.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/14\" class=\"nounderline abstract_t\">Rosen DR, Siddique T, Patterson D, et al. Mutations in Cu/Zn superoxide dismutase gene are associated with familial amyotrophic lateral sclerosis. Nature 1993; 362:59.</a></li><li class=\"breakAll\">The Amyotrophic Lateral Sclerosis Online Genetics Database. http://alsod.iop.kcl.ac.uk/ (Accessed on August 22, 2017).</li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/16\" class=\"nounderline abstract_t\">Abel O, Shatunov A, Jones AR, et al. Development of a Smartphone App for a Genetics Website: The Amyotrophic Lateral Sclerosis Online Genetics Database (ALSoD). JMIR Mhealth Uhealth 2013; 1:e18.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/17\" class=\"nounderline abstract_t\">Andersen PM, Sims KB, Xin WW, et al. Sixteen novel mutations in the Cu/Zn superoxide dismutase gene in amyotrophic lateral sclerosis: a decade of discoveries, defects and disputes. Amyotroph Lateral Scler Other Motor Neuron Disord 2003; 4:62.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/18\" class=\"nounderline abstract_t\">Ratovitski T, Corson LB, Strain J, et al. Variation in the biochemical/biophysical properties of mutant superoxide dismutase 1 enzymes and the rate of disease progression in familial amyotrophic lateral sclerosis kindreds. Hum Mol Genet 1999; 8:1451.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/19\" class=\"nounderline abstract_t\">Cudkowicz ME, McKenna-Yasek D, Chen C, et al. Limited corticospinal tract involvement in amyotrophic lateral sclerosis subjects with the A4V mutation in the copper/zinc superoxide dismutase gene. Ann Neurol 1998; 43:703.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/20\" class=\"nounderline abstract_t\">Andersen PM. Genetics of sporadic ALS. Amyotroph Lateral Scler Other Motor Neuron Disord 2001; 2 Suppl 1:S37.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/21\" class=\"nounderline abstract_t\">Gellera C. Genetics of ALS in Italian families. Amyotroph Lateral Scler Other Motor Neuron Disord 2001; 2 Suppl 1:S43.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/22\" class=\"nounderline abstract_t\">Gellera C, Castellotti B, Riggio MC, et al. Superoxide dismutase gene mutations in Italian patients with familial and sporadic amyotrophic lateral sclerosis: identification of three novel missense mutations. Neuromuscul Disord 2001; 11:404.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/23\" class=\"nounderline abstract_t\">Hays AP, Naini A, He CZ, et al. Sporadic amyotrophic lateral sclerosis and breast cancer: Hyaline conglomerate inclusions lead to identification of SOD1 mutation. J Neurol Sci 2006; 242:67.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/24\" class=\"nounderline abstract_t\">Aksoy H, Dean G, Elian M, et al. A4T mutation in the SOD1 gene causing familial amyotrophic lateral sclerosis. Neuroepidemiology 2003; 22:235.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/25\" class=\"nounderline abstract_t\">Arisato T, Okubo R, Arata H, et al. Clinical and pathological studies of familial amyotrophic lateral sclerosis (FALS) with SOD1 H46R mutation in large Japanese families. Acta Neuropathol 2003; 106:561.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/26\" class=\"nounderline abstract_t\">Rezania K, Yan J, Dellefave L, et al. A rare Cu/Zn superoxide dismutase mutation causing familial amyotrophic lateral sclerosis with variable age of onset, incomplete penetrance and a sensory neuropathy. Amyotroph Lateral Scler Other Motor Neuron Disord 2003; 4:162.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/27\" class=\"nounderline abstract_t\">R&eacute;gal L, Vanopdenbosch L, Tilkin P, et al. The G93C mutation in superoxide dismutase 1: clinicopathologic phenotype and prognosis. Arch Neurol 2006; 63:262.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/28\" class=\"nounderline abstract_t\">Andersen PM, Nilsson P, Ala-Hurula V, et al. Amyotrophic lateral sclerosis associated with homozygosity for an Asp90Ala mutation in CuZn-superoxide dismutase. Nat Genet 1995; 10:61.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/29\" class=\"nounderline abstract_t\">Andersen PM, Forsgren L, Binzer M, et al. Autosomal recessive adult-onset amyotrophic lateral sclerosis associated with homozygosity for Asp90Ala CuZn-superoxide dismutase mutation. A clinical and genealogical study of 36 patients. Brain 1996; 119 ( Pt 4):1153.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/30\" class=\"nounderline abstract_t\">Robberecht W, Aguirre T, Van den Bosch L, et al. D90A heterozygosity in the SOD1 gene is associated with familial and apparently sporadic amyotrophic lateral sclerosis. Neurology 1996; 47:1336.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/31\" class=\"nounderline abstract_t\">Hand CK, Khoris J, Salachas F, et al. A novel locus for familial amyotrophic lateral sclerosis, on chromosome 18q. Am J Hum Genet 2002; 70:251.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/32\" class=\"nounderline abstract_t\">Chance PF, Rabin BA, Ryan SG, et al. Linkage of the gene for an autosomal dominant form of juvenile amyotrophic lateral sclerosis to chromosome 9q34. Am J Hum Genet 1998; 62:633.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/33\" class=\"nounderline abstract_t\">Chen YZ, Bennett CL, Huynh HM, et al. DNA/RNA helicase gene mutations in a form of juvenile amyotrophic lateral sclerosis (ALS4). Am J Hum Genet 2004; 74:1128.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/34\" class=\"nounderline abstract_t\">Asaka T, Yokoji H, Ito J, et al. Autosomal recessive ataxia with peripheral neuropathy and elevated AFP: novel mutations in SETX. Neurology 2006; 66:1580.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/35\" class=\"nounderline abstract_t\">Chen YZ, Hashemi SH, Anderson SK, et al. Senataxin, the yeast Sen1p orthologue: characterization of a unique protein in which recessive mutations cause ataxia and dominant mutations cause motor neuron disease. Neurobiol Dis 2006; 23:97.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/36\" class=\"nounderline abstract_t\">Kwiatkowski TJ Jr, Bosco DA, Leclerc AL, et al. Mutations in the FUS/TLS gene on chromosome 16 cause familial amyotrophic lateral sclerosis. Science 2009; 323:1205.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/37\" class=\"nounderline abstract_t\">Vance C, Rogelj B, Hortob&aacute;gyi T, et al. Mutations in FUS, an RNA processing protein, cause familial amyotrophic lateral sclerosis type 6. Science 2009; 323:1208.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/38\" class=\"nounderline abstract_t\">Yan J, Deng HX, Siddique N, et al. Frameshift and novel mutations in FUS in familial amyotrophic lateral sclerosis and ALS/dementia. Neurology 2010; 75:807.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/39\" class=\"nounderline abstract_t\">Akiyama T, Warita H, Kato M, et al. Genotype-phenotype relationships in familial amyotrophic lateral sclerosis with FUS/TLS mutations in Japan. Muscle Nerve 2016; 54:398.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/40\" class=\"nounderline abstract_t\">Sapp PC, Hosler BA, McKenna-Yasek D, et al. Identification of two novel loci for dominantly inherited familial amyotrophic lateral sclerosis. Am J Hum Genet 2003; 73:397.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/41\" class=\"nounderline abstract_t\">Nishimura AL, Mitne-Neto M, Silva HC, et al. A novel locus for late onset amyotrophic lateral sclerosis/motor neurone disease variant at 20q13. J Med Genet 2004; 41:315.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/42\" class=\"nounderline abstract_t\">Nishimura AL, Mitne-Neto M, Silva HC, et al. A mutation in the vesicle-trafficking protein VAPB causes late-onset spinal muscular atrophy and amyotrophic lateral sclerosis. Am J Hum Genet 2004; 75:822.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/43\" class=\"nounderline abstract_t\">Greenway MJ, Alexander MD, Ennis S, et al. A novel candidate region for ALS on chromosome 14q11.2. Neurology 2004; 63:1936.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/44\" class=\"nounderline abstract_t\">Greenway MJ, Andersen PM, Russ C, et al. ANG mutations segregate with familial and 'sporadic' amyotrophic lateral sclerosis. Nat Genet 2006; 38:411.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/45\" class=\"nounderline abstract_t\">Wu D, Yu W, Kishikawa H, et al. Angiogenin loss-of-function mutations in amyotrophic lateral sclerosis. Ann Neurol 2007; 62:609.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/46\" class=\"nounderline abstract_t\">van Es MA, Diekstra FP, Veldink JH, et al. A case of ALS-FTD in a large FALS pedigree with a K17I ANG mutation. Neurology 2009; 72:287.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/47\" class=\"nounderline abstract_t\">Sreedharan J, Blair IP, Tripathi VB, et al. TDP-43 mutations in familial and sporadic amyotrophic lateral sclerosis. Science 2008; 319:1668.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/48\" class=\"nounderline abstract_t\">Yokoseki A, Shiga A, Tan CF, et al. TDP-43 mutation in familial amyotrophic lateral sclerosis. Ann Neurol 2008; 63:538.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/49\" class=\"nounderline abstract_t\">Van Deerlin VM, Leverenz JB, Bekris LM, et al. TARDBP mutations in amyotrophic lateral sclerosis with TDP-43 neuropathology: a genetic and histopathological analysis. Lancet Neurol 2008; 7:409.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/50\" class=\"nounderline abstract_t\">Kabashi E, Valdmanis PN, Dion P, et al. TARDBP mutations in individuals with sporadic and familial amyotrophic lateral sclerosis. Nat Genet 2008; 40:572.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/51\" class=\"nounderline abstract_t\">K&uuml;hnlein P, Sperfeld AD, Vanmassenhove B, et al. Two German kindreds with familial amyotrophic lateral sclerosis due to TARDBP mutations. Arch Neurol 2008; 65:1185.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/52\" class=\"nounderline abstract_t\">Corcia P, Valdmanis P, Millecamps S, et al. Phenotype and genotype analysis in amyotrophic lateral sclerosis with TARDBP gene mutations. Neurology 2012; 78:1519.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/53\" class=\"nounderline abstract_t\">Borroni B, Archetti S, Del Bo R, et al. TARDBP mutations in frontotemporal lobar degeneration: frequency, clinical features, and disease course. Rejuvenation Res 2010; 13:509.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/54\" class=\"nounderline abstract_t\">Chow CY, Landers JE, Bergren SK, et al. Deleterious variants of FIG4, a phosphoinositide phosphatase, in patients with ALS. Am J Hum Genet 2009; 84:85.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/55\" class=\"nounderline abstract_t\">Maruyama H, Morino H, Ito H, et al. Mutations of optineurin in amyotrophic lateral sclerosis. Nature 2010; 465:223.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/56\" class=\"nounderline abstract_t\">Del Bo R, Tiloca C, Pensato V, et al. Novel optineurin mutations in patients with familial and sporadic amyotrophic lateral sclerosis. J Neurol Neurosurg Psychiatry 2011; 82:1239.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/57\" class=\"nounderline abstract_t\">Belzil VV, Daoud H, Desjarlais A, et al. Analysis of OPTN as a causative gene for amyotrophic lateral sclerosis. Neurobiol Aging 2011; 32:555.e13.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/58\" class=\"nounderline abstract_t\">Johnson JO, Mandrioli J, Benatar M, et al. Exome sequencing reveals VCP mutations as a cause of familial ALS. Neuron 2010; 68:857.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/59\" class=\"nounderline abstract_t\">Gonz&aacute;lez-P&eacute;rez P, Cirulli ET, Drory VE, et al. Novel mutation in VCP gene causes atypical amyotrophic lateral sclerosis. Neurology 2012; 79:2201.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/60\" class=\"nounderline abstract_t\">Watts GD, Wymer J, Kovach MJ, et al. Inclusion body myopathy associated with Paget disease of bone and frontotemporal dementia is caused by mutant valosin-containing protein. Nat Genet 2004; 36:377.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/61\" class=\"nounderline abstract_t\">Nalbandian A, Donkervoort S, Dec E, et al. The multiple faces of valosin-containing protein-associated diseases: inclusion body myopathy with Paget's disease of bone, frontotemporal dementia, and amyotrophic lateral sclerosis. J Mol Neurosci 2011; 45:522.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/62\" class=\"nounderline abstract_t\">Ju JS, Fuentealba RA, Miller SE, et al. Valosin-containing protein (VCP) is required for autophagy and is disrupted in VCP disease. J Cell Biol 2009; 187:875.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/63\" class=\"nounderline abstract_t\">Gydesen S, Brown JM, Brun A, et al. Chromosome 3 linked frontotemporal dementia (FTD-3). Neurology 2002; 59:1585.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/64\" class=\"nounderline abstract_t\">Parkinson N, Ince PG, Smith MO, et al. ALS phenotypes with mutations in CHMP2B (charged multivesicular body protein 2B). Neurology 2006; 67:1074.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/65\" class=\"nounderline abstract_t\">Wu CH, Fallini C, Ticozzi N, et al. Mutations in the profilin 1 gene cause familial amyotrophic lateral sclerosis. Nature 2012; 488:499.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/66\" class=\"nounderline abstract_t\">Mitchell J, Paul P, Chen HJ, et al. Familial amyotrophic lateral sclerosis is associated with a mutation in D-amino acid oxidase. Proc Natl Acad Sci U S A 2010; 107:7556.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/67\" class=\"nounderline abstract_t\">Al-Chalabi A, Andersen PM, Nilsson P, et al. Deletions of the heavy neurofilament subunit tail in amyotrophic lateral sclerosis. Hum Mol Genet 1999; 8:157.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/68\" class=\"nounderline abstract_t\">Kim HJ, Kim NC, Wang YD, et al. Mutations in prion-like domains in hnRNPA2B1 and hnRNPA1 cause multisystem proteinopathy and ALS. Nature 2013; 495:467.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/69\" class=\"nounderline abstract_t\">Gros-Louis F, Meijer IA, Hand CK, et al. An ALS2 gene mutation causes hereditary spastic paraplegia in a Pakistani kindred. Ann Neurol 2003; 53:144.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/70\" class=\"nounderline abstract_t\">Hadano S, Hand CK, Osuga H, et al. A gene encoding a putative GTPase regulator is mutated in familial amyotrophic lateral sclerosis 2. Nat Genet 2001; 29:166.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/71\" class=\"nounderline abstract_t\">Kress JA, K&uuml;hnlein P, Winter P, et al. Novel mutation in the ALS2 gene in juvenile amyotrophic lateral sclerosis. Ann Neurol 2005; 58:800.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/72\" class=\"nounderline abstract_t\">Sheerin UM, Schneider SA, Carr L, et al. ALS2 mutations: juvenile amyotrophic lateral sclerosis and generalized dystonia. Neurology 2014; 82:1065.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/73\" class=\"nounderline abstract_t\">Gascon GG, Chavis P, Yaghmour A, et al. Familial childhood primary lateral sclerosis with associated gaze paresis. Neuropediatrics 1995; 26:313.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/74\" class=\"nounderline abstract_t\">Eymard-Pierre E, Lesca G, Dollet S, et al. Infantile-onset ascending hereditary spastic paralysis is associated with mutations in the alsin gene. Am J Hum Genet 2002; 71:518.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/75\" class=\"nounderline abstract_t\">Hentati A, Ouahchi K, Pericak-Vance MA, et al. Linkage of a commoner form of recessive amyotrophic lateral sclerosis to chromosome 15q15-q22 markers. Neurogenetics 1998; 2:55.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/76\" class=\"nounderline abstract_t\">Orlacchio A, Babalini C, Borreca A, et al. SPATACSIN mutations cause autosomal recessive juvenile amyotrophic lateral sclerosis. Brain 2010; 133:591.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/77\" class=\"nounderline abstract_t\">Stevanin G, Santorelli FM, Azzedine H, et al. Mutations in SPG11, encoding spatacsin, are a major cause of spastic paraplegia with thin corpus callosum. Nat Genet 2007; 39:366.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/78\" class=\"nounderline abstract_t\">Deng HX, Chen W, Hong ST, et al. Mutations in UBQLN2 cause dominant X-linked juvenile and adult-onset ALS and ALS/dementia. Nature 2011; 477:211.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/79\" class=\"nounderline abstract_t\">Gellera C, Tiloca C, Del Bo R, et al. Ubiquilin 2 mutations in Italian patients with amyotrophic lateral sclerosis and frontotemporal dementia. J Neurol Neurosurg Psychiatry 2013; 84:183.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/80\" class=\"nounderline abstract_t\">Fecto F, Yan J, Vemula SP, et al. SQSTM1 mutations in familial and sporadic amyotrophic lateral sclerosis. Arch Neurol 2011; 68:1440.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/81\" class=\"nounderline abstract_t\">Rubino E, Rainero I, Chi&ograve; A, et al. SQSTM1 mutations in frontotemporal lobar degeneration and amyotrophic lateral sclerosis. Neurology 2012; 79:1556.</a></li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/82\" class=\"nounderline abstract_t\">Points to consider: ethical, legal, and psychosocial implications of genetic testing in children and adolescents. American Society of Human Genetics Board of Directors, American College of Medical Genetics Board of Directors. Am J Hum Genet 1995; 57:1233.</a></li><li class=\"breakAll\">Donkervoort S, Siddique T. Amyotrophic lateral sclerosis overview. In: GeneReviews [Internet]. www.ncbi.nlm.nih.gov/bookshelf/br.fcgi?book=gene&amp;part=als-overview (Accessed on October 10, 2011).</li><li><a href=\"https://www.uptodate.com/contents/familial-amyotrophic-lateral-sclerosis/abstract/84\" class=\"nounderline abstract_t\">Benatar M, Stanislaw C, Reyes E, et al. Presymptomatic ALS genetic counseling and testing: Experience and recommendations. Neurology 2016; 86:2295.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 5176 Version 35.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H29079459\"><span>SUMMARY</span></a></li><li><a href=\"#H1\" id=\"outline-link-H1\">INTRODUCTION</a></li><li><a href=\"#H136713437\" id=\"outline-link-H136713437\">OVERVIEW</a></li><li><a href=\"#H2\" id=\"outline-link-H2\">AUTOSOMAL DOMINANT</a><ul><li><a href=\"#H1808661\" id=\"outline-link-H1808661\">C9ORF72 gene</a></li><li><a href=\"#H3\" id=\"outline-link-H3\">ALS1 (SOD1 gene)</a></li><li><a href=\"#H4\" id=\"outline-link-H4\">ALS3</a></li><li><a href=\"#H5\" id=\"outline-link-H5\">ALS4 (SETX gene)</a></li><li><a href=\"#H6\" id=\"outline-link-H6\">ALS6 (FUS gene)</a></li><li><a href=\"#H7\" id=\"outline-link-H7\">ALS7</a></li><li><a href=\"#H8\" id=\"outline-link-H8\">ALS8 (VAPB gene)</a></li><li><a href=\"#H9\" id=\"outline-link-H9\">ALS9 (ANG gene)</a></li><li><a href=\"#H10\" id=\"outline-link-H10\">ALS10 (TARDBP gene)</a></li><li><a href=\"#H11\" id=\"outline-link-H11\">ALS11 (FIG4 gene)</a></li><li><a href=\"#H161409026\" id=\"outline-link-H161409026\">ALS12 (OPTN gene)</a></li><li><a href=\"#H233524289\" id=\"outline-link-H233524289\">ALS13 (ATXN2 gene)</a></li><li><a href=\"#H246948\" id=\"outline-link-H246948\">ALS14 (VCP gene)</a></li><li><a href=\"#H2572382106\" id=\"outline-link-H2572382106\">ALS17 (CHMP2B gene)</a></li><li><a href=\"#H10539001\" id=\"outline-link-H10539001\">ALS18 (PFN1 gene)</a></li><li><a href=\"#H233525093\" id=\"outline-link-H233525093\">DAO gene</a></li><li><a href=\"#H233525261\" id=\"outline-link-H233525261\">NEFH gene</a></li><li><a href=\"#H2187961\" id=\"outline-link-H2187961\">HNRNPA2B1 and HNRNPA1 genes</a></li></ul></li><li><a href=\"#H16\" id=\"outline-link-H16\">AUTOSOMAL RECESSIVE</a><ul><li><a href=\"#H17\" id=\"outline-link-H17\">ALS2 (ALS2 gene)</a></li><li><a href=\"#H18\" id=\"outline-link-H18\">ALS5 (SPG11 gene)</a></li></ul></li><li><a href=\"#H29081274\" id=\"outline-link-H29081274\">X-LINKED DOMINANT</a><ul><li><a href=\"#H233524747\" id=\"outline-link-H233524747\">ALS15 (UBQLN2 gene)</a></li></ul></li><li><a href=\"#H108656424\" id=\"outline-link-H108656424\">INDETERMINATE INHERITANCE</a><ul><li><a href=\"#H106791803\" id=\"outline-link-H106791803\">SQSTM1 gene</a></li></ul></li><li><a href=\"#H19\" id=\"outline-link-H19\">GENETIC TESTING AND COUNSELING</a><ul><li><a href=\"#H233524905\" id=\"outline-link-H233524905\">Recurrence risk within a family</a></li><li><a href=\"#H455427516\" id=\"outline-link-H455427516\">Who should be tested?</a></li><li><a href=\"#H29079488\" id=\"outline-link-H29079488\">How the test is done</a></li><li><a href=\"#H29079495\" id=\"outline-link-H29079495\">Genes to test</a></li><li><a href=\"#H29079502\" id=\"outline-link-H29079502\">Laboratories offering testing</a></li><li><a href=\"#H29079509\" id=\"outline-link-H29079509\">Interpretation</a><ul><li><a href=\"#H29079516\" id=\"outline-link-H29079516\">- Patients with ALS</a></li><li><a href=\"#H29079546\" id=\"outline-link-H29079546\">- Unaffected family members</a><ul><li><a href=\"#H233524177\" id=\"outline-link-H233524177\">Known family mutation</a></li><li><a href=\"#H233524184\" id=\"outline-link-H233524184\">No known family mutation</a></li></ul></li></ul></li><li><a href=\"#H29079596\" id=\"outline-link-H29079596\">Differences between clinical and research genetic testing</a></li></ul></li><li><a href=\"#H29079459\" id=\"outline-link-H29079459\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>GRAPHICS <a href=\"#\" id=\"viewAllGraphicsLink\">View All</a></h2><div id=\"outlineGraphics\"><ul><li><div id=\"NEURO/5176|TAB\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"TABLES\">TABLES</a></div><ul><li><a href=\"image.htm?imageKey=PC/61384\" class=\"graphic graphic_table\">- Benefits and risks of genetic testing</a></li><li><a href=\"image.htm?imageKey=PC/72267\" class=\"graphic graphic_table\">- Genetic terms</a></li></ul></li></ul></div></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=charcot-marie-tooth-disease-genetics-clinical-features-and-diagnosis\" class=\"medical medical_review\">Charcot-Marie-Tooth disease: Genetics, clinical features, and diagnosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=clinical-features-of-amyotrophic-lateral-sclerosis-and-other-forms-of-motor-neuron-disease\" class=\"medical medical_review\">Clinical features of amyotrophic lateral sclerosis and other forms of motor neuron disease</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=diagnosis-of-amyotrophic-lateral-sclerosis-and-other-forms-of-motor-neuron-disease\" class=\"medical medical_review\">Diagnosis of amyotrophic lateral sclerosis and other forms of motor neuron disease</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=disorders-affecting-the-spinal-cord\" class=\"medical medical_review\">Disorders affecting the spinal cord</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=epidemiology-and-pathogenesis-of-amyotrophic-lateral-sclerosis\" class=\"medical medical_review\">Epidemiology and pathogenesis of amyotrophic lateral sclerosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=frontotemporal-dementia-clinical-features-and-diagnosis\" class=\"medical medical_review\">Frontotemporal dementia: Clinical features and diagnosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetic-testing\" class=\"medical medical_review\">Genetic testing</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetics-glossary-of-terms\" class=\"medical medical_review\">Genetics: Glossary of terms</a></li></ul></div></div>","javascript":null}