var data={"title":"Genomic disorders: An overview","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Genomic disorders: An overview</div><dl id=\"topicContributors\"><dt><span> </span>Author:</dt><dd><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/contributors\" class=\"contributor contributor_credentials\">Carlos A Bacino, MD, FACMG</a></dd><dt><span> </span>Section Editors:</dt><dd><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/contributors\" class=\"contributor contributor_credentials\">Helen V Firth, DM, FRCP, DCH</a></dd><dd><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/contributors\" class=\"contributor contributor_credentials\">Benjamin A Raby, MD, MPH</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/contributors\" class=\"contributor contributor_credentials\">Jennifer S Tirnauer, MD</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Mar 24, 2017.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H16722192\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Genomic disorders are diseases that result from the loss or gain of <span class=\"nowrap\">chromosomal/DNA</span> material. The most common and better-delineated genomic disorders are divided in two main categories: those resulting from copy number losses (deletion syndromes) and copy number gains (duplication syndromes).</p><p>An overview of genomic disorders is presented here. Specific syndromic disorders are reviewed separately. </p><p class=\"headingAnchor\" id=\"H16722199\"><span class=\"h1\">COPY NUMBER VARIATIONS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Structural genetic variation refers to a class of sequence alterations spanning typically more than 1000 base pairs (one kilobase or kb) [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/1\" class=\"abstract_t\">1</a>]. However, studies have indicated that these genetic variations can be smaller (eg, in the 450 base pair and up range) and that most individuals have at least 1,000 such variations [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/2\" class=\"abstract_t\">2</a>]. </p><p>These sequence variations include quantitative variations such as copy number variations (CNVs), sequence rearrangements (such as those observed among immunoglobulins), and other less common variations, including chromosomal rearrangements that may or may not alter the genome contents and in some cases result in disease.</p><p>CNVs, the most prevalent type of structural variation, are DNA segments spanning thousands to millions of bases whose copy number varies between different individuals [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/3,4\" class=\"abstract_t\">3,4</a>]. These submicroscopic genomic differences in the number of copies of one or more sections of DNA are the result of DNA gains or losses. Copy number gains can be the result of duplications, triplications, or even multiple copy number gains. Most deletions are one copy loss (heterozygous), but in some instances the loss can affect both copies (homozygous deletions).</p><p>CNVs are most commonly inherited but can occur de novo (ie, as a new event). These were initially thought to be rare events resulting from sporadic mutation and correlated with specific Mendelian diseases [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/5,6\" class=\"abstract_t\">5,6</a>]. These misperceptions about their rarity and absolute disease linkage were primarily due to technical limitations precluding genome-wide assessments in large cohorts. Advances in technology have shown that deviation from the diploid state is widespread and contributes substantially to genetic diversity. Some studies have suggested that CNV differences in the human genome are as extensive as 20 percent, although this may be an overestimation [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/7,8\" class=\"abstract_t\">7,8</a>] and probably closer to 5 percent. It is estimated conservatively that most individuals carry an average of three large-scale CNVs [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/4\" class=\"abstract_t\">4</a>]. The number of known CNVs that contribute to disease pathogenesis continues to increase.</p><p>The physical distribution of CNVs appears to be nonrandom, with both CNV hot and cold spots reported [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/7,9\" class=\"abstract_t\">7,9</a>]. CNV frequency is greatest in regions of segmental duplication (a 4- to 10-fold enrichment for CNVs), consistent with nonallelic homologous recombination as a primary mechanism for CNV mutation [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/3,10-13\" class=\"abstract_t\">3,10-13</a>]. </p><p>CNVs are more commonly observed in gene-rich regions. CNVs appear to be enriched in specific gene families, including immune and inflammatory response genes, cell signaling and cell adhesion molecules, structural proteins, and olfactory receptors [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/7\" class=\"abstract_t\">7</a>]. Most of these differences probably represent benign CNVs that reflect normal variation with no apparent clinical consequence [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/14\" class=\"abstract_t\">14</a>].</p><p>CNVs can be pathogenic if they involve a dose-sensitive gene(s) or if they influence genomic regions through regulatory elements [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/15,16\" class=\"abstract_t\">15,16</a>]. Some pathogenic CNVs cause syndromic disorders with consistent phenotypic features (eg, deletions of elastin in Williams syndrome, duplications of PMP22 in Charcot-Marie-Tooth disease type 1A [CMT1A]), while others are associated with disease susceptibility or resistance (eg, cancer, human immunodeficiency virus [HIV] infection, autoimmune disorders, autism). </p><p>CNVs can be responsible for Mendelian diseases associated with large gains and losses of genetic material or even small losses or gains or losses at the exonic level. Examples include the following: </p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Contiguous gene deletions or duplications as seen in Williams-Beuren syndrome, 22q11 deletion-syndrome, Smith-Magenis syndrome, and Potocki-Lupski syndrome. These recurrent CNVs are mediated by nonallelic homologous recombination (NAHR) at sites of low-copy repeats. (See <a href=\"topic.htm?path=williams-beuren-syndrome-renal-manifestations\" class=\"medical medical_review\">&quot;Williams-Beuren syndrome: Renal manifestations&quot;</a> and <a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-12-to-22#H86262802\" class=\"medical medical_review\">&quot;Microdeletion syndromes (chromosomes 12 to 22)&quot;, section on '17p11.2 deletion syndrome (Smith-Magenis syndrome)'</a> and <a href=\"topic.htm?path=microduplication-syndromes#H12818290\" class=\"medical medical_review\">&quot;Microduplication syndromes&quot;, section on '17p11.2 duplication syndrome (Potocki-Lupski syndrome)'</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Deletions of genes or portions of genes (exons) leading to Mendelian-inherited genetic disorders, including disorders that are autosomal dominant (eg, Rubinstein-Taybi syndrome) and X-linked recessive (eg, Duchenne muscular dystrophy) [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/17\" class=\"abstract_t\">17</a>]. (See <a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-12-to-22#H2157284\" class=\"medical medical_review\">&quot;Microdeletion syndromes (chromosomes 12 to 22)&quot;, section on '16p13.3 deletion syndrome (Rubinstein-Taybi syndrome)'</a> and <a href=\"topic.htm?path=duchenne-and-becker-muscular-dystrophy-genetics-and-pathogenesis#H3815042953\" class=\"medical medical_review\">&quot;Duchenne and Becker muscular dystrophy: Genetics and pathogenesis&quot;, section on 'Genetics and pathogenesis'</a>.)</p><p/><p>However CNVs can also be associated with more complex traits or syndromes in which there are combinations of genetic and environmental factors [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/2\" class=\"abstract_t\">2</a>]. Examples include:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A deletion at the complement factor 4 (C4) locus that confers a 1.6- to 5.3-fold risk for systemic lupus erythematosus [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/18\" class=\"abstract_t\">18</a>]. (See <a href=\"topic.htm?path=epidemiology-and-pathogenesis-of-systemic-lupus-erythematosus\" class=\"medical medical_review\">&quot;Epidemiology and pathogenesis of systemic lupus erythematosus&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A deletion in FCGR3B associated with granulomatosis with polyangiitis (Wegener's) [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/19\" class=\"abstract_t\">19</a>]. (See <a href=\"topic.htm?path=pathogenesis-of-granulomatosis-with-polyangiitis-and-related-vasculitides\" class=\"medical medical_review\">&quot;Pathogenesis of granulomatosis with polyangiitis and related vasculitides&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A deletion of defensin-beta 4 (DEFB4) associated with increased risk of colonic Crohn disease [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/20\" class=\"abstract_t\">20</a>]. Alternatively a deletion of 20 kb upstream of the IRGM gene has also been associated with Crohn disease [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/21\" class=\"abstract_t\">21</a>]. (See <a href=\"topic.htm?path=genetic-factors-in-inflammatory-bowel-disease\" class=\"medical medical_review\">&quot;Genetic factors in inflammatory bowel disease&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>A 32 kb deletion involving two genes that encode envelope proteins (LCE3B and LCE3C) has been associated as a susceptibility factor for psoriasis [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/22\" class=\"abstract_t\">22</a>]. </p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Increased frequency of de novo germ-line CNVs in patients with autism spectrum disorder (ASD) and schizophrenia [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/23,24\" class=\"abstract_t\">23,24</a>]. CNVs in multiple areas of the human genome potentially involved in autism pathogenesis have been described. (See <a href=\"topic.htm?path=autism-spectrum-disorder-terminology-epidemiology-and-pathogenesis\" class=\"medical medical_review\">&quot;Autism spectrum disorder: Terminology, epidemiology, and pathogenesis&quot;</a>.)<br/><br/>This line of research has also identified specific genes and pathways involved in ASD and related syndromes (eg, duplications of the 15q11-q13 region, deletions and duplications of 16p11.2, 1q21 duplications) [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/25-27\" class=\"abstract_t\">25-27</a>]. (See <a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-1-to-11\" class=\"medical medical_review\">&quot;Microdeletion syndromes (chromosomes 1 to 11)&quot;</a> and <a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-12-to-22\" class=\"medical medical_review\">&quot;Microdeletion syndromes (chromosomes 12 to 22)&quot;</a>.)</p><p/><p>In addition, some conditions are associated with multiple CNVs, which may explain their variable phenotypes [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/28\" class=\"abstract_t\">28</a>]. As an example, in a retrospective study, array comparative genomic hybridization (CGH) was used to evaluate CNVs in 2312 children with developmental disabilities who already had one predefined CNV and in 8329 children without developmental disabilities [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/29\" class=\"abstract_t\">29</a>]. This study found that, in comparison to controls without developmental disabilities, individuals with developmental disabilities had an increased number of second site CNVs. This increase in CNVs may have played a causative role in the disabilities (eg, by causing disruption of a new gene or altering gene dosage), or may be an indirect marker of susceptibility to genomic damage. (See <a href=\"#H3390507\" class=\"local\">'Array comparative genomic hybridization'</a> below.)</p><p class=\"headingAnchor\" id=\"H12440911\"><span class=\"h2\">Causes of CNVs</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Low copy repeats are stretches of repetitive DNA sequences (segmental duplications) approximately 10 to 300 kilobases in size that share &ge;95 percent homology. Erroneous pairing of these highly homologous regions can cause misalignment and unequal recombination during meiosis. This can lead to duplication and deletion of chromosomal material resulting in CNVs. This process is known as nonallelic homologous recombination (NAHR) (<a href=\"image.htm?imageKey=PC%2F67644\" class=\"graphic graphic_figure graphicRef67644 \">figure 1</a>), the most common mechanism for the formation of genomic rearrangements [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/30,31\" class=\"abstract_t\">30,31</a>]. </p><p>NAHR can result in either deletions or duplication via the same mechanisms and due to low copy repeats mediated nonallelic homologous recombination. A classic example is the case of Charcot-Marie-Tooth Type I, a peripheral neuropathy caused by duplications of the PMP22 gene on chromosome 17p11.2. The same region when deleted leads to a different neuropathy known as Tomaculous neuropathy also known as hereditary neuropathy with liability to pressure palsies (HNPP). (See <a href=\"topic.htm?path=microduplication-syndromes\" class=\"medical medical_review\">&quot;Microduplication syndromes&quot;</a>.) </p><p>Other mechanisms include nonhomologous end joining and microhomology-mediated break-induced replication, although discussion of these is beyond the scope of this chapter [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/32\" class=\"abstract_t\">32</a>]. </p><p>Despite a great deal of knowledge about the structural details of how CNVs occur, we do not know what predisposes certain individuals to develop these changes more than other individuals.</p><p>Of interest, in a large study of patients with developmental disabilities, parental data provided information about whether CNVs were inherited or arose de novo [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/29\" class=\"abstract_t\">29</a>]. This study suggested that CNVs were more likely to arise de novo in the syndromic disorders (eg, Williams-Beuren syndrome), whereas CNVs were more likely to be inherited in the disorders with variable phenotype (eg, intellectual disability). A potential explanation may be that reproductive fitness is reduced in individuals with the more severe syndromic disorders. (See <a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-1-to-11#H12329278\" class=\"medical medical_review\">&quot;Microdeletion syndromes (chromosomes 1 to 11)&quot;, section on '7q11.23 deletion syndrome (Williams syndrome)'</a>.)</p><p class=\"headingAnchor\" id=\"H12440918\"><span class=\"h2\">Interpreting CNVs</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The interpretation of CNVs has steadily improved due to use of large control databases that allow a direct comparison with apparently normal controls. Examples of these databases include the <a href=\"http://dgv.tcag.ca/dgv/app/home&amp;token=ZWwxzs9Qp/KuLWPOpqxKWBvxEZ9Fd/UNMTRzPXEfWVUohGXikFKQHoSXa2YrI39v&amp;TOPIC_ID=16647\" target=\"_blank\" class=\"external\">Database of Genomic Variants</a> (DGV), large sequencing projects such as the <a href=\"http://www.internationalgenome.org/&amp;token=lmvGwDg43jbWTPc09l4hMGllUvRdVTvC18G4g6UyU6Htrk8rkwumt+55Atifj2nc&amp;TOPIC_ID=16647\" target=\"_blank\" class=\"external\">1000 Genomes Project</a>, or <a href=\"https://www.ncbi.nlm.nih.gov/clinvar/&amp;token=wEsKVFvqlKzpT+qOtddW/y3+Lj0shalBW+L1ZyL4trzGy12ipcVcj41059JRLkL3&amp;TOPIC_ID=16647\" target=\"_blank\" class=\"external\">ClinVar</a> [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/33-35\" class=\"abstract_t\">33-35</a>]. Other databases that include phenotypic information, such as <a href=\"https://decipher.sanger.ac.uk/&amp;token=S4K3ySMKAR5ih4ZompinzIyNEaF8+bf+qNX5W3tolf7P//JZerg3tg2C5xbRFuvj&amp;TOPIC_ID=16647\" target=\"_blank\" class=\"external\">DECIPHER</a> (<strong>D</strong>atabas<strong>E</strong> of <strong>C</strong>hromosomal <strong>I</strong>mbalance and <strong>P</strong>henotype in <strong>H</strong>umans using <strong>E</strong>nsembl <strong>R</strong>esources), are becoming valuable resources for findings detected by array-comparative genomic hybridization [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/36\" class=\"abstract_t\">36</a>]. DECIPHER is a web-based resource that contains array and clinical data deposited by over 247 member centers around the world.</p><p>Interpreting the pathogenicity of CNVs can be quite challenging in the presence of multiple CNVs in one individual. Some studies have shown that multiple rare CNVs, either inherited or de novo, can compound the clinical severity [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/29,37\" class=\"abstract_t\">29,37</a>]. It is also important to highlight the presence of common CNVs in the general population. It is unclear whether many of these common polymorphisms may play a role in common disorders [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/38\" class=\"abstract_t\">38</a>].</p><p class=\"headingAnchor\" id=\"H16722227\"><span class=\"h1\">DISEASE MECHANISMS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>There are different potential mechanisms that can lead to disease in genomic disorders secondary to deletions and duplications. The main mechanism is related to changes in dosage sensitive genes. &ldquo;Haploinsufficiency&rdquo; (&ldquo;haplo&rdquo; = half) defines the concept where loss or gain of one allele of a gene leads to abnormal protein production or function, thereby causing disease [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/39\" class=\"abstract_t\">39</a>].</p><p>Deletions can interfere with the required gene product dose, resulting in disease. One example is Williams-Beuren Syndrome, which is caused by a microdeletion in chromosome 7q11.23 involving multiple genes, including the gene for elastin [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/40,41\" class=\"abstract_t\">40,41</a>]. Having only half of the normal amount (dose) of elastin is enough to cause disruption of the arteries leading to aortic narrowing and multiple other arterial abnormalities. The deletions in Williams-Beuren syndrome, similar to what happens in many other genomic disorders, can be of different sizes. Typically the loss encompasses 1.55 Mb, but in some cases the deletion can be more extensive or even smaller. The size difference is due to the different LCRs around the critical region that can be involved in mediating the rearrangements. (See <a href=\"topic.htm?path=chromosomal-translocations-deletions-and-inversions#H10\" class=\"medical medical_review\">&quot;Chromosomal translocations, deletions, and inversions&quot;, section on 'Deletions'</a> and <a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-1-to-11#H12329278\" class=\"medical medical_review\">&quot;Microdeletion syndromes (chromosomes 1 to 11)&quot;, section on '7q11.23 deletion syndrome (Williams syndrome)'</a> and <a href=\"topic.htm?path=supravalvar-aortic-stenosis\" class=\"medical medical_review\">&quot;Supravalvar aortic stenosis&quot;</a>.)</p><p>The importance of gene dosage in determining the effect of CNVs was illustrated in the sex differences from a large study of CNVs in patients with developmental disabilities [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/29\" class=\"abstract_t\">29</a>]. When compared to females, males had more of the &ldquo;variable phenotype genomic disorders&rdquo; (eg, intellectual disability) but not syndromic disorders (eg, autism spectrum disorders). Females may be protected from these more genetically multifactorial disorders due to sex chromosome bias (ie, the protection of females from weakly deleterious mutations on one X chromosome by the normal corresponding genes on the other X chromosome). </p><p>Duplications may also disrupt a gene and alter the amount of protein produced by interfering with protein synthesis or assembly. Another mechanism includes unmasking recessive alleles [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/42-44\" class=\"abstract_t\">42-44</a>]. When a recessive allele is deleted, it could potentially uncover a pathogenic mutation in the remaining copy (gene) on the other chromosome. This would lead to disease, because there is no normally-functioning copy left for the affected gene. </p><p>Other disease mechanisms include the following:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Interference with imprinted genes, as in the case of paternal duplications of 11p15 that lead to Beckwith-Wiedemann syndrome [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/45\" class=\"abstract_t\">45</a>]. (See <a href=\"topic.htm?path=beckwith-wiedemann-syndrome#H1618409\" class=\"medical medical_review\">&quot;Beckwith-Wiedemann syndrome&quot;, section on 'Genetics and pathogenesis'</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Interference with regulatory elements outside genes, as in brachydactyly type A2 and duplications outside the BMP2 gene [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/46\" class=\"abstract_t\">46</a>]. </p><p/><p>Arrays that contain single nucleotide polymorphisms (SNPs) can further aid in the identification of imprinting disorders caused by uniparental disomy. One such example is Angelman syndrome (AS) and uniparental disomy (UPD) caused by isodisomy. Isodisomies result from either nondisjunction in meiosis II or postzygotic duplication (monosomy rescue). A small number of AS cases are the result of UPD. In those cases there is absence of the maternal contribution for a region of chromosome 15 (15q11-q13). These cases are typically associated with monosomy rescue (duplication of a chromosome from a monosomic zygote), where there are two identical copies of the paternal chromosome 15 and no maternal contribution. Other examples of UPD can be seen in patients with Prader-Willi syndrome. (See <a href=\"topic.htm?path=epidemiology-and-genetics-of-prader-willi-syndrome\" class=\"medical medical_review\">&quot;Epidemiology and genetics of Prader-Willi syndrome&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H174214015\"><span class=\"h2\">Contiguous gene syndromes</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Contiguous gene syndromes can occur when large CNVs affect several contiguous genes [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/47,48\" class=\"abstract_t\">47,48</a>]. For example, Williams-Beuren Syndrome is caused by a 1.5-1.8 Mb deletion on chromosome 7q11 that typically encompasses nine genes. In WAGR syndrome, a genomic disorder consisting of <strong>W</strong>ilms tumor, <strong>A</strong>niridia, <strong>G</strong>enitourinary anomalies, and mental <strong>R</strong>etardation, clinical features are attributable to the loss of individual genes by a large deletion: deletions of <em>WT1</em> are responsible for Wilms tumor, while <em>PAX6</em> deletions are responsible for the aniridia findings. Both genes are contiguously located within the short arm of chromosome 11. (See <a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-1-to-11#H2473577\" class=\"medical medical_review\">&quot;Microdeletion syndromes (chromosomes 1 to 11)&quot;, section on '11p13 deletion syndrome (WAGR syndrome)'</a>.)</p><p class=\"headingAnchor\" id=\"H16722234\"><span class=\"h1\">DETECTION OF GENOMIC DISORDERS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Genomic disorders are typically detected by array comparative genomic hybridization (array CGH). Most laboratories confirm gains or losses detected on an array with an independent method such as fluorescence in situ hybridization (FISH), multiple ligation dependent probe amplification (MLPA), or quantitative PCR (Q-PCR). Parental FISH testing, and in selected cases array CGH, is warranted in all cases of detected genomic abnormalities as many are inherited and relevant for future pregnancies [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/49\" class=\"abstract_t\">49</a>]. Whereas the genomic position of a loss is clear from an array, gains may be either tandem duplications or insertions. If the latter arise as a consequence of a parental insertional translocation (IT), this may have important implications for future pregnancies [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/50\" class=\"abstract_t\">50</a>]. Next generation sequencing (NGS) technology that is being used for exome sequencing may also identify CNVs and might become a resource for detecting CNVs and sequence variations simultaneously [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/51\" class=\"abstract_t\">51</a>]. (See <a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">&quot;Next-generation DNA sequencing (NGS): Principles and clinical applications&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H3390507\"><span class=\"h2\">Array comparative genomic hybridization</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Array comparative genomic hybridization (array CGH), also known as chromosome microarray or microarray-based comparative genomic hybridization, is the gold standard laboratory test for the detection of CNVs that cause genomic disorders (<a href=\"image.htm?imageKey=PC%2F58888\" class=\"graphic graphic_figure graphicRef58888 \">figure 2</a>). Array CGH allows detection of small losses or gains of genomic material down to several kilobases (kb) and even at the exonic level. It is widely used in the evaluation of patients with intellectual disabilities <span class=\"nowrap\">and/or</span> congenital malformations [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/52-57\" class=\"abstract_t\">52-57</a>]. (See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics#H15538652\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Cytogenetics and molecular genetics&quot;, section on 'Array comparative genomic hybridization'</a>.) &#160;</p><p>The two main platforms currently used for CNV detection are oligonucleotide arrays (oligonucleotides are stretches of DNA ranging from 25 to 60 base pairs) (<a href=\"image.htm?imageKey=PC%2F58888\" class=\"graphic graphic_figure graphicRef58888 \">figure 2</a>), and single nucleotide polymorphism arrays (SNP arrays) [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/58\" class=\"abstract_t\">58</a>]. There are approximately 10 million polymorphic SNPs throughout the human genome. Both SNPs and oligonucleotide arrays can detect copy number variations, but SNP arrays can be used in addition to determine lack of heterozygosity or presence of homozygosity, as seen in cases of consanguinity [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/59\" class=\"abstract_t\">59</a>] and in cases of uniparental disomy [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/60\" class=\"abstract_t\">60</a>] when there is inheritance of regions or entire chromosomes from one single parent instead of the normal biparental contribution. SNPs can also detect loss of heterozygosity (LOH) typically seen in somatic cancer cell changes. SNPs can also be very useful for the detection of somatic mosaicism, a situation where two or more cell lines can be present in a single individual, and triploidies, a rare instance when there could be a total of 69 chromosomes (3 haploid sets) that can be detected in the prenatal setting. Some current platforms are combining the use of array CGH and SNPs integrated in one single platform.</p><p class=\"headingAnchor\" id=\"H1726088\"><span class=\"h2\">Other molecular diagnostic techniques</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Other molecular techniques used for the detection of genomic disorders include fluorescence in situ hybridization (FISH), PCR-based studies like Q-PCR (quantitative PCR), and MLPA (Multiple Ligation Dependent Probe Amplification). Protocols using sequencing of the whole genomes or exome (all regions of the genome that encode proteins) will likely become viable options for detecting CNV variations [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/61,62\" class=\"abstract_t\">61,62</a>]. (See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Cytogenetics and molecular genetics&quot;</a>.)</p><p>FISH uses larger stretches of DNA (approximately 50 to 200 kilobase probes) labeled with fluorescence reagents to target specific genome regions. The use of FISH, however, requires knowledge of what specific area is being targeted and is dependent upon a clinical diagnosis [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/63,64\" class=\"abstract_t\">63,64</a>]. MLPA uses a cocktail of multiple probes available in kits and targets specific chromosomes or disease regions. A single reaction allows simultaneous hybridization of multiple probes to multiple regions or even multiple exons within a gene. </p><p>The advantage of MLPA over FISH is the lower cost and more comprehensive coverage for copy number <span class=\"nowrap\">gains/losses</span> detection due to the use of multiple probes. However, MLPA does not provide location. FISH studies, on the other hand, can determine location if visualized in metaphase chromosome spreads. A copy number gain can be the result of chromosome duplication immediately adjacent to the area of interest, part of a marker chromosome (a structurally abnormal chromosome that makes up a partial trisomy), or the result of an insertion or translocation. Duplications that are very close to each other may be challenging to detect by metaphase FISH and are best detected by interphase FISH.</p><p>Sequencing of the whole genome or exome can also be used to detect genomic disorders. This was demonstrated in a study that used sequencing from 849 individuals to identify areas of CNVs and their role in gene dosage [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/61\" class=\"abstract_t\">61</a>]. Genome sequencing has also been used to map breakpoints in the genome responsible for duplications of chromosomal regions [<a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/65\" class=\"abstract_t\">65</a>]. Sequencing has the potential to provide improved resolution, but the sensitivity and specificity of sequencing for detecting CNVs is variable. Information about sequencing methods, collectively referred to as next-generation sequencing (NGS), is presented separately. (See <a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">&quot;Next-generation DNA sequencing (NGS): Principles and clinical applications&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H1726355\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Genomic disorders are diseases that result from the loss or gain of <span class=\"nowrap\">chromosomal/DNA</span> material. The most common and better-delineated genomic disorders are divided in two main categories: those resulting from copy number losses (deletion syndromes) and copy number gains (duplication syndromes). (See <a href=\"#H16722192\" class=\"local\">'Introduction'</a> above and <a href=\"topic.htm?path=congenital-cytogenetic-abnormalities\" class=\"medical medical_review\">&quot;Congenital cytogenetic abnormalities&quot;</a>.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Copy number variations (CNVs) are submicroscopic genomic differences in the number of copies of one or more sections of DNA that result in DNA gains or losses (<a href=\"image.htm?imageKey=PC%2F67644\" class=\"graphic graphic_figure graphicRef67644 \">figure 1</a>). Some pathogenic CNVs cause syndromic disorders with consistent phenotypic features. Other CNVs are associated with disease susceptibility or resistance. (See <a href=\"#H16722199\" class=\"local\">'Copy number variations'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The main mechanism that leads to disease in genomic disorders secondary to deletions and duplications is changes in dosage-sensitive genes. Other disease mechanisms include interference with imprinted genes and with regulatory elements outside genes. (See <a href=\"#H16722227\" class=\"local\">'Disease mechanisms'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Genomic disorders are typically detected by array comparative genomic hybridization (array CGH). Many laboratories confirm gains or losses detected with an independent method such as fluorescence in situ hybridization (FISH), multiple ligation dependent probe amplification (MLPA), or quantitative PCR (Q-PCR). Next generation sequencing (NGS) is likely to become a popular way to detect CNVs. (See <a href=\"#H16722234\" class=\"local\">'Detection of genomic disorders'</a> above.)</p></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/1\" class=\"nounderline abstract_t\">Feuk L, Carson AR, Scherer SW. Structural variation in the human genome. Nat Rev Genet 2006; 7:85.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/2\" class=\"nounderline abstract_t\">Conrad DF, Pinto D, Redon R, et al. Origins and functional impact of copy number variation in the human genome. Nature 2010; 464:704.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/3\" class=\"nounderline abstract_t\">Redon R, Ishikawa S, Fitch KR, et al. Global variation in copy number in the human genome. Nature 2006; 444:444.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/4\" class=\"nounderline abstract_t\">Tuzun E, Sharp AJ, Bailey JA, et al. Fine-scale structural variation of the human genome. Nat Genet 2005; 37:727.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/5\" class=\"nounderline abstract_t\">Ewart AK, Morris CA, Atkinson D, et al. Hemizygosity at the elastin locus in a developmental disorder, Williams syndrome. Nat Genet 1993; 5:11.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/6\" class=\"nounderline abstract_t\">Magenis RE, Toth-Fejel S, Allen LJ, et al. Comparison of the 15q deletions in Prader-Willi and Angelman syndromes: specific regions, extent of deletions, parental origin, and clinical consequences. Am J Med Genet 1990; 35:333.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/7\" class=\"nounderline abstract_t\">Cooper GM, Nickerson DA, Eichler EE. Mutational and selective effects on copy-number variants in the human genome. Nat Genet 2007; 39:S22.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/8\" class=\"nounderline abstract_t\">Sharp AJ. Emerging themes and new challenges in defining the role of structural variation in human disease. Hum Mutat 2009; 30:135.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/9\" class=\"nounderline abstract_t\">Perry GH, Tchinda J, McGrath SD, et al. Hotspots for copy number variation in chimpanzees and humans. Proc Natl Acad Sci U S A 2006; 103:8006.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/10\" class=\"nounderline abstract_t\">Sharp AJ, Cheng Z, Eichler EE. Structural variation of the human genome. Annu Rev Genomics Hum Genet 2006; 7:407.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/11\" class=\"nounderline abstract_t\">Sharp AJ, Locke DP, McGrath SD, et al. Segmental duplications and copy-number variation in the human genome. Am J Hum Genet 2005; 77:78.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/12\" class=\"nounderline abstract_t\">Sebat J, Lakshmi B, Troge J, et al. Large-scale copy number polymorphism in the human genome. Science 2004; 305:525.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/13\" class=\"nounderline abstract_t\">Wong KK, deLeeuw RJ, Dosanjh NS, et al. A comprehensive analysis of common copy-number variations in the human genome. Am J Hum Genet 2007; 80:91.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/14\" class=\"nounderline abstract_t\">Wellcome Trust Case Control Consortium, Craddock N, Hurles ME, et al. Genome-wide association study of CNVs in 16,000 cases of eight common diseases and 3,000 shared controls. Nature 2010; 464:713.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/15\" class=\"nounderline abstract_t\">Girirajan S, Eichler EE. Phenotypic variability and genetic susceptibility to genomic disorders. Hum Mol Genet 2010; 19:R176.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/16\" class=\"nounderline abstract_t\">Fanciulli M, Petretto E, Aitman TJ. Gene copy number variation and common human disease. Clin Genet 2010; 77:201.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/17\" class=\"nounderline abstract_t\">Boone PM, Bacino CA, Shaw CA, et al. Detection of clinically relevant exonic copy-number changes by array CGH. Hum Mutat 2010; 31:1326.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/18\" class=\"nounderline abstract_t\">Yang Y, Chung EK, Wu YL, et al. Gene copy-number variation and associated polymorphisms of complement component C4 in human systemic lupus erythematosus (SLE): low copy number is a risk factor for and high copy number is a protective factor against SLE susceptibility in European Americans. Am J Hum Genet 2007; 80:1037.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/19\" class=\"nounderline abstract_t\">Fanciulli M, Norsworthy PJ, Petretto E, et al. FCGR3B copy number variation is associated with susceptibility to systemic, but not organ-specific, autoimmunity. Nat Genet 2007; 39:721.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/20\" class=\"nounderline abstract_t\">Fellermann K, Stange DE, Schaeffeler E, et al. A chromosome 8 gene-cluster polymorphism with low human beta-defensin 2 gene copy number predisposes to Crohn disease of the colon. Am J Hum Genet 2006; 79:439.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/21\" class=\"nounderline abstract_t\">McCarroll SA, Huett A, Kuballa P, et al. Deletion polymorphism upstream of IRGM associated with altered IRGM expression and Crohn's disease. Nat Genet 2008; 40:1107.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/22\" class=\"nounderline abstract_t\">de Cid R, Riveira-Munoz E, Zeeuwen PL, et al. Deletion of the late cornified envelope LCE3B and LCE3C genes as a susceptibility factor for psoriasis. Nat Genet 2009; 41:211.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/23\" class=\"nounderline abstract_t\">Sebat J, Lakshmi B, Malhotra D, et al. Strong association of de novo copy number mutations with autism. Science 2007; 316:445.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/24\" class=\"nounderline abstract_t\">Stefansson H, Rujescu D, Cichon S, et al. Large recurrent microdeletions associated with schizophrenia. Nature 2008; 455:232.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/25\" class=\"nounderline abstract_t\">Girirajan S, Dennis MY, Baker C, et al. Refinement and discovery of new hotspots of copy-number variation associated with autism spectrum disorder. Am J Hum Genet 2013; 92:221.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/26\" class=\"nounderline abstract_t\">Barber JC, Hall V, Maloney VK, et al. 16p11.2-p12.2 duplication syndrome; a genomic condition differentiated from euchromatic variation of 16p11.2. Eur J Hum Genet 2013; 21:182.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/27\" class=\"nounderline abstract_t\">Hogart A, Wu D, LaSalle JM, Schanen NC. The comorbidity of autism with the genomic disorders of chromosome 15q11.2-q13. Neurobiol Dis 2010; 38:181.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/28\" class=\"nounderline abstract_t\">Sanders SJ, Ercan-Sencicek AG, Hus V, et al. Multiple recurrent de novo CNVs, including duplications of the 7q11.23 Williams syndrome region, are strongly associated with autism. Neuron 2011; 70:863.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/29\" class=\"nounderline abstract_t\">Girirajan S, Rosenfeld JA, Coe BP, et al. Phenotypic heterogeneity of genomic disorders and rare copy-number variants. N Engl J Med 2012; 367:1321.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/30\" class=\"nounderline abstract_t\">Stankiewicz P, Inoue K, Bi W, et al. Genomic disorders: genome architecture results in susceptibility to DNA rearrangements causing common human traits. Cold Spring Harb Symp Quant Biol 2003; 68:445.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/31\" class=\"nounderline abstract_t\">Stankiewicz P, Shaw CJ, Dapper JD, et al. Genome architecture catalyzes nonrecurrent chromosomal rearrangements. Am J Hum Genet 2003; 72:1101.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/32\" class=\"nounderline abstract_t\">Hastings PJ, Lupski JR, Rosenberg SM, Ira G. Mechanisms of change in gene copy number. Nat Rev Genet 2009; 10:551.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/33\" class=\"nounderline abstract_t\">1000 Genomes Project Consortium, Abecasis GR, Altshuler D, et al. A map of human genome variation from population-scale sequencing. Nature 2010; 467:1061.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/34\" class=\"nounderline abstract_t\">Zhang J, Feuk L, Duggan GE, et al. Development of bioinformatics resources for display and analysis of copy number and other structural variants in the human genome. Cytogenet Genome Res 2006; 115:205.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/35\" class=\"nounderline abstract_t\">Landrum MJ, Lee JM, Riley GR, et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Nucleic Acids Res 2014; 42:D980.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/36\" class=\"nounderline abstract_t\">Firth HV, Richards SM, Bevan AP, et al. DECIPHER: Database of Chromosomal Imbalance and Phenotype in Humans Using Ensembl Resources. Am J Hum Genet 2009; 84:524.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/37\" class=\"nounderline abstract_t\">Girirajan S, Brkanac Z, Coe BP, et al. Relative burden of large CNVs on a range of neurodevelopmental phenotypes. PLoS Genet 2011; 7:e1002334.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/38\" class=\"nounderline abstract_t\">Gijsbers AC, Schoumans J, Ruivenkamp CA. Interpretation of array comparative genome hybridization data: a major challenge. Cytogenet Genome Res 2011; 135:222.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/39\" class=\"nounderline abstract_t\">Seidman JG, Seidman C. Transcription factor haploinsufficiency: when half a loaf is not enough. J Clin Invest 2002; 109:451.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/40\" class=\"nounderline abstract_t\">Wu YQ, Sutton VR, Nickerson E, et al. Delineation of the common critical region in Williams syndrome and clinical correlation of growth, heart defects, ethnicity, and parental origin. Am J Med Genet 1998; 78:82.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/41\" class=\"nounderline abstract_t\">Schubert C. The genomic basis of the Williams-Beuren syndrome. Cell Mol Life Sci 2009; 66:1178.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/42\" class=\"nounderline abstract_t\">Ropers HH. Genetics of intellectual disability. Curr Opin Genet Dev 2008; 18:241.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/43\" class=\"nounderline abstract_t\">Wong LJ, Dimmock D, Geraghty MT, et al. Utility of oligonucleotide array-based comparative genomic hybridization for detection of target gene deletions. Clin Chem 2008; 54:1141.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/44\" class=\"nounderline abstract_t\">Kurth I, Klopocki E, Stricker S, et al. Duplications of noncoding elements 5' of SOX9 are associated with brachydactyly-anonychia. Nat Genet 2009; 41:862.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/45\" class=\"nounderline abstract_t\">Russo S, Finelli P, Recalcati MP, et al. Molecular and genomic characterisation of cryptic chromosomal alterations leading to paternal duplication of the 11p15.5 Beckwith-Wiedemann region. J Med Genet 2006; 43:e39.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/46\" class=\"nounderline abstract_t\">Dathe K, Kjaer KW, Brehm A, et al. Duplications involving a conserved regulatory element downstream of BMP2 are associated with brachydactyly type A2. Am J Hum Genet 2009; 84:483.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/47\" class=\"nounderline abstract_t\">Ballabio A. Contiguous deletion syndromes. Curr Opin Genet Dev 1991; 1:25.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/48\" class=\"nounderline abstract_t\">Punnett HH, Zakai EH. Old syndromes and new cytogenetics. Dev Med Child Neurol 1990; 32:824.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/49\" class=\"nounderline abstract_t\">Kang SH, Shaw C, Ou Z, et al. Insertional translocation detected using FISH confirmation of array-comparative genomic hybridization (aCGH) results. Am J Med Genet A 2010; 152A:1111.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/50\" class=\"nounderline abstract_t\">Nowakowska BA, de Leeuw N, Ruivenkamp CA, et al. Parental insertional balanced translocations are an important cause of apparently de novo CNVs in patients with developmental anomalies. Eur J Hum Genet 2012; 20:166.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/51\" class=\"nounderline abstract_t\">Chen Y, Zhao L, Wang Y, et al. SeqCNV: a novel method for identification of copy number variations in targeted next-generation sequencing data. BMC Bioinformatics 2017; 18:147.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/52\" class=\"nounderline abstract_t\">Bejjani BA, Theisen AP, Ballif BC, Shaffer LG. Array-based comparative genomic hybridization in clinical diagnosis. Expert Rev Mol Diagn 2005; 5:421.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/53\" class=\"nounderline abstract_t\">Cheung SW, Shaw CA, Yu W, et al. Development and validation of a CGH microarray for clinical cytogenetic diagnosis. Genet Med 2005; 7:422.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/54\" class=\"nounderline abstract_t\">Lu X, Shaw CA, Patel A, et al. Clinical implementation of chromosomal microarray analysis: summary of 2513 postnatal cases. PLoS One 2007; 2:e327.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/55\" class=\"nounderline abstract_t\">Lu XY, Phung MT, Shaw CA, et al. Genomic imbalances in neonates with birth defects: high detection rates by using chromosomal microarray analysis. Pediatrics 2008; 122:1310.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/56\" class=\"nounderline abstract_t\">Miller DT, Adam MP, Aradhya S, et al. Consensus statement: chromosomal microarray is a first-tier clinical diagnostic test for individuals with developmental disabilities or congenital anomalies. Am J Hum Genet 2010; 86:749.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/57\" class=\"nounderline abstract_t\">Stankiewicz P, Beaudet AL. Use of array CGH in the evaluation of dysmorphology, malformations, developmental delay, and idiopathic mental retardation. Curr Opin Genet Dev 2007; 17:182.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/58\" class=\"nounderline abstract_t\">Cooper GM, Mefford HC. Detection of copy number variation using SNP genotyping. Methods Mol Biol 2011; 767:243.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/59\" class=\"nounderline abstract_t\">Schaaf CP, Scott DA, Wiszniewska J, Beaudet AL. Identification of incestuous parental relationships by SNP-based DNA microarrays. Lancet 2011; 377:555.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/60\" class=\"nounderline abstract_t\">Papenhausen P, Schwartz S, Risheg H, et al. UPD detection using homozygosity profiling with a SNP genotyping microarray. Am J Med Genet A 2011; 155A:757.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/61\" class=\"nounderline abstract_t\">Handsaker RE, Van Doren V, Berman JR, et al. Large multiallelic copy number variations in humans. Nat Genet 2015; 47:296.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/62\" class=\"nounderline abstract_t\">de Ligt J, Boone PM, Pfundt R, et al. Detection of clinically relevant copy number variants with whole-exome sequencing. Hum Mutat 2013; 34:1439.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/63\" class=\"nounderline abstract_t\">Shaffer LG, Kennedy GM, Spikes AS, Lupski JR. Diagnosis of CMT1A duplications and HNPP deletions by interphase FISH: implications for testing in the cytogenetics laboratory. Am J Med Genet 1997; 69:325.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/64\" class=\"nounderline abstract_t\">Eijk-Van Os PG, Schouten JP. Multiplex Ligation-dependent Probe Amplification (MLPA&reg;) for the detection of copy number variation in genomic sequences. Methods Mol Biol 2011; 688:97.</a></li><li><a href=\"https://www.uptodate.com/contents/genomic-disorders-an-overview/abstract/65\" class=\"nounderline abstract_t\">Newman S, Hermetz KE, Weckselblatt B, Rudd MK. Next-generation sequencing of duplication CNVs reveals that most are tandem and some create fusion genes at breakpoints. Am J Hum Genet 2015; 96:208.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 16647 Version 12.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H1726355\"><span>SUMMARY</span></a></li><li><a href=\"#H16722192\" id=\"outline-link-H16722192\">INTRODUCTION</a></li><li><a href=\"#H16722199\" id=\"outline-link-H16722199\">COPY NUMBER VARIATIONS</a><ul><li><a href=\"#H12440911\" id=\"outline-link-H12440911\">Causes of CNVs</a></li><li><a href=\"#H12440918\" id=\"outline-link-H12440918\">Interpreting CNVs</a></li></ul></li><li><a href=\"#H16722227\" id=\"outline-link-H16722227\">DISEASE MECHANISMS</a><ul><li><a href=\"#H174214015\" id=\"outline-link-H174214015\">Contiguous gene syndromes</a></li></ul></li><li><a href=\"#H16722234\" id=\"outline-link-H16722234\">DETECTION OF GENOMIC DISORDERS</a><ul><li><a href=\"#H3390507\" id=\"outline-link-H3390507\">Array comparative genomic hybridization</a></li><li><a href=\"#H1726088\" id=\"outline-link-H1726088\">Other molecular diagnostic techniques</a></li></ul></li><li><a href=\"#H1726355\" id=\"outline-link-H1726355\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>GRAPHICS <a href=\"#\" id=\"viewAllGraphicsLink\">View All</a></h2><div id=\"outlineGraphics\"><ul><li><div id=\"PC/16647|FIG\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"FIGURES\">FIGURES</a></div><ul><li><a href=\"image.htm?imageKey=PC/67644\" class=\"graphic graphic_figure\">- Nonallelic homologous recombination</a></li><li><a href=\"image.htm?imageKey=PC/58888\" class=\"graphic graphic_figure\">- Schematic CGH for microarray setup</a></li></ul></li></ul></div></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=autism-spectrum-disorder-terminology-epidemiology-and-pathogenesis\" class=\"medical medical_review\">Autism spectrum disorder: Terminology, epidemiology, and pathogenesis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=beckwith-wiedemann-syndrome\" class=\"medical medical_review\">Beckwith-Wiedemann syndrome</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=chromosomal-translocations-deletions-and-inversions\" class=\"medical medical_review\">Chromosomal translocations, deletions, and inversions</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=congenital-cytogenetic-abnormalities\" class=\"medical medical_review\">Congenital cytogenetic abnormalities</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=duchenne-and-becker-muscular-dystrophy-genetics-and-pathogenesis\" class=\"medical medical_review\">Duchenne and Becker muscular dystrophy: Genetics and pathogenesis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=epidemiology-and-genetics-of-prader-willi-syndrome\" class=\"medical medical_review\">Epidemiology and genetics of Prader-Willi syndrome</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=epidemiology-and-pathogenesis-of-systemic-lupus-erythematosus\" class=\"medical medical_review\">Epidemiology and pathogenesis of systemic lupus erythematosus</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetic-factors-in-inflammatory-bowel-disease\" class=\"medical medical_review\">Genetic factors in inflammatory bowel disease</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-1-to-11\" class=\"medical medical_review\">Microdeletion syndromes (chromosomes 1 to 11)</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=microdeletion-syndromes-chromosomes-12-to-22\" class=\"medical medical_review\">Microdeletion syndromes (chromosomes 12 to 22)</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=microduplication-syndromes\" class=\"medical medical_review\">Microduplication syndromes</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">Next-generation DNA sequencing (NGS): Principles and clinical applications</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=pathogenesis-of-granulomatosis-with-polyangiitis-and-related-vasculitides\" class=\"medical medical_review\">Pathogenesis of granulomatosis with polyangiitis and related vasculitides</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=supravalvar-aortic-stenosis\" class=\"medical medical_review\">Supravalvar aortic stenosis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">Tools for genetics and genomics: Cytogenetics and molecular genetics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=williams-beuren-syndrome-renal-manifestations\" class=\"medical medical_review\">Williams-Beuren syndrome: Renal manifestations</a></li></ul></div></div>","javascript":null}