var data={"title":"Genetics: Glossary of terms","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Genetics: Glossary of terms</div><dl id=\"topicContributors\"><dt><span> </span>Authors:</dt><dd><a href=\"https://www.uptodate.com/contents/genetics-glossary-of-terms/contributors\" class=\"contributor contributor_credentials\">Benjamin A Raby, MD, MPH</a></dd><dd><a href=\"https://www.uptodate.com/contents/genetics-glossary-of-terms/contributors\" class=\"contributor contributor_credentials\">Robert D Blank, MD, PhD</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/genetics-glossary-of-terms/contributors\" class=\"contributor contributor_credentials\">Anne Slavotinek, MBBS, PhD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/genetics-glossary-of-terms/contributors\" class=\"contributor contributor_credentials\">Jennifer S Tirnauer, MD</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/genetics-glossary-of-terms/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Jan 05, 2018.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H1\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>One of the greatest obstacles clinicians experience in reading about and understanding genetics is the extensive use of technical language and jargon. It should be noted that genetic terms are frequently used imprecisely in published clinical literature. The following is a compilation of some of the most important technical terms.</p><p>A more extensive discussion of terms can be accessed in standard genetics reference texts [<a href=\"https://www.uptodate.com/contents/genetics-glossary-of-terms/abstract/1\" class=\"abstract_t\">1</a>]. In addition, a guide for the conventions regarding the proper names of genes and alleles in humans can be found at <a href=\"http://www.genenames.org/guidelines.html&amp;token=7+GJrCvx/pgnT2Hm8UDNmG3Z5+sv46TDq7H6w74pTGCybuE+wM7hFdu3tK/o2GkF&amp;TOPIC_ID=2898\" target=\"_blank\" class=\"external\">www.genenames.org/guidelines.html</a>.</p><p>Glossaries of epidemiological terms and terms that apply to systematic reviews and meta-analyses are presented separately in UpToDate. (See <a href=\"topic.htm?path=glossary-of-common-biostatistical-and-epidemiological-terms\" class=\"medical medical_review\">&quot;Glossary of common biostatistical and epidemiological terms&quot;</a> and <a href=\"topic.htm?path=systematic-review-and-meta-analysis#H2326537\" class=\"medical medical_review\">&quot;Systematic review and meta-analysis&quot;, section on 'Glossary of terms'</a>.)</p><p class=\"headingAnchor\" id=\"H2\"><span class=\"h1\">LIST OF TERMS</span></p><p class=\"headingAnchor\" id=\"H3942702307\"><span class=\"h2\">Allele</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>An allele is one of a series of alternative forms (genotypes) at locus, or a specific region of a chromosome. At the DNA level, different alleles have different base sequences.</p><p class=\"headingAnchor\" id=\"H1783841946\"><span class=\"h2\">Allelic fraction</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The allelic fraction can be defined as the number of times a mutated base is observed, divided by the total number of times any base is observed at the locus [<a href=\"https://www.uptodate.com/contents/genetics-glossary-of-terms/abstract/2\" class=\"abstract_t\">2</a>]. Allelic fraction is generally applied to a single mutation in a tumor, and thus is distinct from allelic frequency, which examines the frequency of an allele in a population (see <a href=\"#H1537087732\" class=\"local\">'Allele frequency'</a> below). Mutation fraction can be defined as the ratio between mutant and wild-type alleles in a tumor sample.</p><p class=\"headingAnchor\" id=\"H1537087732\"><span class=\"h2\">Allele frequency</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The proportion of chromosomes in a population harboring a specific allele. &quot;Minor allele frequency&quot; typically refers to the less common variant at a biallelic locus and is usually used to refer to the frequency of a single nucleotide polymorphism (SNP). This population frequency is distinguished from allelic ratio, which applies to a single person (eg, with a malignancy).</p><p class=\"headingAnchor\" id=\"H3789757172\"><span class=\"h2\">Allelic heterogeneity</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Allelic heterogeneity refers to the common occurrence of multiple pathogenic variants in one gene that all result in the same disease or syndrome. As an example, more than 1500 variants in the cystic fibrosis transmembrane conductance regulator (<em>CFTR</em>) gene cause cystic fibrosis. Note that this term differs from genetic heterogeneity, in which variants in multiple genes can cause the same disease phenotype. (See <a href=\"#H3479936033\" class=\"local\">'Genetic heterogeneity'</a> below.)</p><p class=\"headingAnchor\" id=\"H2454124624\"><span class=\"h2\">Allelic ratio</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Allelic ratio measures the relative abundance of mutated to normal or wildtype alleles within a tumor. Higher allelic ratios (ie, a greater fraction of mutant alleles) have been reported to be associated with poorer prognosis. Unlike allele frequency, which is a characteristic of a population (see <a href=\"#H1537087732\" class=\"local\">'Allele frequency'</a> above), allelic ratio is a property of cells within a tumor in a single individual. Allelic ratio is of necessity an inexact concept because it is rare (for solid tumors at least) to avoid substantial contamination by non-tumor cells from blood, stroma, and vasculature. Amplification of mutant sequences in a tumor can also have a large impact on allelic ratios.</p><p class=\"headingAnchor\" id=\"H4099420019\"><span class=\"h2\">Aneuploidy</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The state of having an abnormal number of chromosomes. A euploid human karyotype has 46 chromosomes (<a href=\"image.htm?imageKey=PC%2F72134\" class=\"graphic graphic_figure graphicRef72134 \">figure 1</a>). Aneuploidy can affect the entire somatic cell population, as in trisomy 21, or it can affect a subset of cells, as in a tumor.</p><p class=\"headingAnchor\" id=\"H3218144220\"><span class=\"h2\">Anticipation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A phenomenon whereby the symptoms of a genetically-based condition appear at an earlier age, or with greater severity, in successive generations. Expansion of trinucleotide repeats is a known molecular cause for specific diseases (such as myotonic dystrophy, fragile X syndrome, Huntington&rsquo;s chorea) that manifest anticipation.</p><p class=\"headingAnchor\" id=\"H1368106711\"><span class=\"h2\">Association</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Genetic association is a property of alleles. It refers to the non-random relationship between an allele and a phenotype in a population. Genetic association between a marker allele and a phenotype can result either because the allele is a direct causal variant, because the allele is in linkage disequilibrium, or segregating with a causal variant in close proximity, or because of stratification of the population. Association may be determined in a genome-wide association study. (See <a href=\"#H3856285139\" class=\"local\">'Genome-wide association study (GWAS)'</a> below.)</p><p class=\"headingAnchor\" id=\"H4245358129\"><span class=\"h2\">Autosome</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A chromosome other than X or Y. The human genome has 44 autosomes (22 pairs of autosomes) (<a href=\"image.htm?imageKey=PC%2F72134\" class=\"graphic graphic_figure graphicRef72134 \">figure 1</a>).</p><p class=\"headingAnchor\" id=\"H3829231885\"><span class=\"h2\">Autosomal</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A gene is autosomal if it is located on an autosome rather than a sex chromosome. A gene's inheritance pattern is also referred to as autosomal if the pattern corresponds to that of known autosomal genes (rather than sex-linked). (See <a href=\"#H817773338\" class=\"local\">'Sex-linked'</a> below.)</p><p class=\"headingAnchor\" id=\"H2994976941\"><span class=\"h2\">Benign variant</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>(See <a href=\"#H2504463718\" class=\"local\">'Variant'</a> below.)</p><p class=\"headingAnchor\" id=\"H3984329775\"><span class=\"h2\">Biome</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Humans are colonized by a multitude of microorganisms, which vary by age and location in the body. The biome (or microbiome) is the totality of colonizing microorganisms in a specific environmental milieu. Biomes may be studied genetically using metagenomics. (See <a href=\"#H2788605268\" class=\"local\">'Metagenomics'</a> below.)</p><p class=\"headingAnchor\" id=\"H103000573\"><span class=\"h2\">Carrier</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>An individual who is heterozygous for a risk or disease allele. The term is typically used to describe someone who is heterozygous for a gene variant that causes autosomal recessive or X-linked recessive disease, but it is also used to describe heterozygotes for risk alleles of complex traits with variable penetrance, regardless of inheritance type.</p><p class=\"headingAnchor\" id=\"H2161571447\"><span class=\"h2\">Carrier rate</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The frequency of carriers in a population.</p><p class=\"headingAnchor\" id=\"H3240578466\"><span class=\"h2\">Carrier testing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Clinical method of genotyping at risk populations or family members to identify individuals, usually asymptomatic, who have a pathogenic or likely pathogenic variant for an autosomal recessive or X-linked disorder. One example is prenatal screening for Tay-Sachs disease-associated variants in people of Ashkenazi Jewish ancestry. (See <a href=\"topic.htm?path=genetic-testing\" class=\"medical medical_review\">&quot;Genetic testing&quot;</a> and <a href=\"topic.htm?path=carrier-screening-for-genetic-disease-in-the-ashkenazi-jewish-population\" class=\"medical medical_review\">&quot;Carrier screening for genetic disease in the Ashkenazi Jewish population&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H2680527670\"><span class=\"h2\">Centromere</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A condensed chromosome region that mediates attachment of chromosomes to the microtubules of the mitotic or meiotic spindle. The centromere is important in preserving normal chromosome number.</p><p class=\"headingAnchor\" id=\"H3724164370\"><span class=\"h2\">Chromatid</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>One of two replications, or copies, of a chromosome formed prior to cell division and joined together at their centromeres. The centromere is the last portion of a chromosome to replicate during cell division. Sister chromatids are a pair of chromatids attached at the centromere.</p><p class=\"headingAnchor\" id=\"H2759312189\"><span class=\"h2\">Chromatin</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A complex structure composed of DNA, RNA, and proteins that facilitates efficient packaging of DNA in cells. The primary structure of chromatin is the nucleosome, consisting of double-stranded DNA coiled around a core of histone proteins. Nucleosomes packed tightly together form a &quot;bead-on-string&quot; configuration, which in turn assembles in hierarchical looping structures to create densely-packaged chromatin. The regulation of gene transcription is governed by the uncoiling of packed chromatin (heterochromatin) into exposed DNA (euchromatin). (See <a href=\"topic.htm?path=principles-of-epigenetics\" class=\"medical medical_review\">&quot;Principles of epigenetics&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H2427059189\"><span class=\"h2\">Clonal</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Arising from a single clone, or cell. Examples include clonal selection of lymphocytes during immune development and clonal origin of leukemia cells or other tumor cells. (See <a href=\"topic.htm?path=immunoglobulin-genetics\" class=\"medical medical_review\">&quot;Immunoglobulin genetics&quot;</a> and <a href=\"topic.htm?path=pathogenesis-of-acute-myeloid-leukemia\" class=\"medical medical_review\">&quot;Pathogenesis of acute myeloid leukemia&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H614101173\"><span class=\"h2\">Cloning</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Production of a genetically identical copy. Can refer to a single gene or to an entire organism.</p><p class=\"headingAnchor\" id=\"H1618877980\"><span class=\"h2\">Coding region</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Portion of a gene that encodes a protein.</p><p class=\"headingAnchor\" id=\"H4114138417\"><span class=\"h2\">Coding mutation or polymorphism</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A genetic variation in the open reading frame (protein-encoding region) of a gene. Coding variants that alter amino acid composition of a protein are called non-synonymous or missense variants (<a href=\"image.htm?imageKey=ALLRG%2F55798\" class=\"graphic graphic_figure graphicRef55798 \">figure 2</a>). Variants that do not alter amino acid composition are called synonymous variants. Nonsense variants are coding variants that result in the introduction of a stop codon (<a href=\"image.htm?imageKey=ALLRG%2F78399\" class=\"graphic graphic_figure graphicRef78399 \">figure 3</a>). A frameshift mutation results from an insertion or deletion of a number of bases not divisible by three, resulting in shifting of the reading frame (<a href=\"image.htm?imageKey=ALLRG%2F78767\" class=\"graphic graphic_figure graphicRef78767 \">figure 4</a>). Variants are also classified according to their pathogenicity. (See <a href=\"#H2504463718\" class=\"local\">'Variant'</a> below.)</p><p class=\"headingAnchor\" id=\"H3605286535\"><span class=\"h2\">Codon</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A three-nucleotide sequence that codes either for a specific amino acid or for chain initiation or termination during protein synthesis.</p><p class=\"headingAnchor\" id=\"H2828463269\"><span class=\"h2\">Complementation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The restoration of normal phenotype by gene replacement. The replaced gene can either be an intact copy of a defective gene (direct replacement), or an alternate gene with function that can compensate for the defective gene's aberrant function.</p><p class=\"headingAnchor\" id=\"H3440247137\"><span class=\"h2\">Complex trait/complex disease</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Trait or disease for which interactions between more than one gene <span class=\"nowrap\">and/or</span> environmental factors also play a role in the phenotype.</p><p class=\"headingAnchor\" id=\"H745843411\"><span class=\"h2\">Compound heterozygote</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>An individual bearing two different pathogenic variants in the same gene that together are sufficient to manifest an autosomal recessive phenotype. This differs from &quot;homozygote,&quot; which refers to an individual in whom both pathogenic variants are the same, and from &quot;double heterozygote,&quot; which refers to an individual who is heterozygous for pathogenic variants at two separate genetic loci, which together manifest disease.</p><p class=\"headingAnchor\" id=\"H2250302843\"><span class=\"h2\">Consanguinity</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Reproduction between two individuals from the same bloodline (eg, first cousin, second cousin). Consanguineous parentage increases the probability of a rare recessive disease, resulting from higher probability of both parents sharing the same rare deleterious sequence variant.</p><p class=\"headingAnchor\" id=\"H1375728442\"><span class=\"h2\">Copy number variation (CNV)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The most prevalent type of chromosomal structural variation, in which the number of copies of a large chromosomal or DNA segment (usually measuring thousands to millions of bases) varies between individuals. (See <a href=\"topic.htm?path=genomic-disorders-an-overview#H16722199\" class=\"medical medical_review\">&quot;Genomic disorders: An overview&quot;, section on 'Copy number variations'</a>.)</p><p class=\"headingAnchor\" id=\"H444668741\"><span class=\"h2\">Coupling</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The presence of two specified alleles at two linked loci on the same homologous chromosome (ie, &quot;in cis&quot;), and the two alternative alleles on the other chromosome. For illustration, in the case of dominant and recessive alleles, the coupling gametes formed are AB and ab (<a href=\"image.htm?imageKey=ALLRG%2F58385\" class=\"graphic graphic_figure graphicRef58385 \">figure 5</a>). In contrast, repulsion refers to the presence of the specified alleles at two linked loci on different chromosomes (ie, in trans). (See <a href=\"#H1891457425\" class=\"local\">'Repulsion'</a> below.)</p><p class=\"headingAnchor\" id=\"H1815524937\"><span class=\"h2\">CRISPR</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>CRISPR (clustered regularly interspaced short palindromic repeats; pronounced &quot;crisper&quot;; sometimes referred to as CRISPR-Cas9) is a component of a bacterial defense system that has been adapted for use in combination with an endonuclease such as Cas9 or Cpf1 for genome editing. (See <a href=\"#H4056953126\" class=\"local\">'Genome editing'</a> below.)</p><p class=\"headingAnchor\" id=\"H3305482665\"><span class=\"h2\">Crossing-over</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The exchange of chromosome segments through the process of recombination that occurs between two homologous chromosomes during meiosis. The site on the chromosome where the exchange occurs is called a crossover.</p><p class=\"headingAnchor\" id=\"H366724202\"><span class=\"h2\">De novo mutation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A novel genetic sequence variant introduced by a germline mutation in the proband's DNA. Often used to distinguish familial from sporadic cases of genetic disease.</p><p class=\"headingAnchor\" id=\"H3273258581\"><span class=\"h2\">Digenic inheritance</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Diseases caused by co-inheritance of mutations at two distinct genetic loci (ie, in two different genes).</p><p class=\"headingAnchor\" id=\"H4242890491\"><span class=\"h2\">Diploid</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Possessing two copies of each autosomal chromosome and two sex chromosomes. Most human cells are diploid. Hepatocytes are frequently polyploid (tetraploid or greater). Gametes are haploid (one copy of each autosome and one sex chromosome) (<a href=\"image.htm?imageKey=PC%2F72134\" class=\"graphic graphic_figure graphicRef72134 \">figure 1</a>). (See <a href=\"#H390400923\" class=\"local\">'Haploid'</a> below and <a href=\"#H2921582365\" class=\"local\">'Ploidy'</a> below.)</p><p class=\"headingAnchor\" id=\"H2977017041\"><span class=\"h2\">DNA barcoding</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A collection of methods developed to facilitate the analysis of complex mixtures of pooled samples, whereby short, unique DNA sequences (referred to as tags or barcodes) are added to each of the pooled DNA samples (eg, from distinct individuals). Barcoding is used routinely in next-generation sequencing applications, including single-cell RNA sequencing and exome sequencing.</p><p>Barcoding also refers to methods for determining the species of origin of a DNA sample on the basis of the DNA sequence itself. A clinical example is identification of the ingredients in an herbal preparation.</p><p class=\"headingAnchor\" id=\"H2335629778\"><span class=\"h2\">Dominant negative</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Dominant negative alleles are alleles that cause an abnormal phenotype or disease by a mechanism that depends on the presence of an abnormal gene product interfering with the function of the products from a normal gene. In other words, the variant allele confers a loss of function by interfering with the remaining normal allele. In contrast to most loss-of-function variants that confer phenotype only when both alleles are defective (ie, recessive inheritance), dominant-negative mutations act dominantly, meaning that only a single allele with the mutation is sufficient to cause the disease phenotype.</p><p class=\"headingAnchor\" id=\"H3021797604\"><span class=\"h2\">Double heterozygote</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>An individual who is heterozygous for two mutations at two separate genetic loci that together are sufficient to manifest a phenotype. Differs from compound heterozygote.</p><p class=\"headingAnchor\" id=\"H2932917426\"><span class=\"h2\">Embryonic stem cell</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A pluripotential cell derived from the inner cell mass of an early-stage embryo that is capable of differentiating into cells derived from all three germ layers.</p><p class=\"headingAnchor\" id=\"H1151712393\"><span class=\"h2\">Epigenetic change</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A modification of a chromosome that does not alter the nucleotide base sequence, but alters the expression of a gene. Epigenetic changes may be stable in an individual, but may be reversed during gametogenesis or development. DNA methylation and histone acetylation are common epigenetic changes. Epigenetic changes form the mechanistic basis of imprinting. Some medications alter epigenetic regulation (eg, histone deacetylase [HDAC] inhibitors). (See <a href=\"topic.htm?path=principles-of-epigenetics\" class=\"medical medical_review\">&quot;Principles of epigenetics&quot;</a>.)</p><p>Epigenetic modifications are removed when cells are treated in the laboratory to generate induced pluripotent stem cells (iPSCs). (See <a href=\"#H992606025\" class=\"local\">'Induced pluripotent stem cell (iPSC)'</a> below.)</p><p class=\"headingAnchor\" id=\"H1786466759\"><span class=\"h2\">Epistasis</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The process by which variations at two or more genetic loci interact to produce phenotypes different from the individual effects of each variant. This process is often referred to as either a gene-gene interaction or a genetic modifier effect.</p><p class=\"headingAnchor\" id=\"H2701193320\"><span class=\"h2\">Exome</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The portion of the genome that consists of exons. (See <a href=\"#H3520484613\" class=\"local\">'Exon'</a> below.)</p><p class=\"headingAnchor\" id=\"H2032140166\"><span class=\"h2\">Exome sequencing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A sequencing strategy that provides the DNA sequence corresponding to all exons (which represent approximately 1 to 2 percent of the genome), excluding introns and non-coding genomic sequence. Though the complete exome includes non-coding 5&rsquo; and 3&rsquo; untranslated regions (UTRs), most exome sequencing assays are enriched for the coding exons and largely exclude the non-coding regions.</p><p class=\"headingAnchor\" id=\"H3520484613\"><span class=\"h2\">Exon</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A segment of DNA that is transcribed and present in mature messenger RNA (mRNA). Many exons encode a portion of a protein, but non-coding exons also exist. This is in contrast to an intron, the DNA sequence between exons that does not become part of mature mRNA. Exons constitute only a small percent of the genome (about 1 to 2 percent).</p><p class=\"headingAnchor\" id=\"H86594006\"><span class=\"h2\">Expressivity</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A parameter used in genetic models that quantifies the degree to which an inherited characteristic is expressed in an organism.</p><p class=\"headingAnchor\" id=\"H711473475\"><span class=\"h2\">Frameshift mutation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A frameshift mutation is a change in DNA sequence that results from an insertion or deletion of a number of bases that is not divisible by three, resulting in a shift of the reading frame (<a href=\"image.htm?imageKey=ALLRG%2F78767\" class=\"graphic graphic_figure graphicRef78767 \">figure 4</a>) and thus altering synthesis of the protein.</p><p class=\"headingAnchor\" id=\"H2526278640\"><span class=\"h2\">Fusion gene</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A fusion gene is a functional gene product that results from the fusion of DNA segments from two physically distinct genes. The fusion occurs as a consequence of chromosomal rearrangements such as translocations, inversions, segmental deletions, or duplications. Examples include the <em>BCR-ABL</em> and the <em>FIP1L1-PDGFRA</em> oncogenes.</p><p class=\"headingAnchor\" id=\"H3419736662\"><span class=\"h2\">Gene</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A gene is a unit of DNA sequence that encodes specific function. Classical definitions limit genes to those elements that code for proteins. However, non-protein coding genes (such as non-coding RNAs or pseudogenes) are also genes.</p><p class=\"headingAnchor\" id=\"H3828842802\"><span class=\"h2\">Gene editing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Gene editing refers to the use of nucleases to alter the DNA sequence of a gene, as discussed in more detail below. (See <a href=\"#H4056953126\" class=\"local\">'Genome editing'</a> below.)</p><p class=\"headingAnchor\" id=\"H3479936033\"><span class=\"h2\">Genetic heterogeneity</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Genetic heterogeneity refers to a phenomenon in which variants in different genes result in the same phenotype or disease. Examples include the multiple genetic causes of sensorineural deafness. This differs from allelic heterogeneity, in which multiple variants in the same gene can lead to the same phenotype. (See <a href=\"#H3789757172\" class=\"local\">'Allelic heterogeneity'</a> above.)</p><p class=\"headingAnchor\" id=\"H3465061226\"><span class=\"h2\">Genetic polymorphism</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A genetic polymorphism is a DNA segment for which two or more alternate forms can be found in a population. The common types of polymorphisms include single nucleotide variants (single base pair changes, also called single nucleotide polymorphisms [SNPs]), indels <span class=\"nowrap\">(insertion/deletion</span> polymorphisms) or larger structural changes like copy number variants. Most commonly, genetic polymorphism refers a common single base-pair change or single nucleotide polymorphism (SNP). (See <a href=\"#H79497895\" class=\"local\">'Polymorphism'</a> below and <a href=\"#H2407636019\" class=\"local\">'Single nucleotide polymorphism (SNP)'</a> below.)</p><p class=\"headingAnchor\" id=\"H361643240\"><span class=\"h2\">Genotype</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A genotype is the combination of two alleles at one genomic location (locus) or base pair in an individual (<a href=\"image.htm?imageKey=ALLRG%2F58385\" class=\"graphic graphic_figure graphicRef58385 \">figure 5</a>).</p><p class=\"headingAnchor\" id=\"H4056953126\"><span class=\"h2\">Genome editing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Genome editing refers to the use of nucleases to insert or remove DNA from a genome. There are several common technologies that make use of genome editing, including clustered regularly interspaced short palindromic repeats (CRISPR), transcription activator-like effector nucleases (TALENs) and zinc finger nucleases (ZFNs). CRISPR is increasingly employed and is an RNA-guided gene editing method that uses a bacterially-derived protein (Cas9) and a specifically-designed synthetic guide RNA (gRNA; also known as a small guide RNA [sgRNA] or a single guide RNA) to introduce a double-strand break at a precise location in the gene of interest. The sgRNA directs the position of the double-strand break by hybridization to its matching sequence. Genome editing is used as a tool for genetic perturbation in research. Therapeutic applications for the correction of inherited genetic variation are under investigation. (See <a href=\"topic.htm?path=overview-of-gene-therapy-gene-editing-and-gene-silencing\" class=\"medical medical_review\">&quot;Overview of gene therapy, gene editing, and gene silencing&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H3856285139\"><span class=\"h2\">Genome-wide association study (GWAS)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A GWAS (pronounced &quot;gee-wass&quot;) study is a type of genetic mapping study design that assesses for evidence of association between genetic variants and heritable traits across the entire genome. Typical studies consist of genotyping hundreds of thousands of common SNPs, using DNA microarrays or other methodologies in large case-control populations, with the goal of identifying specific risk alleles that are more prevalent in cases than in controls. (See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-gene-expression-profiling\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Gene expression profiling&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H406115043\"><span class=\"h2\">Germline</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Germline refers to the gametes (ova and spermatozoa and their precursors) that have the capacity to give rise to offspring.</p><p class=\"headingAnchor\" id=\"H390400923\"><span class=\"h2\">Haploid</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Cells or organisms possessing one copy of each autosomal chromosome and one sex chromosome (and therefore effectively one copy of each gene). Gametes (ova and sperm) are haploid. Fertilization of a haploid ovum by a haploid sperm results in formation of a diploid embryo. Many microorganisms are haploid. In contrast, diploid organisms possess two of each autosome and two sex chromosomes. (See <a href=\"#H4242890491\" class=\"local\">'Diploid'</a> above.)</p><p class=\"headingAnchor\" id=\"H1042558214\"><span class=\"h2\">Haploinsufficiency</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Having only a single functional copy of a gene due to inactivation of the second allele by a deleterious variant. In a diploid cell, the single functional copy of the gene does not produce sufficient protein, resulting in disease. All haploinsufficient loci are hemizygous, but not all hemizygous loci are haploinsufficient. (See <a href=\"#H773063789\" class=\"local\">'Hemizygous'</a> below.)</p><p class=\"headingAnchor\" id=\"H2732486409\"><span class=\"h2\">Haplotype</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The physical combination or sequence of alleles present on a single chromosome. By definition, alleles on one haplotype are in &quot;cis&quot; (<a href=\"image.htm?imageKey=ALLRG%2F58385\" class=\"graphic graphic_figure graphicRef58385 \">figure 5</a>).</p><p class=\"headingAnchor\" id=\"H773063789\"><span class=\"h2\">Hemizygous</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The state of carrying only one copy of a genomic region due to deletion or altered function of the corresponding region on the other chromosome. Carriers of large-scale deletions are hemizygotes. Hemizygosity can confer disease if having one normally functioning copy is insufficient for normal cellular function (haploinsufficiency), but if a single functional copy of the gene is sufficient for normal cellular function, the phenotype may not be abnormal. Hemizygosity can also confer disease if a pathogenic mutation is present within the hemizygous region. (See <a href=\"#H1042558214\" class=\"local\">'Haploinsufficiency'</a> above.)</p><p class=\"headingAnchor\" id=\"H3157173185\"><span class=\"h2\">Heritability</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The proportion of phenotypic variation that is explained by genetic (or in some cases, epigenetic) factors.</p><p class=\"headingAnchor\" id=\"H1052502515\"><span class=\"h2\">Heteroplasmy</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The occurrence in a single cell of more than one different population of mitochondrial DNA sequence.</p><p class=\"headingAnchor\" id=\"H3344030654\"><span class=\"h2\">Identity by descent</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Alleles are identical by descent if they can be traced back to a common ancestor. Identity by descent is a more stringent classification than identity by state (see <a href=\"#H158047186\" class=\"local\">'Identity by state'</a> below). Identity by descent is the basis for establishing linkage.</p><p class=\"headingAnchor\" id=\"H158047186\"><span class=\"h2\">Identity by state</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Alleles are identical by state if the assay being used to distinguish alleles determines that they are identical.</p><p class=\"headingAnchor\" id=\"H1026529710\"><span class=\"h2\">Imprinting</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Gamete-specific gene silencing, in which only the allele from the mother or only the allele from the father is expressed, leading to observed parent-of-origin effects in offspring. Examples include the Prader-Willi syndrome and Angelman Syndrome locus and a gene involved in pseudohyperparathyroidism. (See <a href=\"topic.htm?path=epidemiology-and-genetics-of-prader-willi-syndrome\" class=\"medical medical_review\">&quot;Epidemiology and genetics of Prader-Willi syndrome&quot;</a> and <a href=\"topic.htm?path=congenital-cytogenetic-abnormalities\" class=\"medical medical_review\">&quot;Congenital cytogenetic abnormalities&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H2065670240\"><span class=\"h2\">Indel</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A class of common polymorphism or deleterious sequence variant defined by an extra copy or a missing copy of a short genetic or chromosomal sequence. (See <a href=\"topic.htm?path=chromosomal-translocations-deletions-and-inversions\" class=\"medical medical_review\">&quot;Chromosomal translocations, deletions, and inversions&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H992606025\"><span class=\"h2\">Induced pluripotent stem cell (iPSC)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A pluripotent cell derived by in vitro reprogramming of a somatic cell that is capable of both self-renewal and differentiation to mature lineages. (See <a href=\"topic.htm?path=overview-of-stem-cells#H3982137\" class=\"medical medical_review\">&quot;Overview of stem cells&quot;, section on 'Induced pluripotent stem (iPS) cells'</a>.)</p><p class=\"headingAnchor\" id=\"H648637710\"><span class=\"h2\">Intron</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A segment of DNA between two exons that is transcribed to pre-mRNA, but is removed through the process of splicing and is therefore not part of mature mRNA. Introns may contain regulatory DNA or serve other functions.</p><p class=\"headingAnchor\" id=\"H1734657361\"><span class=\"h2\">Inversion</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A chromosomal rearrangement characterized by rotation and reintegration of a DNA segment, resulting in an inverted orientation of the segment relative to its typical state.</p><p class=\"headingAnchor\" id=\"H1747533387\"><span class=\"h2\">Karyotype</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Karyotype refers to the complete set of chromosomes in an organism or tumor. Karyotype is determined by visual examination and counting of condensed chromosomes from several representative cells to determine the number of copies of each chromosome as well as any translocations. Determination of the karyotype of a tumor is also called &quot;cytogenetic analysis.&quot; (See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Cytogenetics and molecular genetics&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H4031196770\"><span class=\"h2\">Likely benign variant</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>(See <a href=\"#H2504463718\" class=\"local\">'Variant'</a> below.)</p><p class=\"headingAnchor\" id=\"H999314707\"><span class=\"h2\">Likely pathogenic variant</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>(See <a href=\"#H2504463718\" class=\"local\">'Variant'</a> below.)</p><p class=\"headingAnchor\" id=\"H3139062263\"><span class=\"h2\">Linkage</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The relationship that exists between two loci that violate the Mendelian law of independent assortment and therefore segregate in families in a non-random fashion. Non-independent assortment results because linked loci reside together on the same chromosome (ie, they are syntenic). However, most syntenic loci are not linked due to mandatory recombination during meiosis. Linkage therefore implies the linked loci are in close physical proximity to each other. The genetic linkage distance is expressed as the recombination fraction, which is measured in centiMorgans (cM). Note that this is not necessarily proportional to the physical distance (base pairs) separating the loci.</p><p class=\"headingAnchor\" id=\"H692109156\"><span class=\"h2\">Linkage analysis</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Method of gene mapping that tests for the non-random segregation of disease phenotypes with discrete chromosomal segments. Identification of linked regions implies the existence of disease-causing (pathogenic) variants within or proximal to the linked region. The process of disease-gene identification within this region is termed positional cloning.</p><p class=\"headingAnchor\" id=\"H4097651555\"><span class=\"h2\">Linkage disequilibrium</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The non-random association of alleles at two or more loci in a population. Linkage disequilibrium is present when the observed haplotype distribution of two or more markers in a population is significantly different from the expected haplotype distribution (which can be derived from the cross-product of observed allele frequencies) (<a href=\"image.htm?imageKey=ALLRG%2F76528\" class=\"graphic graphic_figure graphicRef76528 \">figure 6</a>).</p><p class=\"headingAnchor\" id=\"H169768065\"><span class=\"h2\">Locus</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A locus (plural = loci) is a specific chromosomal or genomic location.</p><p class=\"headingAnchor\" id=\"H3229945422\"><span class=\"h2\">LOD score</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The &quot;logarithm of the odds&quot; (LOD) score is a quantitative measure of the statistical evidence of linkage between two genes. The LOD score depends on both the probability of cosegregation of the two genes during meiosis and the size and structure of the population in which the linkage analysis is performed. By convention, LOD scores &gt;3 are considered to be evidence of linkage in human studies. In some studies, the threshold LOD scores for linkage can be established via permutation testing.</p><p class=\"headingAnchor\" id=\"H3964589819\"><span class=\"h2\">Lyonization</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>(See <a href=\"#H832339165\" class=\"local\">'X-inactivation'</a> below.)</p><p class=\"headingAnchor\" id=\"H1962140536\"><span class=\"h2\">Manhattan plot</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A type of plot used to display results of a GWAS study (see <a href=\"#H3856285139\" class=\"local\">'Genome-wide association study (GWAS)'</a> above). Genomic coordinates are shown on the X-axis and the negative logarithm of the P-value for each SNP on the Y-axis. SNPs with the strongest association will have the lowest P-values, and hence the tallest profiles. Named for the appearance of the skyline in Manhattan in the United States (<a href=\"image.htm?imageKey=PC%2F90262\" class=\"graphic graphic_figure graphicRef90262 \">figure 7</a>).</p><p class=\"headingAnchor\" id=\"H2616546832\"><span class=\"h2\">Marker</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A locus with alternative alleles that can be used in genetic mapping experiments.</p><p class=\"headingAnchor\" id=\"H4053855015\"><span class=\"h2\">Meiosis</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The cell division process in germline cells by which the chromosomal complement is reduced from the diploid to the haploid number (<a href=\"image.htm?imageKey=ALLRG%2F79125\" class=\"graphic graphic_figure graphicRef79125 \">figure 8</a>).</p><p class=\"headingAnchor\" id=\"H42565124\"><span class=\"h2\">Mendelian inheritance</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A trait is said to have Mendelian inheritance if its genetic transmission can be explained by a Mendelian model of inheritance, such as autosomal dominant, autosomal recessive, or X-linked recessive or dominant inheritance. This is in contrast to non-Mendelian inheritance patterns such as digenic inheritance, or quantitative traits. (See <a href=\"#H3273258581\" class=\"local\">'Digenic inheritance'</a> above.)</p><p class=\"headingAnchor\" id=\"H2788605268\"><span class=\"h2\">Metagenomics</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The study of complex microbial populations (biomes) using genomic approaches. Human tissues such as the skin and gut have multiple heterogeneous populations of microorganisms that differ from each other with respect to phyla composition and abundance in a tissue-specific manner. These abundances can be estimated by sequencing the mixed population of microorganisms, either through targeted sequencing of 16S ribosomes (for bacterial characterization) or whole-genome approaches (for bacteria, viruses, fungi, and other organisms).</p><p class=\"headingAnchor\" id=\"H1887986224\"><span class=\"h2\">Methylation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The addition of methyl groups to cytosine in DNA. Methylation followed by deamination is a major pathway for mutation to thymine. Methylation also correlates with reduced gene transcription and is an important mechanism for gene imprinting and X-inactivation. (See <a href=\"#H1026529710\" class=\"local\">'Imprinting'</a> above and <a href=\"#H832339165\" class=\"local\">'X-inactivation'</a> below.)</p><p class=\"headingAnchor\" id=\"H1184179623\"><span class=\"h2\">Micro-RNA (miR)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A small, non-coding RNA that regulates the stability or translation of a set of mRNAs.</p><p class=\"headingAnchor\" id=\"H1967539924\"><span class=\"h2\">Microsatellite</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A tandem array of short sequences of DNA (typically two to four bases). Microsatellites are numerous and widely distributed in the genome. There is often polymorphism in their length, making them useful markers in genetic studies, including genome mapping and family-based linkage analysis. Microsatellites are also known as short tandem repeat markers (STRs) or short tandem repeat polymorphisms (STRPs).</p><p class=\"headingAnchor\" id=\"H2663343461\"><span class=\"h2\">Mitochondrial genome</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The genetic material carried within mitochondria, known as mitochondrial DNA (mtDNA). At fertilization, all the mitochondria are derived from the egg, so mitochondrial genes display maternal inheritance.</p><p class=\"headingAnchor\" id=\"H380597366\"><span class=\"h2\">Mitosis</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The process of cell division occurring in somatic cells, in which each daughter cell receives a full chromosome complement.</p><p class=\"headingAnchor\" id=\"H838501563\"><span class=\"h2\">Monogenic trait/monogenic disease</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Trait or disease with inheritance that can be explained by a single gene, in contrast to polygenic and complex diseases. (See <a href=\"#H1586030136\" class=\"local\">'Polygenic trait/polygenic disease'</a> below.)</p><p class=\"headingAnchor\" id=\"H4081848015\"><span class=\"h2\">Mutation, mutant</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>An altered version of a gene that affects function. These terms are used in several different senses, depending on context:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In human genetics, a mutant is a genetic variant of low population frequency, in contrast to a polymorphism (often a single nuclear polymorphism [SNP]) with an allele frequency of 1 percent or greater. Types of gene mutations include nonsense (creates premature stop codon) (<a href=\"image.htm?imageKey=ALLRG%2F78399\" class=\"graphic graphic_figure graphicRef78399 \">figure 3</a>); missense (creates amino acid change) (<a href=\"image.htm?imageKey=ALLRG%2F55798\" class=\"graphic graphic_figure graphicRef55798 \">figure 2</a>); silent (no associated change in protein sequence); and frameshift (shifts the reading frame of the DNA and alters protein translation, resulting in an entirely new protein sequence downstream of the mutation) (<a href=\"image.htm?imageKey=ALLRG%2F78767\" class=\"graphic graphic_figure graphicRef78767 \">figure 4</a>).</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>In human disease, mutation implies a change associated with abnormal function (eg, sickle cell mutation of the hemoglobin beta chain). A disease-causing mutation is also called a pathogenic variant. (See <a href=\"#H2504463718\" class=\"local\">'Variant'</a> below.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>When used in the context of inheritance, mutation implies a recent sequence change (either germline or somatic), in contrast to inheritance from a carrier parent.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>When used to refer to an organism or population of organisms, a mutant refers to a population that harbors a specific, atypical variant (eg, antibiotic-resistant mutants).</p><p/><p class=\"headingAnchor\" id=\"H3120166530\"><span class=\"h2\">Mutation fraction</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Synonymous with allelic fraction or allelic ratio. (See <a href=\"#H1783841946\" class=\"local\">'Allelic fraction'</a> above and <a href=\"#H2454124624\" class=\"local\">'Allelic ratio'</a> above.)</p><p class=\"headingAnchor\" id=\"H978729645\"><span class=\"h2\">Next-generation sequencing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Any of several high-throughput DNA sequencing methods that rely on parallel analysis of multiple DNA fragments (eg, whole genome sequencing, exome sequencing). These methods have resulted in dramatic decreases in the cost and time needed for sequencing projects and are used in some clinical settings. (See <a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">&quot;Next-generation DNA sequencing (NGS): Principles and clinical applications&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H2012181996\"><span class=\"h2\">Non-coding variant</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Genetic variation that does not map to gene regions that code for protein. These variants can be functional if they reside in and disrupt functional elements, such as non-coding RNA sequences or regulatory sites (eg, promoters, enhancers, suppressors, or splice-sites).</p><p class=\"headingAnchor\" id=\"H3970454575\"><span class=\"h2\">Oncogene</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Gene that contributes to the production of cancer. Oncogenes typically act in a dominant manner (ie, an oncogenic mutation at one allele is sufficient to promote tumorigenesis). In contrast, tumor suppressor genes typically act in a recessive manner. (See <a href=\"#H2005622782\" class=\"local\">'Tumor suppressor gene'</a> below.)</p><p><strong>Pathogenic variant</strong> &ndash; Genetic change associated with disease or strongly suspected of being associated with disease. (See <a href=\"#H2504463718\" class=\"local\">'Variant'</a> below.)</p><p class=\"headingAnchor\" id=\"H1722947932\"><span class=\"h2\">Pedigree</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A diagram or other graphic representation of a family that shows the family relationships, sex of each family member, and presence or absence of one or more diseases in each individual (<a href=\"image.htm?imageKey=PC%2F55908\" class=\"graphic graphic_figure graphicRef55908 \">figure 9</a>).</p><p class=\"headingAnchor\" id=\"H1359853734\"><span class=\"h2\">Penetrance</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The probability that an individual harboring a pathogenic variant will develop the associated disease or condition. Incomplete (or variable) penetrance occurs when an individual with a pathogenic variant does not manifest features of the disorder. There are many causes of incomplete penetrance, including absence of environmental or genetic co-factors, epigenetic effects such as imprinting, sex-specific effects, or age-related expression differences.</p><p class=\"headingAnchor\" id=\"H3765863976\"><span class=\"h2\">Phenotype</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A characteristic of an organism (as opposed to the organism&rsquo;s genotype). Phenotypes are sensitive to the assays used to assign or measure them. They may be categorical, such as presence or absence of a disease; or quantitative, such as systolic blood pressure. Further complexities in phenotypic description involve the physiological state of the organism at the time of measurement, age, or use of provocative stimuli. Most phenotypes are variable, and this variability leads to the concepts of penetrance and expressivity. (See <a href=\"topic.htm?path=inheritance-patterns-of-monogenic-disorders-mendelian-and-non-mendelian#H1919293997\" class=\"medical medical_review\">&quot;Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)&quot;, section on 'Penetrance and expressivity'</a>.)</p><p class=\"headingAnchor\" id=\"H2978875878\"><span class=\"h2\">Pleiotropy</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The association of variant(s) in a single gene with multiple phenotypic effects, often in different tissues or organs. An example is Marfan syndrome, in which mutations in the fibrillin 1 (<em>FBN1</em>) gene can cause cardiac, ocular, and connective tissue findings.</p><p class=\"headingAnchor\" id=\"H2921582365\"><span class=\"h2\">Ploidy</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The number of sets of chromosomes present in an organism or cell. Ploidy varies among different organisms, including those that are always haploid (eg, bacteria), either haploid or diploid (eg, Saccharomyces species [yeast]), consistently diploid (eg, mammals) (see <a href=\"#H4242890491\" class=\"local\">'Diploid'</a> above), or polyploid (eg, hexaploid wheat). Different tissues in multicellular organisms may have different ploidies (eg, mammalian hepatocytes may be tetraploid). The gametes (ova and sperm) are haploid (See <a href=\"#H390400923\" class=\"local\">'Haploid'</a> above.). The designation of ploidy is based on the predominant ploidy of cells in the organism.</p><p class=\"headingAnchor\" id=\"H79497895\"><span class=\"h2\">Polymorphism</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Polymorphism can refer to a genetic polymorphism. (See <a href=\"#H3465061226\" class=\"local\">'Genetic polymorphism'</a> above.)</p><p>It can also refer to any biologic marker (DNA, RNA, or protein) with two or more states. Protein polymorphisms (varying amino acid sequence) can result from DNA polymorphisms or from differential RNA splicing (different isoforms), which in turn can result from sequence variation, epigenetic phenomena, or <span class=\"nowrap\">temporal/spatial/environmental</span> differences.</p><p class=\"headingAnchor\" id=\"H1586030136\"><span class=\"h2\">Polygenic trait/polygenic disease</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>In contrast to monogenic diseases, polygenic diseases are those for which the inherited trait(s) is explained by more than one gene. (See <a href=\"#H838501563\" class=\"local\">'Monogenic trait/monogenic disease'</a> above.)</p><p class=\"headingAnchor\" id=\"H23062511\"><span class=\"h2\">Polymerase chain reaction (PCR)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A method of specifically amplifying a unique target sequence (DNA or RNA) in the laboratory. PCR uses specific primers and repeated cycles of heating and cooling with a heat-stable DNA polymerase to replicate the template material exponentially. (See <a href=\"topic.htm?path=tools-for-genetics-and-genomics-polymerase-chain-reaction\" class=\"medical medical_review\">&quot;Tools for genetics and genomics: Polymerase chain reaction&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H3919422493\"><span class=\"h2\">Quantitative traits and quantitative trait loci (QTL)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>&quot;Quantitative&quot; traits are distinguished from discrete traits. The population varies continuously for quantitative traits and falls into obvious phenotypic classes for discrete traits. Quantitative traits are sometimes referred to as &quot;complex&quot; traits, reflecting the fact that multiple genes, the environment, and gene-environment interactions all contribute to an individual's trait value. Many traits are quantitative, and their inheritance is much more challenging to unravel than discrete traits. A quantitative trait locus (QTL) is a genomic region linked or associated with a quantitative trait.</p><p class=\"headingAnchor\" id=\"H92059740\"><span class=\"h2\">Read depth</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>In genomic or gene sequencing, the number of independent times each base in a targeted region has been sequenced. Typically expressed as an average X coverage (for example 20X = an average of 20 sequence reads per base). A minimum read depth of 30X is often required for clinical-grade sequencing. (See <a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">&quot;Next-generation DNA sequencing (NGS): Principles and clinical applications&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H2624951772\"><span class=\"h2\">Reading frame</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The starting point in translating the DNA sequence to protein. Since each codon includes three nucleotides, the reading frame can be initiated at one of three nucleotides. Offsetting the reading frame changes the amino acid composition of the encoded protein.</p><p class=\"headingAnchor\" id=\"H2322458728\"><span class=\"h2\">Recombinant</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Recombinant has different meanings in different contexts. For inheritance patterns, recombinant refers to offspring whose genotype and phenotype combinations differ from their parents, implying genetic recombination between the loci under study.</p><p>For laboratory techniques, recombinant technologies (also called genetic engineering), are molecular genetic approaches that use the process of homologous recombination to manipulate genotypes for experimental purposes. Examples include transgenic models where specific genetic loci are either knocked-out (removed) or knocked-in (introduced) to enable study of the locus; recombinant inbred mouse strains; recombinant viral transfection for synthesis of protein.</p><p class=\"headingAnchor\" id=\"H217868057\"><span class=\"h2\">Recombination</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The process of exchanging DNA sequence between two homologous chromosome regions. Mandatory recombination occurs at least once per aligned chromosome pair during meiosis. The exchange results in the creation of novel haplotypes that are combinations of the grandparental haplotypes present in a diploid cell. Exchange of unequal sequence content (ie, non-homologous recombination) can introduce DNA gains and losses of thousands or millions of bases. These gains and losses result in structural genetic variation and copy number variants (CNVs). (See <a href=\"#H1375728442\" class=\"local\">'Copy number variation (CNV)'</a> above.)</p><p class=\"headingAnchor\" id=\"H1891457425\"><span class=\"h2\">Repulsion</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The state in which alleles at two distinct loci are on physically opposing chromosomal strands. By definition, these variants are not part of the same haplotype (<a href=\"image.htm?imageKey=ALLRG%2F58385\" class=\"graphic graphic_figure graphicRef58385 \">figure 5</a>). In the example of dominant and recessive alleles, repulsion gametes formed are Ab and aB. The opposite relationship is coupling. (See <a href=\"#H444668741\" class=\"local\">'Coupling'</a> above.)</p><p class=\"headingAnchor\" id=\"H741224604\"><span class=\"h2\">Risk allele</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>An allele associated with a disease phenotype that typically acts in combination with other genetic or environmental factors. Though a risk allele is often that which is least common (ie, the minor allele), risk alleles associated with some complex traits may be the more common allele.</p><p class=\"headingAnchor\" id=\"H3875175161\"><span class=\"h2\">RNA interference (RNAi)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A ubiquitous intracellular process mediated by small RNA species, whereby specific RNAs are targeted for editing, degradation, or clearance. RNAi has important roles in the regulation of gene expression, developmental processes, cellular defense, and epigenetic effects.</p><p>RNAi technology (also called antisense technology) has been used in the laboratory to test the function of a gene by preventing its expression. Its use has been attempted clinically as a means of posttranscriptional gene silencing to reduce the expression of viral or cancer genes, or to lower cholesterol. Early attempts at developing therapeutic applications are ongoing in the fields of hematology, oncology, and neurodegenerative disease. (See <a href=\"topic.htm?path=overview-of-gene-therapy-gene-editing-and-gene-silencing\" class=\"medical medical_review\">&quot;Overview of gene therapy, gene editing, and gene silencing&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H868338761\"><span class=\"h2\">Sequencing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Determination of the nucleotide base sequence of a gene or collection of genes that determines the amino acid sequence of a protein. (See <a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">&quot;Next-generation DNA sequencing (NGS): Principles and clinical applications&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H2542927585\"><span class=\"h2\">Sex chromosomes</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Refers to the X and Y chromosomes, which are different in females (XX) and males (XY).</p><p class=\"headingAnchor\" id=\"H817773338\"><span class=\"h2\">Sex-linked</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A gene is sex-linked if it is located on a sex chromosome rather than on an autosome. A gene's inheritance pattern is also referred to as sex-linked if the pattern corresponds to that of known sex-linked genes (rather than autosomal genes). (See <a href=\"#H3829231885\" class=\"local\">'Autosomal'</a> above.)</p><p class=\"headingAnchor\" id=\"H1888984689\"><span class=\"h2\">Silencing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Regulation that prevents the expression of a gene. Mechanisms of silencing include gene methylation (see <a href=\"#H1887986224\" class=\"local\">'Methylation'</a> above), destruction of messenger RNA, or prevention of protein translation.</p><p class=\"headingAnchor\" id=\"H2407636019\"><span class=\"h2\">Single nucleotide polymorphism (SNP)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A single nucleotide polymorphism (pronounced &quot;snip&quot;) is a polymorphism (difference in base pair) that affects a single base pair, with a population frequency of at least 1 percent. Single base pair changes that occur at a lower population frequency are called pathogenic variants or mutations if they cause disease or affect protein function. (See <a href=\"#H79497895\" class=\"local\">'Polymorphism'</a> above.)</p><p class=\"headingAnchor\" id=\"H940268600\"><span class=\"h2\">Somatic</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Referring to tissues that are not within the germline. Somatic mutations arise in somatic tissues and are therefore not passed from parent to offspring. Somatic mutations are common in neoplasms.</p><p class=\"headingAnchor\" id=\"H2952035007\"><span class=\"h2\">Structural genetic variation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A term that encompasses a variety of large-scale genomic aberrations, including segmental rearrangements, translocations, or inversions and copy-number variants (CNVs) (see <a href=\"#H1375728442\" class=\"local\">'Copy number variation (CNV)'</a> above). Large rearrangements or deletions can be visualized through karyotyping. Smaller variants, particularly CNVs, segmental duplications, and interchromosomal interstitial rearrangements, are assessed by array comparative genomic hybridization (array CGH) or SNP arrays.</p><p class=\"headingAnchor\" id=\"H2929531485\"><span class=\"h2\">Syntenic</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Describing genetic loci that reside on the same chromosome. As an example, the genes causing Birt-Hogg-Dub&eacute; syndrome (Folliculin [<em>FLCN</em>], at chromosome 17p11) and early-onset breast cancer (<em>BRCA1</em>, at chromosome 17q21) are syntenic to each other on chromosome 17. However, because they are far apart from each other, they are not linked. (See <a href=\"#H3139062263\" class=\"local\">'Linkage'</a> above.)</p><p class=\"headingAnchor\" id=\"H3656629141\"><span class=\"h2\">Telomere</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Region at the ends of a chromosome that prevents the loss of genetic material or the accidental fusion of two chromosomes together during cell division. Telomeres of chromosomes in most cells shorten as an individual ages. Telomere length is maintained by the enzyme telomerase. (See <a href=\"#H987986146\" class=\"local\">'Telomerase'</a> below.)</p><p class=\"headingAnchor\" id=\"H987986146\"><span class=\"h2\">Telomerase</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Multicomponent enzyme that extends the length of telomeres. Telomerase mutations are seen in some inherited &quot;telomere syndromes.&quot; (See <a href=\"topic.htm?path=dyskeratosis-congenita-and-other-short-telomere-syndromes\" class=\"medical medical_review\">&quot;Dyskeratosis congenita and other short telomere syndromes&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H3486029516\"><span class=\"h2\">Translocation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A translocation is a structural chromosomal abnormality whereby chromosome segments are exchanged (swapped) between two non-homologous chromosomes. This form of rearrangement can be balanced, when the translocation does not result in any significant loss or gain of genetic material in the resultant gamete or cell; or unbalanced, when there is a gain or loss of genetic material in the resultant gamete or cell. (See <a href=\"topic.htm?path=chromosomal-translocations-deletions-and-inversions#H6\" class=\"medical medical_review\">&quot;Chromosomal translocations, deletions, and inversions&quot;, section on 'Translocations'</a>.)</p><p class=\"headingAnchor\" id=\"H2005622782\"><span class=\"h2\">Tumor suppressor gene</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A tumor suppressor gene is a gene that protects against the development or growth of tumors. Tumor suppressor genes typically act in a recessive manner (ie, both normal copies must be lost for a tumor to develop). In contrast, oncogenes typically act in a dominant manner. (See <a href=\"#H3970454575\" class=\"local\">'Oncogene'</a> above.)</p><p class=\"headingAnchor\" id=\"H138467379\"><span class=\"h2\">Uniparental disomy</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The inheritance of two copies of a chromosome (or part of a chromosome) from one parent, and no copy from the other parent, due either to nondisjunction errors during either the first or second phases of meiosis, or to chromosomal alterations in early fetal development. Nondisjunction during the first phase of meiosis (meiosis I) will result in inheritance of each of the grandparental chromosomes from one parent, termed &quot;heterodisomy.&quot; In contrast, nondisjunction during meiosis II results in inheritance of two identical copies of one grandparental chromosome, termed &quot;isodisomy.&quot;</p><p class=\"headingAnchor\" id=\"H2504463718\"><span class=\"h2\">Variant</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The term variant is used to refer to a specific change in either DNA or protein sequence. The American College of Medical Genetics and Genomics (ACMG) and the Association for Molecular Pathology have recommended use of a five-tier terminology system for the clinical classification of genetic variants, consisting of the following designations [<a href=\"https://www.uptodate.com/contents/genetics-glossary-of-terms/abstract/3\" class=\"abstract_t\">3</a>]:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Pathogenic variant &ndash; A disease-causing variant, as determined by very strong genetic and experimental evidence, including consistent familial co-segregation with disease and definitive functional studies.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Likely pathogenic variant &ndash; A variant with strong, but not definitive, evidence of pathogenicity based on its similarity to known pathogenic variants, co-segregation with disease in families or populations, and functional evidence.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Variant of unknown significance &ndash; A variant for which the specific criteria for the other four criteria are not met, or when contradictory lines of evidence in support of both benign or pathogenic classifications are present.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Likely benign variant &ndash; A variant with multiple supporting (but not conclusive) lines of evidence to suggest it is not disease-causing.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Benign variant &ndash; A variant with conclusive evidence as not disease-causing, as determined typically (but not only) by a high prevalence of the variant in the general (healthy) population, at a prevalence that exceeds that of the suspected disease.</p><p/><p>Additional information about this classification and its application to genomic testing is presented separately. (See <a href=\"topic.htm?path=secondary-findings-from-genetic-testing#H70129\" class=\"medical medical_review\">&quot;Secondary findings from genetic testing&quot;, section on 'Definitions and classification of variants'</a>.)</p><p class=\"headingAnchor\" id=\"H3552163370\"><span class=\"h2\">Variant of unknown significance (VUS)</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A classification term used in clinical DNA sequencing reports to signify genetic polymorphisms for which the pathogenicity (likelihood of causing disease) cannot be determined easily. VUS are variants that cannot be readily classified as &quot;pathogenic,&quot; &quot;likely pathogenic,&quot; &quot;benign, or &quot;likely benign.&quot; (See <a href=\"#H2504463718\" class=\"local\">'Variant'</a> above.)</p><p class=\"headingAnchor\" id=\"H3721925573\"><span class=\"h2\">Whole genome sequencing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A sequencing strategy that provides the DNA sequence for the entire genome, including exons, introns, and other non-coding sequence. In contrast, exome sequencing only determines the sequence of gene-coding regions.</p><p class=\"headingAnchor\" id=\"H832339165\"><span class=\"h2\">X-inactivation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>An epigenetic process that occurs in all female mammalian cells, whereby one of the two X chromosomes are randomly rendered inactive, such that all subsequent gene expression is derived from the other (active) X chromosome. This is sometimes called lyonization, after Mary Lyon, who did important early work on this phenomenon.</p><p class=\"headingAnchor\" id=\"H3472001134\"><span class=\"h1\">SUMMARY</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Commonly used terms applicable to genetics concepts are defined above, with links to relevant UpToDate topic reviews.</p></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li class=\"breakAll\">Brenner's Encyclopedia of Genetics, 2nd ed, Maloy S, Hughes K (Eds), Elsevier, 2013.</li><li><a href=\"https://www.uptodate.com/contents/genetics-glossary-of-terms/abstract/2\" class=\"nounderline abstract_t\">Van Loo P, Campbell PJ. ABSOLUTE cancer genomics. Nat Biotechnol 2012; 30:620.</a></li><li><a href=\"https://www.uptodate.com/contents/genetics-glossary-of-terms/abstract/3\" class=\"nounderline abstract_t\">Richards S, Aziz N, Bale S, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Genet Med 2015; 17:405.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 2898 Version 31.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H3472001134\"><span>SUMMARY</span></a></li><li><a href=\"#H1\" id=\"outline-link-H1\">INTRODUCTION</a></li><li><a href=\"#H2\" id=\"outline-link-H2\">LIST OF TERMS</a><ul><li><a href=\"#H3942702307\" id=\"outline-link-H3942702307\">Allele</a></li><li><a href=\"#H1783841946\" id=\"outline-link-H1783841946\">Allelic fraction</a></li><li><a href=\"#H1537087732\" id=\"outline-link-H1537087732\">Allele frequency</a></li><li><a href=\"#H3789757172\" id=\"outline-link-H3789757172\">Allelic heterogeneity</a></li><li><a href=\"#H2454124624\" id=\"outline-link-H2454124624\">Allelic ratio</a></li><li><a href=\"#H4099420019\" id=\"outline-link-H4099420019\">Aneuploidy</a></li><li><a href=\"#H3218144220\" id=\"outline-link-H3218144220\">Anticipation</a></li><li><a href=\"#H1368106711\" id=\"outline-link-H1368106711\">Association</a></li><li><a href=\"#H4245358129\" id=\"outline-link-H4245358129\">Autosome</a></li><li><a href=\"#H3829231885\" id=\"outline-link-H3829231885\">Autosomal</a></li><li><a href=\"#H2994976941\" id=\"outline-link-H2994976941\">Benign variant</a></li><li><a href=\"#H3984329775\" id=\"outline-link-H3984329775\">Biome</a></li><li><a href=\"#H103000573\" id=\"outline-link-H103000573\">Carrier</a></li><li><a href=\"#H2161571447\" id=\"outline-link-H2161571447\">Carrier rate</a></li><li><a href=\"#H3240578466\" id=\"outline-link-H3240578466\">Carrier testing</a></li><li><a href=\"#H2680527670\" id=\"outline-link-H2680527670\">Centromere</a></li><li><a href=\"#H3724164370\" id=\"outline-link-H3724164370\">Chromatid</a></li><li><a href=\"#H2759312189\" id=\"outline-link-H2759312189\">Chromatin</a></li><li><a href=\"#H2427059189\" id=\"outline-link-H2427059189\">Clonal</a></li><li><a href=\"#H614101173\" id=\"outline-link-H614101173\">Cloning</a></li><li><a href=\"#H1618877980\" id=\"outline-link-H1618877980\">Coding region</a></li><li><a href=\"#H4114138417\" id=\"outline-link-H4114138417\">Coding mutation or polymorphism</a></li><li><a href=\"#H3605286535\" id=\"outline-link-H3605286535\">Codon</a></li><li><a href=\"#H2828463269\" id=\"outline-link-H2828463269\">Complementation</a></li><li><a href=\"#H3440247137\" id=\"outline-link-H3440247137\">Complex trait/complex disease</a></li><li><a href=\"#H745843411\" id=\"outline-link-H745843411\">Compound heterozygote</a></li><li><a href=\"#H2250302843\" id=\"outline-link-H2250302843\">Consanguinity</a></li><li><a href=\"#H1375728442\" id=\"outline-link-H1375728442\">Copy number variation (CNV)</a></li><li><a href=\"#H444668741\" id=\"outline-link-H444668741\">Coupling</a></li><li><a href=\"#H1815524937\" id=\"outline-link-H1815524937\">CRISPR</a></li><li><a href=\"#H3305482665\" id=\"outline-link-H3305482665\">Crossing-over</a></li><li><a href=\"#H366724202\" id=\"outline-link-H366724202\">De novo mutation</a></li><li><a href=\"#H3273258581\" id=\"outline-link-H3273258581\">Digenic inheritance</a></li><li><a href=\"#H4242890491\" id=\"outline-link-H4242890491\">Diploid</a></li><li><a href=\"#H2977017041\" id=\"outline-link-H2977017041\">DNA barcoding</a></li><li><a href=\"#H2335629778\" id=\"outline-link-H2335629778\">Dominant negative</a></li><li><a href=\"#H3021797604\" id=\"outline-link-H3021797604\">Double heterozygote</a></li><li><a href=\"#H2932917426\" id=\"outline-link-H2932917426\">Embryonic stem cell</a></li><li><a href=\"#H1151712393\" id=\"outline-link-H1151712393\">Epigenetic change</a></li><li><a href=\"#H1786466759\" id=\"outline-link-H1786466759\">Epistasis</a></li><li><a href=\"#H2701193320\" id=\"outline-link-H2701193320\">Exome</a></li><li><a href=\"#H2032140166\" id=\"outline-link-H2032140166\">Exome sequencing</a></li><li><a href=\"#H3520484613\" id=\"outline-link-H3520484613\">Exon</a></li><li><a href=\"#H86594006\" id=\"outline-link-H86594006\">Expressivity</a></li><li><a href=\"#H711473475\" id=\"outline-link-H711473475\">Frameshift mutation</a></li><li><a href=\"#H2526278640\" id=\"outline-link-H2526278640\">Fusion gene</a></li><li><a href=\"#H3419736662\" id=\"outline-link-H3419736662\">Gene</a></li><li><a href=\"#H3828842802\" id=\"outline-link-H3828842802\">Gene editing</a></li><li><a href=\"#H3479936033\" id=\"outline-link-H3479936033\">Genetic heterogeneity</a></li><li><a href=\"#H3465061226\" id=\"outline-link-H3465061226\">Genetic polymorphism</a></li><li><a href=\"#H361643240\" id=\"outline-link-H361643240\">Genotype</a></li><li><a href=\"#H4056953126\" id=\"outline-link-H4056953126\">Genome editing</a></li><li><a href=\"#H3856285139\" id=\"outline-link-H3856285139\">Genome-wide association study (GWAS)</a></li><li><a href=\"#H406115043\" id=\"outline-link-H406115043\">Germline</a></li><li><a href=\"#H390400923\" id=\"outline-link-H390400923\">Haploid</a></li><li><a href=\"#H1042558214\" id=\"outline-link-H1042558214\">Haploinsufficiency</a></li><li><a href=\"#H2732486409\" id=\"outline-link-H2732486409\">Haplotype</a></li><li><a href=\"#H773063789\" id=\"outline-link-H773063789\">Hemizygous</a></li><li><a href=\"#H3157173185\" id=\"outline-link-H3157173185\">Heritability</a></li><li><a href=\"#H1052502515\" id=\"outline-link-H1052502515\">Heteroplasmy</a></li><li><a href=\"#H3344030654\" id=\"outline-link-H3344030654\">Identity by descent</a></li><li><a href=\"#H158047186\" id=\"outline-link-H158047186\">Identity by state</a></li><li><a href=\"#H1026529710\" id=\"outline-link-H1026529710\">Imprinting</a></li><li><a href=\"#H2065670240\" id=\"outline-link-H2065670240\">Indel</a></li><li><a href=\"#H992606025\" id=\"outline-link-H992606025\">Induced pluripotent stem cell (iPSC)</a></li><li><a href=\"#H648637710\" id=\"outline-link-H648637710\">Intron</a></li><li><a href=\"#H1734657361\" id=\"outline-link-H1734657361\">Inversion</a></li><li><a href=\"#H1747533387\" id=\"outline-link-H1747533387\">Karyotype</a></li><li><a href=\"#H4031196770\" id=\"outline-link-H4031196770\">Likely benign variant</a></li><li><a href=\"#H999314707\" id=\"outline-link-H999314707\">Likely pathogenic variant</a></li><li><a href=\"#H3139062263\" id=\"outline-link-H3139062263\">Linkage</a></li><li><a href=\"#H692109156\" id=\"outline-link-H692109156\">Linkage analysis</a></li><li><a href=\"#H4097651555\" id=\"outline-link-H4097651555\">Linkage disequilibrium</a></li><li><a href=\"#H169768065\" id=\"outline-link-H169768065\">Locus</a></li><li><a href=\"#H3229945422\" id=\"outline-link-H3229945422\">LOD score</a></li><li><a href=\"#H3964589819\" id=\"outline-link-H3964589819\">Lyonization</a></li><li><a href=\"#H1962140536\" id=\"outline-link-H1962140536\">Manhattan plot</a></li><li><a href=\"#H2616546832\" id=\"outline-link-H2616546832\">Marker</a></li><li><a href=\"#H4053855015\" id=\"outline-link-H4053855015\">Meiosis</a></li><li><a href=\"#H42565124\" id=\"outline-link-H42565124\">Mendelian inheritance</a></li><li><a href=\"#H2788605268\" id=\"outline-link-H2788605268\">Metagenomics</a></li><li><a href=\"#H1887986224\" id=\"outline-link-H1887986224\">Methylation</a></li><li><a href=\"#H1184179623\" id=\"outline-link-H1184179623\">Micro-RNA (miR)</a></li><li><a href=\"#H1967539924\" id=\"outline-link-H1967539924\">Microsatellite</a></li><li><a href=\"#H2663343461\" id=\"outline-link-H2663343461\">Mitochondrial genome</a></li><li><a href=\"#H380597366\" id=\"outline-link-H380597366\">Mitosis</a></li><li><a href=\"#H838501563\" id=\"outline-link-H838501563\">Monogenic trait/monogenic disease</a></li><li><a href=\"#H4081848015\" id=\"outline-link-H4081848015\">Mutation, mutant</a></li><li><a href=\"#H3120166530\" id=\"outline-link-H3120166530\">Mutation fraction</a></li><li><a href=\"#H978729645\" id=\"outline-link-H978729645\">Next-generation sequencing</a></li><li><a href=\"#H2012181996\" id=\"outline-link-H2012181996\">Non-coding variant</a></li><li><a href=\"#H3970454575\" id=\"outline-link-H3970454575\">Oncogene</a></li><li><a href=\"#H1722947932\" id=\"outline-link-H1722947932\">Pedigree</a></li><li><a href=\"#H1359853734\" id=\"outline-link-H1359853734\">Penetrance</a></li><li><a href=\"#H3765863976\" id=\"outline-link-H3765863976\">Phenotype</a></li><li><a href=\"#H2978875878\" id=\"outline-link-H2978875878\">Pleiotropy</a></li><li><a href=\"#H2921582365\" id=\"outline-link-H2921582365\">Ploidy</a></li><li><a href=\"#H79497895\" id=\"outline-link-H79497895\">Polymorphism</a></li><li><a href=\"#H1586030136\" id=\"outline-link-H1586030136\">Polygenic trait/polygenic disease</a></li><li><a href=\"#H23062511\" id=\"outline-link-H23062511\">Polymerase chain reaction (PCR)</a></li><li><a href=\"#H3919422493\" id=\"outline-link-H3919422493\">Quantitative traits and quantitative trait loci (QTL)</a></li><li><a href=\"#H92059740\" id=\"outline-link-H92059740\">Read depth</a></li><li><a href=\"#H2624951772\" id=\"outline-link-H2624951772\">Reading frame</a></li><li><a href=\"#H2322458728\" id=\"outline-link-H2322458728\">Recombinant</a></li><li><a href=\"#H217868057\" id=\"outline-link-H217868057\">Recombination</a></li><li><a href=\"#H1891457425\" id=\"outline-link-H1891457425\">Repulsion</a></li><li><a href=\"#H741224604\" id=\"outline-link-H741224604\">Risk allele</a></li><li><a href=\"#H3875175161\" id=\"outline-link-H3875175161\">RNA interference (RNAi)</a></li><li><a href=\"#H868338761\" id=\"outline-link-H868338761\">Sequencing</a></li><li><a href=\"#H2542927585\" id=\"outline-link-H2542927585\">Sex chromosomes</a></li><li><a href=\"#H817773338\" id=\"outline-link-H817773338\">Sex-linked</a></li><li><a href=\"#H1888984689\" id=\"outline-link-H1888984689\">Silencing</a></li><li><a href=\"#H2407636019\" id=\"outline-link-H2407636019\">Single nucleotide polymorphism (SNP)</a></li><li><a href=\"#H940268600\" id=\"outline-link-H940268600\">Somatic</a></li><li><a href=\"#H2952035007\" id=\"outline-link-H2952035007\">Structural genetic variation</a></li><li><a href=\"#H2929531485\" id=\"outline-link-H2929531485\">Syntenic</a></li><li><a href=\"#H3656629141\" id=\"outline-link-H3656629141\">Telomere</a></li><li><a href=\"#H987986146\" id=\"outline-link-H987986146\">Telomerase</a></li><li><a href=\"#H3486029516\" id=\"outline-link-H3486029516\">Translocation</a></li><li><a href=\"#H2005622782\" id=\"outline-link-H2005622782\">Tumor suppressor gene</a></li><li><a href=\"#H138467379\" id=\"outline-link-H138467379\">Uniparental disomy</a></li><li><a href=\"#H2504463718\" id=\"outline-link-H2504463718\">Variant</a></li><li><a href=\"#H3552163370\" id=\"outline-link-H3552163370\">Variant of unknown significance (VUS)</a></li><li><a href=\"#H3721925573\" id=\"outline-link-H3721925573\">Whole genome sequencing</a></li><li><a href=\"#H832339165\" id=\"outline-link-H832339165\">X-inactivation</a></li></ul></li><li><a href=\"#H3472001134\" id=\"outline-link-H3472001134\">SUMMARY</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>GRAPHICS <a href=\"#\" id=\"viewAllGraphicsLink\">View All</a></h2><div id=\"outlineGraphics\"><ul><li><div id=\"PC/2898|FIG\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"FIGURES\">FIGURES</a></div><ul><li><a href=\"image.htm?imageKey=PC/72134\" class=\"graphic graphic_figure\">- Human karyotype</a></li><li><a href=\"image.htm?imageKey=ALLRG/55798\" class=\"graphic graphic_figure\">- Missense mutation</a></li><li><a href=\"image.htm?imageKey=ALLRG/78399\" class=\"graphic graphic_figure\">- Nonsense mutation</a></li><li><a href=\"image.htm?imageKey=ALLRG/78767\" class=\"graphic graphic_figure\">- Frameshift mutation</a></li><li><a href=\"image.htm?imageKey=ALLRG/58385\" class=\"graphic graphic_figure\">- Haplotypes coupling repulsion</a></li><li><a href=\"image.htm?imageKey=ALLRG/76528\" class=\"graphic graphic_figure\">- Linkage disequilibrium</a></li><li><a href=\"image.htm?imageKey=PC/90262\" class=\"graphic graphic_figure\">- Manhattan plot</a></li><li><a href=\"image.htm?imageKey=ALLRG/79125\" class=\"graphic graphic_figure\">- Essential features meiosis</a></li><li><a href=\"image.htm?imageKey=PC/55908\" class=\"graphic graphic_figure\">- 4 generation pedigree</a></li></ul></li></ul></div></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=carrier-screening-for-genetic-disease-in-the-ashkenazi-jewish-population\" class=\"medical medical_review\">Carrier screening for genetic disease in the Ashkenazi Jewish population</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=chromosomal-translocations-deletions-and-inversions\" class=\"medical medical_review\">Chromosomal translocations, deletions, and inversions</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=congenital-cytogenetic-abnormalities\" class=\"medical medical_review\">Congenital cytogenetic abnormalities</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=dyskeratosis-congenita-and-other-short-telomere-syndromes\" class=\"medical medical_review\">Dyskeratosis congenita and other short telomere syndromes</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=epidemiology-and-genetics-of-prader-willi-syndrome\" class=\"medical medical_review\">Epidemiology and genetics of Prader-Willi syndrome</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetic-testing\" class=\"medical medical_review\">Genetic testing</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genomic-disorders-an-overview\" class=\"medical medical_review\">Genomic disorders: An overview</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=glossary-of-common-biostatistical-and-epidemiological-terms\" class=\"medical medical_review\">Glossary of common biostatistical and epidemiological terms</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=immunoglobulin-genetics\" class=\"medical medical_review\">Immunoglobulin genetics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=inheritance-patterns-of-monogenic-disorders-mendelian-and-non-mendelian\" class=\"medical medical_review\">Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=next-generation-dna-sequencing-ngs-principles-and-clinical-applications\" class=\"medical medical_review\">Next-generation DNA sequencing (NGS): Principles and clinical applications</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=overview-of-gene-therapy-gene-editing-and-gene-silencing\" class=\"medical medical_review\">Overview of gene therapy, gene editing, and gene silencing</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=overview-of-stem-cells\" class=\"medical medical_review\">Overview of stem cells</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=pathogenesis-of-acute-myeloid-leukemia\" class=\"medical medical_review\">Pathogenesis of acute myeloid leukemia</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=principles-of-epigenetics\" class=\"medical medical_review\">Principles of epigenetics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=secondary-findings-from-genetic-testing\" class=\"medical medical_review\">Secondary findings from genetic testing</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=systematic-review-and-meta-analysis\" class=\"medical medical_review\">Systematic review and meta-analysis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-cytogenetics-and-molecular-genetics\" class=\"medical medical_review\">Tools for genetics and genomics: Cytogenetics and molecular genetics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-gene-expression-profiling\" class=\"medical medical_review\">Tools for genetics and genomics: Gene expression profiling</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=tools-for-genetics-and-genomics-polymerase-chain-reaction\" class=\"medical medical_review\">Tools for genetics and genomics: Polymerase chain reaction</a></li></ul></div></div>","javascript":null}