var data={"title":"Human leukocyte antigens (HLA): A roadmap","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Human leukocyte antigens (HLA): A roadmap</div><dl id=\"topicContributors\"><dt><span> </span>Author:</dt><dd><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/contributors\" class=\"contributor contributor_credentials\">Sebastien Viatte, MD, PhD</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/contributors\" class=\"contributor contributor_credentials\">Peter H Schur, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/contributors\" class=\"contributor contributor_credentials\">Paul L Romain, MD</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Dec 02, 2017.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H1\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The major histocompatibility complex (MHC) is a term used to describe a group of genes in animals and humans that encode a variety of cell surface markers, antigen-presenting molecules, and other proteins involved in immune function. The human leukocyte antigen (HLA) complex is synonymous with the human MHC.</p><p>The earliest HLA associations with rheumatic diseases, such as the association of the HLA-B*27 allele at the HLA-B gene with ankylosing spondylitis (AS) risk and the association of the HLA-DRB1*04 allele at the HLA-DRB1 gene with rheumatoid arthritis (RA), were discovered several decades ago. As the study of HLA genetics has evolved and expanded, the nomenclature has been continually altered, posing challenges for those attempting to follow the science. However, knowledge concerning this genetic region has evolved sufficiently, so the overall nomenclature should be more stable in the future, even though new alleles will be identified and catalogued.</p><p>The genetics, nomenclature, and typing of HLA, as well as the relationships between HLA and rheumatic disease, are discussed here. The older nomenclature that may still be encountered in the literature is defined. The specific function of the MHC system, including the mechanisms of antigen presentation, is discussed separately. (See <a href=\"topic.htm?path=major-histocompatibility-complex-mhc-structure-and-function\" class=\"medical medical_review\">&quot;Major histocompatibility complex (MHC) structure and function&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H2\"><span class=\"h1\">GENETIC STRUCTURE OF THE HLA REGION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The major histocompatibility complex (MHC) in humans refers to a genetic region containing hundreds of genes, including the human leukocyte antigen (HLA) genes (<a href=\"image.htm?imageKey=RHEUM%2F109459\" class=\"graphic graphic_figure graphicRef109459 \">figure 1</a>). Thus, the human MHC region is also referred to as the HLA region. HLA genes express their gene products on the surface of white blood cells (hence the name &quot;human leukocyte antigen,&quot; although HLA class I genes (see <a href=\"#H4\" class=\"local\">'Class I region'</a> below) are also expressed on all nucleated cells) and were originally recognized to contain the genes encoding &quot;tissue antigens&quot; or &quot;tissue types.&quot; The function of these genes was revealed in rodent studies, in which they were identified as the factors responsible for rejection of tissue grafts between unmatched individuals (hence the name &quot;major histocompatibility&quot;).</p><p>The HLA region lies on the short arm of chromosome six at position 6p21.3. The <strong>classical MHC</strong> spans 3.6 megabases (Mb) and comprises more than 200 genes, including many immune system genes, but also many genes without any known immune function. The localization of genes relevant to the MHC outside the classical boundaries of this region and confirmation of extended linkage disequilibrium have since led to the proposal for an <strong>extended MHC</strong><em> </em><strong>(xMHC)</strong>. This region spans 7.6 Mb and contains over 400 loci. The complete structure and gene map of the HLA region have been published [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/1,2\" class=\"abstract_t\">1,2</a>].</p><p class=\"headingAnchor\" id=\"H3\"><span class=\"h1\">MHC ORGANIZATION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The human leukocyte antigen (HLA) region has been subdivided into three regions: class I, class II, and class III. Each region contains numerous gene loci, including expressed genes and pseudogenes. Some HLA loci are highly polymorphic; for example, over 4300 alleles are known for HLA-B and over 1900 alleles for HLA-DRB1. (See <a href=\"#H4\" class=\"local\">'Class I region'</a> below and <a href=\"#H5\" class=\"local\">'Class II region'</a> below and <a href=\"#H6\" class=\"local\">'Class III region'</a> below.)</p><p>A detailed description of the major histocompatibility complex (MHC), including a listing of the genes in each region, a discussion of standards for nomenclature, and additional information, is available online through the European Bioinformatics Institute and the <a href=\"http://www.ebi.ac.uk/ipd/imgt/hla/&amp;token=vg79WD99YENq89BSCip/KLBxBk0P/UHz4ewiKuLez/ME5OPPVJ7fMcodnBSKvLD2&amp;TOPIC_ID=3987\" target=\"_blank\" class=\"external\">International Immunogenetics Project</a> [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/3\" class=\"abstract_t\">3</a>]. A brief review of MHC organization is provided in this section.</p><p class=\"headingAnchor\" id=\"H4\"><span class=\"h2\">Class I region</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The class I region contains the genes encoding the &quot;classical&quot; class I HLA antigens: HLA-A, B, and C. Class I antigens are expressed on almost all cells of the body, except erythrocytes and trophoblasts, at varying density [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/4\" class=\"abstract_t\">4</a>]. The class I antigens are made up of a heavy chain (alpha chain), which combines non-covalently with the non-polymorphic light chain (beta chain) to form the final dimerized molecule. The class I alpha chains are coded for by genes within the MHC (eg, HLA-A, HLA-B), whereas the beta chain, beta-2 microglobulin, is encoded on chromosome 15, not in the MHC.</p><p>The class I region also contains other HLA class I genes such as HLA-E, HLA-F and HLA-G; MHC class I polypeptide-related (MIC) genes; and a variety of other genes, not all of which are immune-related. (See <a href=\"topic.htm?path=immunology-of-the-maternal-fetal-interface\" class=\"medical medical_review\">&quot;Immunology of the maternal-fetal interface&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H5\"><span class=\"h2\">Class II region</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The class II region contains the genes encoding the HLA class II molecules, HLA-DP, DQ, and DR. Class II molecules are constitutively expressed on antigen-presenting cells (APC; for example, dendritic cells, macrophages, or B cells) and can be induced during inflammation on many other cell types that normally have little or no expression.</p><p>As with the class I molecules, the class II molecules also consist of an alpha (heavy) and a beta (light) chain. However, genes within the MHC (eg, HLA-DRA and HLA-DRB) code for both chains.</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The HLA-DQ and -DP proteins each have polymorphic alpha and beta chains, which can dimerize in various combinations.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>By contrast, HLA-DR dimers all share an essentially invariant alpha chain, while the beta chain carries the extreme polymorphism characteristic of these antigens.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>To add further complexity, the number of HLA-DR genes can vary among individuals. In some cases, two HLA-DR molecules are expressed on a single haplotype. Both dimers utilize the same invariant DR alpha chain, but one uses the beta chain encoded by the HLA-DRB1 locus gene, and the other uses a beta chain encoded by the second DR locus, called DRB3, DRB4, DRB5, etc. Alleles at this latter locus are usually expressed at much lower levels on the cell surface.</p><p/><p>As with the class I region, the class II region also contains many other genes, several of which assist in the processing and presentation of antigen. The DMA and DMB genes, for example, lie within the class II region and have structural similarities to both class I and class II genes. Analysis of mutant cells lacking DM genes reveals that these genes are critical to the process of loading peptides into class II molecules in endosomal compartments within the cell; when this process is defective, the class II molecules cannot appropriately bind processed antigen, traffic to the cell surface, or present antigen for recognition to the immune system [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/5,6\" class=\"abstract_t\">5,6</a>].</p><p>In the class I antigen processing and presentation system, two other sets of genes in the MHC class II region, which are not structurally related to class I or class II genes, have been shown to play a role. LMP2 and LMP7 encode components of a proteasome complex that degrades cytoplasmic proteins into peptides that are then capable of being bound to and presented by class I molecules [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/6,7\" class=\"abstract_t\">6,7</a>]. These peptide fragments are then transported into the endoplasmic reticulum by products of the transporter associated with antigen processing (TAP) genes, TAP1 and TAP2, each of which exists in several allelic forms [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/7,8\" class=\"abstract_t\">7,8</a>]. Without functional TAP genes, antigenic peptides cannot reach the endoplasmic reticulum, in which they normally bind to class I molecules and subsequently move to the cell surface, again to present peptides for recognition by the immune system.</p><p class=\"headingAnchor\" id=\"H6\"><span class=\"h2\">Class III region</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The region in between class I and class II is known as the class III region. Although this region does not contain any of the HLA genes, it does contain many genes of importance in the immune response, a few examples of which are given below.</p><p class=\"headingAnchor\" id=\"H7\"><span class=\"h3\">Complement</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Several complement components (C2, C4, and factor B) are encoded in this region; diseases like systemic lupus erythematosus (SLE) have been associated with particular null alleles at these loci, although the mechanism whereby they may be important is not understood [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/9,10\" class=\"abstract_t\">9,10</a>]. (See <a href=\"topic.htm?path=inherited-disorders-of-the-complement-system\" class=\"medical medical_review\">&quot;Inherited disorders of the complement system&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H8\"><span class=\"h3\">Tumor necrosis factor</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Several cytokines known to play in role in various inflammatory pathways (tumor necrosis factor-alpha [TNF], lymphotoxin alpha, and lymphotoxin beta) are also encoded in the MHC. TNF polymorphisms have been associated with autoimmune diseases such as SLE, and TNF has been shown to play an important role in regulating the inflammation of rheumatoid arthritis (RA) [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/11-13\" class=\"abstract_t\">11-13</a>]. (See <a href=\"topic.htm?path=role-of-cytokines-in-rheumatic-diseases\" class=\"medical medical_review\">&quot;Role of cytokines in rheumatic diseases&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H9\"><span class=\"h3\">Heat shock protein</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A heat shock protein gene, HSP-70, encodes a chaperone molecule that may be relevant to the pathogenesis of RA [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/14\" class=\"abstract_t\">14</a>].</p><p>The importance to normal and aberrant immune function of these and other genes in the MHC whose function is not yet understood is the subject of ongoing research.</p><p class=\"headingAnchor\" id=\"H10\"><span class=\"h1\">METHODS OF HLA TYPING AND CHANGING NOMENCLATURE</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>As the technology used to genotype human leukocyte antigen (HLA) genes has evolved over the years, the nomenclature for the different alleles at individual HLA genes has evolved. Originally, before the widespread availability of sequence-based approaches, genotyping was done by using immunological techniques to query the protein structures of the HLA gene products on cell surfaces. Different alleles code for different versions of the same protein, and immunological techniques are sensitive to some, but not all, of these differences. The original nomenclature used for different alleles referred to the specific immunological reagents used to type. As those alleles have been subdivided into different sets of alleles with gene sequencing methods, which can identify more subtle differences, the nomenclature has evolved. (See <a href=\"#H3784431638\" class=\"local\">'Nomenclature'</a> below.)</p><p>Thus, the nomenclature has been frequently revised as advances were made in understanding HLA polymorphism. As a result, the literature is full of variations that can be frustrating to the nonspecialist: how would one know that DRB1*15:01 is a DR2 allele? A brief historical overview, using DR4 as an example, may make this clearer.</p><p class=\"headingAnchor\" id=\"H16402732\"><span class=\"h2\">Immunological typing methods</span></p><p class=\"headingAnchor\" id=\"H11\"><span class=\"h3\">Serology</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The earliest typing was performed by serologic methods. Women are exposed to foreign, paternally derived HLA antigens during pregnancy and can develop antibodies to these antigens. Sera from numerous multiparous women were screened for those reacting reproducibly to certain HLA types. The epitopes recognized by these sera were generally shared, or &quot;public,&quot; epitopes, which are present on a family of related alleles. Thus, a person would be labeled &quot;DR4&quot; if their cells reacted with sera characterized as seeing this epitope.</p><p>The presence of a second DR specificity in some individuals was also recognized. Sera were characterized that recognized these less polymorphic specificities. Most individuals positive for DR4 were also positive for the second DR specificity, DRw53. Similarly, haplotypes carrying the DR3, DR5, or DR6 specificities were normally positive for DRw52, which later was split into DRw52a, DRw52b, etc by more specific sera.</p><p class=\"headingAnchor\" id=\"H12\"><span class=\"h3\">Cellular typing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>It soon became clear that human T cells could distinguish more precise &quot;splits&quot; among some of the HLA specificities recognized by these sera. As a result, mixed lymphocyte typing was developed to allow more exact typing. A panel of reference cells was generated that were homozygous for different antigens, called homozygous typing cells (HTCs). Cells from an individual of unknown HLA type were tested for recognition by the panel cells. In this way, the DR4 &quot;type&quot; could be split into several subtypes, such as Dw4, Dw10, Dw14, etc.</p><p>Since both the sera from multiparous women and, especially, the HTCs were continually in need of replenishment and replacement, international histocompatibility workshops, under the auspices of the World Health Organization (WHO), were held periodically to compare and share reagents from around the world in an attempt to ensure some consistency. When consensus was reached on a new antigenic specificity, it would be given a new number with the designation &quot;w&quot; for &quot;workshop.&quot; As a specificity became widely reproducible and accepted, the &quot;w&quot; would be dropped and a &quot;permanent&quot; name would be given. Thus, in one workshop, a newly identified specificity might be given the name DRw4, and in a few years it might be revised to DR4. &quot;DR&quot; implies a serologically-based definition, and &quot;D&quot; means cellularly typed. In this way, one HLA specificity could be DR4,Dw4 or DR4,Dw10.</p><p class=\"headingAnchor\" id=\"H14\"><span class=\"h2\">DNA-based typing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A history of HLA DNA typing methods is provided here [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/15\" class=\"abstract_t\">15</a>]. Briefly, two major developments occurred that have led to molecular-based HLA typing and that allowed a more stable nomenclature system to be developed. First, the gene structure and sequence for the HLA genes were determined, thus allowing the identification of the polymorphic regions of the genes (<a href=\"image.htm?imageKey=RHEUM%2F67233\" class=\"graphic graphic_figure graphicRef67233 \">figure 2</a>). Second, polymerase chain reaction (PCR) technology was developed. This procedure permits amplification of specific DNA fragments that are then sufficiently abundant for analysis. PCR technology has provided the basis for development of convenient and rapid methods of HLA typing, based upon the exact nucleotide sequences of individual alleles.</p><p>Traditional DNA cloning and sequencing had been performed on some HLA genes prior to PCR, but this was difficult and time-consuming. With the advent of PCR, dozens of individuals were rapidly sequenced and a surprising number of new polymorphisms were identified.</p><p>Findings related to DR4 are illustrative. Serological typing allowed HLA-DR4 individuals to be identified. Cellular typing showed that HLA-DR4 could be split into a number of specificities (eg, DR4Dw4 or DR4Dw14), and molecular typing has been able to split these still further. Thus, Dw4 was determined to have a sequence now called DRB1*04:01, while Dw14 was found to include at least two alleles, called DRB1*04:04 and *04:08. Other alleles, such DRB1*04:09, *04:10, and *04:11, had not previously been identified. Over 200 DR4 alleles are known, and more are being discovered, particularly as HLA genes of disparate ethnic groups are being analyzed. A more detailed description of the nomenclature system is available on the internet [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/16,17\" class=\"abstract_t\">16,17</a>].</p><p class=\"headingAnchor\" id=\"H58441692\"><span class=\"h3\">DNA-based typing using sequence-specific oligonucleotide probes</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Full HLA class I and II typing begins with drawing a patient's blood and extracting DNA from the white blood cells. Rapid and simple methods have supplanted the original process of separating the white blood cells by density gradient centrifugation, lysing the cells, and precipitating the DNA. A number of PCR-based methods are available for determining which alleles are present. The most commonly used method for disease studies is sequence-specific oligonucleotide probes (SSOP). In this technique, the DNA undergoes PCR amplification, using a set of carefully selected primers designed to amplify the desired portion of DNA, such as a part of the HLA-DRB1 gene. Different primer sets are used for different loci, such as HLA-DRB, -DQA, -DQB, etc. The amplified section of DNA is then probed with various SSOPs, which are typically labeled by nonradioactive means and which have been designed to bind only to specific sequences that discriminate among alleles. Due to the complexity and number of allelic sequences, a single SSOP rarely can identify a specific allele. Rather, a patchwork of reactivity to various probes is required for determination of the allele.</p><p>DNA-based HLA typing is by no means trivial. However, the results reflect a precise determination of individual alleles and are not dependent upon the interpretation of variable biologic reagents. Furthermore, the reagents can all be synthesized and quality-controlled, and the method lends itself to automation. Commercially available automated or semi-automated HLA typing systems are utilized in many laboratories.</p><p>In many cases, full HLA typing is not necessary, since one may want to know only if a disease-associated susceptibility allele is present in an individual. A semi-automated &quot;kit&quot; to detect the presence of rheumatoid arthritis (RA) susceptibility alleles, for example, has been described [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/18\" class=\"abstract_t\">18</a>]. (See <a href=\"topic.htm?path=hla-and-other-susceptibility-genes-in-rheumatoid-arthritis\" class=\"medical medical_review\">&quot;HLA and other susceptibility genes in rheumatoid arthritis&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H204999001\"><span class=\"h3\">DNA-based typing with direct Sanger sequencing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>DNA-based typing with direct Sanger sequencing, also called &quot;sequence-based typing&quot; (SBT), is a technique that consists of the amplification and direct sequencing of relevant exons. First-generation sequencing or Sanger sequencing is a sequencing method based on the incorporation of chain-terminating dideoxynucleotides by the DNA polymerase during in vitro DNA replication. It will give the exact DNA sequence at the nucleotide level of the DNA fragment under consideration.</p><p class=\"headingAnchor\" id=\"H16402746\"><span class=\"h3\">Next-generation DNA-based typing</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>While highly accurate methods to type HLA genes based on DNA sequence remain expensive and time-consuming, there is every expectation that ongoing advances in DNA sequencing technologies will facilitate rapid and economical typing in the near future [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/19\" class=\"abstract_t\">19</a>]. Several investigators have devised preliminary approaches [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/20-23\" class=\"abstract_t\">20-23</a>]. These technologies are still evolving and are not yet in widespread use.</p><p class=\"headingAnchor\" id=\"H1484198751\"><span class=\"h3\">DNA-based typing with genotyping microarray</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>For research purposes, a newer technique for HLA typing has been developed [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/24\" class=\"abstract_t\">24</a>]. The first step consists of determining the genotype of single nucleotide polymorphisms (SNPs) located within the HLA region by using dense genotyping microarrays. Although dense genotyping is performed, it will not determine the carriage of each nucleotide at each position, and the DNA sequence of the HLA region will therefore be incomplete and contain gaps. Databases are now available (eg, from the 1000 Genome Project) which contain the complete DNA sequence of thousands of individuals (reference panel). By comparing an incomplete DNA sequence from a given person of a given ethnicity with the reference panel of that specific ethnicity, it is therefore possible to fill in the gaps, at least partially. This process is called &quot;imputation.&quot; The second step of genotyping-based HLA typing consists, therefore, of a computational strategy to impute four-digit HLA allele by comparing the available genotypes of SNPs over the HLA region with a large, population-specific reference panel.</p><p class=\"headingAnchor\" id=\"H326181305\"><span class=\"h2\">HLA typing resolution and ambiguity</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>&quot;Low-resolution typing&quot; (antigen or allele family level) was defined in 2011 by the Harmonization of Histocompatibility Typing Terms Working Group as being equivalent to serologic typing, such as typing at the level of the first two digits (eg, HLA-DRB1*01, -DRB1*03, -DRB1*04, etc) [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/25\" class=\"abstract_t\">25</a>].</p><p>SSOP typing is usually considered as &quot;intermediate resolution typing&quot; (unless several SSOP reactions are combined to reach higher resolution and unambiguously define a specific allele).</p><p>&quot;High-resolution typing&quot; is considered to distinguish all the alleles at a specific locus (at the four-digit level [eg, HLA-DRB1*04:01] or protein level).</p><p>&quot;Ambiguity&quot; refers to the inability to assign a single HLA-allele present in the <span class=\"nowrap\">IMGT/HLA</span> sequence database [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/17\" class=\"abstract_t\">17</a>] to the results of HLA typing. For example, ambiguity can typically result from the use of low-resolution typing methods but also from high-resolution typing if polymorphisms are present outside the region being typed. As the number of alleles added to the <span class=\"nowrap\">IMGT/HLA</span> sequence database increases, the problem of ambiguity also becomes more difficult and the length of ambiguity strings (ie, lists of alleles and possible genotypes consistent with the typing data) increases.</p><p class=\"headingAnchor\" id=\"H3784431638\"><span class=\"h2\">Nomenclature</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The 2010 naming convention for HLA alleles (WHO Nomenclature Committee for Factors of the HLA System) assigns a unique identifier to each HLA allele [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/16,17\" class=\"abstract_t\">16,17</a>]. This identifier always starts with HLA, followed by a hyphen, the name of the gene (eg, A, B, or C for class I genes), an asterisk (*), and up to four sets of digits separated by colons (ie, HLA-A*XX:XX:XX:XX) (<a href=\"image.htm?imageKey=RHEUM%2F109457\" class=\"graphic graphic_figure graphicRef109457 \">figure 3</a>). All alleles receive at least a four-digit code, which corresponds to the first two sets of digits. The digits after the asterisk and before the first colon describe the allele group or type, which frequently corresponds to the serological type. The second set of digits is used to define a specific HLA protein. Therefore, a so-called &quot;four-digit HLA type&quot; fully and unequivocally determines the protein structure, by definition. HLA alleles whose identifiers differ in the two first sets of digits must differ in at least one nucleotide substitution that changes the amino acid sequence of the encoded HLA protein. Alleles that differ only by non-coding nucleotide substitutions within the coding sequence are distinguished by the use of the third set of digits (&quot;six-digit HLA typing&quot;). A fourth set of digits is occasionally used to separate further sequence polymorphisms occurring in non-coding regions (introns, 5' or 3' untranslated regions).</p><p class=\"headingAnchor\" id=\"H15\"><span class=\"h1\">INTERPRETATION OF HLA AND DISEASE ASSOCIATIONS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Over one hundred diseases are associated with classical human leukocyte antigen (HLA) class I and II genes, as well as with some of the non-HLA genes in the major histocompatibility complex (MHC) region (eg, C4 complement) [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/1,4\" class=\"abstract_t\">1,4</a>]. More information on the contribution of genetic variation within the MHC to the pathogenesis, susceptibility or severity of specific diseases is presented in topic reviews that discuss these aspects of the relevant disease (see appropriate topics). A more detailed discussion of genetic association studies and their interpretation is provided elsewhere. (See <a href=\"topic.htm?path=genetic-association-studies-principles-and-applications\" class=\"medical medical_review\">&quot;Genetic association studies: Principles and applications&quot;</a>.)</p><p>It is reasonable to ask whether these studies are relevant to clinical practice and why there are so many studies. Part of the confusion arises from the evolution of HLA typing methods as previously described. Many early studies used serologic typing methods, so the results might describe, for example, that rheumatoid arthritis (RA) was associated with DR4.</p><p>However, DR4 encompasses many different alleles, some of which are RA-associated and some of which are not; as a result, the associations were generally weaker than when the precise alleles were subsequently investigated. Subsequent studies using DNA-based methods have consistently shown that, in Caucasians, the alleles DRB1*04:01, *04:04, *04:05, and *04:08 are highly associated with RA. (See <a href=\"topic.htm?path=hla-and-other-susceptibility-genes-in-rheumatoid-arthritis\" class=\"medical medical_review\">&quot;HLA and other susceptibility genes in rheumatoid arthritis&quot;</a>.)</p><p>With ankylosing spondylitis (AS), HLA-B27 is estimated to contribute only 16 to 50 percent of the total genetic risk. Of the large number of documented HLA-B27 alleles, the strongest association with AS is with HLA-B*27:05; among the Japanese and Chinese, it is with HLA-B*27:04. However, there is no association with *27:06 and *27:09. (See <a href=\"topic.htm?path=pathogenesis-of-spondyloarthritis\" class=\"medical medical_review\">&quot;Pathogenesis of spondyloarthritis&quot;</a>.)</p><p>One promising approach in understanding disease association has been the examination of variation at individual amino acid sites, as opposed to HLA alleles. In RA, for example, investigators were able to demonstrate that much of the association of HLA-DRB1 with RA is best explained by differences in the specific amino acid that occurs at position 11 in the DRB1 molecule encoded by the different HLA-DRB1 gene alleles [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/26\" class=\"abstract_t\">26</a>]. Position 11 is located at the base of the DRB1 binding groove and probably modulates differential binding to key antigens involved in autoimmunity.</p><p>Ethnic differences must also be taken into account. The frequency of a particular allele in one population can be very different from that in another population. As an example, DR4 was shown not to be associated with RA in Israeli Jews [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/27\" class=\"abstract_t\">27</a>]. However, RA-associated DR4 alleles are rarely present in this Israeli population. The most common DR4 allele is DRB1*04:02, which is not disease-associated (or, at most, very weakly), thereby explaining the apparent paradox. For the same reason, the control group with which the patient group is compared must be ethnically matched for the results to be valid.</p><p>Definition of the patient group is also responsible for some differences among studies. It now seems clear, for example, that the DRB1*04:01, *04:04, *04:05, and *04:08 alleles are associated with severe forms of RA [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/28\" class=\"abstract_t\">28</a>]. In studies of patient groups acquired from tertiary medical center practices, associations with these alleles are much stronger than in studies in which the patients come from primary care practices, presumably because of clinical differences in patient populations. (See <a href=\"topic.htm?path=hla-and-other-susceptibility-genes-in-rheumatoid-arthritis#H2\" class=\"medical medical_review\">&quot;HLA and other susceptibility genes in rheumatoid arthritis&quot;, section on 'HLA alleles and susceptibility to rheumatoid arthritis'</a>.)</p><p class=\"headingAnchor\" id=\"H3004135\"><span class=\"h2\">HLA and drug-induced hypersensitivity</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The role of HLA genes in drug-induced hypersensitivity (DIH) may have more immediate clinical relevance than HLA disease associations. Numerous studies have shown a relationship between HLA class I and class II molecules and DIH [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/29\" class=\"abstract_t\">29</a>]; the most well-established of such relationships include the association of HLA-B*15:02 with carbamazepine-induced Stevens-Johnson <span class=\"nowrap\">syndrome/toxic</span> epidermal necrolysis <span class=\"nowrap\">(SJS/TEN)</span> in Southeast Asian populations, and HLA-B*57:01 with <a href=\"topic.htm?path=abacavir-drug-information\" class=\"drug drug_general\">abacavir</a> hypersensitivity. Screening for HLA-B*57:01 prior to abacavir use has now been implemented in clinical practice and provides an excellent example of the process that needs to be followed to translate association findings into routine clinical care. (See <a href=\"topic.htm?path=abacavir-hypersensitivity-reaction\" class=\"medical medical_review\">&quot;Abacavir hypersensitivity reaction&quot;</a> and <a href=\"topic.htm?path=overview-of-pharmacogenomics#H9\" class=\"medical medical_review\">&quot;Overview of pharmacogenomics&quot;, section on 'Effect on idiosyncratic reactions'</a>.)</p><p class=\"headingAnchor\" id=\"H16\"><span class=\"h2\">Use of p values</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The methods used to assess statistical significance are also critical. Most straightforward population studies of disease association use the chi-squared test to compare the frequency of a particular allele in the patient population with its frequency in a control population. If the p value is less than 0.05, the association is said to be statistically significant; this means that the chance of this association occurring by chance alone is less than 1 in 20.</p><p>However, if one is analyzing dozens of different alleles for possible association, it becomes likely that one or more alleles will result in a spurious association in which p is less than 0.05 simply because of the many comparisons being made. As a result, the p value must be corrected for the number of tests performed. In other words, if one studies 10 different DR alleles for association with a particular disease, the p value for each one must be multiplied by 10. Thus, if the uncorrected p value is 0.05, the value becomes 0.5, or statistically nonsignificant, after correction. If, on the other hand, the original p value is 0.0005, the corrected value of p = 0.005 is still significant.</p><p class=\"headingAnchor\" id=\"H17\"><span class=\"h2\">Linkage disequilibrium</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>One must also keep in mind the concept of linkage disequilibrium when interpreting association studies. As noted above, a set of alleles are often inherited together. For example, the ancestral 8.1 haplotype spans the MHC region and includes the *01:01 allele at HLA-A, the *07:01 allele at HLA-C, the *08:01 allele at HLA-B, the *03:01 allele at HLA-DRB1, the *01:01 allele at HLA-DRB3, the *05:01 allele at HLA-DQB1, and the *02:01 allele at HLA-DQB1. As a result, all of the alleles can be seen together, and it can therefore be difficult in a genetic study to say which locus is primarily responsible for the association in such cases. An apparent association of a particular disease with, for example, HLA-DRB1*03:01 might actually be due to an association with a linked gene, such as HLA-DQB1*02:01, or even with a non-HLA gene such as a tumor necrosis factor (TNF) polymorphism.</p><p class=\"headingAnchor\" id=\"H18\"><span class=\"h2\">Relative risk</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Finally, one must remember that these are population studies and that the results cannot be easily transferred to an individual patient. The alleles that have been shown to be associated with disease are susceptibility alleles and identical to genes that are present among normal individuals, albeit with lower frequency. One can use the calculation of relative risk (RR) to indicate the odds that the disease will occur among individuals positive for the allele compared with individuals negative for the allele. The RR is calculated from the following formula:</p><p>RR = (the number of patients positive for the allele divided by the number of controls positive for the allele) divided by (the number of patients negative for the allele divided by the number of controls negative for the allele)</p><p>The RR for developing RA in Caucasians if one carries DRB1*04:01, for example, is about 6.</p><p class=\"headingAnchor\" id=\"H19\"><span class=\"h1\">HOW DO HLA ANTIGENS CONFER DISEASE SUSCEPTIBILITY?</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Data regarding the relationship of human leukocyte antigen (HLA) antigens to disease susceptibility remain at the level of associations, not disease mechanisms. Nevertheless, HLA associations that are reproducible and robust are giving us important clues about the development of certain rheumatic diseases. A variety of models have been postulated to explain these associations functionally [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/30\" class=\"abstract_t\">30</a>]. These include the importance of HLA polymorphisms in:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Shaping the T-cell repertoire during development</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Shaping the peripheral T-cell repertoire</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Determining which antigenic peptides are bound and therefore presented to the immune system for recognition [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/31\" class=\"abstract_t\">31</a>]</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Generating molecular mimicry between self-antigens and either the HLA molecule itself or peptides that it recognizes</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Affecting HLA protein presentation of either foreign or self-antigenic peptides to autoreactive T cells [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/1\" class=\"abstract_t\">1</a>]</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Influencing how infection, exogenous agents, or &quot;molecular mimicry&quot; may reactivate silenced T cells in autoimmune diseases [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/1\" class=\"abstract_t\">1</a>]</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Affecting immune suppression and cancer development in important ways through the loss of HLA gene expression because of viral infection, somatic mutations, or other causes [<a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/1\" class=\"abstract_t\">1</a>]</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Influencing antigen processing and presentation</p><p/><p>Identifying the mechanistic basis of these disease associations should lead to novel and specific treatment and even to preventive strategies.</p><p class=\"headingAnchor\" id=\"H75073771\"><span class=\"h1\">SUMMARY AND RECOMMENDATIONS</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The human leukocyte antigen (HLA) complex is synonymous with the human major histocompatibility complex (MHC). These terms describe a group of genes on chromosome 6 that encode a variety of cell surface markers, antigen-presenting molecules, and other proteins; many are involved in immune function, but many other genes located within this genetic region are not known to be involved in immune function. The classical MHC spans 3.6 megabases (Mb) and includes more than 200 genes. The identification of MHC-related genes outside these boundaries and findings of extended linkage disequilibrium have led to the proposal for an extended MHC, which spans 7.6 Mb and which contains over 400 loci. (See <a href=\"#H1\" class=\"local\">'Introduction'</a> above and <a href=\"#H2\" class=\"local\">'Genetic structure of the HLA region'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The HLA region has been subdivided into three regions: class I, class II, and class III. Each region contains numerous gene loci, including expressed genes and pseudogenes. Some HLA loci are highly polymorphic. (See <a href=\"#H3\" class=\"local\">'MHC organization'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The class I region contains the genes encoding the class I HLA antigens, which include HLA A, B, and C. Class I antigens are expressed on almost all cells of the body, except erythrocytes and trophoblasts, at varying density. The class I antigens are made up of a heavy chain (alpha chain) encoded within the MHC, which combines non-covalently with the non-polymorphic light chain (beta chain), also termed beta-2-microglobulin, which is encoded on chromosome 15, not in the MHC, to form the final dimerized molecule. (See <a href=\"#H4\" class=\"local\">'Class I region'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The class II region contains the genes encoding the HLA class II molecules, HLA-DP, -DQ, and -DR. Class II molecules are constitutively expressed on antigen-presenting cells (APC; for example, dendritic cells, macrophages, or B cells) and can be induced during inflammation on many other cell types that normally have little or no expression. As with the class I molecules, the class II molecules also consist of an alpha (heavy) and a beta (light) chain. However, genes within the MHC (eg, HLA-DRA and HLA-DRB) code for both chains. The DQ and DP molecules each have polymorphic alpha and beta chains, which can dimerize in various combinations; by contrast, DR dimers all share an essentially invariant alpha chain, while the beta chain carries the extreme polymorphism characteristic of these antigens. (See <a href=\"#H5\" class=\"local\">'Class II region'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The region in between class I and class II is known as the class III region. Although this region does not contain any of the HLA genes, it does contain many genes of importance in the immune response, including several complement components (C2, C4, and factor B); several cytokines known to play a role in inflammatory pathways, such as tumor necrosis factor (TNF)-alpha, lymphotoxin alpha, and lymphotoxin beta; and a heat shock protein gene, HSP-70, that encodes a chaperone molecule. (See <a href=\"#H6\" class=\"local\">'Class III region'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Determination of HLA types has become much more accurate and has led to frequent revision of the nomenclature. Typing was initially performed by serologic methods, and then additional specificities were detected by mixed lymphocyte typing using homozygous typing cells. Two major developments that have led to molecular-based HLA typing and a more stable nomenclature have been (a) the determination of the gene structure and sequence for the HLA genes and (b) the development of polymerase chain reaction (PCR) technology. PCR has provided the basis for convenient and rapid HLA typing methods, based upon the exact nucleotide sequences of individual alleles. (See <a href=\"#H10\" class=\"local\">'Methods of HLA typing and changing nomenclature'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>The 2010 naming convention for HLA alleles (World Health Organization [WHO] Nomenclature Committee for Factors of the HLA System) assigns a unique identifier to each HLA allele. This identifier follows a strict syntax (HLA-A as an example): HLA-A*XX:XX:XX:XX (<a href=\"image.htm?imageKey=RHEUM%2F109457\" class=\"graphic graphic_figure graphicRef109457 \">figure 3</a>). All alleles receive at least a four-digit code, which fully and unequivocally determines the protein structure, by definition. HLA alleles whose identifiers differ in the two first sets of digits must differ in at least one nucleotide substitution that changes the amino acid sequence of the encoded HLA protein.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Many diseases are associated with classical HLA I and II genes, as well as with some of the non-HLA genes in the MHC region. Results among different ethnicities may vary, and definition of the patient group may influence degrees of association. Attention should be given to the statistical methods used when interpreting association studies. (See <a href=\"#H15\" class=\"local\">'Interpretation of HLA and disease associations'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Data regarding HLA molecules and disease remain at the level of associations, not disease mechanisms. Nevertheless, HLA associations that are reproducible and robust provide important clues about the development of certain disorders. A variety of models have been postulated to explain these associations functionally. (See <a href=\"#H19\" class=\"local\">'How do HLA antigens confer disease susceptibility?'</a> above.)</p><p/><p class=\"headingAnchor\" id=\"H415445320\"><span class=\"h1\">ACKNOWLEDGMENT</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The editorial staff at UpToDate would like to acknowledge Wendy Thomson, MD, and Soumya Raychaudhuri, MD, PhD, who contributed to earlier versions of this topic review.</p></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/1\" class=\"nounderline abstract_t\">Shiina T, Inoko H, Kulski JK. An update of the HLA genomic region, locus information and disease associations: 2004. Tissue Antigens 2004; 64:631.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/2\" class=\"nounderline abstract_t\">Horton R, Wilming L, Rand V, et al. Gene map of the extended human MHC. Nat Rev Genet 2004; 5:889.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/3\" class=\"nounderline abstract_t\">Robinson J, Halliwell JA, Hayhurst JD, et al. The IPD and IMGT/HLA database: allele variant databases. Nucleic Acids Res 2015; 43:D423.</a></li><li class=\"breakAll\">Milford EL, Carpenter CB. Adapative immunity: Histocompatibility antigens and immune response genes. In: ACP Medicine, 2004.</li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/5\" class=\"nounderline abstract_t\">Busch R, Mellins ED. Developing and shedding inhibitions: how MHC class II molecules reach maturity. Curr Opin Immunol 1996; 8:51.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/6\" class=\"nounderline abstract_t\">Kelly A, Trowsdale J. Novel genes in the human major histocompatibility complex class-II region. Int Arch Allergy Immunol 1994; 103:11.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/7\" class=\"nounderline abstract_t\">Belich MP, Trowsdale J. Proteasome and class I antigen processing and presentation. Mol Biol Rep 1995; 21:53.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/8\" class=\"nounderline abstract_t\">Vandevyver C, Geusens P, Cassiman JJ, Raus J. Peptide transporter genes (TAP) polymorphisms and genetic susceptibility to rheumatoid arthritis. Br J Rheumatol 1995; 34:207.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/9\" class=\"nounderline abstract_t\">Reveille JD. Molecular genetics of systemic lupus erythematosus and Sj&ouml;gren's syndrome. Curr Opin Rheumatol 1990; 2:733.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/10\" class=\"nounderline abstract_t\">Christiansen FT, Zhang WJ, Griffiths M, et al. Major histocompatibility complex (MHC) complement deficiency, ancestral haplotypes and systemic lupus erythematosus (SLE): C4 deficiency explains some but not all of the influence of the MHC. J Rheumatol 1991; 18:1350.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/11\" class=\"nounderline abstract_t\">Jacob CO, Fronek Z, Lewis GD, et al. Heritable major histocompatibility complex class II-associated differences in production of tumor necrosis factor alpha: relevance to genetic predisposition to systemic lupus erythematosus. Proc Natl Acad Sci U S A 1990; 87:1233.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/12\" class=\"nounderline abstract_t\">Tomita Y, Hashimoto S, Yamagami K, et al. Restriction fragment length polymorphism (RFLP) analysis in the TNF genes of patients with systemic lupus erythematosus (SLE). Clin Exp Rheumatol 1993; 11:533.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/13\" class=\"nounderline abstract_t\">Feldmann M, Brennan FM, Maini RN. Role of cytokines in rheumatoid arthritis. Annu Rev Immunol 1996; 14:397.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/14\" class=\"nounderline abstract_t\">Auger I, Escola JM, Gorvel JP, Roudier J. HLA-DR4 and HLA-DR10 motifs that carry susceptibility to rheumatoid arthritis bind 70-kD heat shock proteins. Nat Med 1996; 2:306.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/15\" class=\"nounderline abstract_t\">Erlich H. HLA DNA typing: past, present, and future. Tissue Antigens 2012; 80:1.</a></li><li class=\"breakAll\">http://hla.alleles.org/nomenclature/ (Accessed on November 29, 2017).</li><li class=\"breakAll\">http://www.ebi.ac.uk/ipd/imgt/hla/ (Accessed on November 29, 2017).</li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/18\" class=\"nounderline abstract_t\">Nepom GT, Gersuk V, Nepom BS. Prognostic implications of HLA genotyping in the early assessment of patients with rheumatoid arthritis. J Rheumatol Suppl 1996; 44:5.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/19\" class=\"nounderline abstract_t\">Shendure J, Ji H. Next-generation DNA sequencing. Nat Biotechnol 2008; 26:1135.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/20\" class=\"nounderline abstract_t\">Bentley G, Higuchi R, Hoglund B, et al. High-resolution, high-throughput HLA genotyping by next-generation sequencing. Tissue Antigens 2009; 74:393.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/21\" class=\"nounderline abstract_t\">Iqbal Z, Caccamo M, Turner I, et al. De novo assembly and genotyping of variants using colored de Bruijn graphs. Nat Genet 2012; 44:226.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/22\" class=\"nounderline abstract_t\">Holcomb CL, H&ouml;glund B, Anderson MW, et al. A multi-site study using high-resolution HLA genotyping by next generation sequencing. Tissue Antigens 2011; 77:206.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/23\" class=\"nounderline abstract_t\">Erlich RL, Jia X, Anderson S, et al. Next-generation sequencing for HLA typing of class I loci. BMC Genomics 2011; 12:42.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/24\" class=\"nounderline abstract_t\">Jia X, Han B, Onengut-Gumuscu S, et al. Imputing amino acid polymorphisms in human leukocyte antigens. PLoS One 2013; 8:e64683.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/25\" class=\"nounderline abstract_t\">Nunes E, Heslop H, Fernandez-Vina M, et al. Definitions of histocompatibility typing terms. Blood 2011; 118:e180.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/26\" class=\"nounderline abstract_t\">Raychaudhuri S, Sandor C, Stahl EA, et al. Five amino acids in three HLA proteins explain most of the association between MHC and seropositive rheumatoid arthritis. Nat Genet 2012; 44:291.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/27\" class=\"nounderline abstract_t\">Schiff B, Mizrachi Y, Orgad S, et al. Association of HLA-Aw31 and HLA-DR1 with adult rheumatoid arthritis. Ann Rheum Dis 1982; 41:403.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/28\" class=\"nounderline abstract_t\">Viatte S, Plant D, Han B, et al. Association of HLA-DRB1 haplotypes with rheumatoid arthritis severity, mortality, and treatment response. JAMA 2015; 313:1645.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/29\" class=\"nounderline abstract_t\">Karlin E, Phillips E. Genotyping for severe drug hypersensitivity. Curr Allergy Asthma Rep 2014; 14:418.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/30\" class=\"nounderline abstract_t\">Viatte S, Plant D, Raychaudhuri S. Genetics and epigenetics of rheumatoid arthritis. Nat Rev Rheumatol 2013; 9:141.</a></li><li><a href=\"https://www.uptodate.com/contents/human-leukocyte-antigens-hla-a-roadmap/abstract/31\" class=\"nounderline abstract_t\">Scally SW, Petersen J, Law SC, et al. A molecular basis for the association of the HLA-DRB1 locus, citrullination, and rheumatoid arthritis. J Exp Med 2013; 210:2569.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 3987 Version 15.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H75073771\"><span>SUMMARY &amp; RECOMMENDATIONS</span></a></li><li><a href=\"#H1\" id=\"outline-link-H1\">INTRODUCTION</a></li><li><a href=\"#H2\" id=\"outline-link-H2\">GENETIC STRUCTURE OF THE HLA REGION</a></li><li><a href=\"#H3\" id=\"outline-link-H3\">MHC ORGANIZATION</a><ul><li><a href=\"#H4\" id=\"outline-link-H4\">Class I region</a></li><li><a href=\"#H5\" id=\"outline-link-H5\">Class II region</a></li><li><a href=\"#H6\" id=\"outline-link-H6\">Class III region</a><ul><li><a href=\"#H7\" id=\"outline-link-H7\">- Complement</a></li><li><a href=\"#H8\" id=\"outline-link-H8\">- Tumor necrosis factor</a></li><li><a href=\"#H9\" id=\"outline-link-H9\">- Heat shock protein</a></li></ul></li></ul></li><li><a href=\"#H10\" id=\"outline-link-H10\">METHODS OF HLA TYPING AND CHANGING NOMENCLATURE</a><ul><li><a href=\"#H16402732\" id=\"outline-link-H16402732\">Immunological typing methods</a><ul><li><a href=\"#H11\" id=\"outline-link-H11\">- Serology</a></li><li><a href=\"#H12\" id=\"outline-link-H12\">- Cellular typing</a></li></ul></li><li><a href=\"#H14\" id=\"outline-link-H14\">DNA-based typing</a><ul><li><a href=\"#H58441692\" id=\"outline-link-H58441692\">- DNA-based typing using sequence-specific oligonucleotide probes</a></li><li><a href=\"#H204999001\" id=\"outline-link-H204999001\">- DNA-based typing with direct Sanger sequencing</a></li><li><a href=\"#H16402746\" id=\"outline-link-H16402746\">- Next-generation DNA-based typing</a></li><li><a href=\"#H1484198751\" id=\"outline-link-H1484198751\">- DNA-based typing with genotyping microarray</a></li></ul></li><li><a href=\"#H326181305\" id=\"outline-link-H326181305\">HLA typing resolution and ambiguity</a></li><li><a href=\"#H3784431638\" id=\"outline-link-H3784431638\">Nomenclature</a></li></ul></li><li><a href=\"#H15\" id=\"outline-link-H15\">INTERPRETATION OF HLA AND DISEASE ASSOCIATIONS</a><ul><li><a href=\"#H3004135\" id=\"outline-link-H3004135\">HLA and drug-induced hypersensitivity</a></li><li><a href=\"#H16\" id=\"outline-link-H16\">Use of p values</a></li><li><a href=\"#H17\" id=\"outline-link-H17\">Linkage disequilibrium</a></li><li><a href=\"#H18\" id=\"outline-link-H18\">Relative risk</a></li></ul></li><li><a href=\"#H19\" id=\"outline-link-H19\">HOW DO HLA ANTIGENS CONFER DISEASE SUSCEPTIBILITY?</a></li><li><a href=\"#H75073771\" id=\"outline-link-H75073771\">SUMMARY AND RECOMMENDATIONS</a></li><li><a href=\"#H415445320\" id=\"outline-link-H415445320\">ACKNOWLEDGMENT</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>GRAPHICS <a href=\"#\" id=\"viewAllGraphicsLink\">View All</a></h2><div id=\"outlineGraphics\"><ul><li><div id=\"RHEUM/3987|FIG\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"FIGURES\">FIGURES</a></div><ul><li><a href=\"image.htm?imageKey=RHEUM/109459\" class=\"graphic graphic_figure\">- Gene structure of the human major histocompatibility complex</a></li><li><a href=\"image.htm?imageKey=RHEUM/67233\" class=\"graphic graphic_figure\">- DNA-based HLA typing</a></li><li><a href=\"image.htm?imageKey=RHEUM/109457\" class=\"graphic graphic_figure\">- HLA nomenclature</a></li></ul></li></ul></div></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=abacavir-hypersensitivity-reaction\" class=\"medical medical_review\">Abacavir hypersensitivity reaction</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=genetic-association-studies-principles-and-applications\" class=\"medical medical_review\">Genetic association studies: Principles and applications</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=hla-and-other-susceptibility-genes-in-rheumatoid-arthritis\" class=\"medical medical_review\">HLA and other susceptibility genes in rheumatoid arthritis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=immunology-of-the-maternal-fetal-interface\" class=\"medical medical_review\">Immunology of the maternal-fetal interface</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=inherited-disorders-of-the-complement-system\" class=\"medical medical_review\">Inherited disorders of the complement system</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=major-histocompatibility-complex-mhc-structure-and-function\" class=\"medical medical_review\">Major histocompatibility complex (MHC) structure and function</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=overview-of-pharmacogenomics\" class=\"medical medical_review\">Overview of pharmacogenomics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=pathogenesis-of-spondyloarthritis\" class=\"medical medical_review\">Pathogenesis of spondyloarthritis</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=role-of-cytokines-in-rheumatic-diseases\" class=\"medical medical_review\">Role of cytokines in rheumatic diseases</a></li></ul></div></div>","javascript":null}