var data={"title":"Immunoglobulin genetics","body":"<div id=\"topicContent\" class=\"utdArticleSection utdStyle\"><div id=\"topicTitle\">Immunoglobulin genetics</div><dl id=\"topicContributors\"><dt><span> </span>Author:</dt><dd><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/contributors\" class=\"contributor contributor_credentials\">Francisco A Bonilla, MD, PhD</a></dd><dt><span> </span>Section Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/contributors\" class=\"contributor contributor_credentials\">Jennifer M Puck, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/contributors\" class=\"contributor contributor_credentials\">Anna M Feldweg, MD</a></dd></dl><p class=\"disclosureLink\"><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/contributor-disclosure\" class=\"contributor contributor_credentials\">Contributor Disclosures</a></p><div id=\"reviewProcess\"><span>All topics are updated as new evidence becomes available and our <a href=\"https://www.uptodate.com/home/editorial-policy\" target=\"_blank\" class=\"policy policy_editorialpolicy\">peer review process</a> is complete.</span></div><div id=\"literatureReviewDate\"><span class=\"emphasis\">Literature review current through:</span>&#160;Feb 2018.&#160;&#124;&#160;<span class=\"emphasis\">This topic last updated:</span>&#160;Feb 02, 2016.</div><div id=\"topicWhatsNewContainer\"></div><div id=\"topicText\"><p class=\"headingAnchor\" id=\"H1\"><span class=\"h1\">INTRODUCTION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The genetics of immunoglobulin (Ig) during both B cell development in the bone marrow and the induction of the humoral immune response are presented here. Discussions concerning B cell development and the humoral immune response are reviewed separately (see <a href=\"topic.htm?path=normal-b-and-t-lymphocyte-development\" class=\"medical medical_review\">&quot;Normal B and T lymphocyte development&quot;</a> and <a href=\"topic.htm?path=the-humoral-immune-response\" class=\"medical medical_review\">&quot;The humoral immune response&quot;</a>). A <a href=\"http://www.imgt.org/&amp;token=McmUHzbivMBbVmUkgiAMB+LYUddD64inheNV6pn4DLU=&amp;TOPIC_ID=3977\" target=\"_blank\" class=\"external\">website</a> devoted to immunoglobulin genetics is available [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/1\" class=\"abstract_t\">1</a>].</p><p class=\"headingAnchor\" id=\"H2\"><span class=\"h1\">IMMUNOGLOBULIN GENE ORGANIZATION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>There are several classes of immunoglobulin (Ig) that are categorized by a unique heavy chain structure: IgG, IgM, IgA, IgD, and IgE. There are also subclasses of IgG (IgG1 to 4) and of IgA (IgA1,2) (see <a href=\"topic.htm?path=structure-of-immunoglobulins\" class=\"medical medical_review\">&quot;Structure of immunoglobulins&quot;</a>). In addition, there are two different types of antibody light chains, kappa and lambda. Ig heavy chains and each type of light chain are encoded by genes in different loci. The table shows the locations of these gene complexes on human chromosomes (<a href=\"image.htm?imageKey=ALLRG%2F64329\" class=\"graphic graphic_table graphicRef64329 \">table 1</a>).</p><p>Genes capable of encoding a complete antibody heavy or light chain do not exist, as such, within the DNA of most cells. The complete genes are assembled by the <strong>union</strong> of separate gene segments. These segments are widely separated in germ cells and in all somatic cells, except for B lymphocytes. Within B cells, these genes rearrange their relative positions to create a &quot;mature&quot; Ig gene that can encode a functional protein. This rearrangement process is the core of the immune system's ability to generate antibodies capable of recognizing the tremendous variety of antigenic structures in nature.</p><p>An Ig light chain gene is assembled from three types of gene segments. These are the light chain variable region (VL), joining region (JL), and constant region (CL) gene segments (<a href=\"image.htm?imageKey=ALLRG%2F76331\" class=\"graphic graphic_figure graphicRef76331 \">figure 1</a>). Similarly, heavy chain gene segments are VH, JH and CH, and another type of gene segment called D (diversity) (<a href=\"image.htm?imageKey=ALLRG%2F72581\" class=\"graphic graphic_figure graphicRef72581 \">figure 2</a>).</p><p>The nomenclature of the gene segments derives from names assigned to portions of antibody heavy or light chains based upon an analysis of their amino acid sequences (see <a href=\"topic.htm?path=structure-of-immunoglobulins\" class=\"medical medical_review\">&quot;Structure of immunoglobulins&quot;</a>). Thus, the variable region shows great variation from one antibody chain to another. The constant region, as its name implies, is invariant within an Ig class or subclass. The joining and diversity regions were the names given to areas between V and C, each having its own characteristic sequence pattern. The variable regions of the heavy and light chains together form the antibody combining site. This is the portion of the Ig molecule that makes contact with antigen. (See <a href=\"topic.htm?path=overview-of-therapeutic-monoclonal-antibodies\" class=\"medical medical_review\">&quot;Overview of therapeutic monoclonal antibodies&quot;</a>.)</p><p>The organization of Ig light and heavy chain genes is very similar. The kappa locus on chromosome 2 consists of 79 Vkappa genes, only 49 of which are functional (ie, capable of encoding a complete Vkappa light chain segment). The others are pseudogenes that contain lethal mutations that prevent expression or translation of functional protein. The Vkappa genes are followed by five Jkappa genes, and one Ckappa gene (<a href=\"image.htm?imageKey=ALLRG%2F76331\" class=\"graphic graphic_figure graphicRef76331 \">figure 1</a>) [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/1,2\" class=\"abstract_t\">1,2</a>]. The human lambda locus on chromosome 22 is somewhat different, being composed of 73 Vlambda (of which 33 are functional), 7 Jlambda, and 7 Clambda genes (some of which are pseudogenes) (<a href=\"image.htm?imageKey=ALLRG%2F71110\" class=\"graphic graphic_figure graphicRef71110 \">figure 3</a>) [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/1,3\" class=\"abstract_t\">1,3</a>].</p><p>The Ig heavy chain locus on chromosome 14 has been completely sequenced [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/1,4\" class=\"abstract_t\">1,4</a>]. It spans 957 kilobase pairs of DNA and contains a total of 123 VH genes. Only 39 of these genes are functional. Note that not all VH loci in all individuals will contain the same number of VH genes due to deletions in the region that can be inherited allelically (the same is true of VL loci). This leads to some individual variation in the number of V genes that can be expressed at each locus. There are 26 D genes, 6 JH genes (along with 3 nonfunctional JH pseudogenes), and 11 CH region genes (<a href=\"image.htm?imageKey=ALLRG%2F72581\" class=\"graphic graphic_figure graphicRef72581 \">figure 2</a>). Each of these CH genes corresponds to a particular Ig class or subclass. (See <a href=\"topic.htm?path=structure-of-immunoglobulins\" class=\"medical medical_review\">&quot;Structure of immunoglobulins&quot;</a>.)</p><p>Ig V genes may also be found in other chromosomal locations where they cannot be expressed. These loci are called orphons and are believed to have arisen via gene conversions that have been maintained during evolution. VH orphons can be found on chromosomes 15 and 16 [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/5\" class=\"abstract_t\">5</a>]. A Vkappa orphon is found near the centromere of chromosome 2 [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/6\" class=\"abstract_t\">6</a>]. Most genes in orphon loci are pseudogenes. Since they are not used to create mature Ig genes, there is no selection pressure to limit the accumulation of mutations that destroy the ability of these genes to encode a functional protein.</p><p>VH and VL genes are grouped into V gene families based upon nucleotide sequence homology. V genes that are greater than 80 percent homologous are said to belong to the same family. Homology between families is generally less than 75 percent. Human VH genes have been divided into seven families designated V1 to V7 [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/4\" class=\"abstract_t\">4</a>]. Similarly, human Vkappa and Vlambda genes have been divided into 4 and 10 families, respectively [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/2,3\" class=\"abstract_t\">2,3</a>]. D genes are also grouped into six families [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/4\" class=\"abstract_t\">4</a>].</p><p>A typical V gene consists of two exons separated by an intron. The first V gene exon encodes most of the leader sequence, or signal peptide, a structure analogous to those found in many secreted proteins (<a href=\"image.htm?imageKey=ALLRG%2F76331\" class=\"graphic graphic_figure graphicRef76331 \">figure 1</a>). This is the first part of the protein synthesized (the amino terminus), and this peptide interacts with the endoplasmic reticulum, which is important in the intracellular transport and processing of the protein. The signal peptide is removed from the antibody before it appears in the membrane or is secreted. The second V gene exon encodes the majority of the V region, about 95 amino acids. D genes are variable in length and may encode 1 to 15 amino acids. J genes encode 15 to 20 amino acids. (See <a href=\"topic.htm?path=structure-of-immunoglobulins\" class=\"medical medical_review\">&quot;Structure of immunoglobulins&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H3\"><span class=\"h1\">IMMUNOGLOBULIN GENE REARRANGEMENT</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>As previously mentioned, the creation of a mature expressible immunoglobulin heavy or light chain gene requires the assembly of randomly selected gene &quot;modules&quot; taken from groups of similar elements scattered widely over many kilobases of DNA [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/7\" class=\"abstract_t\">7</a>]. With heavy chain gene rearrangement, the 3' end of a diversity (D) gene is brought to the 5' end of a heavy chain joining region (JH) gene (<a href=\"image.htm?imageKey=ALLRG%2F72581\" class=\"graphic graphic_figure graphicRef72581 \">figure 2</a>). This is called D-JH rearrangement. Next, the 3' end of a heavy chain variable region (VH) gene is juxtaposed to DJH (V-DJH rearrangement). Light chains undergo only one rearrangement event, light chain variable region (VL) to light chain joining region (JL) (<a href=\"image.htm?imageKey=ALLRG%2F76331\" class=\"graphic graphic_figure graphicRef76331 \">figure 1</a>).</p><p>Ig gene rearrangement is regulated by complex interactions between special DNA sequences, DNA-binding factors, and DNA-modifying enzymes [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/7\" class=\"abstract_t\">7</a>]. V, D, and J genes all have flanking regions called recombination signal sequences. These are important in the mechanism of rearrangement. The heptamer (seven base pairs, or bp) and nonamer (nine bp) sequences are separated by a spacer of either 12 or 23 bp (<a href=\"image.htm?imageKey=ALLRG%2F62896\" class=\"graphic graphic_figure graphicRef62896 \">figure 4</a>). A gene with a flanking sequence containing a 12 bp spacer may only join to a gene whose flanking sequence has a 23 bp spacer. This is called the 12 to 23 base pair rule. These sequences are recognized by DNA-binding proteins focusing the activity of the nucleases and ligases performing the splicing. The conservation of these flanking sequences in many species indicates their importance in the rearrangement mechanism.</p><p>The process of gene rearrangement begins with complex regulatory mechanisms that affect the chromatin structure of the DNA regions to be cut and ligated. This regulation and selective unmasking or unpackaging of DNA help to ensure that the splicing mechanism does not go awry and lead to pathogenic translocations [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/7\" class=\"abstract_t\">7</a>]. The gene splicing that occurs in Ig gene rearrangement is mediated via one or more complexes of proteins that cut the two DNA sites that will be joined and ligate the DNA, thereby holding the two segments in their newly spliced configuration (<a href=\"image.htm?imageKey=ALLRG%2F62896\" class=\"graphic graphic_figure graphicRef62896 \">figure 4</a>). The protein products of the recombinase activating genes 1 and 2 (RAG1 and RAG2) were the first components of this complex to be described. RAG1 binds directly to the recombination signal sequences, and also binds RAG2 [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/7\" class=\"abstract_t\">7</a>]. RAG1 and RAG2 make a double-stranded break in the DNA and recruit several additional molecules that complete the juxtaposition of the coding DNA ends and their ligation. During this process, the DNA that was between the two rearranging genes is formed into a closed circle (<a href=\"image.htm?imageKey=ALLRG%2F62896\" class=\"graphic graphic_figure graphicRef62896 \">figure 4</a>).</p><p>RAG1 and RAG2 activity are also found in a subset of germinal center B cells undergoing the process of receptor editing of light chains [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/8\" class=\"abstract_t\">8</a>]. Thus, RAG1 and RAG2 play roles not only in the development of B cells in the bone marrow, but also in the generation of the antibody response to antigen (see <a href=\"topic.htm?path=the-humoral-immune-response\" class=\"medical medical_review\">&quot;The humoral immune response&quot;</a>). Furthermore, Ig and T cell receptor genes rearrange via identical mechanisms.</p><p class=\"headingAnchor\" id=\"H28823984\"><span class=\"h2\">KRECs</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The excised DNA circles produced during Ig gene rearrangement are retained in the nucleus of the B cell. They are not replicated during subsequent cell divisions, but they persist in the nucleus of one of the progeny cells after division. Thus, the fraction of the B cells bearing these circles gradually diminishes with successive rounds of cell division during development in the bone marrow. A small fraction of mature B cells in the circulation will have these DNA circles in their nuclei. The excised DNA circles formed during kappa light chain gene rearrangement are called kappa-deleting recombination excision circles (KRECs) [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/9\" class=\"abstract_t\">9</a>]. The number of KRECs in peripheral blood lymphocyte DNA can be measured by quantitative polymerase chain reaction (qPCR) and this can be used as a marker of the rate of B cell generation [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/9\" class=\"abstract_t\">9</a>]. For example, the number of KRECs is very low in patients with agammaglobulinemia, or a subset of patients with common variable immunodeficiency (CVID) [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/10\" class=\"abstract_t\">10</a>]. KREC measurement is being studied as a clinical marker of B cell development and for newborn screening for defects of B cell development [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/11\" class=\"abstract_t\">11</a>]. This type of assay is analogous to measurement of T cell receptor excision circles (TRECs) [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/12\" class=\"abstract_t\">12</a>]. (See <a href=\"topic.htm?path=newborn-screening-for-primary-immunodeficiencies\" class=\"medical medical_review\">&quot;Newborn screening for primary immunodeficiencies&quot;</a> and <a href=\"topic.htm?path=t-cell-receptor-genetics#H6\" class=\"medical medical_review\">&quot;T cell receptor genetics&quot;, section on 'T cell receptor excision circles (TRECs)'</a>.)</p><p class=\"headingAnchor\" id=\"H4\"><span class=\"h1\">ANTIBODY DIVERSITY</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Several mechanisms generate antibody region (combining site) diversity:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Multiple variable (V), diversity (D), and joining (J) genes (germline-encoded diversity)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Rearrangement diversity with imprecise joining of gene segments</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Nongermline encoded nucleotides (N regions)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Pairing of heavy and light chains (combinatorial diversity)</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Somatic mutation [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/13\" class=\"abstract_t\">13</a>]</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Gene replacements (receptor editing) [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/8\" class=\"abstract_t\">8</a>]</p><p/><p>These mechanisms allow for highly efficient use of a relatively small portion of the genome to provide a sufficiently large repertoire of antibody combining sites, a population of antibody specificities so diverse that no matter what antigens are encountered, a complementary antibody may be found.</p><p class=\"headingAnchor\" id=\"H5\"><span class=\"h2\">Germline-encoded diversity</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>A large number of different heavy chain VDJH combinations may be assembled. The completely sequenced heavy chain variable region (VH) locus contains 39 usable VH genes, 26 D, and 6 heavy chain joining region (JH) genes (see <a href=\"#H2\" class=\"local\">'Immunoglobulin gene organization'</a> above). Assuming any VH can join to any D, and any D to any JH, 6084 different heavy chain VDJ units may therefore be assembled. The germline-encoded VH diversity is amplified by 156-fold, simply by combining VH gene segments with various D and JH segments. This may be an overestimate because it appears that some VH genes preferentially rearrange [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/14\" class=\"abstract_t\">14</a>]. However, it is clear that the multiplicative nature of immunoglobulin (Ig) gene rearrangement greatly extends the diversity that can be encoded in the genome.</p><p class=\"headingAnchor\" id=\"H6\"><span class=\"h2\">Rearrangement diversity</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Since the joining of D to JH and VH to DJH (similarly, light chain variable region [VL] to light chain joining region [JL]) is not precise, it does not always occur at the same position in the gene segments. This results in different codons at the joints and in different complementarity-determining region (CDR)3 lengths (<a href=\"image.htm?imageKey=ALLRG%2F62896\" class=\"graphic graphic_figure graphicRef62896 \">figure 4</a>).</p><p class=\"headingAnchor\" id=\"H7\"><span class=\"h2\">Nongermline encoded nucleotides</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Nucleotides that are not encoded by genomic DNA may exist at the gene segment junctions. These nucleotides are believed to be inserted at the junction by the enzyme terminal deoxyribonucleotidyl transferase (TdT) and are designated the N region. This process contributes to diversity by changing the length of the resultant protein, and if a nonmultiple of three nucleotides is added, the reading frame is also shifted.</p><p class=\"headingAnchor\" id=\"H8\"><span class=\"h2\">Combinatorial diversity</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Additional diversity is created by the pairing of the heavy and light chains produced by the cell. Although a given VH cannot pair with all light chain V genes to form a functional antibody [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/15\" class=\"abstract_t\">15</a>], the assembly of antibodies from two different component chains leads to another multiplicative increase in the possible repertoire of combining sites. The total number capable of being generated is equal to the product of the number of different heavy chains and the number of different light chains that can form functional pairs. As an example, 3198 different possible combinations may arise because there are 39 functional VH genes, and 82 Vkappa and Vlambda genes. Even if not all of these pairs are functional, this represents an approximately 15- to 30-fold increase over germline-encoded diversity.</p><p>The parts of the antibody V region that contact antigen are called complementarity-determining regions (CDRs) (see <a href=\"topic.htm?path=structure-of-immunoglobulins\" class=\"medical medical_review\">&quot;Structure of immunoglobulins&quot;</a>). The CDRs 1 and 2 are encoded by the 5' regions of VH genes, and they reflect predominantly germline-encoded sequence diversity. All of the rearrangement mechanisms described above create diversity within the most 3' CDR3, built up from the 3' end of VH, a D, and the 5' end of a JH gene. One analysis estimates the overall number of potential H chain CDR3 sequences to be on the order of 10<sup>14</sup> [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/13\" class=\"abstract_t\">13</a>].</p><p class=\"headingAnchor\" id=\"H9\"><span class=\"h2\">Somatic mutation</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Somatic mutation is another very important mechanism for generating variation in V regions [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/13\" class=\"abstract_t\">13</a>]. At some time after rearrangement is complete, the heavy and light chain V genes may accumulate point mutations in any CDR or framework region. One gene may collect up to 10 or more nucleotide changes, some fraction of which result in an amino acid substitution. These changes may alter antibody affinity, or may even alter specificity. This process is important in the phenomenon of affinity maturation that occurs in germinal centers, although it may also be found outside these areas [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/16\" class=\"abstract_t\">16</a>]. (See <a href=\"topic.htm?path=the-humoral-immune-response\" class=\"medical medical_review\">&quot;The humoral immune response&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H10\"><span class=\"h2\">Gene fusion</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>D genes may sometimes fuse during H chain gene rearrangement [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/13\" class=\"abstract_t\">13</a>].</p><p class=\"headingAnchor\" id=\"H11\"><span class=\"h2\">Gene replacements</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Gene replacements (receptor editing) occur during B cell development and antibody responses. These underlie selective processes critical for the formation of the primary and secondary antibody repertoires. (See <a href=\"topic.htm?path=normal-b-and-t-lymphocyte-development\" class=\"medical medical_review\">&quot;Normal B and T lymphocyte development&quot;</a> and <a href=\"topic.htm?path=the-humoral-immune-response\" class=\"medical medical_review\">&quot;The humoral immune response&quot;</a>.)</p><p>It should be understood that all of these mechanisms that create antibody diversity also add significant cellular wastage. A series of gene rearrangements may create nonfunctional genes by altering the reading frame or by producing an early termination codon. In addition, several germline V genes are pseudogenes and encode nonfunctional proteins. In order to produce an antibody, two successful series of rearrangements are required, one for heavy and one for light chain genes. Each series of rearrangements is susceptible to error. The fraction of developing B cells lost to aberrant Ig gene rearrangements is unknown, but it is probably significantly greater than 50 percent [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/17\" class=\"abstract_t\">17</a>].</p><p class=\"headingAnchor\" id=\"H12\"><span class=\"h1\">ALLELIC EXCLUSION</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Since there are two copies of each chromosome within a cell, it is theoretically possible that a B cell may produce two different immunoglobulin (Ig) heavy chains or two different light chains. This could result in the formation of several different antibodies in a single cell.</p><p>However, this does not occur because of a phenomenon termed allelic exclusion. During the null B cell stage, diversity-heavy chain joining region (D-JH) rearrangement usually occurs in both heavy chain loci before variable-diversity heavy chain joining region (V-DJH) joining. If the first V-DJH rearrangement on one chromosome is successful, a complete immunoglobulin M (IgM) (mu) heavy chain mRNA is made, IgM heavy chain protein is synthesized and complexed with surrogate light chains, then expressed on the cell surface as the pre-B cell receptor (pre-BCR). (See <a href=\"topic.htm?path=normal-b-and-t-lymphocyte-development\" class=\"medical medical_review\">&quot;Normal B and T lymphocyte development&quot;</a>.)</p><p>In addition to triggering rearrangement of light chain genes, the pre-BCR appears also to provide a negative signal inhibiting further heavy chain gene rearrangement on the other chromosome [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/17,18\" class=\"abstract_t\">17,18</a>]. If the first heavy chain gene rearrangement is unsuccessful, no IgM heavy chain protein can be expressed, and the locus on the other chromosome is free to rearrange. If both rearrangements are unsuccessful, no antibody can be formed, and the cell soon dies by apoptosis (programmed cell death). Thus, only cells with one productive heavy chain rearrangement go on to rearrange light chain genes.</p><p>The kappa light chain genes usually rearrange before lambda genes. Successful kappa gene rearrangement allows formation of a complete antibody, which seems to shut down further gene rearrangement, at least for a time (receptor editing may occur if the Ig is specific for self antigen) (see <a href=\"topic.htm?path=normal-b-and-t-lymphocyte-development\" class=\"medical medical_review\">&quot;Normal B and T lymphocyte development&quot;</a>). If kappa rearrangement is unsuccessful (on both chromosomes), lambda genes then recombine. If no rearrangement produces a functional light chain, again no antibody can be formed, and the cell dies. Thus, a single B cell will produce only a single antibody specificity resulting from the combination of one heavy chain with one light chain. This phenomenon is called allelic exclusion. In an individual heterozygous with respect to a particular polymorphic heavy or light chain C gene, a given B cell will produce antibodies of only one allotype. (See <a href=\"topic.htm?path=structure-of-immunoglobulins\" class=\"medical medical_review\">&quot;Structure of immunoglobulins&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H13\"><span class=\"h1\">IMMUNOGLOBULIN CLASS-SWITCHING</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>After the mature B cell stage, a cell may change from production of immunoglobulin M (IgM) and immunoglobulin D (IgD) to synthesis of immunoglobulin G (IgG), immunoglobulin A (IgA), or immunoglobulin E (IgE). This process is called class-switching [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/19\" class=\"abstract_t\">19</a>]. This is a stable change in the cell's genome and is transmitted to all progeny cells. Since class-switching yields the same variable diversity heavy chain joining region (VDJH) in association with a different heavy chain constant region (CH) region, antigen specificity and idiotypic determinants remain unchanged. Only the heavy chain isotype is altered relative to the antibody that was produced before the switch. (See <a href=\"topic.htm?path=structure-of-immunoglobulins\" class=\"medical medical_review\">&quot;Structure of immunoglobulins&quot;</a>.)</p><p>A single cell can produce both IgM and IgD by differential splicing of mRNA (<a href=\"image.htm?imageKey=ALLRG%2F50773\" class=\"graphic graphic_figure graphicRef50773 \">figure 5</a>). The predominant mechanism of class-switching is somewhat different, but is similar to the process of VDJH rearrangement in DNA. In the DNA 5' to each of the CH genes (except Cdelta) are found 1 kb or more of a characteristic repetitive sequence, such as (GAGCT)n or (GGGT)n. These are called switch sequences (or switch regions or S regions). When a cell undergoes the class-switch, the S region 5' to Cm is brought to the S region 5' to a downstream C region (eg, g1) (<a href=\"image.htm?imageKey=ALLRG%2F58440\" class=\"graphic graphic_figure graphicRef58440 \">figure 6</a>). The intervening DNA is either inverted or excised (looped out), just as in V gene recombination, although distinct cellular machinery is required to accomplish this recombination of single stranded mRNA [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/19\" class=\"abstract_t\">19</a>].</p><p>The fate of the DNA intervening between recombined sequences appears to be the same in V gene recombination and class-switching. However, the heptamer and nonamer V gene recombination sequences and S sequences are very different, and recombinase activating genes 1 and 2 (RAG1 and RAG2) are not involved in the class-switch. A description of the cellular interactions and factors regulating class-switching can be found elsewhere. (See <a href=\"topic.htm?path=the-humoral-immune-response\" class=\"medical medical_review\">&quot;The humoral immune response&quot;</a>.)</p><p class=\"headingAnchor\" id=\"H14\"><span class=\"h1\">MEMBRANE VERSUS SECRETED IMMUNOGLOBULIN</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Immunoglobulin (Ig) functions in the following two forms:</p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>As a receptor for antigen on the surface of the B cell, thereby transmitting signals by virtue of its association with transducing molecules.</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>As a secreted protein (made mainly by plasma cells), where it may bind to microbes and be neutralizing, opsonic, complement-fixing, etc. (See <a href=\"topic.htm?path=overview-of-therapeutic-monoclonal-antibodies\" class=\"medical medical_review\">&quot;Overview of therapeutic monoclonal antibodies&quot;</a>.)</p><p/><p>The only difference between these molecules is in the C-terminal portion where one has a transmembrane anchor region and the other does not. These two different forms are generated by differential splicing of mRNA (<a href=\"image.htm?imageKey=ALLRG%2F70768\" class=\"graphic graphic_figure graphicRef70768 \">figure 7</a>) [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/20\" class=\"abstract_t\">20</a>].</p><p class=\"headingAnchor\" id=\"H15\"><span class=\"h1\">CLONAL DISTRIBUTION OF B CELLS</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>Each mature B cell expresses a more or less unique set of immunoglobulin (Ig) heavy (H) and light (L) chain gene rearrangements and H + L combination. Each represents a genetically distinct clone [<a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/17\" class=\"abstract_t\">17</a>]. All of the progeny B cells generated by mitosis of that cell (clonal expansion) are considered as belonging to the same clone, since they are genetically identical. Some divergence may occur during clonal expansion, since an individual cell of a clone may class-switch or somatically mutate its variable (V) genes differently from its siblings. Depending on one's point of view, one may designate the progeny of these genetic events as constituting new clones, yet they all remain clonally related, since they originated from one cell. Thus, the circulating pool of B lymphocytes is comprised of many genetically distinct clones, each of which may have anywhere from one to thousands of members.</p><p class=\"headingAnchor\" id=\"H98246844\"><span class=\"h1\">SUMMARY</span></p><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Genes capable of encoding a complete antibody heavy or light chain do not exist, as such, within the DNA of most cells. Instead, complete genes are assembled within B cells by the union of separate gene segments to create a &quot;mature&quot; immunoglobulin (Ig) gene that can encode a functional protein. This rearrangement is critical to creating a diverse antibody repertoire capable of interacting with the tremendous variety of antigenic structures in nature. (See <a href=\"#H2\" class=\"local\">'Immunoglobulin gene organization'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>An Ig light chain gene is assembled from three types of gene segments. These are the light chain variable region (VL), joining region (JL), and constant region (CL) gene segments (<a href=\"image.htm?imageKey=ALLRG%2F76331\" class=\"graphic graphic_figure graphicRef76331 \">figure 1</a>). Similarly, heavy chain gene segments are VH, JH, and CH, and another type of gene segment called D (diversity) (<a href=\"image.htm?imageKey=ALLRG%2F72581\" class=\"graphic graphic_figure graphicRef72581 \">figure 2</a>). (See <a href=\"#H2\" class=\"local\">'Immunoglobulin gene organization'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Immunoglobulin gene rearrangement is regulated by complex interactions between special DNA sequences, DNA-binding factors, and DNA-modifying enzymes. (See <a href=\"#H3\" class=\"local\">'Immunoglobulin gene rearrangement'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Several mechanisms of generating diversity contribute to the structural variations found within the combining site of an immunoglobulin molecule. These mechanisms include germline-encoded diversity, imprecise joining during rearrangement, N region diversity, combinatorial diversity, somatic mutation, and receptor editing. (See <a href=\"#H4\" class=\"local\">'Antibody diversity'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Once a mature B cell is producing IgM and IgD, it may &quot;class-switch&quot; to make IgG, IgA, or IgE. The cell then continues to make the isotype to which it has committed for the rest of its life, as does all of that cell's progeny. (See <a href=\"#H13\" class=\"local\">'Immunoglobulin class-switching'</a> above.)</p><p/><p class=\"bulletIndent1\"><span class=\"glyph\">&#9679;</span>Each mature B cell expresses a unique set of Ig heavy and light chain gene rearrangements and H + L combination and represents a genetically distinct clone. Thus, the circulating pool of B lymphocytes is comprised of many genetically distinct clones, each of which may contain from one to thousands of members. (See <a href=\"#H15\" class=\"local\">'Clonal distribution of B cells'</a> above.)</p><p/><p class=\"headingAnchor\" id=\"H4014243761\"><span class=\"h1\">ACKNOWLEDGMENT</span><span class=\"headingEndMark\">&#160;&#8212;&#160;</span>The editorial staff at UpToDate would like to acknowledge E Richard Stiehm, MD, who contributed as a Section Editor to an earlier version of this topic review.</p></div><div id=\"topicAgreement\">Use of UpToDate is subject to the <a href=\"https://www.uptodate.com/legal/license\" class=\"licenseLink\" id=\"sla_in_page\">Subscription and License Agreement</a>.</div><div id=\"references\" class=\"headingAnchor\"><h1>REFERENCES</h1><ol id=\"reference\"><li class=\"breakAll\">The International Immunogenetics Information System www.imgt.org (Accessed on October 27, 2010).</li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/2\" class=\"nounderline abstract_t\">Barbi&eacute; V, Lefranc MP. The human immunoglobulin kappa variable (IGKV) genes and joining (IGKJ) segments. Exp Clin Immunogenet 1998; 15:171.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/3\" class=\"nounderline abstract_t\">Pallar&egrave;s N, Frippiat JP, Giudicelli V, Lefranc MP. The human immunoglobulin lambda variable (IGLV) genes and joining (IGLJ) segments. Exp Clin Immunogenet 1998; 15:8.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/4\" class=\"nounderline abstract_t\">Matsuda F, Ishii K, Bourvagnet P, et al. The complete nucleotide sequence of the human immunoglobulin heavy chain variable region locus. J Exp Med 1998; 188:2151.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/5\" class=\"nounderline abstract_t\">Nagaoka H, Ozawa K, Matsuda F, et al. Recent translocation of variable and diversity segments of the human immunoglobulin heavy chain from chromosome 14 to chromosomes 15 and 16. Genomics 1994; 22:189.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/6\" class=\"nounderline abstract_t\">Weichhold GM, Lautner-Rieske A, Zachau HG. Human immunoglobulin genes of the kappa type. The long-range map of an orphon V kappa gene region. Biol Chem Hoppe Seyler 1992; 373:1159.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/7\" class=\"nounderline abstract_t\">Cobb RM, Oestreich KJ, Osipovich OA, Oltz EM. Accessibility control of V(D)J recombination. Adv Immunol 2006; 91:45.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/8\" class=\"nounderline abstract_t\">Azulay-Debby H, Melamed D. B cell receptor editing in tolerance and autoimmunity. Front Biosci 2007; 12:2136.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/9\" class=\"nounderline abstract_t\">van Zelm MC, Szczepanski T, van der Burg M, van Dongen JJ. Replication history of B lymphocytes reveals homeostatic proliferation and extensive antigen-induced B cell expansion. J Exp Med 2007; 204:645.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/10\" class=\"nounderline abstract_t\">Kamae C, Nakagawa N, Sato H, et al. Common variable immunodeficiency classification by quantifying T-cell receptor and immunoglobulin &kappa;-deleting recombination excision circles. J Allergy Clin Immunol 2013; 131:1437.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/11\" class=\"nounderline abstract_t\">Nakagawa N, Imai K, Kanegane H, et al. Quantification of &kappa;-deleting recombination excision circles in Guthrie cards for the identification of early B-cell maturation defects. J Allergy Clin Immunol 2011; 128:223.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/12\" class=\"nounderline abstract_t\">Serana F, Chiarini M, Zanotti C, et al. Use of V(D)J recombination excision circles to identify T- and B-cell defects and to monitor the treatment in primary and acquired immunodeficiencies. J Transl Med 2013; 11:119.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/13\" class=\"nounderline abstract_t\">Maizels N. Immunoglobulin gene diversification. Annu Rev Genet 2005; 39:23.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/14\" class=\"nounderline abstract_t\">Rao SP, Riggs JM, Friedman DF, et al. Biased VH gene usage in early lineage human B cells: evidence for preferential Ig gene rearrangement in the absence of selection. J Immunol 1999; 163:2732.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/15\" class=\"nounderline abstract_t\">Grey HM, Mannik M. Specificity of recombination of H and L chains from human gamma-G-myeloma proteins. J Exp Med 1965; 122:619.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/16\" class=\"nounderline abstract_t\">William J, Euler C, Christensen S, Shlomchik MJ. Evolution of autoantibody responses via somatic hypermutation outside of germinal centers. Science 2002; 297:2066.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/17\" class=\"nounderline abstract_t\">Rajewsky K. Clonal selection and learning in the antibody system. Nature 1996; 381:751.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/18\" class=\"nounderline abstract_t\">Cedar H, Bergman Y. Choreography of Ig allelic exclusion. Curr Opin Immunol 2008; 20:308.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/19\" class=\"nounderline abstract_t\">Stavnezer J, Guikema JE, Schrader CE. Mechanism and regulation of class switch recombination. Annu Rev Immunol 2008; 26:261.</a></li><li><a href=\"https://www.uptodate.com/contents/immunoglobulin-genetics/abstract/20\" class=\"nounderline abstract_t\">Borghesi L, Milcarek C. From B cell to plasma cell: regulation of V(D)J recombination and antibody secretion. Immunol Res 2006; 36:27.</a></li></ol></div><div id=\"topicVersionRevision\">Topic 3977 Version 9.0</div></div>","outline":"<div id=\"outlineSections\"><h2>Topic Outline</h2><ul id=\"innerOutline\"><li class=\"sr-button\"><a href=\"#H98246844\"><span>SUMMARY</span></a></li><li><a href=\"#H1\" id=\"outline-link-H1\">INTRODUCTION</a></li><li><a href=\"#H2\" id=\"outline-link-H2\">IMMUNOGLOBULIN GENE ORGANIZATION</a></li><li><a href=\"#H3\" id=\"outline-link-H3\">IMMUNOGLOBULIN GENE REARRANGEMENT</a><ul><li><a href=\"#H28823984\" id=\"outline-link-H28823984\">KRECs</a></li></ul></li><li><a href=\"#H4\" id=\"outline-link-H4\">ANTIBODY DIVERSITY</a><ul><li><a href=\"#H5\" id=\"outline-link-H5\">Germline-encoded diversity</a></li><li><a href=\"#H6\" id=\"outline-link-H6\">Rearrangement diversity</a></li><li><a href=\"#H7\" id=\"outline-link-H7\">Nongermline encoded nucleotides</a></li><li><a href=\"#H8\" id=\"outline-link-H8\">Combinatorial diversity</a></li><li><a href=\"#H9\" id=\"outline-link-H9\">Somatic mutation</a></li><li><a href=\"#H10\" id=\"outline-link-H10\">Gene fusion</a></li><li><a href=\"#H11\" id=\"outline-link-H11\">Gene replacements</a></li></ul></li><li><a href=\"#H12\" id=\"outline-link-H12\">ALLELIC EXCLUSION</a></li><li><a href=\"#H13\" id=\"outline-link-H13\">IMMUNOGLOBULIN CLASS-SWITCHING</a></li><li><a href=\"#H14\" id=\"outline-link-H14\">MEMBRANE VERSUS SECRETED IMMUNOGLOBULIN</a></li><li><a href=\"#H15\" id=\"outline-link-H15\">CLONAL DISTRIBUTION OF B CELLS</a></li><li><a href=\"#H98246844\" id=\"outline-link-H98246844\">SUMMARY</a></li><li><a href=\"#H4014243761\" id=\"outline-link-H4014243761\">ACKNOWLEDGMENT</a></li><li><a href=\"#references\">REFERENCES</a></li></ul></div><div><h2>GRAPHICS <a href=\"#\" id=\"viewAllGraphicsLink\">View All</a></h2><div id=\"outlineGraphics\"><ul><li><div id=\"ALLRG/3977|FIG\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"FIGURES\">FIGURES</a></div><ul><li><a href=\"image.htm?imageKey=ALLRG/76331\" class=\"graphic graphic_figure\">- Ig light chain gene rearrangement</a></li><li><a href=\"image.htm?imageKey=ALLRG/72581\" class=\"graphic graphic_figure\">- Ig heavy chain gene rearrangement</a></li><li><a href=\"image.htm?imageKey=ALLRG/71110\" class=\"graphic graphic_figure\">- Organization of Ig lambda light chain locus</a></li><li><a href=\"image.htm?imageKey=ALLRG/62896\" class=\"graphic graphic_figure\">- Details of Ig gene rearrangement</a></li><li><a href=\"image.htm?imageKey=ALLRG/50773\" class=\"graphic graphic_figure\">- Differential splicing mu and delta mRNA</a></li><li><a href=\"image.htm?imageKey=ALLRG/58440\" class=\"graphic graphic_figure\">- Ig class switching</a></li><li><a href=\"image.htm?imageKey=ALLRG/70768\" class=\"graphic graphic_figure\">- Differential splicing mRNA</a></li></ul></li><li><div id=\"ALLRG/3977|TAB\" class=\"openRelatedGraphics\"><a href=\"#\" title=\"TABLES\">TABLES</a></div><ul><li><a href=\"image.htm?imageKey=ALLRG/64329\" class=\"graphic graphic_table\">- Chromosomal locations</a></li></ul></li></ul></div></div><div><h2>RELATED TOPICS</h2><div id=\"outlineTopics\"><ul><li class=\"plainItem\"><a href=\"topic.htm?path=newborn-screening-for-primary-immunodeficiencies\" class=\"medical medical_review\">Newborn screening for primary immunodeficiencies</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=normal-b-and-t-lymphocyte-development\" class=\"medical medical_review\">Normal B and T lymphocyte development</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=overview-of-therapeutic-monoclonal-antibodies\" class=\"medical medical_review\">Overview of therapeutic monoclonal antibodies</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=structure-of-immunoglobulins\" class=\"medical medical_review\">Structure of immunoglobulins</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=t-cell-receptor-genetics\" class=\"medical medical_review\">T cell receptor genetics</a></li><li class=\"plainItem\"><a href=\"topic.htm?path=the-humoral-immune-response\" class=\"medical medical_review\">The humoral immune response</a></li></ul></div></div>","javascript":null}